{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Single nucleotide variants (SNVs)\n",
    "\n",
    "**Author**: Katarina Braun \n",
    "\n",
    "\n",
    "## Overview\n",
    "\n",
    "This script maps fasta files (contains the consensus sequence generated from the ARTIC analysis pipeline) to the Wuhan/1 reference (MT039887), generates an alignment file (bam), sorts the bam (sam), generates a pileup file, and calls variants (any position that differs from the reference) using Varscan. The resulting VCF files contain consensus SNVs only. \n",
    "\n",
    "This script cleans up the VCF files and saves each VCF as a new, smaller CSV. \n",
    "\n",
    "This script also generates two figures:\n",
    "1. Consensus SNVs vs # of sequences figure. \n",
    "2. Number of sequences within age bins \n",
    "\n",
    "## Input: \n",
    "\n",
    "Consensus fasta files: \n",
    "\n",
    "`data_raw/UWHC/*.fasta`\n",
    "    - 122 files \n",
    "`data_raw/MHD/*.fasta`\n",
    "    - 125 files\n",
    "\n",
    "\n",
    "## Output: \n",
    "\n",
    "Consensus SNV output files: \n",
    "\n",
    "`data_derived/UWHC/VCFs/*.vcfs`\n",
    "    - 122 files\n",
    "`data_derived/UWHC/VCFs/*.vcfs`\n",
    "    - 125 files \n",
    "\n",
    "Figures:  \n",
    "1. `figures/consensus_SNV_differences.pdf` \n",
    "2. `figures/SNVs_vs_age.pdf`\n",
    "3. `figures/SNVs-per-age-bins.pdf`\n",
    " \n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## import modules and dependencies "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 362,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import glob, os\n",
    "import shutil as sh\n",
    "import numpy as np\n",
    "from sklearn.decomposition import PCA\n",
    "from sklearn import preprocessing\n",
    "import matplotlib.pyplot as plt\n",
    "from matplotlib import pyplot\n",
    "import matplotlib.pyplot as plt\n",
    "from pathlib import Path\n",
    "from os.path import splitext\n",
    "from functools import reduce\n",
    "from pandas.api.types import is_numeric_dtype\n",
    "import warnings\n",
    "from pandas.core.common import SettingWithCopyWarning\n",
    "from matplotlib.font_manager import FontProperties \n",
    "from matplotlib import cm\n",
    "import itertools\n",
    "from matplotlib import container\n",
    "from matplotlib import gridspec\n",
    "from matplotlib.patches import Patch\n",
    "from matplotlib.lines import Line2D\n",
    "import matplotlib.lines as mlines\n",
    "import matplotlib as mpl\n",
    "from pandas.plotting import table\n",
    "from statistics import mean\n",
    "import statistics"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## map reads, modify alignment file, call consensus SNVs\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-02_2020.fasta out=../data_raw/UWHC/USA_WI-UW-02_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-02_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-02_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.784 seconds.\n",
      "Analyzed Index:   \t3.321 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.105 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.346 seconds.\n",
      "Reads/sec:       \t173.55\n",
      "kBases/sec:      \t86.14\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "semiperfect site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.825 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-02_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-03_2020.fasta out=../data_raw/UWHC/USA_WI-UW-03_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-03_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-03_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.705 seconds.\n",
      "Analyzed Index:   \t3.009 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.089 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29870 bases)\n",
      "\n",
      "Mapping:          \t0.281 seconds.\n",
      "Reads/sec:       \t213.47\n",
      "kBases/sec:      \t106.27\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29870\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29870\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.6086% \t       25870\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.6086% \t       25870\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9732% \t       29862\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0268% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0268% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.336 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-03_2020.fasta.pileup\n",
      "29870 bases in pileup file\n",
      "29870 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-04_2020.fasta out=../data_raw/UWHC/USA_WI-UW-04_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-04_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-04_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.038 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.995 seconds.\n",
      "Analyzed Index:   \t4.703 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29843 bases)\n",
      "\n",
      "Mapping:          \t0.475 seconds.\n",
      "Reads/sec:       \t126.42\n",
      "kBases/sec:      \t62.88\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29843\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29843\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2720% \t       26343\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2720% \t       26343\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29836\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t6.601 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-04_2020.fasta.pileup\n",
      "29843 bases in pileup file\n",
      "29843 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-05_2020.fasta out=../data_raw/UWHC/USA_WI-UW-05_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-05_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-05_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.031 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.741 seconds.\n",
      "Analyzed Index:   \t3.706 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.095 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29855 bases)\n",
      "\n",
      "Mapping:          \t0.345 seconds.\n",
      "Reads/sec:       \t173.79\n",
      "kBases/sec:      \t86.48\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29855\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29855\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 87.0876% \t       26000\n",
      "semiperfect site:\t 86.6667% \t       52 \t 87.0876% \t       26000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9732% \t       29847\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0268% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0268% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.186 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-05_2020.fasta.pileup\n",
      "29855 bases in pileup file\n",
      "29855 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-06_2020.fasta out=../data_raw/UWHC/USA_WI-UW-06_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-06_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-06_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.837 seconds.\n",
      "Analyzed Index:   \t3.115 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.094 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29844 bases)\n",
      "\n",
      "Mapping:          \t0.267 seconds.\n",
      "Reads/sec:       \t224.84\n",
      "kBases/sec:      \t111.84\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29844\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29844\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.4443% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.4443% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29835\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.586 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-06_2020.fasta.pileup\n",
      "29844 bases in pileup file\n",
      "29844 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-07_2020.fasta out=../data_raw/UWHC/USA_WI-UW-07_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-07_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-07_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.808 seconds.\n",
      "Analyzed Index:   \t3.551 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.090 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29865 bases)\n",
      "\n",
      "Mapping:          \t0.335 seconds.\n",
      "Reads/sec:       \t178.84\n",
      "kBases/sec:      \t89.02\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29865\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29865\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.7100% \t       25000\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.7100% \t       25000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9665% \t       29855\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0335% \t          10\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0335% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.042 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-07_2020.fasta.pileup\n",
      "29865 bases in pileup file\n",
      "29865 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-08_2020.fasta out=../data_raw/UWHC/USA_WI-UW-08_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-08_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-08_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.962 seconds.\n",
      "Analyzed Index:   \t3.117 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.094 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29843 bases)\n",
      "\n",
      "Mapping:          \t0.282 seconds.\n",
      "Reads/sec:       \t212.90\n",
      "kBases/sec:      \t105.89\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29843\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29843\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 90.4735% \t       27000\n",
      "semiperfect site:\t 90.0000% \t       54 \t 90.4735% \t       27000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29837\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.754 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-08_2020.fasta.pileup\n",
      "29843 bases in pileup file\n",
      "29843 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-09_2020.fasta out=../data_raw/UWHC/USA_WI-UW-09_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-09_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-09_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.057 seconds.\n",
      "Analyzed Index:   \t3.577 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29843 bases)\n",
      "\n",
      "Mapping:          \t0.298 seconds.\n",
      "Reads/sec:       \t201.42\n",
      "kBases/sec:      \t100.18\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29843\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29843\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6228% \t       27343\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6228% \t       27343\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29837\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0201% \t           6\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.306 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-09_2020.fasta.pileup\n",
      "29843 bases in pileup file\n",
      "29843 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-100_2020.fasta out=../data_raw/UWHC/USA_WI-UW-100_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-100_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-100_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.031 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.841 seconds.\n",
      "Analyzed Index:   \t2.982 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.103 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.337 seconds.\n",
      "Reads/sec:       \t177.81\n",
      "kBases/sec:      \t88.26\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.557 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-100_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-101_2020.fasta out=../data_raw/UWHC/USA_WI-UW-101_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-101_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-101_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.791 seconds.\n",
      "Analyzed Index:   \t2.879 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.098 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.318 seconds.\n",
      "Reads/sec:       \t188.81\n",
      "kBases/sec:      \t93.72\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.359 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-101_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-102_2020.fasta out=../data_raw/UWHC/USA_WI-UW-102_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-102_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-102_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.858 seconds.\n",
      "Analyzed Index:   \t3.077 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.096 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.335 seconds.\n",
      "Reads/sec:       \t179.33\n",
      "kBases/sec:      \t89.01\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.642 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-102_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-103_2020.fasta out=../data_raw/UWHC/USA_WI-UW-103_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-103_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-103_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.035 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.065 seconds.\n",
      "Analyzed Index:   \t3.054 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.111 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.329 seconds.\n",
      "Reads/sec:       \t182.17\n",
      "kBases/sec:      \t90.43\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.879 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-103_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-105_2020.fasta out=../data_raw/UWHC/USA_WI-UW-105_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-105_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-105_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.033 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.907 seconds.\n",
      "Analyzed Index:   \t3.802 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.148 seconds.\n",
      "Cleared Memory:    \t0.117 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.332 seconds.\n",
      "Reads/sec:       \t180.79\n",
      "kBases/sec:      \t89.74\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.494 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-105_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-106_2020.fasta out=../data_raw/UWHC/USA_WI-UW-106_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-106_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-106_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.037 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.847 seconds.\n",
      "Analyzed Index:   \t3.327 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.122 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.353 seconds.\n",
      "Reads/sec:       \t169.97\n",
      "kBases/sec:      \t84.37\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0201% \t           6\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.032 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-106_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-107_2020.fasta out=../data_raw/UWHC/USA_WI-UW-107_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-107_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-107_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.920 seconds.\n",
      "Analyzed Index:   \t4.591 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.141 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.404 seconds.\n",
      "Reads/sec:       \t148.55\n",
      "kBases/sec:      \t73.73\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t6.330 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-107_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-108_2020.fasta out=../data_raw/UWHC/USA_WI-UW-108_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-108_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-108_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.621 seconds.\n",
      "Analyzed Index:   \t5.279 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.200 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.436 seconds.\n",
      "Reads/sec:       \t137.75\n",
      "kBases/sec:      \t68.37\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t7.926 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-108_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-109_2020.fasta out=../data_raw/UWHC/USA_WI-UW-109_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-109_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-109_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.047 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.258 seconds.\n",
      "Analyzed Index:   \t4.506 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.105 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.308 seconds.\n",
      "Reads/sec:       \t195.10\n",
      "kBases/sec:      \t96.84\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t6.591 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-109_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-10_2020.fasta out=../data_raw/UWHC/USA_WI-UW-10_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-10_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-10_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.039 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.991 seconds.\n",
      "Analyzed Index:   \t3.309 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.118 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29841 bases)\n",
      "\n",
      "Mapping:          \t0.287 seconds.\n",
      "Reads/sec:       \t209.15\n",
      "kBases/sec:      \t104.02\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29841\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29841\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2712% \t       26341\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2712% \t       26341\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29832\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.059 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-10_2020.fasta.pileup\n",
      "29841 bases in pileup file\n",
      "29841 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-110_2020.fasta out=../data_raw/UWHC/USA_WI-UW-110_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-110_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-110_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.036 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.155 seconds.\n",
      "Analyzed Index:   \t4.011 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.108 seconds.\n",
      "Cleared Memory:    \t0.118 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.316 seconds.\n",
      "Reads/sec:       \t189.68\n",
      "kBases/sec:      \t94.15\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.887 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-110_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-111_2020.fasta out=../data_raw/UWHC/USA_WI-UW-111_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-111_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-111_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.041 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t2.247 seconds.\n",
      "Analyzed Index:   \t5.283 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.119 seconds.\n",
      "Cleared Memory:    \t0.118 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.995 seconds.\n",
      "Reads/sec:       \t60.33\n",
      "kBases/sec:      \t29.95\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t8.954 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-111_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-112_2020.fasta out=../data_raw/UWHC/USA_WI-UW-112_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-112_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-112_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.024 seconds.\n",
      "Analyzed Index:   \t3.372 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.105 seconds.\n",
      "Cleared Memory:    \t0.132 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.517 seconds.\n",
      "Reads/sec:       \t116.15\n",
      "kBases/sec:      \t57.65\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.388 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-112_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-114_2020.fasta out=../data_raw/UWHC/USA_WI-UW-114_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-114_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-114_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.832 seconds.\n",
      "Analyzed Index:   \t3.483 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.153 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.561 seconds.\n",
      "Reads/sec:       \t107.05\n",
      "kBases/sec:      \t53.13\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9631% \t       29771\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.362 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-114_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-115_2020.fasta out=../data_raw/UWHC/USA_WI-UW-115_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-115_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-115_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.075 seconds.\n",
      "Analyzed Index:   \t5.871 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.093 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.408 seconds.\n",
      "Reads/sec:       \t147.07\n",
      "kBases/sec:      \t73.00\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t7.732 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-115_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-116_2020.fasta out=../data_raw/UWHC/USA_WI-UW-116_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-116_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-116_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.057 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.068 seconds.\n",
      "Analyzed Index:   \t4.430 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.124 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.418 seconds.\n",
      "Reads/sec:       \t143.41\n",
      "kBases/sec:      \t71.18\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t6.393 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-116_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-117_2020.fasta out=../data_raw/UWHC/USA_WI-UW-117_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-117_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-117_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.089 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.510 seconds.\n",
      "Analyzed Index:   \t4.443 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.107 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.987 seconds.\n",
      "Reads/sec:       \t60.80\n",
      "kBases/sec:      \t30.18\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t7.495 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-117_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-119_2020.fasta out=../data_raw/UWHC/USA_WI-UW-119_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-119_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-119_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.822 seconds.\n",
      "Analyzed Index:   \t3.357 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.095 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.353 seconds.\n",
      "Reads/sec:       \t170.03\n",
      "kBases/sec:      \t84.40\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.918 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-119_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-120_2020.fasta out=../data_raw/UWHC/USA_WI-UW-120_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-120_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-120_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.031 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.952 seconds.\n",
      "Analyzed Index:   \t3.153 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.107 seconds.\n",
      "Cleared Memory:    \t0.116 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.352 seconds.\n",
      "Reads/sec:       \t170.21\n",
      "kBases/sec:      \t84.49\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.852 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-120_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-122_2020.fasta out=../data_raw/UWHC/USA_WI-UW-122_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-122_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-122_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.043 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.987 seconds.\n",
      "Analyzed Index:   \t3.744 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.401 seconds.\n",
      "Cleared Memory:    \t0.199 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.921 seconds.\n",
      "Reads/sec:       \t65.17\n",
      "kBases/sec:      \t32.35\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.9799% \t       26500\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.9799% \t       26500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.7415% \t       29705\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.2350% \t          70\n",
      "\n",
      "Total time:     \t6.505 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-122_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-124_2020.fasta out=../data_raw/UWHC/USA_WI-UW-124_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-124_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-124_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.079 seconds.\n",
      "Analyzed Index:   \t3.080 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.098 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.308 seconds.\n",
      "Reads/sec:       \t194.83\n",
      "kBases/sec:      \t96.71\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.849 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-124_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-127_2020.fasta out=../data_raw/UWHC/USA_WI-UW-127_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-127_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-127_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.031 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.941 seconds.\n",
      "Analyzed Index:   \t3.063 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.100 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.382 seconds.\n",
      "Reads/sec:       \t157.11\n",
      "kBases/sec:      \t77.98\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "semiperfect site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.773 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-127_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-128_2020.fasta out=../data_raw/UWHC/USA_WI-UW-128_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-128_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-128_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.787 seconds.\n",
      "Analyzed Index:   \t2.826 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.092 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.227 seconds.\n",
      "Reads/sec:       \t264.88\n",
      "kBases/sec:      \t131.48\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.208 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-128_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-129_2020.fasta out=../data_raw/UWHC/USA_WI-UW-129_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-129_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-129_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.813 seconds.\n",
      "Analyzed Index:   \t3.143 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.120 seconds.\n",
      "Cleared Memory:    \t0.116 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.510 seconds.\n",
      "Reads/sec:       \t117.75\n",
      "kBases/sec:      \t58.44\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "semiperfect site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.0061% \t       29486\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.9570% \t         285\n",
      "\n",
      "Total time:     \t4.871 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-129_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-135_2020.fasta out=../data_raw/UWHC/USA_WI-UW-135_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-135_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-135_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.863 seconds.\n",
      "Analyzed Index:   \t3.490 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.101 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.339 seconds.\n",
      "Reads/sec:       \t177.12\n",
      "kBases/sec:      \t87.92\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.111 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-135_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-140_2020.fasta out=../data_raw/UWHC/USA_WI-UW-140_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-140_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-140_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.912 seconds.\n",
      "Analyzed Index:   \t3.314 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.092 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.273 seconds.\n",
      "Reads/sec:       \t220.09\n",
      "kBases/sec:      \t109.25\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.877 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-140_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-144_2020.fasta out=../data_raw/UWHC/USA_WI-UW-144_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-144_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-144_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.762 seconds.\n",
      "Analyzed Index:   \t2.970 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.122 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.294 seconds.\n",
      "Reads/sec:       \t203.86\n",
      "kBases/sec:      \t101.19\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "semiperfect site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.388 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-144_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-145_2020.fasta out=../data_raw/UWHC/USA_WI-UW-145_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-145_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-145_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.043 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.812 seconds.\n",
      "Analyzed Index:   \t2.932 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.318 seconds.\n",
      "Reads/sec:       \t188.96\n",
      "kBases/sec:      \t93.80\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.441 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-145_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-146_2020.fasta out=../data_raw/UWHC/USA_WI-UW-146_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-146_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-146_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.036 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.729 seconds.\n",
      "Analyzed Index:   \t2.997 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.268 seconds.\n",
      "Reads/sec:       \t224.02\n",
      "kBases/sec:      \t111.20\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "semiperfect site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.0061% \t       29486\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.9570% \t         285\n",
      "\n",
      "Total time:     \t4.383 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-146_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-147_2020.fasta out=../data_raw/UWHC/USA_WI-UW-147_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-147_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-147_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.713 seconds.\n",
      "Analyzed Index:   \t3.030 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.105 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.347 seconds.\n",
      "Reads/sec:       \t173.02\n",
      "kBases/sec:      \t85.88\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.451 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-147_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-148_2020.fasta out=../data_raw/UWHC/USA_WI-UW-148_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-148_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-148_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.830 seconds.\n",
      "Analyzed Index:   \t3.266 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.093 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.326 seconds.\n",
      "Reads/sec:       \t183.88\n",
      "kBases/sec:      \t91.27\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "\n",
      "Total time:     \t4.787 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-148_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-150_2020.fasta out=../data_raw/UWHC/USA_WI-UW-150_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-150_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-150_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.838 seconds.\n",
      "Analyzed Index:   \t3.840 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.094 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.331 seconds.\n",
      "Reads/sec:       \t181.31\n",
      "kBases/sec:      \t90.00\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.2647% \t       29563\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.7118% \t         212\n",
      "\n",
      "Total time:     \t5.356 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-150_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-151_2020.fasta out=../data_raw/UWHC/USA_WI-UW-151_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-151_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-151_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.870 seconds.\n",
      "Analyzed Index:   \t3.121 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.095 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.294 seconds.\n",
      "Reads/sec:       \t204.42\n",
      "kBases/sec:      \t101.47\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 93.3333% \t       56 \t 93.2845% \t       27782\n",
      "semiperfect site:\t 93.3333% \t       56 \t 93.2845% \t       27782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9866% \t       29778\n",
      "Error Rate:      \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "Sub Rate:        \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.667 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-151_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "4 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-152_2020.fasta out=../data_raw/UWHC/USA_WI-UW-152_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-152_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-152_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.033 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.015 seconds.\n",
      "Analyzed Index:   \t3.153 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.158 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.361 seconds.\n",
      "Reads/sec:       \t166.06\n",
      "kBases/sec:      \t82.43\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.048 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-152_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-153_2020.fasta out=../data_raw/UWHC/USA_WI-UW-153_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-153_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-153_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.960 seconds.\n",
      "Analyzed Index:   \t3.062 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.103 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.314 seconds.\n",
      "Reads/sec:       \t190.80\n",
      "kBases/sec:      \t94.70\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.724 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-153_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-155_2020.fasta out=../data_raw/UWHC/USA_WI-UW-155_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-155_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-155_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.759 seconds.\n",
      "Analyzed Index:   \t2.932 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.260 seconds.\n",
      "Reads/sec:       \t230.74\n",
      "kBases/sec:      \t114.53\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  1.6667% \t        1 \t  1.6789% \t         500\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9658% \t       29272\n",
      "Error Rate:      \t 13.5593% \t        8 \t  0.0342% \t          10\n",
      "Sub Rate:        \t 13.5593% \t        8 \t  0.0342% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.293 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-155_2020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-259_2020.fasta out=../data_raw/UWHC/USA_WI-UW-259_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-259_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-259_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.047 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.046 seconds.\n",
      "Analyzed Index:   \t3.007 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29611 bases)\n",
      "\n",
      "Mapping:          \t0.282 seconds.\n",
      "Reads/sec:       \t212.90\n",
      "kBases/sec:      \t105.07\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.1801% \t       26111\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.1801% \t       26111\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9696% \t       29602\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0304% \t           9\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0304% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.824 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-259_2020.fasta.pileup\n",
      "29611 bases in pileup file\n",
      "29611 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-25_2020.fasta out=../data_raw/UWHC/USA_WI-UW-25_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-25_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-25_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.832 seconds.\n",
      "Analyzed Index:   \t3.979 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.116 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.319 seconds.\n",
      "Reads/sec:       \t188.15\n",
      "kBases/sec:      \t93.39\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.504 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-25_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-260_2020.fasta out=../data_raw/UWHC/USA_WI-UW-260_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-260_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-260_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.043 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.862 seconds.\n",
      "Analyzed Index:   \t3.618 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.277 seconds.\n",
      "Reads/sec:       \t216.40\n",
      "kBases/sec:      \t107.42\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0235% \t           7\n",
      "\n",
      "Total time:     \t5.131 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-260_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-26_2020.fasta out=../data_raw/UWHC/USA_WI-UW-26_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-26_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-26_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.835 seconds.\n",
      "Analyzed Index:   \t2.931 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29611 bases)\n",
      "\n",
      "Mapping:          \t0.273 seconds.\n",
      "Reads/sec:       \t219.45\n",
      "kBases/sec:      \t108.30\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.5572% \t       27111\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.5572% \t       27111\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9831% \t       29606\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0169% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0169% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.388 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-26_2020.fasta.pileup\n",
      "29611 bases in pileup file\n",
      "29611 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-27_2020.fasta out=../data_raw/UWHC/USA_WI-UW-27_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-27_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-27_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.720 seconds.\n",
      "Analyzed Index:   \t2.921 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.292 seconds.\n",
      "Reads/sec:       \t205.67\n",
      "kBases/sec:      \t102.09\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.9799% \t       26500\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.9799% \t       26500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0101% \t           3\n",
      "\n",
      "Total time:     \t4.285 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-27_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-28_2020.fasta out=../data_raw/UWHC/USA_WI-UW-28_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-28_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-28_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.751 seconds.\n",
      "Analyzed Index:   \t2.821 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.090 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.311 seconds.\n",
      "Reads/sec:       \t192.66\n",
      "kBases/sec:      \t95.63\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "semiperfect site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.231 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-28_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-29_2020.fasta out=../data_raw/UWHC/USA_WI-UW-29_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-29_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-29_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.034 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.841 seconds.\n",
      "Analyzed Index:   \t3.179 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.285 seconds.\n",
      "Reads/sec:       \t210.77\n",
      "kBases/sec:      \t104.62\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.698 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-29_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-31_2020.fasta out=../data_raw/UWHC/USA_WI-UW-31_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-31_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-31_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.773 seconds.\n",
      "Analyzed Index:   \t3.131 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.104 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.280 seconds.\n",
      "Reads/sec:       \t214.51\n",
      "kBases/sec:      \t106.48\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.547 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-31_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-32_2020.fasta out=../data_raw/UWHC/USA_WI-UW-32_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-32_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-32_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.756 seconds.\n",
      "Analyzed Index:   \t2.952 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.268 seconds.\n",
      "Reads/sec:       \t224.13\n",
      "kBases/sec:      \t111.25\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.343 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-32_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-33_2020.fasta out=../data_raw/UWHC/USA_WI-UW-33_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-33_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-33_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.713 seconds.\n",
      "Analyzed Index:   \t2.966 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.287 seconds.\n",
      "Reads/sec:       \t209.04\n",
      "kBases/sec:      \t103.76\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9631% \t       29771\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.307 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-33_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-347_2020.fasta out=../data_raw/UWHC/USA_WI-UW-347_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-347_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-347_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.033 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.002 seconds.\n",
      "Analyzed Index:   \t3.013 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.290 seconds.\n",
      "Reads/sec:       \t206.58\n",
      "kBases/sec:      \t102.54\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.694 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-347_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-34_2020.fasta out=../data_raw/UWHC/USA_WI-UW-34_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-34_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-34_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.051 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.799 seconds.\n",
      "Analyzed Index:   \t3.433 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.102 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.279 seconds.\n",
      "Reads/sec:       \t215.12\n",
      "kBases/sec:      \t106.78\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.9433% \t       25000\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.9433% \t       25000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9631% \t       29771\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.987 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-34_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-351_2020.fasta out=../data_raw/UWHC/USA_WI-UW-351_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-351_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-351_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.742 seconds.\n",
      "Analyzed Index:   \t3.020 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.291 seconds.\n",
      "Reads/sec:       \t205.96\n",
      "kBases/sec:      \t102.23\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.399 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-351_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-356_2020.fasta out=../data_raw/UWHC/USA_WI-UW-356_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-356_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-356_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.766 seconds.\n",
      "Analyzed Index:   \t3.515 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.088 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.260 seconds.\n",
      "Reads/sec:       \t230.49\n",
      "kBases/sec:      \t114.41\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.891 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-356_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-359_2020.fasta out=../data_raw/UWHC/USA_WI-UW-359_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-359_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-359_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.810 seconds.\n",
      "Analyzed Index:   \t3.360 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.095 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.375 seconds.\n",
      "Reads/sec:       \t160.15\n",
      "kBases/sec:      \t79.49\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.916 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-359_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-35_2020.fasta out=../data_raw/UWHC/USA_WI-UW-35_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-35_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-35_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.767 seconds.\n",
      "Analyzed Index:   \t3.272 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.128 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.373 seconds.\n",
      "Reads/sec:       \t161.03\n",
      "kBases/sec:      \t79.93\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.825 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-35_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-362_2020.fasta out=../data_raw/UWHC/USA_WI-UW-362_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-362_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-362_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.541 seconds.\n",
      "Analyzed Index:   \t3.058 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.523 seconds.\n",
      "Reads/sec:       \t114.73\n",
      "kBases/sec:      \t56.95\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.532 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-362_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-37_2020.fasta out=../data_raw/UWHC/USA_WI-UW-37_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-37_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-37_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.031 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.921 seconds.\n",
      "Analyzed Index:   \t3.952 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.093 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.304 seconds.\n",
      "Reads/sec:       \t197.64\n",
      "kBases/sec:      \t98.10\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t5.592 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-37_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-38_2020.fasta out=../data_raw/UWHC/USA_WI-UW-38_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-38_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-38_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.047 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.873 seconds.\n",
      "Analyzed Index:   \t3.064 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.238 seconds.\n",
      "Reads/sec:       \t252.60\n",
      "kBases/sec:      \t125.38\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.812 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-38_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-39_2020.fasta out=../data_raw/UWHC/USA_WI-UW-39_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-39_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-39_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.756 seconds.\n",
      "Analyzed Index:   \t3.454 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.306 seconds.\n",
      "Reads/sec:       \t196.09\n",
      "kBases/sec:      \t97.33\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0201% \t           6\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.862 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-39_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-40_2020.fasta out=../data_raw/UWHC/USA_WI-UW-40_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-40_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-40_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.793 seconds.\n",
      "Analyzed Index:   \t4.116 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.241 seconds.\n",
      "Cleared Memory:    \t0.186 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.491 seconds.\n",
      "Reads/sec:       \t122.08\n",
      "kBases/sec:      \t60.60\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t6.032 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-40_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-41_2020.fasta out=../data_raw/UWHC/USA_WI-UW-41_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-41_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-41_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.034 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.868 seconds.\n",
      "Analyzed Index:   \t3.718 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.337 seconds.\n",
      "Reads/sec:       \t177.79\n",
      "kBases/sec:      \t88.25\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.323 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-41_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-42_2020.fasta out=../data_raw/UWHC/USA_WI-UW-42_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-42_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-42_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.031 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.964 seconds.\n",
      "Analyzed Index:   \t2.936 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.090 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.282 seconds.\n",
      "Reads/sec:       \t212.71\n",
      "kBases/sec:      \t105.58\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.565 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-42_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-43_2020.fasta out=../data_raw/UWHC/USA_WI-UW-43_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-43_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-43_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.801 seconds.\n",
      "Analyzed Index:   \t3.441 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.119 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.320 seconds.\n",
      "Reads/sec:       \t187.66\n",
      "kBases/sec:      \t93.15\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.005 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-43_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-44_2020.fasta out=../data_raw/UWHC/USA_WI-UW-44_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-44_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-44_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.031 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.005 seconds.\n",
      "Analyzed Index:   \t3.765 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.094 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.284 seconds.\n",
      "Reads/sec:       \t210.98\n",
      "kBases/sec:      \t104.72\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.432 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-44_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-45_2020.fasta out=../data_raw/UWHC/USA_WI-UW-45_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-45_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-45_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.081 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.096 seconds.\n",
      "Analyzed Index:   \t3.876 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.147 seconds.\n",
      "Cleared Memory:    \t0.119 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.377 seconds.\n",
      "Reads/sec:       \t159.20\n",
      "kBases/sec:      \t79.02\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t6.048 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-45_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-46_2020.fasta out=../data_raw/UWHC/USA_WI-UW-46_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-46_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-46_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.743 seconds.\n",
      "Analyzed Index:   \t4.118 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.089 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.564 seconds.\n",
      "Reads/sec:       \t106.47\n",
      "kBases/sec:      \t52.85\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.805 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-46_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-48_2020.fasta out=../data_raw/UWHC/USA_WI-UW-48_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-48_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-48_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.044 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.212 seconds.\n",
      "Analyzed Index:   \t3.985 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.233 seconds.\n",
      "Reads/sec:       \t257.94\n",
      "kBases/sec:      \t128.03\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.880 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-48_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-49_2020.fasta out=../data_raw/UWHC/USA_WI-UW-49_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-49_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-49_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.003 seconds.\n",
      "Analyzed Index:   \t3.516 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.267 seconds.\n",
      "Reads/sec:       \t224.93\n",
      "kBases/sec:      \t111.65\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.129 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-49_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-50_2020.fasta out=../data_raw/UWHC/USA_WI-UW-50_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-50_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-50_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.723 seconds.\n",
      "Analyzed Index:   \t3.362 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.102 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.458 seconds.\n",
      "Reads/sec:       \t131.06\n",
      "kBases/sec:      \t65.06\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.901 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-50_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-51_2020.fasta out=../data_raw/UWHC/USA_WI-UW-51_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-51_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-51_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.039 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.379 seconds.\n",
      "Analyzed Index:   \t3.179 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.203 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.280 seconds.\n",
      "Reads/sec:       \t213.91\n",
      "kBases/sec:      \t106.18\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.366 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-51_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-52_2020.fasta out=../data_raw/UWHC/USA_WI-UW-52_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-52_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-52_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.045 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.839 seconds.\n",
      "Analyzed Index:   \t4.195 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.093 seconds.\n",
      "Cleared Memory:    \t0.118 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.278 seconds.\n",
      "Reads/sec:       \t216.21\n",
      "kBases/sec:      \t107.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.770 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-52_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-53_2020.fasta out=../data_raw/UWHC/USA_WI-UW-53_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-53_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-53_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.829 seconds.\n",
      "Analyzed Index:   \t3.300 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.163 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.306 seconds.\n",
      "Reads/sec:       \t196.40\n",
      "kBases/sec:      \t97.49\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.878 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-53_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-54_2020.fasta out=../data_raw/UWHC/USA_WI-UW-54_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-54_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-54_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.767 seconds.\n",
      "Analyzed Index:   \t3.309 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.116 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.349 seconds.\n",
      "Reads/sec:       \t172.10\n",
      "kBases/sec:      \t85.42\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9631% \t       29771\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.809 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-54_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-55_2020.fasta out=../data_raw/UWHC/USA_WI-UW-55_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-55_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-55_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.759 seconds.\n",
      "Analyzed Index:   \t3.046 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.106 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.286 seconds.\n",
      "Reads/sec:       \t210.02\n",
      "kBases/sec:      \t104.25\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "semiperfect site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9866% \t       29778\n",
      "Error Rate:      \t  5.0000% \t        3 \t  0.0134% \t           4\n",
      "Sub Rate:        \t  5.0000% \t        3 \t  0.0134% \t           4\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.434 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-55_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "4 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-56_2020.fasta out=../data_raw/UWHC/USA_WI-UW-56_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-56_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-56_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.773 seconds.\n",
      "Analyzed Index:   \t2.573 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.258 seconds.\n",
      "Reads/sec:       \t232.23\n",
      "kBases/sec:      \t115.27\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.947 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-56_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-58_2020.fasta out=../data_raw/UWHC/USA_WI-UW-58_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-58_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-58_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.725 seconds.\n",
      "Analyzed Index:   \t2.446 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.248 seconds.\n",
      "Reads/sec:       \t242.12\n",
      "kBases/sec:      \t120.18\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.743 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-58_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-59_2020.fasta out=../data_raw/UWHC/USA_WI-UW-59_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-59_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-59_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.765 seconds.\n",
      "Analyzed Index:   \t3.138 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.097 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.265 seconds.\n",
      "Reads/sec:       \t226.46\n",
      "kBases/sec:      \t112.41\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.527 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-59_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-60_2020.fasta out=../data_raw/UWHC/USA_WI-UW-60_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-60_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-60_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.273 seconds.\n",
      "Analyzed Index:   \t3.039 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.098 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.328 seconds.\n",
      "Reads/sec:       \t183.12\n",
      "kBases/sec:      \t90.89\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t5.133 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-60_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-61_2020.fasta out=../data_raw/UWHC/USA_WI-UW-61_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-61_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-61_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.818 seconds.\n",
      "Analyzed Index:   \t3.715 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.104 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.481 seconds.\n",
      "Reads/sec:       \t124.87\n",
      "kBases/sec:      \t61.98\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.425 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-61_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-62_2020.fasta out=../data_raw/UWHC/USA_WI-UW-62_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-62_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-62_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.907 seconds.\n",
      "Analyzed Index:   \t3.569 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.276 seconds.\n",
      "Reads/sec:       \t217.31\n",
      "kBases/sec:      \t107.87\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t5.195 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-62_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-63_2020.fasta out=../data_raw/UWHC/USA_WI-UW-63_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-63_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-63_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.787 seconds.\n",
      "Analyzed Index:   \t2.599 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.093 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.263 seconds.\n",
      "Reads/sec:       \t227.74\n",
      "kBases/sec:      \t113.04\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.006 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-63_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-64_2020.fasta out=../data_raw/UWHC/USA_WI-UW-64_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-64_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-64_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.866 seconds.\n",
      "Analyzed Index:   \t3.473 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.089 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.232 seconds.\n",
      "Reads/sec:       \t258.59\n",
      "kBases/sec:      \t128.35\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.932 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-64_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-65_2020.fasta out=../data_raw/UWHC/USA_WI-UW-65_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-65_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-65_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.867 seconds.\n",
      "Analyzed Index:   \t2.930 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.108 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.328 seconds.\n",
      "Reads/sec:       \t182.65\n",
      "kBases/sec:      \t90.66\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.487 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-65_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-66_2020.fasta out=../data_raw/UWHC/USA_WI-UW-66_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-66_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-66_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.756 seconds.\n",
      "Analyzed Index:   \t2.929 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.155 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.282 seconds.\n",
      "Reads/sec:       \t212.70\n",
      "kBases/sec:      \t105.58\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.9433% \t       25000\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.9433% \t       25000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0134% \t           4\n",
      "\n",
      "Total time:     \t4.428 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-66_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-67_2020.fasta out=../data_raw/UWHC/USA_WI-UW-67_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-67_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-67_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.913 seconds.\n",
      "Analyzed Index:   \t4.159 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.119 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.430 seconds.\n",
      "Reads/sec:       \t139.46\n",
      "kBases/sec:      \t69.22\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "semiperfect site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.892 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-67_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-68_2020.fasta out=../data_raw/UWHC/USA_WI-UW-68_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-68_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-68_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.820 seconds.\n",
      "Analyzed Index:   \t3.807 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.096 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.320 seconds.\n",
      "Reads/sec:       \t187.62\n",
      "kBases/sec:      \t93.13\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.326 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-68_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-69_2020.fasta out=../data_raw/UWHC/USA_WI-UW-69_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-69_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-69_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.034 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.934 seconds.\n",
      "Analyzed Index:   \t2.532 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.289 seconds.\n",
      "Reads/sec:       \t207.63\n",
      "kBases/sec:      \t103.06\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.146 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-69_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-70_2020.fasta out=../data_raw/UWHC/USA_WI-UW-70_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-70_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-70_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.035 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.944 seconds.\n",
      "Analyzed Index:   \t2.820 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.115 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.341 seconds.\n",
      "Reads/sec:       \t176.05\n",
      "kBases/sec:      \t87.38\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "semiperfect site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.536 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-70_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-71_2020.fasta out=../data_raw/UWHC/USA_WI-UW-71_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-71_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-71_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.865 seconds.\n",
      "Analyzed Index:   \t3.117 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.256 seconds.\n",
      "Reads/sec:       \t234.50\n",
      "kBases/sec:      \t116.40\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "semiperfect site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.607 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-71_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-72_2020.fasta out=../data_raw/UWHC/USA_WI-UW-72_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-72_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-72_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.052 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.025 seconds.\n",
      "Analyzed Index:   \t3.135 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.304 seconds.\n",
      "Reads/sec:       \t197.54\n",
      "kBases/sec:      \t98.05\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.953 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-72_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-73_2020.fasta out=../data_raw/UWHC/USA_WI-UW-73_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-73_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-73_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.723 seconds.\n",
      "Analyzed Index:   \t3.036 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.248 seconds.\n",
      "Reads/sec:       \t241.86\n",
      "kBases/sec:      \t120.05\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "semiperfect site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.339 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-73_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-74_2020.fasta out=../data_raw/UWHC/USA_WI-UW-74_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-74_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-74_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.748 seconds.\n",
      "Analyzed Index:   \t2.722 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.278 seconds.\n",
      "Reads/sec:       \t215.87\n",
      "kBases/sec:      \t107.15\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.085 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-74_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-75_2020.fasta out=../data_raw/UWHC/USA_WI-UW-75_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-75_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-75_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.969 seconds.\n",
      "Analyzed Index:   \t2.930 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.070 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.251 seconds.\n",
      "Reads/sec:       \t239.21\n",
      "kBases/sec:      \t118.74\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.486 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-75_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-76_2020.fasta out=../data_raw/UWHC/USA_WI-UW-76_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-76_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-76_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.774 seconds.\n",
      "Analyzed Index:   \t2.675 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.088 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.303 seconds.\n",
      "Reads/sec:       \t198.04\n",
      "kBases/sec:      \t98.30\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 93.3333% \t       56 \t 93.2845% \t       27782\n",
      "semiperfect site:\t 93.3333% \t       56 \t 93.2845% \t       27782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  6.6667% \t        4 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  6.6667% \t        4 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.091 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-76_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-77_2020.fasta out=../data_raw/UWHC/USA_WI-UW-77_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-77_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-77_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.775 seconds.\n",
      "Analyzed Index:   \t3.248 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.247 seconds.\n",
      "Reads/sec:       \t242.44\n",
      "kBases/sec:      \t120.34\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.9433% \t       25000\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.9433% \t       25000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.628 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-77_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-78_2020.fasta out=../data_raw/UWHC/USA_WI-UW-78_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-78_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-78_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.696 seconds.\n",
      "Analyzed Index:   \t3.162 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.199 seconds.\n",
      "Reads/sec:       \t301.37\n",
      "kBases/sec:      \t149.59\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.381 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-78_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-79_2020.fasta out=../data_raw/UWHC/USA_WI-UW-79_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-79_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-79_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.757 seconds.\n",
      "Analyzed Index:   \t3.274 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.237 seconds.\n",
      "Reads/sec:       \t253.36\n",
      "kBases/sec:      \t125.76\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.594 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-79_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-80_2020.fasta out=../data_raw/UWHC/USA_WI-UW-80_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-80_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-80_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.726 seconds.\n",
      "Analyzed Index:   \t3.066 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.094 seconds.\n",
      "Cleared Memory:    \t0.120 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.265 seconds.\n",
      "Reads/sec:       \t226.49\n",
      "kBases/sec:      \t112.42\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.409 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-80_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-81_2020.fasta out=../data_raw/UWHC/USA_WI-UW-81_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-81_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-81_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.674 seconds.\n",
      "Analyzed Index:   \t3.002 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.276 seconds.\n",
      "Reads/sec:       \t217.22\n",
      "kBases/sec:      \t107.82\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.273 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-81_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-82_2020.fasta out=../data_raw/UWHC/USA_WI-UW-82_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-82_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-82_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.729 seconds.\n",
      "Analyzed Index:   \t2.468 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.253 seconds.\n",
      "Reads/sec:       \t236.92\n",
      "kBases/sec:      \t117.60\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "semiperfect site:\t 91.6667% \t       55 \t 91.6057% \t       27282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.777 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-82_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-84_2020.fasta out=../data_raw/UWHC/USA_WI-UW-84_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-84_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-84_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.691 seconds.\n",
      "Analyzed Index:   \t2.829 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.368 seconds.\n",
      "Reads/sec:       \t163.02\n",
      "kBases/sec:      \t80.92\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.234 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-84_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-85_2020.fasta out=../data_raw/UWHC/USA_WI-UW-85_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-85_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-85_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.675 seconds.\n",
      "Analyzed Index:   \t2.998 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.334 seconds.\n",
      "Reads/sec:       \t179.58\n",
      "kBases/sec:      \t89.14\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.338 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-85_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-86_2020.fasta out=../data_raw/UWHC/USA_WI-UW-86_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-86_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-86_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.704 seconds.\n",
      "Analyzed Index:   \t3.010 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.270 seconds.\n",
      "Reads/sec:       \t222.11\n",
      "kBases/sec:      \t110.25\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.311 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-86_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-87_2020.fasta out=../data_raw/UWHC/USA_WI-UW-87_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-87_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-87_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.696 seconds.\n",
      "Analyzed Index:   \t2.486 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.235 seconds.\n",
      "Reads/sec:       \t255.80\n",
      "kBases/sec:      \t126.97\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.738 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-87_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-88_2020.fasta out=../data_raw/UWHC/USA_WI-UW-88_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-88_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-88_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.716 seconds.\n",
      "Analyzed Index:   \t3.031 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.090 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.281 seconds.\n",
      "Reads/sec:       \t213.27\n",
      "kBases/sec:      \t105.86\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "semiperfect site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.0061% \t       29486\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.9570% \t         285\n",
      "\n",
      "Total time:     \t4.366 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-88_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-89_2020.fasta out=../data_raw/UWHC/USA_WI-UW-89_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-89_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-89_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.772 seconds.\n",
      "Analyzed Index:   \t3.038 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.284 seconds.\n",
      "Reads/sec:       \t211.50\n",
      "kBases/sec:      \t104.98\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.445 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-89_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-90_2020.fasta out=../data_raw/UWHC/USA_WI-UW-90_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-90_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-90_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.737 seconds.\n",
      "Analyzed Index:   \t3.002 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.290 seconds.\n",
      "Reads/sec:       \t206.94\n",
      "kBases/sec:      \t102.72\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.9799% \t       26500\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.9799% \t       26500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9496% \t       29767\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0269% \t           8\n",
      "\n",
      "Total time:     \t4.365 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-90_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-91_2020.fasta out=../data_raw/UWHC/USA_WI-UW-91_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-91_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-91_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.721 seconds.\n",
      "Analyzed Index:   \t3.147 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.088 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29611 bases)\n",
      "\n",
      "Mapping:          \t0.299 seconds.\n",
      "Reads/sec:       \t200.75\n",
      "kBases/sec:      \t99.07\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.1801% \t       26111\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.1801% \t       26111\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9629% \t       29600\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0371% \t          11\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0371% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.508 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-91_2020.fasta.pileup\n",
      "29611 bases in pileup file\n",
      "29611 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-92_2020.fasta out=../data_raw/UWHC/USA_WI-UW-92_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-92_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-92_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.707 seconds.\n",
      "Analyzed Index:   \t2.477 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.249 seconds.\n",
      "Reads/sec:       \t240.79\n",
      "kBases/sec:      \t119.52\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.763 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-92_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-93_2020.fasta out=../data_raw/UWHC/USA_WI-UW-93_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-93_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-93_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.035 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.771 seconds.\n",
      "Analyzed Index:   \t3.612 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.337 seconds.\n",
      "Reads/sec:       \t178.21\n",
      "kBases/sec:      \t88.46\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.091 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-93_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-94_2020.fasta out=../data_raw/UWHC/USA_WI-UW-94_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-94_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-94_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.719 seconds.\n",
      "Analyzed Index:   \t2.984 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.306 seconds.\n",
      "Reads/sec:       \t196.00\n",
      "kBases/sec:      \t97.29\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.347 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-94_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-95_2020.fasta out=../data_raw/UWHC/USA_WI-UW-95_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-95_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-95_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.034 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.725 seconds.\n",
      "Analyzed Index:   \t2.832 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.265 seconds.\n",
      "Reads/sec:       \t226.75\n",
      "kBases/sec:      \t112.55\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.171 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-95_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-96_2020.fasta out=../data_raw/UWHC/USA_WI-UW-96_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-96_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-96_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.705 seconds.\n",
      "Analyzed Index:   \t2.970 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.074 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.254 seconds.\n",
      "Reads/sec:       \t235.91\n",
      "kBases/sec:      \t117.10\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 80.0000% \t       48 \t 80.5856% \t       24000\n",
      "semiperfect site:\t 80.0000% \t       48 \t 80.5856% \t       24000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9563% \t       29769\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.253 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-96_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-97_2020.fasta out=../data_raw/UWHC/USA_WI-UW-97_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-97_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-97_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.716 seconds.\n",
      "Analyzed Index:   \t2.999 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29611 bases)\n",
      "\n",
      "Mapping:          \t0.252 seconds.\n",
      "Reads/sec:       \t237.72\n",
      "kBases/sec:      \t117.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.8686% \t       26611\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.8686% \t       26611\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9764% \t       29604\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0236% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0236% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.297 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-97_2020.fasta.pileup\n",
      "29611 bases in pileup file\n",
      "29611 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-98_2020.fasta out=../data_raw/UWHC/USA_WI-UW-98_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-98_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-98_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.737 seconds.\n",
      "Analyzed Index:   \t2.987 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.075 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.250 seconds.\n",
      "Reads/sec:       \t240.14\n",
      "kBases/sec:      \t119.20\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.310 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-98_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI-UW-99_2020.fasta out=../data_raw/UWHC/USA_WI-UW-99_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI-UW-99_2020.fasta, out=../data_raw/UWHC/USA_WI-UW-99_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.694 seconds.\n",
      "Analyzed Index:   \t3.730 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.128 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.315 seconds.\n",
      "Reads/sec:       \t190.32\n",
      "kBases/sec:      \t94.47\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.124 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI-UW-99_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/UWHC/USA_WI1_2020.fasta out=../data_raw/UWHC/USA_WI1_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/UWHC/USA_WI1_2020.fasta, out=../data_raw/UWHC/USA_WI1_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.743 seconds.\n",
      "Analyzed Index:   \t3.215 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.076 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29879 bases)\n",
      "\n",
      "Mapping:          \t0.447 seconds.\n",
      "Reads/sec:       \t134.21\n",
      "kBases/sec:      \t66.84\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29879\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29879\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 96.6532% \t       28879\n",
      "semiperfect site:\t 96.6667% \t       58 \t 96.6532% \t       28879\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9866% \t       29878\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0134% \t           4\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0033% \t           1\n",
      "Del Rate:        \t  1.6667% \t        1 \t  0.0100% \t           3\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.737 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/UWHC/USA_WI1_2020.fasta.pileup\n",
      "29882 bases in pileup file\n",
      "29882 met minimum coverage of 1x\n",
      "1 SNPs predicted\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# UWHC samples, this takes ~10 mins to run \n",
    "\n",
    "for file in $(ls ../data_raw/UWHC/*.fasta)\n",
    "do \n",
    "    bbmap.sh in=$file out=$file.bam ref=ref.fa #map to Wuhan reference\n",
    "    samtools sort $file.bam -o $file.bam # sort the bam \n",
    "    samtools mpileup -f ref.fa $file.bam > $file.pileup # generate a pileup file\n",
    "    java -jar VarScan.jar pileup2snp $file.pileup --min-coverage 1 --min-reads2 1 > $file.vcf\n",
    "done\n",
    "\n",
    "##"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-156_2020.fasta out=../data_raw/MHD/USA_WI-UW-156_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-156_2020.fasta, out=../data_raw/MHD/USA_WI-UW-156_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.740 seconds.\n",
      "Analyzed Index:   \t3.478 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.404 seconds.\n",
      "Reads/sec:       \t148.35\n",
      "kBases/sec:      \t73.64\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.968 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-156_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-157_2020.fasta out=../data_raw/MHD/USA_WI-UW-157_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-157_2020.fasta, out=../data_raw/MHD/USA_WI-UW-157_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.743 seconds.\n",
      "Analyzed Index:   \t2.429 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.093 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.281 seconds.\n",
      "Reads/sec:       \t213.78\n",
      "kBases/sec:      \t106.11\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.803 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-157_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-158_2020.fasta out=../data_raw/MHD/USA_WI-UW-158_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-158_2020.fasta, out=../data_raw/MHD/USA_WI-UW-158_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.731 seconds.\n",
      "Analyzed Index:   \t2.454 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.264 seconds.\n",
      "Reads/sec:       \t227.49\n",
      "kBases/sec:      \t112.92\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.791 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-158_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-159_2020.fasta out=../data_raw/MHD/USA_WI-UW-159_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-159_2020.fasta, out=../data_raw/MHD/USA_WI-UW-159_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.716 seconds.\n",
      "Analyzed Index:   \t3.014 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.216 seconds.\n",
      "Reads/sec:       \t277.59\n",
      "kBases/sec:      \t137.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.278 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-159_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-160_2020.fasta out=../data_raw/MHD/USA_WI-UW-160_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-160_2020.fasta, out=../data_raw/MHD/USA_WI-UW-160_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.732 seconds.\n",
      "Analyzed Index:   \t2.504 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.089 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.260 seconds.\n",
      "Reads/sec:       \t231.16\n",
      "kBases/sec:      \t114.74\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.844 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-160_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-161_2020.fasta out=../data_raw/MHD/USA_WI-UW-161_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-161_2020.fasta, out=../data_raw/MHD/USA_WI-UW-161_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.737 seconds.\n",
      "Analyzed Index:   \t3.029 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.293 seconds.\n",
      "Reads/sec:       \t204.85\n",
      "kBases/sec:      \t101.68\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "semiperfect site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9899% \t       29779\n",
      "Error Rate:      \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Sub Rate:        \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.395 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-161_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "3 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-162_2020.fasta out=../data_raw/MHD/USA_WI-UW-162_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-162_2020.fasta, out=../data_raw/MHD/USA_WI-UW-162_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.703 seconds.\n",
      "Analyzed Index:   \t3.042 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.310 seconds.\n",
      "Reads/sec:       \t193.49\n",
      "kBases/sec:      \t96.04\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.388 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-162_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-165_2020.fasta out=../data_raw/MHD/USA_WI-UW-165_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-165_2020.fasta, out=../data_raw/MHD/USA_WI-UW-165_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.725 seconds.\n",
      "Analyzed Index:   \t3.181 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.095 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.283 seconds.\n",
      "Reads/sec:       \t212.23\n",
      "kBases/sec:      \t105.34\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.555 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-165_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-166_2020.fasta out=../data_raw/MHD/USA_WI-UW-166_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-166_2020.fasta, out=../data_raw/MHD/USA_WI-UW-166_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.727 seconds.\n",
      "Analyzed Index:   \t3.029 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.234 seconds.\n",
      "Reads/sec:       \t256.66\n",
      "kBases/sec:      \t127.40\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.354 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-166_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-168_2020.fasta out=../data_raw/MHD/USA_WI-UW-168_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-168_2020.fasta, out=../data_raw/MHD/USA_WI-UW-168_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.035 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.859 seconds.\n",
      "Analyzed Index:   \t3.278 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.098 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.315 seconds.\n",
      "Reads/sec:       \t190.66\n",
      "kBases/sec:      \t94.64\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.967 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-168_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-170_2020.fasta out=../data_raw/MHD/USA_WI-UW-170_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-170_2020.fasta, out=../data_raw/MHD/USA_WI-UW-170_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.789 seconds.\n",
      "Analyzed Index:   \t2.773 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.095 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.258 seconds.\n",
      "Reads/sec:       \t232.78\n",
      "kBases/sec:      \t115.54\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.196 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-170_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-171_2020.fasta out=../data_raw/MHD/USA_WI-UW-171_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-171_2020.fasta, out=../data_raw/MHD/USA_WI-UW-171_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.031 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.775 seconds.\n",
      "Analyzed Index:   \t3.401 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.283 seconds.\n",
      "Reads/sec:       \t211.96\n",
      "kBases/sec:      \t105.21\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.819 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-171_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-172_2020.fasta out=../data_raw/MHD/USA_WI-UW-172_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-172_2020.fasta, out=../data_raw/MHD/USA_WI-UW-172_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.693 seconds.\n",
      "Analyzed Index:   \t2.532 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.094 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.232 seconds.\n",
      "Reads/sec:       \t258.15\n",
      "kBases/sec:      \t128.14\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.803 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-172_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-173_2020.fasta out=../data_raw/MHD/USA_WI-UW-173_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-173_2020.fasta, out=../data_raw/MHD/USA_WI-UW-173_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.721 seconds.\n",
      "Analyzed Index:   \t3.333 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.168 seconds.\n",
      "Cleared Memory:    \t0.133 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.262 seconds.\n",
      "Reads/sec:       \t228.95\n",
      "kBases/sec:      \t113.64\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.772 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-173_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-176_2020.fasta out=../data_raw/MHD/USA_WI-UW-176_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-176_2020.fasta, out=../data_raw/MHD/USA_WI-UW-176_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.891 seconds.\n",
      "Analyzed Index:   \t2.585 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.254 seconds.\n",
      "Reads/sec:       \t236.22\n",
      "kBases/sec:      \t117.25\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.069 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-176_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-177_2020.fasta out=../data_raw/MHD/USA_WI-UW-177_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-177_2020.fasta, out=../data_raw/MHD/USA_WI-UW-177_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.729 seconds.\n",
      "Analyzed Index:   \t3.008 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.254 seconds.\n",
      "Reads/sec:       \t236.48\n",
      "kBases/sec:      \t117.38\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.326 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-177_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-178_2020.fasta out=../data_raw/MHD/USA_WI-UW-178_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-178_2020.fasta, out=../data_raw/MHD/USA_WI-UW-178_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.713 seconds.\n",
      "Analyzed Index:   \t2.750 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.251 seconds.\n",
      "Reads/sec:       \t239.16\n",
      "kBases/sec:      \t118.71\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.060 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-178_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-179_2020.fasta out=../data_raw/MHD/USA_WI-UW-179_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-179_2020.fasta, out=../data_raw/MHD/USA_WI-UW-179_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.778 seconds.\n",
      "Analyzed Index:   \t2.702 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.261 seconds.\n",
      "Reads/sec:       \t229.53\n",
      "kBases/sec:      \t113.93\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "semiperfect site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9899% \t       29779\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.112 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-179_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-181_2020.fasta out=../data_raw/MHD/USA_WI-UW-181_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-181_2020.fasta, out=../data_raw/MHD/USA_WI-UW-181_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.770 seconds.\n",
      "Analyzed Index:   \t3.300 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.265 seconds.\n",
      "Reads/sec:       \t226.76\n",
      "kBases/sec:      \t112.56\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.681 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-181_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-182_2020.fasta out=../data_raw/MHD/USA_WI-UW-182_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-182_2020.fasta, out=../data_raw/MHD/USA_WI-UW-182_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.747 seconds.\n",
      "Analyzed Index:   \t3.018 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.088 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.252 seconds.\n",
      "Reads/sec:       \t237.80\n",
      "kBases/sec:      \t118.04\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.367 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-182_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-183_2020.fasta out=../data_raw/MHD/USA_WI-UW-183_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-183_2020.fasta, out=../data_raw/MHD/USA_WI-UW-183_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.725 seconds.\n",
      "Analyzed Index:   \t3.027 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.264 seconds.\n",
      "Reads/sec:       \t227.36\n",
      "kBases/sec:      \t112.85\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "semiperfect site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9899% \t       29779\n",
      "Error Rate:      \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Sub Rate:        \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.373 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-183_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "3 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-184_2020.fasta out=../data_raw/MHD/USA_WI-UW-184_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-184_2020.fasta, out=../data_raw/MHD/USA_WI-UW-184_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.718 seconds.\n",
      "Analyzed Index:   \t3.393 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.097 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.206 seconds.\n",
      "Reads/sec:       \t291.66\n",
      "kBases/sec:      \t144.77\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 80.0000% \t       48 \t 80.5856% \t       24000\n",
      "semiperfect site:\t 80.0000% \t       48 \t 80.5856% \t       24000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9563% \t       29769\n",
      "Error Rate:      \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 20.0000% \t       12 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.672 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-184_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-185_2020.fasta out=../data_raw/MHD/USA_WI-UW-185_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-185_2020.fasta, out=../data_raw/MHD/USA_WI-UW-185_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.761 seconds.\n",
      "Analyzed Index:   \t2.611 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.265 seconds.\n",
      "Reads/sec:       \t226.61\n",
      "kBases/sec:      \t112.48\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.004 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-185_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-186_2020.fasta out=../data_raw/MHD/USA_WI-UW-186_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-186_2020.fasta, out=../data_raw/MHD/USA_WI-UW-186_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.725 seconds.\n",
      "Analyzed Index:   \t3.008 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.271 seconds.\n",
      "Reads/sec:       \t221.78\n",
      "kBases/sec:      \t110.08\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.375 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-186_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-187_2020.fasta out=../data_raw/MHD/USA_WI-UW-187_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-187_2020.fasta, out=../data_raw/MHD/USA_WI-UW-187_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.753 seconds.\n",
      "Analyzed Index:   \t3.036 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.099 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.268 seconds.\n",
      "Reads/sec:       \t224.08\n",
      "kBases/sec:      \t111.22\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.409 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-187_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-188_2020.fasta out=../data_raw/MHD/USA_WI-UW-188_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-188_2020.fasta, out=../data_raw/MHD/USA_WI-UW-188_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.720 seconds.\n",
      "Analyzed Index:   \t3.030 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.100 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.201 seconds.\n",
      "Reads/sec:       \t297.91\n",
      "kBases/sec:      \t147.87\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9624% \t       29271\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0239% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0239% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0137% \t           4\n",
      "\n",
      "Total time:     \t4.307 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-188_2020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-189_2020.fasta out=../data_raw/MHD/USA_WI-UW-189_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-189_2020.fasta, out=../data_raw/MHD/USA_WI-UW-189_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.727 seconds.\n",
      "Analyzed Index:   \t3.071 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.257 seconds.\n",
      "Reads/sec:       \t233.50\n",
      "kBases/sec:      \t115.90\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.9433% \t       25000\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.9433% \t       25000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9563% \t       29769\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.420 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-189_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-190_2020.fasta out=../data_raw/MHD/USA_WI-UW-190_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-190_2020.fasta, out=../data_raw/MHD/USA_WI-UW-190_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.033 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.737 seconds.\n",
      "Analyzed Index:   \t3.082 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.088 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.269 seconds.\n",
      "Reads/sec:       \t223.38\n",
      "kBases/sec:      \t110.88\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.464 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-190_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-191_2020.fasta out=../data_raw/MHD/USA_WI-UW-191_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-191_2020.fasta, out=../data_raw/MHD/USA_WI-UW-191_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.703 seconds.\n",
      "Analyzed Index:   \t3.024 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.095 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.270 seconds.\n",
      "Reads/sec:       \t222.56\n",
      "kBases/sec:      \t110.47\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "semiperfect site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9899% \t       29779\n",
      "Error Rate:      \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Sub Rate:        \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.345 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-191_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "3 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-192_2020.fasta out=../data_raw/MHD/USA_WI-UW-192_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-192_2020.fasta, out=../data_raw/MHD/USA_WI-UW-192_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.713 seconds.\n",
      "Analyzed Index:   \t2.683 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.225 seconds.\n",
      "Reads/sec:       \t267.08\n",
      "kBases/sec:      \t132.57\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.954 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-192_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-195_2020.fasta out=../data_raw/MHD/USA_WI-UW-195_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-195_2020.fasta, out=../data_raw/MHD/USA_WI-UW-195_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.712 seconds.\n",
      "Analyzed Index:   \t2.510 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.255 seconds.\n",
      "Reads/sec:       \t235.04\n",
      "kBases/sec:      \t116.66\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.816 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-195_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-196_2020.fasta out=../data_raw/MHD/USA_WI-UW-196_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-196_2020.fasta, out=../data_raw/MHD/USA_WI-UW-196_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.723 seconds.\n",
      "Analyzed Index:   \t3.045 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.259 seconds.\n",
      "Reads/sec:       \t231.64\n",
      "kBases/sec:      \t114.98\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.372 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-196_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-197_2020.fasta out=../data_raw/MHD/USA_WI-UW-197_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-197_2020.fasta, out=../data_raw/MHD/USA_WI-UW-197_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.760 seconds.\n",
      "Analyzed Index:   \t3.005 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.124 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.250 seconds.\n",
      "Reads/sec:       \t240.46\n",
      "kBases/sec:      \t119.36\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.373 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-197_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-200_2020.fasta out=../data_raw/MHD/USA_WI-UW-200_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-200_2020.fasta, out=../data_raw/MHD/USA_WI-UW-200_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.699 seconds.\n",
      "Analyzed Index:   \t2.995 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.260 seconds.\n",
      "Reads/sec:       \t230.45\n",
      "kBases/sec:      \t114.39\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.319 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-200_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-205_2020.fasta out=../data_raw/MHD/USA_WI-UW-205_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-205_2020.fasta, out=../data_raw/MHD/USA_WI-UW-205_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.728 seconds.\n",
      "Analyzed Index:   \t2.998 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.332 seconds.\n",
      "Reads/sec:       \t180.66\n",
      "kBases/sec:      \t89.68\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.394 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-205_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-207_2020.fasta out=../data_raw/MHD/USA_WI-UW-207_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-207_2020.fasta, out=../data_raw/MHD/USA_WI-UW-207_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.720 seconds.\n",
      "Analyzed Index:   \t3.105 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.203 seconds.\n",
      "Reads/sec:       \t296.10\n",
      "kBases/sec:      \t146.97\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "semiperfect site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9866% \t       29778\n",
      "Error Rate:      \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "Sub Rate:        \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.386 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-207_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "4 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-208_2020.fasta out=../data_raw/MHD/USA_WI-UW-208_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-208_2020.fasta, out=../data_raw/MHD/USA_WI-UW-208_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.684 seconds.\n",
      "Analyzed Index:   \t3.019 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.252 seconds.\n",
      "Reads/sec:       \t238.19\n",
      "kBases/sec:      \t118.23\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.297 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-208_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-209_2020.fasta out=../data_raw/MHD/USA_WI-UW-209_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-209_2020.fasta, out=../data_raw/MHD/USA_WI-UW-209_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.702 seconds.\n",
      "Analyzed Index:   \t3.177 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.212 seconds.\n",
      "Reads/sec:       \t283.63\n",
      "kBases/sec:      \t140.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.439 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-209_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-210_2020.fasta out=../data_raw/MHD/USA_WI-UW-210_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-210_2020.fasta, out=../data_raw/MHD/USA_WI-UW-210_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.727 seconds.\n",
      "Analyzed Index:   \t3.075 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.092 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.206 seconds.\n",
      "Reads/sec:       \t290.93\n",
      "kBases/sec:      \t144.41\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9899% \t       29779\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0101% \t           3\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0101% \t           3\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.371 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-210_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "3 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-211_2020.fasta out=../data_raw/MHD/USA_WI-UW-211_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-211_2020.fasta, out=../data_raw/MHD/USA_WI-UW-211_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.675 seconds.\n",
      "Analyzed Index:   \t2.983 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.095 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.254 seconds.\n",
      "Reads/sec:       \t235.78\n",
      "kBases/sec:      \t117.03\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.260 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-211_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-212_2020.fasta out=../data_raw/MHD/USA_WI-UW-212_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-212_2020.fasta, out=../data_raw/MHD/USA_WI-UW-212_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.688 seconds.\n",
      "Analyzed Index:   \t2.998 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.106 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.237 seconds.\n",
      "Reads/sec:       \t253.22\n",
      "kBases/sec:      \t125.69\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.286 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-212_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-213_2020.fasta out=../data_raw/MHD/USA_WI-UW-213_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-213_2020.fasta, out=../data_raw/MHD/USA_WI-UW-213_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.697 seconds.\n",
      "Analyzed Index:   \t3.052 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.268 seconds.\n",
      "Reads/sec:       \t224.29\n",
      "kBases/sec:      \t111.33\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.354 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-213_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-215_2020.fasta out=../data_raw/MHD/USA_WI-UW-215_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-215_2020.fasta, out=../data_raw/MHD/USA_WI-UW-215_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.701 seconds.\n",
      "Analyzed Index:   \t3.048 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.247 seconds.\n",
      "Reads/sec:       \t243.12\n",
      "kBases/sec:      \t120.68\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.331 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-215_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-216_2020.fasta out=../data_raw/MHD/USA_WI-UW-216_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-216_2020.fasta, out=../data_raw/MHD/USA_WI-UW-216_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.715 seconds.\n",
      "Analyzed Index:   \t3.080 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.243 seconds.\n",
      "Reads/sec:       \t246.65\n",
      "kBases/sec:      \t122.43\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9899% \t       29779\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0101% \t           3\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0101% \t           3\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.374 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-216_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "3 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-217_2020.fasta out=../data_raw/MHD/USA_WI-UW-217_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-217_2020.fasta, out=../data_raw/MHD/USA_WI-UW-217_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.736 seconds.\n",
      "Analyzed Index:   \t3.248 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.258 seconds.\n",
      "Reads/sec:       \t232.48\n",
      "kBases/sec:      \t115.39\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.569 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-217_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-218_2020.fasta out=../data_raw/MHD/USA_WI-UW-218_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-218_2020.fasta, out=../data_raw/MHD/USA_WI-UW-218_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.708 seconds.\n",
      "Analyzed Index:   \t3.085 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.088 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.262 seconds.\n",
      "Reads/sec:       \t229.30\n",
      "kBases/sec:      \t113.82\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "semiperfect site:\t 85.0000% \t       51 \t 85.6222% \t       25500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9631% \t       29771\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.394 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-218_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-219_2020.fasta out=../data_raw/MHD/USA_WI-UW-219_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-219_2020.fasta, out=../data_raw/MHD/USA_WI-UW-219_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.716 seconds.\n",
      "Analyzed Index:   \t3.010 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.242 seconds.\n",
      "Reads/sec:       \t247.59\n",
      "kBases/sec:      \t122.89\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.332 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-219_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-221_2020.fasta out=../data_raw/MHD/USA_WI-UW-221_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-221_2020.fasta, out=../data_raw/MHD/USA_WI-UW-221_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.788 seconds.\n",
      "Analyzed Index:   \t3.006 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.246 seconds.\n",
      "Reads/sec:       \t244.23\n",
      "kBases/sec:      \t121.23\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.381 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-221_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-222_2020.fasta out=../data_raw/MHD/USA_WI-UW-222_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-222_2020.fasta, out=../data_raw/MHD/USA_WI-UW-222_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.708 seconds.\n",
      "Analyzed Index:   \t3.069 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.259 seconds.\n",
      "Reads/sec:       \t231.26\n",
      "kBases/sec:      \t114.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.383 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-222_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-223_2020.fasta out=../data_raw/MHD/USA_WI-UW-223_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-223_2020.fasta, out=../data_raw/MHD/USA_WI-UW-223_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.714 seconds.\n",
      "Analyzed Index:   \t3.015 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.242 seconds.\n",
      "Reads/sec:       \t247.46\n",
      "kBases/sec:      \t122.83\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.310 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-223_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-225_2020.fasta out=../data_raw/MHD/USA_WI-UW-225_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-225_2020.fasta, out=../data_raw/MHD/USA_WI-UW-225_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.710 seconds.\n",
      "Analyzed Index:   \t3.066 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.267 seconds.\n",
      "Reads/sec:       \t224.51\n",
      "kBases/sec:      \t111.44\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.371 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-225_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-226_2020.fasta out=../data_raw/MHD/USA_WI-UW-226_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-226_2020.fasta, out=../data_raw/MHD/USA_WI-UW-226_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.731 seconds.\n",
      "Analyzed Index:   \t3.013 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.092 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.240 seconds.\n",
      "Reads/sec:       \t250.45\n",
      "kBases/sec:      \t124.31\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.322 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-226_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-227_2020.fasta out=../data_raw/MHD/USA_WI-UW-227_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-227_2020.fasta, out=../data_raw/MHD/USA_WI-UW-227_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.701 seconds.\n",
      "Analyzed Index:   \t2.939 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.244 seconds.\n",
      "Reads/sec:       \t245.75\n",
      "kBases/sec:      \t121.98\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "semiperfect site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.224 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-227_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-228_2020.fasta out=../data_raw/MHD/USA_WI-UW-228_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-228_2020.fasta, out=../data_raw/MHD/USA_WI-UW-228_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.693 seconds.\n",
      "Analyzed Index:   \t2.953 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.245 seconds.\n",
      "Reads/sec:       \t245.37\n",
      "kBases/sec:      \t121.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "semiperfect site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9899% \t       29779\n",
      "Error Rate:      \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Sub Rate:        \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.227 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-228_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "3 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-229_2020.fasta out=../data_raw/MHD/USA_WI-UW-229_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-229_2020.fasta, out=../data_raw/MHD/USA_WI-UW-229_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.727 seconds.\n",
      "Analyzed Index:   \t3.100 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.256 seconds.\n",
      "Reads/sec:       \t234.60\n",
      "kBases/sec:      \t116.45\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.417 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-229_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-230_2020.fasta out=../data_raw/MHD/USA_WI-UW-230_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-230_2020.fasta, out=../data_raw/MHD/USA_WI-UW-230_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.694 seconds.\n",
      "Analyzed Index:   \t3.075 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.253 seconds.\n",
      "Reads/sec:       \t236.83\n",
      "kBases/sec:      \t117.55\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.357 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-230_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-231_2020.fasta out=../data_raw/MHD/USA_WI-UW-231_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-231_2020.fasta, out=../data_raw/MHD/USA_WI-UW-231_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.697 seconds.\n",
      "Analyzed Index:   \t2.423 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.242 seconds.\n",
      "Reads/sec:       \t248.10\n",
      "kBases/sec:      \t123.15\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.689 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-231_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-232_2020.fasta out=../data_raw/MHD/USA_WI-UW-232_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-232_2020.fasta, out=../data_raw/MHD/USA_WI-UW-232_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.731 seconds.\n",
      "Analyzed Index:   \t2.474 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.255 seconds.\n",
      "Reads/sec:       \t235.29\n",
      "kBases/sec:      \t116.79\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.790 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-232_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-233_2020.fasta out=../data_raw/MHD/USA_WI-UW-233_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-233_2020.fasta, out=../data_raw/MHD/USA_WI-UW-233_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.761 seconds.\n",
      "Analyzed Index:   \t3.015 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.235 seconds.\n",
      "Reads/sec:       \t255.39\n",
      "kBases/sec:      \t126.77\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.349 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-233_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-234_2020.fasta out=../data_raw/MHD/USA_WI-UW-234_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-234_2020.fasta, out=../data_raw/MHD/USA_WI-UW-234_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.716 seconds.\n",
      "Analyzed Index:   \t3.019 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.275 seconds.\n",
      "Reads/sec:       \t218.03\n",
      "kBases/sec:      \t108.22\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.349 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-234_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-235_2020.fasta out=../data_raw/MHD/USA_WI-UW-235_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-235_2020.fasta, out=../data_raw/MHD/USA_WI-UW-235_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.671 seconds.\n",
      "Analyzed Index:   \t2.479 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.228 seconds.\n",
      "Reads/sec:       \t262.95\n",
      "kBases/sec:      \t130.52\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "semiperfect site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.720 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-235_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-236_2020.fasta out=../data_raw/MHD/USA_WI-UW-236_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-236_2020.fasta, out=../data_raw/MHD/USA_WI-UW-236_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.680 seconds.\n",
      "Analyzed Index:   \t2.587 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.251 seconds.\n",
      "Reads/sec:       \t239.22\n",
      "kBases/sec:      \t118.74\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.848 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-236_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-237_2020.fasta out=../data_raw/MHD/USA_WI-UW-237_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-237_2020.fasta, out=../data_raw/MHD/USA_WI-UW-237_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.709 seconds.\n",
      "Analyzed Index:   \t3.030 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.088 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.249 seconds.\n",
      "Reads/sec:       \t240.91\n",
      "kBases/sec:      \t119.58\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "semiperfect site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Sub Rate:        \t  5.0000% \t        3 \t  0.0101% \t           3\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.338 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-237_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "3 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-238_2020.fasta out=../data_raw/MHD/USA_WI-UW-238_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-238_2020.fasta, out=../data_raw/MHD/USA_WI-UW-238_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.640 seconds.\n",
      "Analyzed Index:   \t3.037 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.090 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.210 seconds.\n",
      "Reads/sec:       \t285.48\n",
      "kBases/sec:      \t141.70\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.229 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-238_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-240_2020.fasta out=../data_raw/MHD/USA_WI-UW-240_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-240_2020.fasta, out=../data_raw/MHD/USA_WI-UW-240_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.709 seconds.\n",
      "Analyzed Index:   \t3.100 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.090 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.231 seconds.\n",
      "Reads/sec:       \t259.91\n",
      "kBases/sec:      \t129.01\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.376 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-240_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-241_2020.fasta out=../data_raw/MHD/USA_WI-UW-241_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-241_2020.fasta, out=../data_raw/MHD/USA_WI-UW-241_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.737 seconds.\n",
      "Analyzed Index:   \t3.020 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.253 seconds.\n",
      "Reads/sec:       \t237.49\n",
      "kBases/sec:      \t117.88\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.347 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-241_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-242_2020.fasta out=../data_raw/MHD/USA_WI-UW-242_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-242_2020.fasta, out=../data_raw/MHD/USA_WI-UW-242_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.713 seconds.\n",
      "Analyzed Index:   \t2.363 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.244 seconds.\n",
      "Reads/sec:       \t245.89\n",
      "kBases/sec:      \t122.05\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.657 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-242_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-245_2020.fasta out=../data_raw/MHD/USA_WI-UW-245_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-245_2020.fasta, out=../data_raw/MHD/USA_WI-UW-245_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.713 seconds.\n",
      "Analyzed Index:   \t3.005 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.253 seconds.\n",
      "Reads/sec:       \t237.48\n",
      "kBases/sec:      \t117.88\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 80.0000% \t       48 \t 80.5856% \t       24000\n",
      "semiperfect site:\t 80.0000% \t       48 \t 80.5856% \t       24000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9530% \t       29768\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.322 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-245_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-246_2020.fasta out=../data_raw/MHD/USA_WI-UW-246_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-246_2020.fasta, out=../data_raw/MHD/USA_WI-UW-246_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.696 seconds.\n",
      "Analyzed Index:   \t3.051 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.281 seconds.\n",
      "Reads/sec:       \t213.55\n",
      "kBases/sec:      \t106.00\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "semiperfect site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.4194% \t       29112\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.5771% \t         169\n",
      "\n",
      "Total time:     \t4.376 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-246_2020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-248_2020.fasta out=../data_raw/MHD/USA_WI-UW-248_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-248_2020.fasta, out=../data_raw/MHD/USA_WI-UW-248_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.738 seconds.\n",
      "Analyzed Index:   \t3.078 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.094 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.237 seconds.\n",
      "Reads/sec:       \t252.64\n",
      "kBases/sec:      \t125.40\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.413 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-248_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-270_2020.fasta out=../data_raw/MHD/USA_WI-UW-270_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-270_2020.fasta, out=../data_raw/MHD/USA_WI-UW-270_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.715 seconds.\n",
      "Analyzed Index:   \t3.024 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.094 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29611 bases)\n",
      "\n",
      "Mapping:          \t0.278 seconds.\n",
      "Reads/sec:       \t215.52\n",
      "kBases/sec:      \t106.36\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29611\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t100.0000% \t       60 \t100.0000% \t       29611\n",
      "semiperfect site:\t100.0000% \t       60 \t100.0000% \t       29611\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t100.0000% \t       29611\n",
      "Error Rate:      \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Sub Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.398 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-270_2020.fasta.pileup\n",
      "29611 bases in pileup file\n",
      "29611 met minimum coverage of 1x\n",
      "0 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-272_2020.fasta out=../data_raw/MHD/USA_WI-UW-272_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-272_2020.fasta, out=../data_raw/MHD/USA_WI-UW-272_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.704 seconds.\n",
      "Analyzed Index:   \t3.028 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.099 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.250 seconds.\n",
      "Reads/sec:       \t240.43\n",
      "kBases/sec:      \t119.34\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.339 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-272_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-273_2020.fasta out=../data_raw/MHD/USA_WI-UW-273_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-273_2020.fasta, out=../data_raw/MHD/USA_WI-UW-273_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.689 seconds.\n",
      "Analyzed Index:   \t3.267 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.271 seconds.\n",
      "Reads/sec:       \t221.34\n",
      "kBases/sec:      \t109.86\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.584 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-273_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-274_2020.fasta out=../data_raw/MHD/USA_WI-UW-274_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-274_2020.fasta, out=../data_raw/MHD/USA_WI-UW-274_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.695 seconds.\n",
      "Analyzed Index:   \t2.839 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.244 seconds.\n",
      "Reads/sec:       \t245.63\n",
      "kBases/sec:      \t121.92\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "semiperfect site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9866% \t       29778\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.117 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-274_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-275_2020.fasta out=../data_raw/MHD/USA_WI-UW-275_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-275_2020.fasta, out=../data_raw/MHD/USA_WI-UW-275_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.706 seconds.\n",
      "Analyzed Index:   \t2.543 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.084 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.284 seconds.\n",
      "Reads/sec:       \t211.27\n",
      "kBases/sec:      \t104.87\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.876 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-275_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-276_2020.fasta out=../data_raw/MHD/USA_WI-UW-276_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-276_2020.fasta, out=../data_raw/MHD/USA_WI-UW-276_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.707 seconds.\n",
      "Analyzed Index:   \t2.720 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.116 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.260 seconds.\n",
      "Reads/sec:       \t230.61\n",
      "kBases/sec:      \t114.47\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.022 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-276_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-277_2020.fasta out=../data_raw/MHD/USA_WI-UW-277_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-277_2020.fasta, out=../data_raw/MHD/USA_WI-UW-277_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.037 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.704 seconds.\n",
      "Analyzed Index:   \t2.389 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.260 seconds.\n",
      "Reads/sec:       \t230.41\n",
      "kBases/sec:      \t114.37\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.707 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-277_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-278_2020.fasta out=../data_raw/MHD/USA_WI-UW-278_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-278_2020.fasta, out=../data_raw/MHD/USA_WI-UW-278_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.690 seconds.\n",
      "Analyzed Index:   \t2.769 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.238 seconds.\n",
      "Reads/sec:       \t251.96\n",
      "kBases/sec:      \t125.06\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "semiperfect site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.021 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-278_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-279_2020.fasta out=../data_raw/MHD/USA_WI-UW-279_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-279_2020.fasta, out=../data_raw/MHD/USA_WI-UW-279_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.701 seconds.\n",
      "Analyzed Index:   \t2.411 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.247 seconds.\n",
      "Reads/sec:       \t242.65\n",
      "kBases/sec:      \t120.44\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "semiperfect site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.682 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-279_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-280_2020.fasta out=../data_raw/MHD/USA_WI-UW-280_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-280_2020.fasta, out=../data_raw/MHD/USA_WI-UW-280_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.686 seconds.\n",
      "Analyzed Index:   \t2.902 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.239 seconds.\n",
      "Reads/sec:       \t250.58\n",
      "kBases/sec:      \t124.38\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "unambiguous:     \t 98.3333% \t       59 \t 98.3211% \t       29282\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  1.6667% \t        1 \t  1.6789% \t         500\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9795% \t       29276\n",
      "Error Rate:      \t  8.4746% \t        5 \t  0.0205% \t           6\n",
      "Sub Rate:        \t  8.4746% \t        5 \t  0.0205% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.160 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-280_2020.fasta.pileup\n",
      "29282 bases in pileup file\n",
      "29282 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-285_2020.fasta out=../data_raw/MHD/USA_WI-UW-285_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-285_2020.fasta, out=../data_raw/MHD/USA_WI-UW-285_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.691 seconds.\n",
      "Analyzed Index:   \t3.055 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.117 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.238 seconds.\n",
      "Reads/sec:       \t252.23\n",
      "kBases/sec:      \t125.20\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.320 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-285_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-288_2020.fasta out=../data_raw/MHD/USA_WI-UW-288_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-288_2020.fasta, out=../data_raw/MHD/USA_WI-UW-288_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.717 seconds.\n",
      "Analyzed Index:   \t2.372 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.117 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.274 seconds.\n",
      "Reads/sec:       \t219.24\n",
      "kBases/sec:      \t108.83\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.706 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-288_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-289_2020.fasta out=../data_raw/MHD/USA_WI-UW-289_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-289_2020.fasta, out=../data_raw/MHD/USA_WI-UW-289_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.681 seconds.\n",
      "Analyzed Index:   \t2.974 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.115 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.250 seconds.\n",
      "Reads/sec:       \t239.85\n",
      "kBases/sec:      \t119.05\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "semiperfect site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.238 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-289_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-290_2020.fasta out=../data_raw/MHD/USA_WI-UW-290_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-290_2020.fasta, out=../data_raw/MHD/USA_WI-UW-290_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.707 seconds.\n",
      "Analyzed Index:   \t2.410 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.094 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.304 seconds.\n",
      "Reads/sec:       \t197.12\n",
      "kBases/sec:      \t97.85\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "semiperfect site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  6.6667% \t        4 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  6.6667% \t        4 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.761 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-290_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-291_2020.fasta out=../data_raw/MHD/USA_WI-UW-291_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-291_2020.fasta, out=../data_raw/MHD/USA_WI-UW-291_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.700 seconds.\n",
      "Analyzed Index:   \t2.822 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.119 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.259 seconds.\n",
      "Reads/sec:       \t231.47\n",
      "kBases/sec:      \t114.89\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "semiperfect site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.119 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-291_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-293_2020.fasta out=../data_raw/MHD/USA_WI-UW-293_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-293_2020.fasta, out=../data_raw/MHD/USA_WI-UW-293_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.681 seconds.\n",
      "Analyzed Index:   \t2.400 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.283 seconds.\n",
      "Reads/sec:       \t211.73\n",
      "kBases/sec:      \t105.09\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 90.6588% \t       27000\n",
      "semiperfect site:\t 90.0000% \t       54 \t 90.6588% \t       27000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.702 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-293_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-296_2020.fasta out=../data_raw/MHD/USA_WI-UW-296_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-296_2020.fasta, out=../data_raw/MHD/USA_WI-UW-296_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.721 seconds.\n",
      "Analyzed Index:   \t2.945 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.195 seconds.\n",
      "Reads/sec:       \t307.47\n",
      "kBases/sec:      \t152.62\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.192 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-296_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-299_2020.fasta out=../data_raw/MHD/USA_WI-UW-299_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-299_2020.fasta, out=../data_raw/MHD/USA_WI-UW-299_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.694 seconds.\n",
      "Analyzed Index:   \t2.986 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.257 seconds.\n",
      "Reads/sec:       \t233.19\n",
      "kBases/sec:      \t115.75\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "semiperfect site:\t 91.6667% \t       55 \t 92.3377% \t       27500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9832% \t       29777\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.285 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-299_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-301_2020.fasta out=../data_raw/MHD/USA_WI-UW-301_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-301_2020.fasta, out=../data_raw/MHD/USA_WI-UW-301_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.694 seconds.\n",
      "Analyzed Index:   \t2.956 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.116 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.196 seconds.\n",
      "Reads/sec:       \t306.66\n",
      "kBases/sec:      \t152.21\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "semiperfect site:\t 86.6667% \t       52 \t 87.3011% \t       26000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.177 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-301_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-302_2020.fasta out=../data_raw/MHD/USA_WI-UW-302_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-302_2020.fasta, out=../data_raw/MHD/USA_WI-UW-302_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.705 seconds.\n",
      "Analyzed Index:   \t3.007 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.080 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.258 seconds.\n",
      "Reads/sec:       \t232.18\n",
      "kBases/sec:      \t115.25\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "semiperfect site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9631% \t       29771\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0369% \t          11\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0369% \t          11\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.305 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-302_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "11 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-303_2020.fasta out=../data_raw/MHD/USA_WI-UW-303_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-303_2020.fasta, out=../data_raw/MHD/USA_WI-UW-303_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.693 seconds.\n",
      "Analyzed Index:   \t2.952 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.252 seconds.\n",
      "Reads/sec:       \t237.80\n",
      "kBases/sec:      \t118.04\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "semiperfect site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9866% \t       29778\n",
      "Error Rate:      \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "Sub Rate:        \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.228 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-303_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "4 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-304_2020.fasta out=../data_raw/MHD/USA_WI-UW-304_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-304_2020.fasta, out=../data_raw/MHD/USA_WI-UW-304_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.731 seconds.\n",
      "Analyzed Index:   \t3.154 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.260 seconds.\n",
      "Reads/sec:       \t230.47\n",
      "kBases/sec:      \t114.40\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.478 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-304_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-305_2020.fasta out=../data_raw/MHD/USA_WI-UW-305_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-305_2020.fasta, out=../data_raw/MHD/USA_WI-UW-305_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.749 seconds.\n",
      "Analyzed Index:   \t3.936 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.141 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.362 seconds.\n",
      "Reads/sec:       \t165.68\n",
      "kBases/sec:      \t82.24\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 90.6588% \t       27000\n",
      "semiperfect site:\t 90.0000% \t       54 \t 90.6588% \t       27000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0201% \t           6\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.465 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-305_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "6 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-306_2020.fasta out=../data_raw/MHD/USA_WI-UW-306_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-306_2020.fasta, out=../data_raw/MHD/USA_WI-UW-306_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.032 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.418 seconds.\n",
      "Analyzed Index:   \t4.283 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.164 seconds.\n",
      "Cleared Memory:    \t0.125 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.231 seconds.\n",
      "Reads/sec:       \t259.81\n",
      "kBases/sec:      \t128.96\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 90.6588% \t       27000\n",
      "semiperfect site:\t 90.0000% \t       54 \t 90.6588% \t       27000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9799% \t       29776\n",
      "Error Rate:      \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Sub Rate:        \t  8.3333% \t        5 \t  0.0168% \t           5\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t6.432 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-306_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "5 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-308_2020.fasta out=../data_raw/MHD/USA_WI-UW-308_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-308_2020.fasta, out=../data_raw/MHD/USA_WI-UW-308_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.707 seconds.\n",
      "Analyzed Index:   \t2.838 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.269 seconds.\n",
      "Reads/sec:       \t223.05\n",
      "kBases/sec:      \t110.72\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.247 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-308_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-309_2020.fasta out=../data_raw/MHD/USA_WI-UW-309_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-309_2020.fasta, out=../data_raw/MHD/USA_WI-UW-309_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.044 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.865 seconds.\n",
      "Analyzed Index:   \t3.876 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.219 seconds.\n",
      "Cleared Memory:    \t0.118 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.396 seconds.\n",
      "Reads/sec:       \t151.67\n",
      "kBases/sec:      \t75.28\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.674 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-309_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-310_2020.fasta out=../data_raw/MHD/USA_WI-UW-310_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-310_2020.fasta, out=../data_raw/MHD/USA_WI-UW-310_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.797 seconds.\n",
      "Analyzed Index:   \t2.719 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.112 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.281 seconds.\n",
      "Reads/sec:       \t213.84\n",
      "kBases/sec:      \t106.14\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "semiperfect site:\t 81.6667% \t       49 \t 82.2645% \t       24500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9631% \t       29771\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "\n",
      "Total time:     \t4.153 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-310_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-313_2020.fasta out=../data_raw/MHD/USA_WI-UW-313_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-313_2020.fasta, out=../data_raw/MHD/USA_WI-UW-313_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.028 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.722 seconds.\n",
      "Analyzed Index:   \t2.801 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.228 seconds.\n",
      "Reads/sec:       \t262.82\n",
      "kBases/sec:      \t130.45\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "semiperfect site:\t 83.3333% \t       50 \t 83.2113% \t       24782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 16.6667% \t       10 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 16.6667% \t       10 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.104 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-313_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-316_2020.fasta out=../data_raw/MHD/USA_WI-UW-316_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-316_2020.fasta, out=../data_raw/MHD/USA_WI-UW-316_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.739 seconds.\n",
      "Analyzed Index:   \t2.702 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.085 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.298 seconds.\n",
      "Reads/sec:       \t201.08\n",
      "kBases/sec:      \t99.81\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.074 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-316_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-319_2020.fasta out=../data_raw/MHD/USA_WI-UW-319_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-319_2020.fasta, out=../data_raw/MHD/USA_WI-UW-319_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.702 seconds.\n",
      "Analyzed Index:   \t2.407 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.272 seconds.\n",
      "Reads/sec:       \t220.39\n",
      "kBases/sec:      \t109.39\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t3.710 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-319_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-320_2020.fasta out=../data_raw/MHD/USA_WI-UW-320_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-320_2020.fasta, out=../data_raw/MHD/USA_WI-UW-320_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.712 seconds.\n",
      "Analyzed Index:   \t2.990 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.082 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.284 seconds.\n",
      "Reads/sec:       \t211.00\n",
      "kBases/sec:      \t104.73\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9664% \t       29772\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0336% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.327 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-320_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-321_2020.fasta out=../data_raw/MHD/USA_WI-UW-321_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-321_2020.fasta, out=../data_raw/MHD/USA_WI-UW-321_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.775 seconds.\n",
      "Analyzed Index:   \t2.827 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.314 seconds.\n",
      "Reads/sec:       \t190.88\n",
      "kBases/sec:      \t94.75\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.246 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-321_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-322_2020.fasta out=../data_raw/MHD/USA_WI-UW-322_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-322_2020.fasta, out=../data_raw/MHD/USA_WI-UW-322_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.026 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.731 seconds.\n",
      "Analyzed Index:   \t2.391 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.090 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.269 seconds.\n",
      "Reads/sec:       \t222.80\n",
      "kBases/sec:      \t110.59\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.732 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-322_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-323_2020.fasta out=../data_raw/MHD/USA_WI-UW-323_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-323_2020.fasta, out=../data_raw/MHD/USA_WI-UW-323_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.806 seconds.\n",
      "Analyzed Index:   \t2.435 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.081 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.239 seconds.\n",
      "Reads/sec:       \t250.68\n",
      "kBases/sec:      \t124.43\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.825 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-323_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-324_2020.fasta out=../data_raw/MHD/USA_WI-UW-324_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-324_2020.fasta, out=../data_raw/MHD/USA_WI-UW-324_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.747 seconds.\n",
      "Analyzed Index:   \t2.439 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.086 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.210 seconds.\n",
      "Reads/sec:       \t285.77\n",
      "kBases/sec:      \t141.85\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t3.732 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-324_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-325_2020.fasta out=../data_raw/MHD/USA_WI-UW-325_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-325_2020.fasta, out=../data_raw/MHD/USA_WI-UW-325_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.766 seconds.\n",
      "Analyzed Index:   \t3.028 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.288 seconds.\n",
      "Reads/sec:       \t208.58\n",
      "kBases/sec:      \t103.53\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "semiperfect site:\t 93.3333% \t       56 \t 94.0165% \t       28000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9866% \t       29778\n",
      "Error Rate:      \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "Sub Rate:        \t  6.6667% \t        4 \t  0.0134% \t           4\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.417 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-325_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "4 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-328_2020.fasta out=../data_raw/MHD/USA_WI-UW-328_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-328_2020.fasta, out=../data_raw/MHD/USA_WI-UW-328_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.049 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.945 seconds.\n",
      "Analyzed Index:   \t3.057 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.079 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.436 seconds.\n",
      "Reads/sec:       \t137.76\n",
      "kBases/sec:      \t68.38\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.936 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-328_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-329_2020.fasta out=../data_raw/MHD/USA_WI-UW-329_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-329_2020.fasta, out=../data_raw/MHD/USA_WI-UW-329_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.030 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.809 seconds.\n",
      "Analyzed Index:   \t4.121 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.093 seconds.\n",
      "Cleared Memory:    \t0.113 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.293 seconds.\n",
      "Reads/sec:       \t204.54\n",
      "kBases/sec:      \t101.53\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "semiperfect site:\t 81.6667% \t       49 \t 81.5325% \t       24282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9563% \t       29769\n",
      "Error Rate:      \t 18.3333% \t       11 \t  0.0437% \t          13\n",
      "Sub Rate:        \t 18.3333% \t       11 \t  0.0437% \t          13\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.583 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-329_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "13 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-330_2020.fasta out=../data_raw/MHD/USA_WI-UW-330_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-330_2020.fasta, out=../data_raw/MHD/USA_WI-UW-330_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.949 seconds.\n",
      "Analyzed Index:   \t4.164 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.126 seconds.\n",
      "Cleared Memory:    \t0.114 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29611 bases)\n",
      "\n",
      "Mapping:          \t0.368 seconds.\n",
      "Reads/sec:       \t163.17\n",
      "kBases/sec:      \t80.53\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 96.6667% \t       58 \t 96.6229% \t       28611\n",
      "unambiguous:     \t 96.6667% \t       58 \t 96.6229% \t       28611\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  1.6667% \t        1 \t  1.6886% \t         500\n",
      "\n",
      "perfect best site:\t 70.0000% \t       42 \t 69.6059% \t       20611\n",
      "semiperfect site:\t 70.0000% \t       42 \t 69.6059% \t       20611\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 97.9099% \t       28013\n",
      "Error Rate:      \t 16.9492% \t       10 \t  0.0350% \t          10\n",
      "Sub Rate:        \t 16.9492% \t       10 \t  0.0350% \t          10\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 11.8644% \t        7 \t  2.0552% \t         588\n",
      "\n",
      "Total time:     \t5.880 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-330_2020.fasta.pileup\n",
      "28611 bases in pileup file\n",
      "28611 met minimum coverage of 1x\n",
      "10 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-331_2020.fasta out=../data_raw/MHD/USA_WI-UW-331_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-331_2020.fasta, out=../data_raw/MHD/USA_WI-UW-331_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.046 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.955 seconds.\n",
      "Analyzed Index:   \t3.585 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.109 seconds.\n",
      "Cleared Memory:    \t0.117 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.291 seconds.\n",
      "Reads/sec:       \t206.14\n",
      "kBases/sec:      \t102.32\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.323 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-331_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-332_2020.fasta out=../data_raw/MHD/USA_WI-UW-332_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-332_2020.fasta, out=../data_raw/MHD/USA_WI-UW-332_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.061 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.212 seconds.\n",
      "Analyzed Index:   \t3.442 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.290 seconds.\n",
      "Cleared Memory:    \t0.130 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.353 seconds.\n",
      "Reads/sec:       \t169.84\n",
      "kBases/sec:      \t84.30\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.697 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-332_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-333_2020.fasta out=../data_raw/MHD/USA_WI-UW-333_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-333_2020.fasta, out=../data_raw/MHD/USA_WI-UW-333_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.029 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.823 seconds.\n",
      "Analyzed Index:   \t4.060 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.141 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.405 seconds.\n",
      "Reads/sec:       \t148.18\n",
      "kBases/sec:      \t73.55\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.770 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-333_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-334_2020.fasta out=../data_raw/MHD/USA_WI-UW-334_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-334_2020.fasta, out=../data_raw/MHD/USA_WI-UW-334_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.022 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t1.005 seconds.\n",
      "Analyzed Index:   \t2.693 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.252 seconds.\n",
      "Reads/sec:       \t238.21\n",
      "kBases/sec:      \t118.24\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.277 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-334_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-335_2020.fasta out=../data_raw/MHD/USA_WI-UW-335_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-335_2020.fasta, out=../data_raw/MHD/USA_WI-UW-335_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.709 seconds.\n",
      "Analyzed Index:   \t3.213 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.366 seconds.\n",
      "Reads/sec:       \t164.08\n",
      "kBases/sec:      \t81.44\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "semiperfect site:\t 90.0000% \t       54 \t 89.9268% \t       26782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 10.0000% \t        6 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.624 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-335_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-336_2020.fasta out=../data_raw/MHD/USA_WI-UW-336_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-336_2020.fasta, out=../data_raw/MHD/USA_WI-UW-336_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.733 seconds.\n",
      "Analyzed Index:   \t2.808 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.309 seconds.\n",
      "Reads/sec:       \t194.10\n",
      "kBases/sec:      \t96.35\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9597% \t       29770\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0403% \t          12\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0403% \t          12\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.183 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-336_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "12 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-337_2020.fasta out=../data_raw/MHD/USA_WI-UW-337_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-337_2020.fasta, out=../data_raw/MHD/USA_WI-UW-337_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.752 seconds.\n",
      "Analyzed Index:   \t2.871 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.089 seconds.\n",
      "Cleared Memory:    \t0.108 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.263 seconds.\n",
      "Reads/sec:       \t228.13\n",
      "kBases/sec:      \t113.23\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8902% \t       25282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9698% \t       29773\n",
      "Error Rate:      \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Sub Rate:        \t 15.0000% \t        9 \t  0.0302% \t           9\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.225 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-337_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "9 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-338_2020.fasta out=../data_raw/MHD/USA_WI-UW-338_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-338_2020.fasta, out=../data_raw/MHD/USA_WI-UW-338_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.758 seconds.\n",
      "Analyzed Index:   \t3.026 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.088 seconds.\n",
      "Cleared Memory:    \t0.107 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.300 seconds.\n",
      "Reads/sec:       \t199.79\n",
      "kBases/sec:      \t99.17\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.2479% \t       26282\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.431 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-338_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-339_2020.fasta out=../data_raw/MHD/USA_WI-UW-339_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-339_2020.fasta, out=../data_raw/MHD/USA_WI-UW-339_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.025 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.743 seconds.\n",
      "Analyzed Index:   \t3.019 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.078 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29611 bases)\n",
      "\n",
      "Mapping:          \t0.317 seconds.\n",
      "Reads/sec:       \t188.99\n",
      "kBases/sec:      \t93.27\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 95.0000% \t       57 \t 94.9343% \t       28111\n",
      "unambiguous:     \t 95.0000% \t       57 \t 94.9343% \t       28111\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  3.3333% \t        2 \t  3.3771% \t        1000\n",
      "\n",
      "perfect best site:\t 76.6667% \t       46 \t 76.3601% \t       22611\n",
      "semiperfect site:\t 76.6667% \t       46 \t 76.3601% \t       22611\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 95.8628% \t       26948\n",
      "Error Rate:      \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Sub Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 18.9655% \t       11 \t  4.1372% \t        1163\n",
      "\n",
      "Total time:     \t4.414 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-339_2020.fasta.pileup\n",
      "28111 bases in pileup file\n",
      "28111 met minimum coverage of 1x\n",
      "0 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-340_2020.fasta out=../data_raw/MHD/USA_WI-UW-340_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-340_2020.fasta, out=../data_raw/MHD/USA_WI-UW-340_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.748 seconds.\n",
      "Analyzed Index:   \t3.155 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.104 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.276 seconds.\n",
      "Reads/sec:       \t217.19\n",
      "kBases/sec:      \t107.81\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.536 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-340_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-341_2020.fasta out=../data_raw/MHD/USA_WI-UW-341_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-341_2020.fasta, out=../data_raw/MHD/USA_WI-UW-341_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.754 seconds.\n",
      "Analyzed Index:   \t4.093 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.111 seconds.\n",
      "Cleared Memory:    \t0.140 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.304 seconds.\n",
      "Reads/sec:       \t197.15\n",
      "kBases/sec:      \t97.86\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "semiperfect site:\t 86.6667% \t       52 \t 86.5691% \t       25782\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9731% \t       29774\n",
      "Error Rate:      \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Sub Rate:        \t 13.3333% \t        8 \t  0.0269% \t           8\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t5.547 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-341_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "8 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-342_2020.fasta out=../data_raw/MHD/USA_WI-UW-342_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-342_2020.fasta, out=../data_raw/MHD/USA_WI-UW-342_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.069 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.931 seconds.\n",
      "Analyzed Index:   \t3.026 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.077 seconds.\n",
      "Cleared Memory:    \t0.109 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.278 seconds.\n",
      "Reads/sec:       \t215.87\n",
      "kBases/sec:      \t107.15\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "semiperfect site:\t 98.3333% \t       59 \t 99.0531% \t       29500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9966% \t       29781\n",
      "Error Rate:      \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Sub Rate:        \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.714 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-342_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-343_2020.fasta out=../data_raw/MHD/USA_WI-UW-343_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-343_2020.fasta, out=../data_raw/MHD/USA_WI-UW-343_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.027 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.754 seconds.\n",
      "Analyzed Index:   \t3.028 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.243 seconds.\n",
      "Reads/sec:       \t246.74\n",
      "kBases/sec:      \t122.47\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "semiperfect site:\t 95.0000% \t       57 \t 95.6954% \t       28500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9899% \t       29779\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  1.6667% \t        1 \t  0.0034% \t           1\n",
      "\n",
      "Total time:     \t4.366 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-343_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-344_2020.fasta out=../data_raw/MHD/USA_WI-UW-344_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-344_2020.fasta, out=../data_raw/MHD/USA_WI-UW-344_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.024 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.711 seconds.\n",
      "Analyzed Index:   \t3.092 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.091 seconds.\n",
      "Cleared Memory:    \t0.112 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.239 seconds.\n",
      "Reads/sec:       \t251.05\n",
      "kBases/sec:      \t124.61\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 88.3333% \t       53 \t 88.9799% \t       26500\n",
      "semiperfect site:\t 88.3333% \t       53 \t 88.9799% \t       26500\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9765% \t       29775\n",
      "Error Rate:      \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Sub Rate:        \t 11.6667% \t        7 \t  0.0235% \t           7\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.382 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-344_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "7 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-345_2020.fasta out=../data_raw/MHD/USA_WI-UW-345_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-345_2020.fasta, out=../data_raw/MHD/USA_WI-UW-345_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.788 seconds.\n",
      "Analyzed Index:   \t2.782 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.087 seconds.\n",
      "Cleared Memory:    \t0.111 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29611 bases)\n",
      "\n",
      "Mapping:          \t0.262 seconds.\n",
      "Reads/sec:       \t229.40\n",
      "kBases/sec:      \t113.21\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t 96.6667% \t       58 \t 96.6229% \t       28611\n",
      "unambiguous:     \t 96.6667% \t       58 \t 96.6229% \t       28611\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  3.3333% \t        2 \t  3.3771% \t        1000\n",
      "\n",
      "perfect best site:\t 85.0000% \t       51 \t 84.8029% \t       25111\n",
      "semiperfect site:\t 85.0000% \t       51 \t 84.8029% \t       25111\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 98.1755% \t       28089\n",
      "Error Rate:      \t  1.7241% \t        1 \t  0.0035% \t           1\n",
      "Sub Rate:        \t  1.7241% \t        1 \t  0.0035% \t           1\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t 10.3448% \t        6 \t  1.8210% \t         521\n",
      "\n",
      "Total time:     \t4.165 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-345_2020.fasta.pileup\n",
      "28611 bases in pileup file\n",
      "28611 met minimum coverage of 1x\n",
      "1 SNPs predicted\n",
      "Max memory cannot be determined.  Attempting to use 3200 MB.\n",
      "If this fails, please add the -Xmx flag (e.g. -Xmx24g) to your command, \n",
      "or run this program qsubbed or from a qlogin session on Genepool, or set ulimit to an appropriate value.\n",
      "java -Djava.library.path=/Users/katbraun/anaconda3/opt/bbmap-38.22-1/jni/ -ea -Xmx3200m -cp /Users/katbraun/anaconda3/opt/bbmap-38.22-1/current/ align2.BBMap build=1 overwrite=true fastareadlen=500 in=../data_raw/MHD/USA_WI-UW-346_2020.fasta out=../data_raw/MHD/USA_WI-UW-346_2020.fasta.bam ref=ref.fa\n",
      "Executing align2.BBMap [build=1, overwrite=true, fastareadlen=500, in=../data_raw/MHD/USA_WI-UW-346_2020.fasta, out=../data_raw/MHD/USA_WI-UW-346_2020.fasta.bam, ref=ref.fa]\n",
      "Version 38.22\n",
      "\n",
      "Retaining first best site only for ambiguous mappings.\n",
      "NOTE:\tIgnoring reference file because it already appears to have been processed.\n",
      "NOTE:\tIf you wish to regenerate the index, please manually delete ref/genome/1/summary.txt\n",
      "Set genome to 1\n",
      "\n",
      "Loaded Reference:\t0.023 seconds.\n",
      "Loading index for chunk 1-1, build 1\n",
      "Generated Index:\t0.747 seconds.\n",
      "Analyzed Index:   \t2.984 seconds.\n",
      "Found samtools 1.9\n",
      "Started output stream:\t0.083 seconds.\n",
      "Cleared Memory:    \t0.110 seconds.\n",
      "Processing reads in single-ended mode.\n",
      "Started read stream.\n",
      "Started 4 mapping threads.\n",
      "Detecting finished threads: 0, 1, 2, 3\n",
      "\n",
      "   ------------------   Results   ------------------   \n",
      "\n",
      "Genome:                \t1\n",
      "Key Length:            \t13\n",
      "Max Indel:             \t16000\n",
      "Minimum Score Ratio:  \t0.56\n",
      "Mapping Mode:         \tnormal\n",
      "Reads Used:           \t60\t(29782 bases)\n",
      "\n",
      "Mapping:          \t0.262 seconds.\n",
      "Reads/sec:       \t228.93\n",
      "kBases/sec:      \t113.63\n",
      "\n",
      "\n",
      "Read 1 data:      \tpct reads\tnum reads \tpct bases\t   num bases\n",
      "\n",
      "mapped:          \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "unambiguous:     \t100.0000% \t       60 \t100.0000% \t       29782\n",
      "ambiguous:       \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "low-Q discards:  \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "perfect best site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "semiperfect site:\t 96.6667% \t       58 \t 97.3743% \t       29000\n",
      "\n",
      "Match Rate:      \t      NA \t       NA \t 99.9933% \t       29780\n",
      "Error Rate:      \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Sub Rate:        \t  3.3333% \t        2 \t  0.0067% \t           2\n",
      "Del Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "Ins Rate:        \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "N Rate:          \t  0.0000% \t        0 \t  0.0000% \t           0\n",
      "\n",
      "Total time:     \t4.326 seconds.\n",
      "[mpileup] 1 samples in 1 input files\n",
      "Warning: No p-value threshold provided, so p-values will not be calculated\n",
      "Min coverage:\t1\n",
      "Min reads2:\t1\n",
      "Min var freq:\t0.01\n",
      "Min avg qual:\t15\n",
      "P-value thresh:\t0.99\n",
      "Reading input from ../data_raw/MHD/USA_WI-UW-346_2020.fasta.pileup\n",
      "29782 bases in pileup file\n",
      "29782 met minimum coverage of 1x\n",
      "2 SNPs predicted\n"
     ]
    }
   ],
   "source": [
    "%%bash \n",
    "\n",
    "# MHD samples, this takes ~10 mins to run \n",
    "\n",
    "for file in $(ls ../data_raw/MHD/*.fasta)\n",
    "do \n",
    "    bbmap.sh in=$file out=$file.bam ref=ref.fa #map to Wuhan reference\n",
    "    samtools sort $file.bam -o $file.bam # sort the bam \n",
    "    samtools mpileup -f ref.fa $file.bam > $file.pileup # generate a pileup file\n",
    "    java -jar VarScan.jar pileup2snp $file.pileup --min-coverage 1 --min-reads2 1 > $file.vcf\n",
    "done\n",
    "\n",
    "##"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {},
   "outputs": [],
   "source": [
    "# move *.vcf files from data_raw to data_derived \n",
    "# UWHC\n",
    "\n",
    "root_path = \"../data_raw/Dane/\"\n",
    "dest_path = \"../data_derived/Dane/VCFs/\"\n",
    "\n",
    "for dirpath, dnames, fnames in os.walk(root_path):    \n",
    "    for f in fnames:\n",
    "        if f.endswith(\".vcf\"):\n",
    "            source_file_path =  os.path.join(dirpath, f)\n",
    "            dest_file_path   =  os.path.join(dest_path, f)\n",
    "            sh.copyfile(source_file_path, dest_file_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "# move *.vcf files from data_raw to data_derived \n",
    "# MHD\n",
    "\n",
    "root_path = \"../data_raw/MKE/\"\n",
    "dest_path = \"../data_derived/MKE/VCFs/\"\n",
    "\n",
    "for dirpath, dnames, fnames in os.walk(root_path):    \n",
    "    for f in fnames:\n",
    "        if f.endswith(\".vcf\"):\n",
    "            source_file_path =  os.path.join(dirpath, f)\n",
    "            dest_file_path   =  os.path.join(dest_path, f)\n",
    "            sh.copyfile(source_file_path, dest_file_path)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## cleanup data_raw directories\n",
    "\n",
    "Get rid of *bam* and *pileup* files generated above. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [],
   "source": [
    "# UWHC cleanup \n",
    "\n",
    "dir_name = \"../data_raw/Dane/\"\n",
    "test = os.listdir(dir_name)\n",
    "\n",
    "for item in test:\n",
    "    if item.endswith(\".bam\"):\n",
    "        os.remove(os.path.join(dir_name, item))\n",
    "        \n",
    "for item in test:\n",
    "    if item.endswith(\".pileup\"):\n",
    "        os.remove(os.path.join(dir_name, item))\n",
    "        \n",
    "for item in test:\n",
    "    if item.endswith(\".vcf\"):\n",
    "        os.remove(os.path.join(dir_name, item))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [],
   "source": [
    "# MHD cleanup \n",
    "\n",
    "dir_name = \"../data_raw/MKE/\"\n",
    "test = os.listdir(dir_name)\n",
    "\n",
    "for item in test:\n",
    "    if item.endswith(\".bam\"):\n",
    "        os.remove(os.path.join(dir_name, item))\n",
    "        \n",
    "for item in test:\n",
    "    if item.endswith(\".pileup\"):\n",
    "        os.remove(os.path.join(dir_name, item))\n",
    "        \n",
    "for item in test:\n",
    "    if item.endswith(\".vcf\"):\n",
    "        os.remove(os.path.join(dir_name, item))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## load VCF files into dataframes"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {},
   "outputs": [],
   "source": [
    "def load_dataframes(path):\n",
    "    filelist = [os.path.splitext(file)[0] for file in os.listdir(path) if file.endswith('.vcf')]\n",
    "    list_of_DFs = []\n",
    "    for file in filelist: \n",
    "#         print(file)\n",
    "        DF = path + '/' + file + '.vcf'\n",
    "        file = pd.read_csv(DF, header=0, sep='\\t', low_memory=False)\n",
    "        file['sample'] = DF\n",
    "        list_of_DFs.append(file)\n",
    "    print(filelist)\n",
    "    return(list_of_DFs)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {},
   "outputs": [],
   "source": [
    "# there are two MDH samples (270 and 339) that don't have any consensus SNVs -- these are identical to Wuhan/1\n",
    "# I am going to write a NaN-only line into the VCFs so the cleaning functions to spit out errors \n",
    "\n",
    "with open('../data_derived/MKE/VCFs/USA_WI-UW-270_2020.fasta.vcf', 'a') as file:\n",
    "    file.write('NaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN')\n",
    "    \n",
    "with open('../data_derived/MKE/VCFs/USA_WI-UW-339_2020.fasta.vcf', 'a') as file:\n",
    "    file.write('NaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN\tNaN')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['USA_WI-UW-356_2020.fasta', 'USA_WI-UW-28_2020.fasta', 'USA_WI-UW-59_2020.fasta', 'USA_WI-UW-44_2020.fasta', 'USA_WI-UW-51_2020.fasta', 'USA_WI-UW-35_2020.fasta', 'USA_WI-UW-116_2020.fasta', 'USA_WI-UW-103_2020.fasta', 'USA_WI-UW-351_2020.fasta', 'USA_WI-UW-119_2020.fasta', 'USA_WI-UW-56_2020.fasta', 'USA_WI-UW-43_2020.fasta', 'USA_WI-UW-359_2020.fasta', 'USA_WI-UW-111_2020.fasta', 'USA_WI-UW-32_2020.fasta', 'USA_WI-UW-27_2020.fasta', 'USA_WI-UW-42_2020.fasta', 'USA_WI-UW-26_2020.fasta', 'USA_WI-UW-33_2020.fasta', 'USA_WI-UW-110_2020.fasta', 'USA_WI-UW-105_2020.fasta', 'USA_WI-UW-50_2020.fasta', 'USA_WI-UW-45_2020.fasta', 'USA_WI-UW-102_2020.fasta', 'USA_WI-UW-117_2020.fasta', 'USA_WI-UW-34_2020.fasta', 'USA_WI-UW-29_2020.fasta', 'USA_WI-UW-58_2020.fasta', 'USA_WI-UW-100_2020.fasta', 'USA_WI-UW-115_2020.fasta', 'USA_WI-UW-52_2020.fasta', 'USA_WI-UW-108_2020.fasta', 'USA_WI-UW-112_2020.fasta', 'USA_WI-UW-107_2020.fasta', 'USA_WI-UW-259_2020.fasta', 'USA_WI-UW-31_2020.fasta', 'USA_WI-UW-40_2020.fasta', 'USA_WI-UW-55_2020.fasta', 'USA_WI-UW-48_2020.fasta', 'USA_WI-UW-39_2020.fasta', 'USA_WI-UW-347_2020.fasta', 'USA_WI-UW-49_2020.fasta', 'USA_WI-UW-38_2020.fasta', 'USA_WI-UW-25_2020.fasta', 'USA_WI-UW-106_2020.fasta', 'USA_WI-UW-54_2020.fasta', 'USA_WI-UW-41_2020.fasta', 'USA_WI-UW-109_2020.fasta', 'USA_WI-UW-114_2020.fasta', 'USA_WI-UW-101_2020.fasta', 'USA_WI-UW-37_2020.fasta', 'USA_WI-UW-46_2020.fasta', 'USA_WI-UW-53_2020.fasta', 'USA_WI-UW-85_2020.fasta', 'USA_WI-UW-90_2020.fasta', 'USA_WI-UW-128_2020.fasta', 'USA_WI-UW-03_2020.fasta', 'USA_WI-UW-120_2020.fasta', 'USA_WI-UW-135_2020.fasta', 'USA_WI-UW-98_2020.fasta', 'USA_WI-UW-72_2020.fasta', 'USA_WI-UW-67_2020.fasta', 'USA_WI-UW-144_2020.fasta', 'USA_WI-UW-151_2020.fasta', 'USA_WI-UW-68_2020.fasta', 'USA_WI-UW-97_2020.fasta', 'USA_WI-UW-82_2020.fasta', 'USA_WI-UW-127_2020.fasta', 'USA_WI-UW-04_2020.fasta', 'USA_WI-UW-60_2020.fasta', 'USA_WI-UW-75_2020.fasta', 'USA_WI-UW-10_2020.fasta', 'USA_WI-UW-05_2020.fasta', 'USA_WI-UW-74_2020.fasta', 'USA_WI-UW-61_2020.fasta', 'USA_WI-UW-69_2020.fasta', 'USA_WI-UW-96_2020.fasta', 'USA_WI-UW-99_2020.fasta', 'USA_WI-UW-02_2020.fasta', 'USA_WI-UW-150_2020.fasta', 'USA_WI-UW-145_2020.fasta', 'USA_WI-UW-66_2020.fasta', 'USA_WI-UW-73_2020.fasta', 'USA_WI-UW-129_2020.fasta', 'USA_WI-UW-91_2020.fasta', 'USA_WI-UW-84_2020.fasta', 'USA_WI-UW-64_2020.fasta', 'USA_WI-UW-71_2020.fasta', 'USA_WI-UW-152_2020.fasta', 'USA_WI-UW-147_2020.fasta', 'USA_WI-UW-93_2020.fasta', 'USA_WI-UW-86_2020.fasta', 'USA_WI-UW-08_2020.fasta', 'USA_WI-UW-260_2020.fasta', 'USA_WI-UW-79_2020.fasta', 'USA_WI-UW-140_2020.fasta', 'USA_WI-UW-155_2020.fasta', 'USA_WI-UW-76_2020.fasta', 'USA_WI-UW-63_2020.fasta', 'USA_WI-UW-89_2020.fasta', 'USA_WI-UW-124_2020.fasta', 'USA_WI-UW-07_2020.fasta', 'USA_WI-UW-81_2020.fasta', 'USA_WI-UW-94_2020.fasta', 'USA_WI-UW-148_2020.fasta', 'USA_WI-UW-95_2020.fasta', 'USA_WI-UW-80_2020.fasta', 'USA_WI-UW-62_2020.fasta', 'USA_WI-UW-77_2020.fasta', 'USA_WI-UW-06_2020.fasta', 'USA_WI1_2020.fasta', 'USA_WI-UW-88_2020.fasta', 'USA_WI-UW-09_2020.fasta', 'USA_WI-UW-362_2020.fasta', 'USA_WI-UW-87_2020.fasta', 'USA_WI-UW-92_2020.fasta', 'USA_WI-UW-78_2020.fasta', 'USA_WI-UW-146_2020.fasta', 'USA_WI-UW-153_2020.fasta', 'USA_WI-UW-70_2020.fasta', 'USA_WI-UW-65_2020.fasta', 'USA_WI-UW-122_2020.fasta']\n",
      "['USA_WI-UW-343_2020.fasta', 'USA_WI-UW-240_2020.fasta', 'USA_WI-UW-185_2020.fasta', 'USA_WI-UW-190_2020.fasta', 'USA_WI-UW-332_2020.fasta', 'USA_WI-UW-231_2020.fasta', 'USA_WI-UW-172_2020.fasta', 'USA_WI-UW-248_2020.fasta', 'USA_WI-UW-197_2020.fasta', 'USA_WI-UW-182_2020.fasta', 'USA_WI-UW-344_2020.fasta', 'USA_WI-UW-223_2020.fasta', 'USA_WI-UW-236_2020.fasta', 'USA_WI-UW-168_2020.fasta', 'USA_WI-UW-335_2020.fasta', 'USA_WI-UW-320_2020.fasta', 'USA_WI-UW-160_2020.fasta', 'USA_WI-UW-328_2020.fasta', 'USA_WI-UW-161_2020.fasta', 'USA_WI-UW-329_2020.fasta', 'USA_WI-UW-345_2020.fasta', 'USA_WI-UW-246_2020.fasta', 'USA_WI-UW-183_2020.fasta', 'USA_WI-UW-196_2020.fasta', 'USA_WI-UW-321_2020.fasta', 'USA_WI-UW-334_2020.fasta', 'USA_WI-UW-237_2020.fasta', 'USA_WI-UW-222_2020.fasta', 'USA_WI-UW-166_2020.fasta', 'USA_WI-UW-173_2020.fasta', 'USA_WI-UW-238_2020.fasta', 'USA_WI-UW-191_2020.fasta', 'USA_WI-UW-184_2020.fasta', 'USA_WI-UW-241_2020.fasta', 'USA_WI-UW-342_2020.fasta', 'USA_WI-UW-225_2020.fasta', 'USA_WI-UW-230_2020.fasta', 'USA_WI-UW-333_2020.fasta', 'USA_WI-UW-171_2020.fasta', 'USA_WI-UW-339_2020.fasta', 'USA_WI-UW-179_2020.fasta', 'USA_WI-UW-331_2020.fasta', 'USA_WI-UW-324_2020.fasta', 'USA_WI-UW-227_2020.fasta', 'USA_WI-UW-232_2020.fasta', 'USA_WI-UW-340_2020.fasta', 'USA_WI-UW-186_2020.fasta', 'USA_WI-UW-189_2020.fasta', 'USA_WI-UW-176_2020.fasta', 'USA_WI-UW-228_2020.fasta', 'USA_WI-UW-235_2020.fasta', 'USA_WI-UW-323_2020.fasta', 'USA_WI-UW-336_2020.fasta', 'USA_WI-UW-181_2020.fasta', 'USA_WI-UW-337_2020.fasta', 'USA_WI-UW-322_2020.fasta', 'USA_WI-UW-221_2020.fasta', 'USA_WI-UW-234_2020.fasta', 'USA_WI-UW-346_2020.fasta', 'USA_WI-UW-245_2020.fasta', 'USA_WI-UW-195_2020.fasta', 'USA_WI-UW-188_2020.fasta', 'USA_WI-UW-229_2020.fasta', 'USA_WI-UW-162_2020.fasta', 'USA_WI-UW-177_2020.fasta', 'USA_WI-UW-233_2020.fasta', 'USA_WI-UW-226_2020.fasta', 'USA_WI-UW-178_2020.fasta', 'USA_WI-UW-325_2020.fasta', 'USA_WI-UW-330_2020.fasta', 'USA_WI-UW-187_2020.fasta', 'USA_WI-UW-192_2020.fasta', 'USA_WI-UW-242_2020.fasta', 'USA_WI-UW-341_2020.fasta', 'USA_WI-UW-170_2020.fasta', 'USA_WI-UW-165_2020.fasta', 'USA_WI-UW-338_2020.fasta', 'USA_WI-UW-289_2020.fasta', 'USA_WI-UW-159_2020.fasta', 'USA_WI-UW-304_2020.fasta', 'USA_WI-UW-207_2020.fasta', 'USA_WI-UW-212_2020.fasta', 'USA_WI-UW-276_2020.fasta', 'USA_WI-UW-319_2020.fasta', 'USA_WI-UW-215_2020.fasta', 'USA_WI-UW-200_2020.fasta', 'USA_WI-UW-303_2020.fasta', 'USA_WI-UW-316_2020.fasta', 'USA_WI-UW-279_2020.fasta', 'USA_WI-UW-293_2020.fasta', 'USA_WI-UW-156_2020.fasta', 'USA_WI-UW-208_2020.fasta', 'USA_WI-UW-278_2020.fasta', 'USA_WI-UW-209_2020.fasta', 'USA_WI-UW-157_2020.fasta', 'USA_WI-UW-302_2020.fasta', 'USA_WI-UW-270_2020.fasta', 'USA_WI-UW-280_2020.fasta', 'USA_WI-UW-213_2020.fasta', 'USA_WI-UW-158_2020.fasta', 'USA_WI-UW-305_2020.fasta', 'USA_WI-UW-310_2020.fasta', 'USA_WI-UW-288_2020.fasta', 'USA_WI-UW-277_2020.fasta', 'USA_WI-UW-219_2020.fasta', 'USA_WI-UW-275_2020.fasta', 'USA_WI-UW-211_2020.fasta', 'USA_WI-UW-290_2020.fasta', 'USA_WI-UW-285_2020.fasta', 'USA_WI-UW-308_2020.fasta', 'USA_WI-UW-272_2020.fasta', 'USA_WI-UW-216_2020.fasta', 'USA_WI-UW-273_2020.fasta', 'USA_WI-UW-299_2020.fasta', 'USA_WI-UW-301_2020.fasta', 'USA_WI-UW-217_2020.fasta', 'USA_WI-UW-309_2020.fasta', 'USA_WI-UW-291_2020.fasta', 'USA_WI-UW-274_2020.fasta', 'USA_WI-UW-205_2020.fasta', 'USA_WI-UW-210_2020.fasta', 'USA_WI-UW-313_2020.fasta', 'USA_WI-UW-306_2020.fasta', 'USA_WI-UW-296_2020.fasta', 'USA_WI-UW-218_2020.fasta']\n"
     ]
    }
   ],
   "source": [
    "# generate a list of dataframes\n",
    "\n",
    "list_of_Dane_DFs = load_dataframes('../data_derived/Dane/VCFs')\n",
    "\n",
    "list_of_MKE_DFs = load_dataframes('../data_derived/MKE/VCFs')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {},
   "outputs": [],
   "source": [
    "# clean up the VCF files and save as CSV files \n",
    "\n",
    "def clean_Dane_dataframes(path, list_of_DFs):\n",
    "    if not os.path.exists(path + '/cleaned'):\n",
    "            os.makedirs(path + '/cleaned')\n",
    "    for DF in list_of_DFs:\n",
    "        DF = DF[['sample', 'Position', 'Ref', 'Cons']]\n",
    "        DF['sample'] = DF['sample'].map(lambda x: x.lstrip('../data_derived/Dane/VCFs/'))\n",
    "        DF['sample'] = DF['sample'].map(lambda x: x.rstrip('.fasta.vcf'))\n",
    "        DF['sample'] = DF['sample'].map(lambda x: x.lstrip('SA_'))\n",
    "        sample = DF['sample'].values[0]\n",
    "        DF.to_csv(path + 'cleaned/' + sample + '-cleaned.csv', sep='\\t')\n",
    "        \n",
    "def clean_MKE_dataframes(path, list_of_DFs):\n",
    "    if not os.path.exists(path + '/cleaned'):\n",
    "            os.makedirs(path + '/cleaned')\n",
    "    for DF in list_of_DFs:\n",
    "        DF = DF[['sample', 'Position', 'Ref', 'Cons']]\n",
    "        DF['sample'] = DF['sample'].map(lambda x: x.lstrip('../data_derived/MKE/VCFs/'))\n",
    "        DF['sample'] = DF['sample'].map(lambda x: x.rstrip('.fasta.vcf'))\n",
    "        DF['sample'] = DF['sample'].map(lambda x: x.lstrip('USA_'))\n",
    "        sample = DF['sample'].values[0]\n",
    "        DF.to_csv(path + 'cleaned/' + sample + '-cleaned.csv', sep='\\t')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:8: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy\n",
      "  \n",
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:9: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy\n",
      "  if __name__ == '__main__':\n",
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:10: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy\n",
      "  # Remove the CWD from sys.path while we load stuff.\n",
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:19: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy\n",
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:20: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy\n",
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:21: SettingWithCopyWarning: \n",
      "A value is trying to be set on a copy of a slice from a DataFrame.\n",
      "Try using .loc[row_indexer,col_indexer] = value instead\n",
      "\n",
      "See the caveats in the documentation: http://pandas.pydata.org/pandas-docs/stable/indexing.html#indexing-view-versus-copy\n"
     ]
    }
   ],
   "source": [
    "# clean up all of the dataframes and save to new CSV files \n",
    "\n",
    "clean_Dane_dataframes('../data_derived/Dane/', list_of_Dane_DFs) # Dane County\n",
    "\n",
    "clean_MKE_dataframes('../data_derived/MKE/', list_of_MKE_DFs) # Milwaukee County "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "122\n",
      "125\n"
     ]
    }
   ],
   "source": [
    "# check to make sure there is the correct number of cleaned CSV files in each cleaned directory \n",
    "\n",
    "path, dirs, files = next(os.walk(\"../data_derived/Dane/cleaned\"))\n",
    "Dane_len = len(files)\n",
    "\n",
    "path, dirs, files = next(os.walk(\"../data_derived/MKE/cleaned\"))\n",
    "MKE_len = len(files)\n",
    "\n",
    "print(Dane_len) # should be 122\n",
    "print(MKE_len) # should be 125"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 278,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:6: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version\n",
      "of pandas will change to not sort by default.\n",
      "\n",
      "To accept the future behavior, pass 'sort=False'.\n",
      "\n",
      "To retain the current behavior and silence the warning, pass 'sort=True'.\n",
      "\n",
      "  \n"
     ]
    }
   ],
   "source": [
    "# merge all SNVs into a single dataframe\n",
    "\n",
    "Dane_all = pd.concat([pd.read_csv(f, sep='\\t') for f in glob.glob(\"../data_derived/Dane/cleaned/*.csv\")], ignore_index=True)\n",
    "Dane_all = Dane_all[['sample', 'Position']]\n",
    "\n",
    "MKE_all = pd.concat([pd.read_csv(f, sep='\\t') for f in glob.glob(\"../data_derived/MKE/cleaned/*.csv\")], ignore_index=True)\n",
    "MKE_all = MKE_all[['sample', 'Position']]\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 229,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/ipykernel_launcher.py:1: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version\n",
      "of pandas will change to not sort by default.\n",
      "\n",
      "To accept the future behavior, pass 'sort=False'.\n",
      "\n",
      "To retain the current behavior and silence the warning, pass 'sort=True'.\n",
      "\n",
      "  \"\"\"Entry point for launching an IPython kernel.\n"
     ]
    }
   ],
   "source": [
    "MKEannots = pd.concat([pd.read_csv(f, sep='\\t') for f in glob.glob(\"../data_derived/MKE/cleaned/*.csv\")], ignore_index=True)\n",
    "\n",
    "MKEannots = MKEannots.drop_duplicates(subset='Position', keep=\"first\")\n",
    "\n",
    "MKEannots = MKEannots[['Cons', 'Position', 'Ref', 'sample']]\n",
    "\n",
    "MKEannots.to_csv(\"/Users/katbraun/Desktop/MKEannots.csv\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 245,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total # of Dane SNVs: 933\n",
      "Total # of MKE SNVs: 627\n",
      "# of unique Dane SNVs: 155\n",
      "# of unique MKE SNVs: 148\n"
     ]
    }
   ],
   "source": [
    "# print the total number of SNVs as well as the total number of unique SNVs in each catchment area \n",
    "\n",
    "Dane_unique = Dane_all.drop_duplicates(subset='Position', keep=\"first\")\n",
    "MKE_unique = MKE_all.drop_duplicates(subset='Position', keep=\"first\")\n",
    "\n",
    "print(\"Total # of Dane SNVs:\", len(Dane_all)) # total number of UWHC SNVs \n",
    "print(\"Total # of MKE SNVs:\", len(MKE_all)) # total number of MHD SNVs\n",
    "\n",
    "print(\"# of unique Dane SNVs:\", len(Dane_unique)) # total number of UWHC SNVs \n",
    "print(\"# of unique MKE SNVs:\", len(MKE_unique)) # total number of MHD SNVs\n",
    "\n",
    "#"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## load metadata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 246,
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata = pd.read_csv('../data_raw/metadata.csv')\n",
    "\n",
    "metadata = metadata[['Position', 'Annotations']]\n",
    "\n",
    "metadata_dict = dict(zip(metadata.Position, metadata.Annotations))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 247,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "temp = Dane_all.groupby('Position')['sample'].apply(','.join)\n",
    "\n",
    "Dane_group = pd.DataFrame(temp)\n",
    "\n",
    "Dane_group = Dane_group['sample'].str.split(pat=\",\", expand=True)\n",
    "\n",
    "Dane_group['Sample_count'] = Dane_group.apply(lambda x: x.count(), axis=1)\n",
    "\n",
    "Dane_group = Dane_group[['Sample_count']]\n",
    "\n",
    "Dane_group = Dane_group.reset_index()\n",
    "\n",
    "Dane_group['Annotation']= Dane_group['Position'].map(metadata_dict)\n",
    "\n",
    "# Dane_group\n",
    "\n",
    "#"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 248,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "temp1 = MKE_all.groupby('Position')['sample'].apply(','.join)\n",
    "\n",
    "MKE_group = pd.DataFrame(temp1)\n",
    "\n",
    "MKE_group = MKE_group['sample'].str.split(pat=\",\", expand=True)\n",
    "\n",
    "MKE_group['Sample_count'] = MKE_group.apply(lambda x: x.count(), axis=1)\n",
    "\n",
    "MKE_group = MKE_group[['Sample_count']]\n",
    "\n",
    "MKE_group = MKE_group.reset_index()\n",
    "\n",
    "MKE_group['Annotation']= MKE_group['Position'].map(metadata_dict)\n",
    "\n",
    "#"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 249,
   "metadata": {},
   "outputs": [],
   "source": [
    "# split dataframe into synonymous + intergenic (open datapoints)\n",
    "\n",
    "Dane_group_syn_temp = Dane_group.loc[Dane_group.Annotation=='synonymous']\n",
    "\n",
    "Dane_group_intergenic = Dane_group.loc[Dane_group.Annotation=='intergenic']\n",
    "\n",
    "Dane_frames_syn = [Dane_group_syn_temp, Dane_group_intergenic]\n",
    "\n",
    "Dane_group_syn = pd.concat(Dane_frames_syn)\n",
    "\n",
    "Dane_group_syn = Dane_group_syn.sort_values('Position')\n",
    "\n",
    "# split dataframe into nonsynonymous (closed datapoints)\n",
    "\n",
    "Dane_group_nonsyn = Dane_group.loc[Dane_group.Annotation=='nonsynonymous']\n",
    "\n",
    "Dane_group_nonsyn = Dane_group_nonsyn.sort_values('Position')\n",
    "\n",
    "#"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 250,
   "metadata": {},
   "outputs": [],
   "source": [
    "# split dataframe into synonymous + intergenic (open datapoints)\n",
    "\n",
    "MKE_group_syn_temp = MKE_group.loc[MKE_group.Annotation=='synonymous']\n",
    "\n",
    "MKE_group_intergenic = MKE_group.loc[MKE_group.Annotation=='intergenic']\n",
    "\n",
    "MKE_frames_syn = [MKE_group_syn_temp, MKE_group_intergenic]\n",
    "\n",
    "MKE_group_syn = pd.concat(MKE_frames_syn)\n",
    "\n",
    "MKE_group_syn = MKE_group_syn.sort_values('Position')\n",
    "\n",
    "# split dataframe into nonsynonymous (closed datapoints)\n",
    "\n",
    "MKE_group_nonsyn = MKE_group.loc[MKE_group.Annotation=='nonsynonymous']\n",
    "\n",
    "MKE_group_nonsyn = MKE_group_nonsyn.sort_values('Position')\n",
    "\n",
    "\n",
    "#"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 256,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# of synonymous SNVs in Dane county is  73\n",
      "# of nonsynonymous SNVs in Dane county is  82\n",
      "# of synonymous SNVs in MKE county is  53\n",
      "# of nonsynonymous SNVs in MKE county is  94\n",
      "ratio of nonsyn/syn in Dane county is  1.1232876712328768\n",
      "ratio of nonsyn/syn in MKE county is  1.7735849056603774\n"
     ]
    }
   ],
   "source": [
    "Dane_syn = len(Dane_group_syn)\n",
    "Dane_nonsyn = len(Dane_group_nonsyn)\n",
    "\n",
    "MKE_syn = len(MKE_group_syn)\n",
    "MKE_nonsyn = len(MKE_group_nonsyn)\n",
    "\n",
    "Dane_ratio = Dane_nonsyn / Dane_syn\n",
    "MKE_ratio = MKE_nonsyn / MKE_syn\n",
    "\n",
    "print(\"# of synonymous SNVs in Dane county is \", len(Dane_group_syn))\n",
    "print(\"# of nonsynonymous SNVs in Dane county is \", len(Dane_group_nonsyn))\n",
    "\n",
    "print(\"# of synonymous SNVs in MKE county is \", len(MKE_group_syn))\n",
    "print(\"# of nonsynonymous SNVs in MKE county is \", len(MKE_group_nonsyn))\n",
    "\n",
    "print(\"ratio of nonsyn/syn in Dane county is \", Dane_ratio)\n",
    "print(\"ratio of nonsyn/syn in MKE county is \", MKE_ratio)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "#SNV Count and Mean Inter-patient pairwise difference\n",
    "\n",
    "temp3 = Dane_all.groupby('sample').Position.apply(lambda x: ','.join(map(str, x)))\n",
    "\n",
    "Dane_group = pd.DataFrame(temp3)\n",
    "\n",
    "Dane_group['SNV_count'] = Dane_group.Position.str.count(',') + 1\n",
    "\n",
    "Dane_group = Dane_group[['SNV_count']]\n",
    "\n",
    "temp4 = MKE_all.groupby('sample').Position.apply(lambda x: ','.join(map(str, x)))\n",
    "\n",
    "MKE_group = pd.DataFrame(temp4)\n",
    "\n",
    "MKE_group['SNV_count'] = MKE_group.Position.str.count(',') + 1\n",
    "\n",
    "MKE_group = MKE_group[['SNV_count']]\n",
    "\n",
    "print(\"average interpatient SNV distance for MKE is \", MKE_group['SNV_count'].mean())\n",
    "print(\"std interpatient SNV distance for MKE is \", MKE_group['SNV_count'].std())\n",
    "\n",
    "print(\"average interpatient SNV distance for Dane is \", Dane_group['SNV_count'].mean())\n",
    "print(\"std interpatient SNV distance for Dane is \", Dane_group['SNV_count'].std())\n",
    "\n",
    "#"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# generate the SNV figure"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 252,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/matplotlib/figure.py:445: UserWarning: Matplotlib is currently using module://ipykernel.pylab.backend_inline, which is a non-GUI backend, so cannot show the figure.\n",
      "  % get_backend())\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAn8AAAGwCAYAAADYGAMHAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAPYQAAD2EBqD+naQAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4zLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvnQurowAAIABJREFUeJzs3Xd4FFXbwOHftvQeID0hofcWgiBNqiCiKIICiiiKgr5YP9sL2BBFRSzoi4qACoggICqoWBCpIXSCdElPSEhvu9nd+f7Y7JKlZjUh7bmvay/YyezsMzNnZ545c84ZlaIoCkIIIYQQokFQ13QAQgghhBDi2pHkTwghhBCiAZHkTwghhBCiAZHkTwghhBCiAZHkTwghhBCiAZHkTwghhBCiAZHkTwghhBCiAZHkTwghhBCiAZHkTwghhBCiAZHkTwghhBCiAZHkTwghhBCiAanR5C8zM5PmzZuzefNm27Rdu3bRo0cPPDw8iIyMZNGiRXafWbp0Kc2bN8fd3Z3o6Gh27Njh0HcqikJ+fj7ySGMhhBBCNEQ1lvxt27aNnj17curUKdu0nJwchg8fzj333ENubi6LFi3i8ccfJzY2FoDNmzfz6KOPsnTpUnJzcxk/fjwjR46kuLi40t9bUFCAt7c3BQUFVb5OQgghhBC1XY0kf0uXLmXcuHHMnj3bbvo333yDv78/06ZNQ6vVMmDAAMaPH8+CBQsA+PTTT7nzzju5/vrr0el0PP744zRq1IiVK1fWxGoIIYQQQtQ5NZL8DR06lFOnTjF27Fi76fHx8XTo0MFuWtu2bTlw4ECl/n4per2e/Px8u5cQQgghRENVI8lfYGAgWq32oukFBQW4u7vbTXNzc6OwsLBSf7+UOXPm4O3tbXuFhYVVwRoIIYQQQtRNtaq3r7u7+0Xt94qLi/H09KzU3y/lueeeIy8vz/ZKSkqq+sCFEEIIIeqIWpX8tW/fnvj4eLtpR44coX379pX6+6U4Ozvj5eVl9xJCCCGEaKhqVfJ32223kZ6ezvz58ykrK+P3339n2bJl3HfffQDcd999LFu2jN9//52ysjLmz59PRkYGo0aNquHIhRBCCCHqhlqV/Pn7+7Np0yZWrVqFv78/kydP5r333uOGG24AYODAgXz44Yc8/PDD+Pr6smLFCjZu3Iifn18NRy6EEEIIUTeolAY22nF+fj7e3t7k5eXJLWAhhBBCNDi1quZPCCGEEEJUL0n+hBBCCCEaEEn+hBACMBuNFGSkY3DgcZFCVCeTwUBBehrG0tKaDkXUMxePtNzA5CQmkJ+SjHdoGD5h4TUdjhDiGlMUhYNfr2DPF0soPpeFWqejxaCh9H38aZyvMIaoENXFbDQSu2ghh775Gn1BATo3N9refCu9pv4HjZNTTYcn6oEGm/yV5uWxeeZzJO7aYZsW3qMnQ16eg0sd7QgiiawQjjuwcjlb332btiNHEdXvBrJPn2LP559RkJbCqA8/RaVS1XSIooHZ9v47HPzmazqPHUdodAzphw6yd9lSSvNyGTzr1ZoOT9QDDba374qHJpN7aD8hOvDQqCg0KaSUQXB0DCPnL6jpMB1SmpfHz7Oer1eJrBDXgslYxtJbhtO0d18GPDfDNj1hxza+e+JRRn34CSFdutVghKKhKcnNYfHIG4m57wGi751sm354zWo2vzWHe1avxys4pAYjFPVBg23zlxQXS4gO/HRqnNQq/HRqQnSQuGsHuUmJNR2eQ36e9TypcbFEuGho564lwkVDalwsP898rqZDE6JWK8rMpDj7HM36D7CbHn5dL7QuLpw9+lcNRSYaquzTpzGXlRF1QZmM6j8AFIWzx47WUGSiPmmwyR9Yavwu9T4vue48/zcnMYHEXTvqTSIrxLXk4uWFWqMl+8zfdtML0lIxlpbi5udfQ5GJhsqt/KEFOReUSet7dymTogo06OSv0KRc8r13aFhNhPOP5KckA/UjkRXiWnNy96D5oMHELVlEYuxOFEUhPy2VX159ERdvH6L69a/pEEUD49s0kqCOndn2/nwy/joCwLlTJ/nj7dfxi4wisGOnGo5Q1AcNtsNHWHQMKYf2A2a7Nn/hPXrWqc4SXiGhgCVx9VOfTwDrYiIrRE3o+/j/8X3KdNZPn4rOzY2y4mKcvby5ae48dC6uNR2eaIAGv/gq3z3+CKvum4DOzZ2y4iI8A4O4+Z33pQOSqBINtsNHRlISO9+aUy86Sax/bBqp5W0Y63rnFSFqgqIopOyN4+zRv3D39yeq3wB0rpL4iZpjNhpJ2LGNnIQzeIWEEtm7LxqdrqbDEvVEg03+rM/2zU1KJC85qU4Pj1Kan8/P9WzYGiGEEEJUjwaf/NUn9SGRFUIIIUT1arBt/uojn7BwSfqEEEIIcUUNurevEEIIIURDI8mfEEIIIUQDIsmfEEIIIUQDIsmfEEIIIUQDIsmfEEIIIUQDIsmfEEIIIUQD4lDyV1hYyPz58wE4cuQIPXr04KabbiIlJaVaghNCCCGEEFXLoUGe7733Xvbv38/+/fvp168fAQEBuLi4kJeXx7fffludcVaZ+jzIsxBCCCHE1TiU/EVGRrJnzx5UKhWNGzcmISEBf39/goKCyMnJqc44q4wkf0IIIYRoyBy67Zufn4+fnx+//vorzZo1IyQkBJVKhUqlqq74hBBCCCFEFXLo8W7t27fn1VdfZePGjYwYMYKCggJeeOEFunXrVl3xCSGEEEKIKuRQzd9HH33EL7/8gpeXF7NmzWLv3r1s3ryZBQsWVFlAy5Ytw8PDw+7l5OSEs7MzAMOGDcPFxcXu7z/++GOVfb8QQgghRH3mUJu/CymKUu23fFNSUujevTtz585lwoQJNG7cmNWrV9OvX79/tDxp8yeEEEKIhszhcf42bdrELbfcQrdu3cjIyOCpp57CaDRWR2woisLdd9/NTTfdxIQJE/j777/Jzs6ma9eu1fJ9QgghhBD1nUPJ3/Llyxk/fjzt27fn5MmTAKxfv57nn3++WoL78ssviY+PZ968eQDs3r0bT09Pxo4dS+PGjWnfvj2fffbZFZeh1+vJz8+3ewkhhBBCNFQOJX9z5sxh/fr1zJ49G7VaTWBgID/88APLly+v8sDMZjOvvPIKL7zwAp6enoAlkevZsyezZ88mNTWVefPmMX36dFatWnXFmL29vW2vsLCwKo9VCCGEEKKucKjNn6+vL9nZ2ahUKvz8/MjOzkZRFHx9fcnNza3SwH799VduvfVW0tPTcXd3v+x806ZNIyMjg9WrV1/y73q9Hr1eb3ufn59PWFiYtPkTQgghRIPkUM1fy5YtWb9+vd20X375hRYtWlRpUADffPMNo0aNskv8Pvvss4tq+fR6Pa6urpddjrOzM15eXnYvIYQQQoiGyqHkb/bs2YwbN47x48dTWlrK1KlTueOOO3j55ZerPLCtW7fSt29fu2l5eXk88sgj7Nu3D7PZbLvl/OCDD1b59wshhBBC1EcODfI8aNAgduzYwcKFC7nhhhswmUz8/PPPxMTEVHlgp0+fJiQkxG7aY489RlFREaNGjeLs2bNERUXx+eef06dPnyr/fiGEEEKI+sjhcf42bdpE165d8ff3Z8OGDTg5OTFo0KDqiq/KyTh/QgghhGjIHLrtu2DBAu644w7S0tIAOHv2LGPGjOHzzz+vluCEEEIIIUTVcqjmr1mzZqxatcpukOU9e/Ywbtw4jh07Vi0BVjWp+RNCCCFEQ+ZQ8ufp6UleXh5q9fkKQ7PZjJ+fX5UP9VJdJPkTQgghREPm0G3ftm3b8sUXX9hNW758Oa1bt67SoIQQQgghRPVwqLfvq6++ysiRI/nkk0+IiIggMTGRvXv3smHDhuqKTwghhBBCVCGHe/seP36cr776irS0NMLCwrjzzjuJioqqrviqnNz2FUIIIURD5nDyV9dJ8ieEEEKIhsyh277x8fE8/fTTHD9+HLPZbPe306dPV2lgQgghhBCi6jmU/E2ZMgU3NzeeffZZdDpddcUkhBBCCCGqiUPJ38GDB0lJScHT07O64hFCCCGEENXIoaFegoODKS0tra5YhBBCCCFENXOo5u+RRx5h1KhRTJ8+nYCAALu/9e3bt0oDE0IIIYQQVc+h3r4Vn+xhtxCVCpPJVGVBVSfp7SuEEEKIhsyhmr8Le/gKIYQQQoi6xaE2fwAGg4G1a9cyf/58iouLOXDgQHXEJYQQQgghqoFDNX+nTp1iyJAhGAwGcnJyGD58ONHR0axdu5YRI0ZUV4xCCCGEEKKKOFTzN336dCZNmkRiYiI6nY6WLVvy6aefMnPmzOqKTwghhBBCVCGHOnw0atSI1NRUnJyc8PPzIzs7G7PZjJ+fH7m5udUZZ5WRDh9CCCGEaMgcqvnz9vYmPT3dblpaWhp+fn5VGpQQQgghhKgeDiV/48eP57bbbmPTpk2YzWZiY2OZMGECd955Z3XFJ4QQQgghqpBDHT5mzJhBSUkJt912G0VFRfTv35/777+fWbNmVVd8QgghhBCiCjnU5q+izMxMGjVqhEqlquqYqpW0+RNCCCFEQ+ZQzd/nn39+2b/dc889/zoYIYQQQghRvRyq+YuMjLR7n52dTVFREb1792bz5s1VHVu1kJo/IYQQQjRkDtX8/f3333bvFUXhjTfeIDs7u0qDEkIIIYQQ1eMft/mzMplMhISEXDQETG0lNX9CCCGEaMgcfrbvhY4dO1blnT5WrlyJVqvFw8PD9rr77rsB2LBhAx06dMDd3Z02bdrw/fffV+l3CyGEEELUZw7d9r3hhhvsEj2DwcDBgweZMGFClQa1e/du7r77bhYvXmw3/cSJE9x+++2sWLGCESNGsGbNGsaMGcOJEycICQmp0hiEEEIIIeojh5K//v37273XaDQ8/vjj3HrrrVUZE7t372bMmDEXTV+6dCl9+vSxfd+YMWNYvHgxH3/8MS+99FKVxiCEEEIIUR85lPxdi8GczWYze/fuxd3dnblz52IymRg+fDhvvPEG8fHxdOjQwW7+tm3bcuDAgcsuT6/Xo9frbe/z8/OrLXYhhBBCiNrOoeTvvvvuq9R8n3322T8KBiyDR3fp0oXRo0ezevVqsrKymDhxIhMmTECv1+Pu7m43v5ubG4WFhZdd3pw5c6RWUAghhBCinEMdPlxcXPjyyy/Jzc0lICCAsrIyVqxYQWpqKoqi2F7/RkBAAFu2bOG+++7Dzc2N8PBw5s6dy8aNG1EUheLiYrv5i4uL8fT0vOzynnvuOfLy8myvpKSkfxWfEEIIIURd5lDN3+nTp1mxYgW33367bdrEiRN58803L+qc8U8dPHiQ5cuXM2fOHFvnEr1ej1qtJiYmhv3799vNf+TIEaKjoy+7PGdnZ5ydnaskNiGEEEKIus6hcf48PT3Jy8tDrT5fYWgymWjcuHGVDfScnJxM69atmTlzJk888QSpqamMHTuWdu3a8dRTT9GlSxeWLl3Kbbfdxpo1a5g4cSIHDhygZcuWlVq+jPMnhBBCiIbModu+ERERfP3113bTFi9eTJs2baosoNDQUH744QfWrVuHn58f0dHRdO/enQ8++IDWrVuzbt06XnvtNXx9fXn55Zf55ptvKp34CSGEEEI0dA7V/K1fv5477riDmJgYwsPDOX36NEePHuWXX36hW7du1RlnlZGaPyGEEEI0ZA4/3u3o0aN89dVXnD17lqioKCZMmEBgYGB1xVflJPkTQgghREP2j57tm5OTw+nTp+nSpQtGoxEnJ6fqiK1aSPInhBBCiIbMoTZ/hYWFjBs3Dn9/f/r27cuJEydo1qwZx44dq674hBBCCCFEFXIo+Xv66acpKiri6NGjODk5ERUVxc0338z06dOrKz4hhBBCCFGFHLrtGxoayqFDh/D19cXPz4/s7GxKSkoICQmpsqFeqpvc9hVCCCH+naKsTE5t/g2jXk94zHU0aiGjbtQlDg3ybDKZbAMmW3NGs9ksgygLIYQQDcThtavZ8vZcUIFaq2X7B/NpNWwEA1+YhVqjqenwRCU4dNt34MCBTJs2jeLiYtvTN/773//Sv3//6ohNCCGEELVI1onjbH5zDm1H3sr9G37lgU1/cMNzMzj+0wYOrV5Z0+GJSnIo+Zs3bx5//fUXvr6+5OXl4enpyR9//MHbb79dXfEJIYQQopY48v23uPs3ou8T/4ezpycarY52I0fRfOBg4tevq+nwRCU5dNu3SZMm7Nixg7i4OM6cOUNoaCgxMTFopJpXCCGEqPdKss/hHRaOWmufPvhGRJIcF1tDUQlHOVTzB7B37166d+/O4MGDWbduHe+88w5Go7E6YhNCCCFELdK4dVvSDx+k8OxZ2zSz0cjpP36nSeu2NRiZcIRDNX+zZ89m7ty55OXl8Z///Ie4uDjUajXJycnMnz+/umIUQgghRC3QdsQtHPhqGWunTqbr3ffi7OnF4bWrOXf6JH2feLqmwxOV5NBQL+3atWP58uW0bt0aX19fdu7cSWBgIB07diQ9Pb0646wyMtSLEEII8c/lJiWyZd5cEnduB8C/WXN6TZtORM/razgyUVkOJX++vr7k5OTw22+/MX78eNLS0gDw8vIiPz+/2oKsSpL8CSGEEP9eaX4+pjIDbn7+thFARN3g0G3fkJAQtmzZwtKlSxk0aBAAK1asICoqqlqCE0IIIUTt5CIVKHWWQ8nfSy+9xI033oibmxvbtm3jt99+Y9KkSXzzzTfVFZ8QQgghhKhCDt32BSgtLQXAxcWFgoICCgsLCQoKqpbgqoPc9hVCCCFEQ+ZQzR9Ykj4rT09PPD09qzQgIYQQQghRfRxO/gSYTSb+/nMzf2/dgkqlJqpvf5pe3weV2uFhE4UQQgghrilJ/hxkKitjw7NPkrB9K/7NW6CYzfz1/bc0u2EQQ1+ZIw+1FkIIIUStVqmqqnvuuQeA3377rVqDqQuOrF9L4s7tjHj7Pe76YiXjlq3ixtlzOfX7Lxz/aWNNhyeEEEIIcUWV6vDh6elJdnY2/v7+dWY8v8v5tx0+vnnoPpw9vRjxpv0TTdZOexCtszM3z3u/qkIVolYpOpfFgZUrSNmzG52rKy2G3Eib4Tdf9IxPIYQQtVuljtpt2rQhLCyM4uLiy47pd/r06SoNrLYylpbiHRJ20XQXb29KcnNrICIhql9BehqrH5xEWUkxkb37UZKbw++vv0rCjm0Mmz23zrV3zUlMID8lGe/QMHzCwms6HCH+NSnTwhGVSv7Wr1/Pr7/+yuTJk5k1a1Z1x1SrhXXvweF131CY+QgejRsDkJ+WSsL2bXSbeF8NRydE9Yj9dCEoCuNXfIN7I0u5P/n7r/z4/NMk7tpRZx7rVJqXx8+znidx1w7btPAePRny8hwZsFbUSVKmxT/h0Dh/77//Po8++mh1xlPt/u1t36JzWXw9aQKKyUTrm25GMZn4a8N3OLl7MGbRF7h4e1dD1KI+qYtX6J8MvYEOt93BdVOm2qYpisLyu24nuEs3bnjmhRqMrvLWPzaN1LhYQnTgoVFRaFJIKYPg6BhGzl9Q0+EJ4TAp045RFIX4dd9wcPXXFKSn4dc0ki7j76H5gEE1Hdo15dC9mkcffZRvv/2W4cOH07ZtWwYOHMjy5curK7Zayd2/EaM/WULTPn05uuF7jv28kRYDBnP7ws8k8RNXVJqXx/rHprFs7Ci+e+JRvhxzK+sfm0ZpHWhHa3lu58XXiQ6OEV+jchITSNy1gxAd+OnUOKlV+OnUhOggcdcOcpMSazpEIRwiZdpxOz58j81zX8M3oind73sAJw8Pfnzh/zj0zdc1Hdo15VBL7eXLlzN16lSmTJnCyJEjOXnyJA8//DDFxcVMnjy5umKsdTwDAhnw7IyaDkPUMT/Pep7UuFgiXDTnr9DjYvl55nO1/go9sk8/jnz3Le1H3YFHkyYAnPx1E7mJCfR57Kkajq5y8lOSAUvtSEXW93nJSXWmJlYIkDLtqKKsTPavWEaPB6fSfZIlZ+ky7m5+n/MKOz/+iDYjbkHr7FzDUV4bDtX8vfHGG6xdu5Y33niDhx56iLfeeot169Yxb968Kg3qwIEDDB48GD8/PwIDA7nnnnvIysoC4OGHH8bZ2RkPDw/b6+OPP67S7xeiqtX1K/SYyVNQa7Usu+s2fpr5HOsefYifZjxL84GDCe/Rs6bDqxSvkFAACk32tZXW996hF3fkEqI2kzLtmNT9ezGbjLS/9XbbNJVKRbtbb0efn0fWyeM1GN215VDyl5CQQP/+/e2m9evXj+Tk5CoLqKSkhGHDhtGrVy/S09OJj4/n3LlzTJo0CYDdu3fz8ccfU1hYaHs9+OCDVfb9QlSHylyh12aeAYGM+exLOo0ZR0F6OiqNmoH/fYkhL71WZ3r6+oZHEN6jJyllkF1mxmBWyC4zk1JmaSAvNSSirpEy7RidqxsAJbk5dtNL83Lt/t4QOHTUDg0NZcuWLXbTtmzZQnh41RWwxMREOnXqxMyZM3FycsLf358pU6awZcsW9Ho9hw4dIjo6usq+T4hroT5cobv5+XHdlKmM/ngxt8z/kDY33Vznnmgz5OU5BEfHkFBqIr7ISEKpieDoGIa8PKemQxPiH5EyXXmh0TG4+Piw7f13MBQVAVCSk8POjz/Ev3kL/CIvPZRdfeRQb99PP/2Up59+milTphAVFcWpU6dYuHAh77zzjq1mrjrcc889JCcn8/rrr3PdddcxatQotm7dire3N/fffz9PP/006svUPuj1evR6ve19fn4+YWFh/7i3rxD/lPTKqz1ykxLJS06qUz2uhbgSKdOVk7BjGxuefQqNTodfZBSZx4+hdXHhlncX0KR125oO75pxKPkDWLJkCYsXLyYjI4OmTZsyefJkRo8eXS3BKYrCjBkzWLBgAVu2bCE9PZ05c+Ywa9YsevXqxb59+xg1ahSPPfYYTz/99CWX8eKLL/LSSy9dNF2SP3Gtlebn8/PM52Q8LiGEqEEF6Wkc+f5bCtLS8IuMpM1Nt+Dq61vTYV1TDid/10p+fj6TJk1iz549fPfdd3To0OGS87355pusXLmSuLi4S/5dav5EbSNX6EIIIWpSrXwo56lTpxg+fDjh4eHExcXRqFEjANatW0dGRgZTpkyxzavX63F1db3sspydnXFuIF23Rd3gExYuSZ8QQogaU+uSv5ycHAYMGMCAAQNYtGiRXVs+RVF4/PHHad68OQMGDGDnzp28++67vPPOOzUYsRBCVK2M+MPELlpI8p44dG6utBoyjJjJD+Hs6Vlt31kXnzxTF2OuLxrats86eYLYT/5H4q4daJycaD5wMD0eeBg3Pz+HllNbtlutu+07b948nnzySdzc3MqfKnBeYWEhCxcuZN68eSQnJxMYGMiTTz7J1KlTL7O0i/3bx7sJIUR1yjgSz5qHJ+MTFkarYSMoyT5H/Ldr8Q4LY/THS9DodFX6fXXx2bB1Meb6oiFu++wzp1l1/0TcGzWizU0jMRQXEb9uDa4+Ptyx6Auc3N2vuozatt0cSv5mzJjBK6+8AsCsWbMu2ZGitpPkTwhRm3335H8oSE9nzGdf2J42kHEknlX3383gF2fTauiwKv2+utgLvS7GXF80xG2/6aUZpO7fy11ffm1L9HLO/M3yCWPo89hTdBw99qrLqG3brVLj/HXu3JkZM2Ywf/58iouLAXj//ferNTAhhGiIUvftodWNw+0eMxXQth3+zVuQsu/SHdv+qbr45Jm6GHN90VC3fcreOFoMHmpXw+fbNJLgTl1I2Xv132Rt3G6VSv4++eQTdDodxcXFBAUF0bNnT0pLS1m5ciXHjzecx6EIIUR107m5U3wuy26a2WikJDcHJ7er315yRF188kxdjLm+aKjb3sndneJz5+ymKYpC8bmsSv0ma+N2q1Ty1717d2bOnIm7uztZWVm8+eabAKxZs4YRI0bg7e1drUEKIURD0XLIMI58t460gwcAS+IXu+hjirOyaFnFt3zr4pNn6mLM9UVD3fYthwzj+KYfSYzdCYDZZGL/ii/JSThTqd9kbdxulerte9dddzFixAgAdDodvXv3xsXFhZUrVwKWjhhCCCH+vZj7HyDt4D6+mTIJv8goSvPyKM4+R48Hp9KkVZsq/S7bs2HjYgGzXVuk2vps2LoYc33RULd957smkLI3jvXTp+IT3pSykmKKMs/S+a4JhEbHXPXztXG7VarDxxdffMHGjRv56quvCA8Pp2/fvqxZs4YtW7bQoUMHdFXc+6w6SYcPIURtZzKWcXrz76TsjUPn5kbLIcNo3LJVtXxXXXzyTF2Mub5oqNvebDKRsH0riTu3o3ZyosWgIQS2u/TDJy6ltm03h3r7enp6cvDgQf744w8eeugh2rZty/Hjx2nRogX79u2rzjirjCR/Qghxsbr45Jm6GHN9Idv+n6kt282h5K9z587s378fgG7durFnzx4MBgOHDx+ma9eu1RZkVZLkTwghhBANWa0b5Lm6SfInhBBCiIasUr19hRBCCCFE/SDJnwDAUFxMTsIZDEXSc1sIIYSozyo11Iuov0xlZWz/YD7x69diLC1F4+RMmxEj6f3o42hdXGo6PCGEEEJUMUn+Grg/3nqdoxu/J3rifYR0iSbt4H7ilizCUFTIkBdn13R4QgghhKhikvw1YEVZmfz1w3qunzadzndNACCkazdcvLzZ/NYcrpsyDa+g4BqOUoj6I+vkcQrS0vBtGinDYwghaowkfw3YudOnUEwmIvv0s5se2bcfm998jXMnT0jyJ0QVKMrK5KcZz5K6//x4qJF9+jF41is4uXvUYGRCiIZIOnw0YB6NmwCQdfKE3fSsE8cBcGvU+JrHJER9oygKG597mryUFIa/MY9J3/3EoJkvk7JvD7+//mpNh1ctchITSNixjdykxJoORdQCUh5qH6n5a8D8IqMI6tSZre++jZuvH4EdO5Fx5DBb5s2lcavWNGldtc8RFaIhOvvXEdIPH2TE2+/RtFdvAFoPG4GxpJQ/3n6d6/+TiUfj+nGhVZqXx8+znq81j7ASNUvKQ+0lNX8N3JCXXsPJ3Z1vHrqPj/r0YPXkiag1Gm6cPReVSlXT4QlR5+WlJAMQ3Kmz3fSgTp1RzGYKUlNqIqxq8fOs50mNiyXCRUM7dy108L+eAAAgAElEQVQRLhpS42L5eeZzNR2aqAFSHmovqflr4DwDArnz869IioslNzEB75BQwmKuQ63R1HRoQtQLPmFhACTviSOqb3/b9JS9cag0GrxCQ6/4+ZzEBPJTkmv8WaBXk5OYQOKuHUS4aPDTWeoV/NQqwEzCrh3kJiXW6vhF1bpUeXAD/ExmEqU81DhJ/gQqtZrwmOsIj7mupkMRot5p3KoNwZ278vvrr2IsLSWgXXuSYney438f0HLwUNz9G13yc9V9y8xkLOPvLX+QGLuT4qxM1FotngGBNB8wmKBOnR1OOvPLazg9NPZ3DKzv85KT5GTfgFQsD0ZFIaHERL7p/NNkf5rxHLe899E1u/2bHn+IE7/8jLG0lLDoGCL79Uej1V00n6G4mOM/beDs0b9w8/Oj9fCb62W5lWf7inrPqNeTm5SAi5c3Hk0CajqcOsFYWkpuciIuXj54NGlS0+HUeSU5OWx66b/nEzmVihaDhjDguZnoXF0v+Zn1j00jNS6WEJ3lBFpoUkgpg+DoGEbOX/Cv4ikrKeG7Jx4hdf8+tK6uGEtKANC5uVNWXIRnYCAF6em2+SuTdOYkJrBs7Ci7mh6A7DIzCaUmJny9rl6eRMWlVSwPOWVmikwKoS6aCmVZITi6x78uy5Wx46P32fP5YtwbN8HZ05Ps06cI6tSZke8ssPv95aelsu6RKRSkp9GoRUvyU1MxFBcx8IVZtB42otrjvJak5k/UW4qicOCrZcQt/YzSvFwAQqNjGPD8TBnC5jIURWHfsqXs+XwJ+oJ8AII6daHdLaMIbN9RTt5XcbnaMldfX0bOX0BuUiIFaWn4RETgGRB4xeVU5y3UPZ8v5uxff9H25ls5+uMP3PzOB2QdP8aOj97HI8CS+DXRqWnspLacqMvbaV3pRO0bHkF4j56kxMUCZruENbxHTyk7DYy1PCTv3oXJrNRYc4DUA/vY8/lirnvoEbpOsLRpT9m3h+8ef9QyfcpU27xb5s1FMZsY/9UafMLCMZaW8vvc1/j99VcJ79ELNz+/aovzWpMOH6LeOrJ+LVvfm0ezGwZy2/8WMXjWK+SlJPPtow9hMhhqOrxa6eDqlWxf8B6tbhzOiLffwy+qGWkH9vHLyzP5csytrH9sGqX5+TUdZq1TmpfH+semsWzsKL574tHLbiufsHDCYnpcMfGDyt1C/TeOb/qRVsNuIj3+EC0GDSXiul50nTARj4AACjPS0anAqCg4qVX46dSE6LC107qSIS/PITg6hoRSE/FFRhJKTQRHxzDk5Tn/Kl5RNw15eQ5+LVoC1VeWr+b4TxvxCg6h29332tqyh3TpRqthN3H85422+fSFBZzZ9idd755kS0a1Li70nv4EisnM6c2/XfW76tKQNlLzV856xW42mzCV6vEMCqZJm7Z1ssdrYWYm6YcP4uTuTkjXbpds11DV8tNSOfvXEVy8vQnu3LXGO4woisLeL5YQ0bM3UX374+bnT3CnLjRq2ZoV4+/g5O+/0mrosBqNsbooZjOp+/dRkpNN49Zt8A4536HgSu24FLOZfV8upfVNN9P3if9j/WPTyE84Q6CTmnSDmcY6ta2n3rW4VVPbXGnbWXs1hjlbrqdLzZCye9c/3lZe5fus0KSU15Jgew/gHRr2T1cDgLLiItz8/Gz/gqXtr87FcgvMSaXCVD5vqVnB2jrocu32DEWFJO+JQ6VScePsuRRnnyMvOemibVWVnVfqSkeYa+nfbBPrZ1VqNYrZ/K+3q/U81GH0WH6b/RIZBjPeWhWeGhUqleqKZbmy65GblMiZ7VsxG8uI7N0P34imGPV6kvfsxmTQE9y5K4biIlx9fVGpLb9Nk7GMlL17KM3LRV9YYFtWWUkpKMpFtXvOHp5onHQYiosuG8eV2ucWZqSTm5iAV0gIWlc30vbvA5UlAa3Jctvgk79L7TSrgHbtGfbaW3WmzZNiNrPtg/kc+HoFisly6HZv3IQhL80mpEu3avlOk7GMP+bO4cj330L5CcIrJJQbZ79Bk1Y1N05g0dmz5KUkk5eSTMKOrcD5H6NnUDDnTp0A6l/yl3XyOBuf/z/yrFeeKhWtbryJXlP/w6+vzrpi54HSgnwKz2bQtFefC247asgqM6NRQYiOBtdz82odL6zbyk+rIkWvYC6fRwX/uFdjdd9CDekSzfGffySoY2dO/voz3SdNJj8tlZzEBACKzApBWjWnio12jfT3LP2MgHYd7Nr+xX+7hq3vzaOsuBiwtBvs89iTtL35Vts8Vdl5RcaOu9i/2SZXOgf+k+2qmM1sfW8eB1evtJ2HALLKzGSVgVYF/jo150yqi8pyZdfDqNez6cUXOFWhNm77B+/iG9GU4pxs9OU17honJyJ6Xk/GkXiyThzHUFTITzOfpyjzrOVDKhV/zn+L3v95AvdGjfAJb8qR774lqu8NtmTxxC8/UVZSQki37pdd54pD2lh/q8m7d/HFHSNtsVxKaHR3bpz9Zo2U2wbb4WPXyhXEjLmTNQ/dT8ahAzhhxmAGT62KXKP9JlFptTi7e2A2m9C5uBDVbwC9pv4HrYsLR75bx+G1qynMyKBR8xZ0mTDxqr1m89NSiVuyiIQd21BrtTQfMIhu99x3UQEozs5m0yszSd69y/IjUqnwCAik6/h78A4JxTs0DGcvL/YsXcypzb+iz8/HUFRI1wkT6XzXBIoyM/nz3bfJiD+Eb9NICjPS8WgSQMc77rQ7MAMk7d7F3i+XknXiOK4+PoRGx9Bh9Fh8wyOuuC47F37I3i+WcP30J2g5eCh5yUlsmTeXgvR07l69Hic3N85s38q+5Z+Tffo0XkHBdLh9DK2G3XTJWlXFbObwum+IX7eGwrPpeAYF02nsOALatr/klWBOYgJxSxaRtHsnOmcXWgweStcJ97Lx+adJit2Jp0ZFuIvGduIM6NSZtIMH6P2fJ+h4x51XXLfLudRVaVFWJnFLP+PvP/9ApYLIPv3pNvE+u56cxtJS9i3/gkNrV1GSk2NLlhVApSigVuMZEEj0xPtpc/MtDtc6G0tL+eKOW3D19aPfk89gMpvY9u48Mo/9ZamJNZsJdVLhqVXbdR7o88T/kZ+SjLOnJ2umPoDW2Rm1WkNpfh4tXDU4qVXEFxkJc9bgpbX8f8hLr5ERf4jTW37HVFZG41Zt6HbPJII7dbHFc3zTTxz8egV5Kcn4RkTQ+c4JRPW7ocq2eVUwGQzsW/Elh9euojg7GxUqPIOC6DR2HO1vvR2VWn3VjhcJO7bx3ROPAuCnU+GrVWMwK+QaFQpMCr2mTafrhIkOr0thViZrH55sd1sssH1HRrz9nu1YkZOYQOq+vRfVJFxYi1OxNgdg6/vvcGbrFtsJTufqislgQOfmjqEgH7PJhDNgAHQqMCmgU4NeUeETHoFap6MkOxvP4GAyDh+i2Q2DiOh1PZ4BgRz7aQNHf/iO2/63iOBOXchJTODnmc+Tc/IYIToV7mrIMJg5Z1RQa3UEd+pMl/H3ENHz+krtszUP3U/6wf14qKHErGBSLL+hwI6duX3hZ1f9fNG5LOKWLOLvP//AaNBTVlyMSa8HwMnDk4H/fZGoPv2I/3aN5TiUeZbGLVvRdcJEQqNjLlre33/+wb4VX5Jz5m+8QkLoOPpOWg658aLfb2X3+5XmO/HLz+z5Ygn5qSl4h4YSPfE+/KKa223fC8uo9fdtKC7i2I8bOHskHhcfX9refAsdR4/l+6emkxoXi04xYTBDIyc1RSaFYpPlQsa3aSTjlq2ylRUrU1kZ+1Z8ycGvV1CSm4NKpcLNvxGmMoOlbAQG0aRtO07//ivuAYEUZmSAYrZ9PqRrNMPmvGV33rvcb61xuw5E33u/bZv88uqLHP1hPWBpu+ahgSITmABXH19GffQJTm4eHFy1gr1fLsUjIBBDYQFGvQFXX19MxjLKioppd+ttHPx6BSHdulOQnoY+Px99QT5NWrfFLzKKhF3bKcnOxsXbm87jJnL8x+/JPvM3KAoqtRoXL2+cPDzJS05EheWCT6MCN42KEpOCQYHoSQ9weM0qW7tzDRDhoibXqJBjVAjs1Jnb/3f1clvV6mTyd/bsWR588EE2b96MVqtlwoQJvPXWW2i1V6/ItCZ/c6M74qotv//vrCZFbybQSc1ZgxkFy8HkSpw9PWkx6EYOr11FZN/+NG7ZioQd28g4Es+Ql16j5eChl/xcQXoaq+6/B9QqWg0djrG0lGM/bsAzMJDbP16Ck5sbAHnJySwbdzvmsrIrxqFzdUWlVtNq2AiO/bgBk0GPi5c3dyz6Ao8mTfhz/lscWLncrkcfgH9Uc0Z99CkuXl6c/HUTP854lsYtWmEoKrQNSgtXvvIzm0wsGj6I1sNH0Gf6k+e3cVoqn99+MwOenwnAb7NfIqhjZ8JiriPjyGEStm+l+6QH6PHgwxctc/Obczi8djWuPr6U5GRfcp2tMZXk5rB68kR0bm60HDIMfUE+x378Aa/gELJPn8JLo6LApBDmrMFXpyLTYCbVYEbn6srEtRtw8fa+4ra90OWuSvs88QzfPT4NQ1ERrW4cDsDRjT/g7OnJHYs+x9XbB7PJxPrHppGybw+KyYRKo7G7KgbspnW+6256/+dxh+I79tNGNr34AuNXrsVQVMTaqZNxb9wEnasrWcePAeCpUdHMVYNKpSLTYCJZb75oOSq1mqBOXUjdtwcV4KIGgxnaeWjJMyoklJrwDArGUFCAk4e7Xa/QkK7dGDbnbeK/XcOOD98jrMd1BHXoTMreOFL2xtH3yWfoOHpspdepOmt5FLOZ759+jMSd21HMZtz8G2E2lqEvKEBRFDqMGk3HseOu2oNVURSWjR2FVgWuKii4YJOGxVzHLe9+6NC6KGYz3z3xKMl742h6fR80Oh3phw9RkpPNqA8/xTs4hI0vPE3Knji7zwV37oJaqyU5bvdl11ut1eLq40tYj+tI2buHgrRUy3SNBp27O1F9b+Dc6ZOcPRIPgJdGhZtGRYFRochsOTJG9OpNkzZt2b9iGWUX3A4Li7mO/LRUGjVrTllJiW19rdswVW8iw2DGVQ0lZmjcqjWZx44yaMbLtB5++R6VpXl5/PB/j5N2cD9OKjBc4iB94+y5NB8w6LLLKMnNYdXkiRgKCwnveT3Hf/zB8geVClcfH8tFGdC0d1/ObN1CVP8BNGregjPb/uTs0b8Y9tpcmvUfaFve4bWr2Tz3NYI7dyU0Oob0wwdJ3LmdHg9Opfukyba4K7PfrzZf7KcLiV208LLrdmEZvdTvW+fqRvvb7qDwbDonf91ERK/enNm6hWAnNanlTTsyyyz7xlurJt9optgMLQYNZegr59ttKorChmee4O+tW2wXsRW5NwnAJyyMlD1xBLTrwIi35rPklmF0GjuORs1b8ssrM7n+kcfoNHac7TOX6i1uVBROFhspqbAagZ26kH7A8oxsL40KZ7WK7DLLeds6W8We5d9On4qhsBCAjCOH7eIM7R5D5tGj6AsLaD9qNE5ubsR/t85SW6coaJycCOzQCUNxEZl/HbnstgdL4nepvMF64QXgrAL9JWYKbN+B0Z8sveLyq1qd7PAxduxYPDw8SE1NJTY2ll9++YV33nnHoWW4qi07C0BXfoVWYFTQqs7vwLa33Gb3GRWWnddEp0JfUMDhtavo+fCj3PTGPGLun8Loj5cQ1e8Gti94F7PReMnv3fvlUhTFzJ1Lv+L6Rx6j31PPcvvCz8g+87ftSgZg/ePTbImfv1ZFhIt9Gzo14KNRUVZSgn+zFvR78hnKiouImTzFckW2/HMKMzM5uHolap0OU2kpES4a2rppcFOrOHf6JD/99xnMJhPbPphPZJ9+uPj4UJSeRoSLhgAnS9Gwtlu6lLLiYvT5eQS262A33SsoGPdGjclLSmLHh+/TcugwbvvfImLuf5Cb336P7pMeYM8XiynOPmf3uZyEMxxeswrf8AjK8vOIcNHgXl5CtSpo66axGyE+bvEnOLm7cdcXK+k19VFueOYFbv1gIdmnTwGWpN5HqyJRb+JAoZFUg+UHGD3pAYcTP7j8aPXfPfEIJbk5jFn8JX0ee4o+jz3F2CXLKD6XxaHVKwFI2L6V5LhY1BotAe07oNZo0Tg7g0qF1sUFjbMzWmdnGrdui1qn48DKZRSkpzkUX0FaKi7ePviGR7Br4QK8Q0K56/OviOzdD4BgJzUFJkttFECuUUGN5aQRVL6/1SoVLt4+pO7bA1h+C6VmCHfRkGe0XIV7h4ZRfO4cfs2aU5KZSYSLhpauGlRA6r69/Pj80+xe9DGd75rALfM/JOb+B7n1g4W0u+U2di5cQFmFi5B/us2r4gkBSXGxJGzfiquvHxG9enPvtxu5d91G/Js1xyc0nENrVllq1bhyY3Xf8Aic3D0wKlBsxi5WFZARf8jhdUnctYPEXTsYPucths95i6Evz2H88tV4h4Sya+ECy7L2xJXXIpxfXtr+faTticNVbalhsP5rncddo8JsNOITEcGg/77ExDXfM2bxl6BS0f+ZF3jgp80MfGEWncbcBUBjnZpmblqCnDWEOp8/XbS/9XZ6TH7IdhDVVfh9pu3ZjbGkhKS4WFLjYmlSfiL30KgoMytkGCwX2lGulov1Hg9OpcXgoWz/8D1Mxstf7P4863kyDh8ELImftey2c9fa2lr+/saVn5V8cPVKSrLPMWbxl2Qd+8vy/VOmoXNxof1tdzDyvY8AOLN1C73/8wTD57xFzP1TuOPTz2naqzfbP3jXdiI3lpay438LaDPiFkZ9+Akx9z/IyHc+oOvd9xK35FNKymt6KrvfrzSfvrCA3Ys/QaVS2f7uUr7trSXzwjJq/X2HO6txUVvOXWZ9KedOHmfoy3MY8PxMzmzdAliOrwB5RjOeGhWtyvd5ZPk+OvHLT+Sc+du27JS9cfz95x+gKKhVKsKc1baaL4Dic1nc/Pb7gKUpitbZBZ+wcMqKi2k55Ea8Q8PIu+DJNpfq5JRQYqmNrLhNrIkfQJiLhlAXDS3dtVRMc+1ryztQeDaDonNZABds392WUQ0UhX5PPUuvadMZ89mXoCj4RTbj4T92MuqDhagr1HqqsJQ95wu2v47zCZVPhdO1tbyApVbQyl+nssWRWX6MuJbqXPJ38uRJNm/ezNy5c3FzcyMqKooZM2bwwQcfOLScEjO2E15Z+ZVLkVnBtcLeqXjL01VtORHqFWjipMG6b8Ov62WbR6VW0/H2MRRmpNvazlwoMXYnzQcOsWtU6t+sOaFdo0mM3QlYroDsCq+zBrfyuNzKwzMDBkXBRQ1pB/eTm5SIb9NIMo7E0+yGgSTF7iRl724UkwlzWRk+GvDTqXHWqAkpP1Am7d5FUlwsBelpRPbpR1LsTkJ0lvkCy7eNt1q5bC8/J3d33Bo1Ijku1m569pnTFGWeRePsTElONh1Hj7W7BdJh9FjMRiMpe+1rLZJ270Kt0ZKTcIYQnaXqvMhsOQEZFUBl3/MwYcd2Wt14E86enrZlOHuev5IuNkNTVy2t3bSEO2toXH4SatZ/wCX3zZVY23VZt0/FXpD5KcmEdOlmN3yMV1Awkb37krjLsk8Td+3AvXETTAY9islEkzZtLLeaFAVTWRkmvZ6Atu0xGfSYy8pQzGaSLtiuV+PbtCmlebmcPfoXSXGxtL35VrQuLuSXH2Q1KDipIN+oUGpWKDQphJVfZZeYFTw0loN4SU42N701nz6PP42Ljy8K8HepydZz083Pn+DOXUg7sM+2Pdy1any1KnQoJO/ZjVFfalfDp1Kp6DB6LIbCQjL+iv/X27wyPU+vJmnXDtz8G1F8LosOt92BWqNB6+JC25tvJTcpAbVGQ+HZDOB8RwurCxurB5U/ui3EWW2L1VerQqMCQ1GRLZmr7Lokxu7EMyiYiPJnAQO22JJid5G4awcKEFq+/5zUlto5BQjQqSgxQ4CTmhKz/TwGs4KXRkXKnjjbdzZp3ZbA9h1JKj/+gKXJCUCFfI8Cs2I70ZnNZnISEygrKkINlCmWCwc/nZpgrUJRViaGwkJCdOBffiwprHDhYR1CBiw9oDvcPobic1lknzp1yX1lLQuBFW7uhFVYr0ZOGrw0KvT5+VcsF0mxu2jauy9eQcHkJiWi1mrpfu/9NO3dl6RdOwnv3sNyUQa0HzXa9jmVWk2H28fY2hEDZB47ij4/76LjW8fRYzEZDKTu21vpMny1+Q6uWoliNhOks0w3A6WK5RxmLZkVy2jF37e3Tk2p2XIeCXVS2b631Y03oXO13GmytnQyKJZ9Y10f6zLVWi2Ju3fZlp+4aye68rtUIU4qdGpL2Qt0snxOMZlIituFZ2AQJr2eM9v+JOfMGXybRlKQnkZuUiJ+EZF2+6ZiJyfrOuSXjw1YcZtUvL9nnddFrbKbbv1dKopCctxuPAIDKcyw3KHw0Khsy3JRnW+nbl3n3IQzAOgL8mzLyzpx3PZ/BUvZM5ZfgFi3uqF8ujVXsHKr8BsqLs8DdSowK9jiCHW+9h0k61zyFx8fj5+fH8HB50+0bdu2JTExkdzc3Ivm1+v15Ofn272sfHVqvDQq0vVmW1JVVKG93+4li1CV91q1v6g6P4+l40CF7yvvPaQtP4BcSOvsjKFCD6OKn7N+Jr/CbVewtLfRl99uqdDxD7VKZTsY5yUn0fmuCZze/Bspe3ajKArZf5+/UrPW5AFUvNlYnGW5GspLsiSb1qsu63HEtUINx4VUajWdx44nfv1advzvA86dOsnpLZvZ8MyTeAYF07S8DY++wH59DUWF5dvCxX7bODljNptscVjX2Rq6dQ2sMarU6ouWbY1Tq4LkUhPZ5R0VVCo4V2amcas2/6jN2NWG3riwFhMs623dp1pnZ4wGfXncGkz680PNWHtGG/V6u17SF26fq2nauy/eYeFsfOH/UGs05CQmsOuTjzj+80a8Q8NINaowKqCgcK68FtS2LQGTotjeq9UaOo25i8jeffEKCeXmee8z4et1jJy/AJ2bm+2WfMXtYeJ8DQKAvvxWi5XhKr+NC1X3cCdaZxeM5W29Kvb60xcWotY5YTab8QwKsnS8KLPc6jWYFbLLzBd1vAjv0ROAHKNCoclMocnM6RKT7aSaXn5lX9l10To5YSwtwWyyv4OgLyxArdNe9Hk4f4yw7gPrvxXnUXP+WGb9TkVRMBQV2pIewHYhk1ZhvUtMiu3I5xsRYds/1nqNYrNCodFMdoVjqIdGhYtahZdGRXKpieLyA8s5g/02tN6Wu1zZsH6Xr05tu/B2v+Dspb1gvS5F6+RkO2ZUvB1nKCy0fbdiKp9W3oHFSl9of9zSODuVT7c/BlmXr3V2rnQZvup8KZb5XMpPANZ9bX3vpj5/vDOY7X/f1iVW/H3nJSdhLC3FbDLiE96UsyaVrSYxz2hfzkO6RqOYzXb7RuvsbLsz5apW2Y7NZRWq3w58tYKQrpaOhr+/8So6Nzdcff34/qnpuPr40nLojXbrauvkVF7mioz2xyirCne2SdKbSdebyDSYsP5SnD09KTybwdmjR/jllVmkHdxPxHWWc5EGOFViJM9optik2H6fkdf3rbBulv2r0lQYJUN18X5RqbCrbQRLmbS0j1Xbton1XFrx2KjCcv6uuLxrrc4lfwUFBbi7u9tNcyu/Aim84GQDMGfOHLy9vW2vsLDz3coLTQoRrhrcNSqKy3dQxUOtk7s7vuFNAUtjUhXgobYc4K0J1MlfN9nGjCvJzWH3Z5/SpE1bu+E1KmoxcDAnf/uFtAP7AcuB99hPGzj71xFaDBoCnL8CskrRm9CVl43C8tLmpLLcDra2hfAODaPNTSNpd+tocpMSyTnzN3uWLrIVWutB16gopOvNtuUFdexEQLv2tl5ThSYFs6KQojeh5nwBudzQEl3G3U30xPs4sHI5KyaMYcMzT+Ds6ckt735I49Zt8IuMInbRQtt4Z0a9nu0L3sXZ04vQ7vaNpyP79kNd3m6zwGjGufzAdtZg6e2oU9tfjUb26cfRjd9ztvz2jaIoZJS3U/LRqnDXqOzGGzMD/Z954ZLrcTUXXpVaWd9nHj1Kwo5ttulntv1JYuxO2z5tPmgo+rw8nNw9MBQVcvboEZw8PABL20lnL2/S4w9RVlqK1tkFnasbTXtVrgG8lUar45Z3P8SjUWNMBgOH16xizxdL6DLubm7732d4BIdgBrLKFM6WH6Wt8fvqLLVE6XpLyfYODSP98EFObPqJVjcOJ6Ln9bZEp8WgIbYrYevn841m8oyKbZ+5+Piy46P3KSu13OI1FBWx8+OP8AoOoUnrtpVan6tt83873EnzgYMxFBbgERBI3JLPKM4+R25SIgdXfYVHkwA0Oici+/Sr1Nh11jsAxSaFE8UmThSbKDErttpma9OIyq5Li0FDKcnJYe/nS2wJSk5iAoe+WWVXG1hxedZtbz2hWf+tOI+PTm3r0OYdGoaiKBz5bh3Zp0/RYuCQ8+vToydO7u44efvY1jun/HPOnl54h4TZ9o8ay7HxdImJEyUmSip8n/W7rcfZzPJyl2owE9Q1miEvz6E0L4/YRR/j37wFPhFNL7mvKpaFwPLqyHSD2TYETbFJscV3pXLRfNAQEnft4Mz2rQR37oZiNrNy0gQSdmyj+cDB/PHW65iNZaBSsX3Bu7bb0MXZ2cQtWURQx8620R8at2yNd1g4uz75yJYAlpWWsOOj93H19SW0W/dKl+Grzdd8wODy7WbCpJz/naUaTDipIKp8+1r3VcXft0ZlSb7PGszklidUnsHB7Pz4Q8wmEzfOnktwdAyl5V99rkyxlfOgrtG4N2mCSqMhqk9/W1wtBg7GVJ78JetNuKktyc25Col/RvxBjm743rJdiorRF+Tz03+fQaXWcMu7H+Lk7nHR/qn4W0vU2x+jrKzrHhIdgxlIM5htbRs9AgJx9vJm3SNT+HrSBBJ2bOWG52bQorwNfhMnta2sHis22tqNuvr52sqSqrwvgFqjtjVRCdNuPl8AACAASURBVLigaVOu8fw5tGKdXZrBjEEBLed7/esv+D2CpYbVt0I2eOE6Xgt1rsPH2rVreeCBB8gqr7ECOHToEB07diQ3NxfvC9py6fV69OVX92Dp8BEWFsb7MR1RqzWElXfNzikzk1amWNrHlZVdshErWKpryxTLbSy/Zs3J/vs0Ll7e+Ec1Iz3+EBonZ2597yMat2p9yc+XlZaw/rFppB3YT0C79pSVlJJ9+iStbhzOoBkv23pUfTt9qt1tmAtFumgwKQpJBssYXP7NmqNxcubsX/EEdepCzP0P4urrR07CGX6a8SyU3yIuvyBEpVYT0t3yaJ2sE8dZ9+hDlrYPZjMqLFc0/joVeWZ1pR4npS8o4NypE7j4+ODXNMo2PT3+EOunT0NRzAS0bW/rbj/0ldcvefv1yPff8tvslwDLLSdre+UgJ8vttIq92Ia++jrrHplC5vFjBLbviD4/j5yEM3g0CaAkK5NQJxW68tuc50zY1vefulxPtKCu0ah1OhK2b7XsdwUyjx+l6fV9GPb6W7ZxFrd/+D57v1h81e9RqdUMfnH2ZTsNVUbqgX38PPN5irIyCerYicLMTPJTkukweixNe/XGOzSMLW+/YVsfdzUk6C2PYNK5uuLXrDkZhw8R0L4Dt7z7ka0jEoDZaOTHF57h9JbfAWyN713UUIalXHUaO44Nzz5lacfYqjVn/zqCYjYx4q33bLUBlVGdjzgDiF30MbGf/s9SC6Qotl58AAOen0mbm0ba5s1NSrzk2HUVY03ZvYvGWkttiElRSDWqbLE6ui67PvmI/2fvvsObKt8+gH+T7pFuaEsHtIVSSssesstSQKYg+GMPRVEUHLwCKksRcQ9wIDIUGQqyhwJlCGUWaKGMlhY6oAO60nQkafK8f5RGwmwg6SDfz3X1uppzkpP7PHmS3HnWObHsZzj5+MKxVi2kn42Fs48vBn3/M/Z+OBupx45ACuhdMiulRAOpRAJriYBKW9ZqrtL+dx95qRapt95UXmFNoFQUIPfqFYT2H4Su09/X675M3L8Xf38wA1b2DpB5eiIvJQUCAhqlEo61PeHs64drp6MBIeBtJYGtRdnwgRulZe81AHedb5pKwMHLCwWZmbB1coJ7YH1knj8HiYUFBnzzAzxDG9/3tSovvzqWAtdVWpSKsoTDSgLdj2Dflq0xcNH9J0VoStXY8e7bSI46BLegBshJuvzfZ71EUvb6W1igw+SpOPzdV7BzdYNbvQBknDsLSztbDPzuJ3jUb6A73vWY07dmekvgGdoYN+IvobSkGL3mf4qAjp314n7Y6/6w+60bO6Js5j7K6lf55BtPKyk8bnWjp6kE3BoE45kPP9F7f1tLyhIeDcqSdys7OyiyMtHprf9D01urHuSlpiAt+gSiFn+L0pIS1GoYgoLMdBRlZ6Pb9A8Q2l9/hYiTK37B0Z/u/R60dnCAEIC2VI3O70xH/a49kJ2YABtHGdwCgx66kkH5ey165TJknYu9qw5ZOjpCKZfDo0FDlMjzocjMgP9T7dH382/KJrMlXIJGqUKthiG6Fsvb64+VtOxYN9QCdh4eKLxxA24BgbBycEDmubNwqVsPBRnpsLS2Rq2GjZBx/hxK72gJvh8pyr4/7aUSFGnvWDkEt1Z3AOBnI9WtvJCm1GJSVHSFjm8sNS75S0hIQHBwMDIyMuDpWXad1nXr1uGdd95BaurDu4Hy8/Ph4uKCec0bw9bSUi/J82vVBp3e/j+kHDmM5KjDyL+eBo1KDSs7W5TI5SjJ/28MgG/L1uj+/hwU5WTj4s5tZZUnMAghffrCwc39gTFo1Gok7Y9E8vGjsLC0QkDnLqjbtp3eVPoSuRx7P5qDtOj/Zu1JLS1h6+Ss18Xo27I1gp/ujdSTxyG0Gvi3aYfACP0LVmdeOI+9H85CQWam3rl2e2+2brZZUU4Ozm1cj0s7t6Potlm2d97vUShu3sTF7ZuRm5wMWW1PNHy2H1wf0PWafjYW+xbMRUFm1j333x5TaUkJLkfuQdrpk7C0tkFQ1x7waBCMfR/P0xszZ4zzKJHLETl/7j2Pa21vjyuH/0VyVNmagnXbd0RAx853LXZ9PeY04rZtQs7ly2XdJhIphNBCW1q2jJBnWBiaPP/CXeNhHoWqqAjx/+wsW/Db3gENej4D7/CmDzwfjwbBcC7vymzdFkER3WBhbX3XsbUaDRL27sbxpT+iKPu/+nh7Oedfv4aL27dCnn4dLr6+CHm2/0OvbHGnB5W5sdbGyjh3Fue3bcHNxHhIpRbwDG2M0L4D4R4UZNRYH+Vc0s/GIGH331AVFcIrrAmCe/aCtYMDSuRy7Jn3Aa7dmpBSzrtJE0gtLXHt1Kl7Hg8AfFu2Qt32HZERdxaWVjYI7NoNfq3a3PMLOTclGRd3bIMiKxOu9eohpE9/lOTm4OKu7SjOy4Vr3QCkx57G9dO3DcK/dU4A7nu+xXm5dx3X0cPjrud/WPnezqdlS/R4f95D64VWo8GVQweRHHUIGq0GivR05F69UvYDtXE4erw/F7ZOTshJSsTFXdtRePMm3IPqI6R333te3qsgKxOXtm9FbmoKnLy80bBPX7jc1vpY0de9IvXnn1kzkH42VrffRuYIZYHivve/83gufn5w8a8HBw8PNOzVB7WC726kKM7NxcVd23EzIR727u4I6fUs3IPq37MsU08ex/5P5ut9ZzjWrg2P+sFwCwhAwz794OTlfd/X4mHuVyYR776H66ejkXzsCCRSCwR06ox67TrctRxNRY7VdeYs3Lh0AYn7I6FRq+HXqg3qd+2OwpxsXNy2GfnXr8PZxwf+T3XA3o9mQ5F19/eSnasrZF5eyL+WBqX8v2EAPs1bQAgtrp8588Dz9GnRAv/7cVmlXlSixiV/ANCpUyf4+vpiyZIluHnzJvr164chQ4Zgzpw5D31sUlISggz8UCciIiIylaysLNSqVavSnq9GXuFj/fr1mDx5MgICAiCVSjF69Gh88MEHFXqs261fbikpKXd1EZNxlXexp6amwslMV96vTCzvysXyrjws68rF8q485WVtfY8eFlOqkcmfp6cn/vzzz0d6bPl6Pc7OzqzUlcTJyYllXYlY3pWL5V15WNaVi+VdeSqzyxeogbN9iYiIiOjRMfkjIiIiMiMWcyoyS+IJY2FhgYiIiApdC5geD8u6crG8KxfLu/KwrCsXy7vyVEVZ18jZvkRERET0aNjtS0RERGRGmPwRERERmREmf0RERERmxGySv6ysLAwcOBAuLi7w8PDA1KlTUVpaWtVh1Rjr1q2DpaUlHB0ddX+jRo0CAOzYsQPh4eFwcHBAo0aNsG3bNr3Hfvrpp/D19YWDgwMiIiJw6dIl3b7CwkKMGzcO7u7ucHZ2xujRo6FQKGCubty4gfr162P//v26bceOHUPbtm3h6OiIgIAA/PLLL3qPWblyJerXrw8HBwe0atUKR44c0e3TaDSYNm0aPD09IZPJMGDAAKSnp+v2m/v74l7lPWnSJNjY2OjV9SVLluj2P059jo+PR/fu3SGTyVCnTh18/PHHlXKeVSkmJgY9e/aEm5sbvLy8MHr0aN212Vm3je9B5c26bXyRkZFo27YtnJyc4OXlhddffx3FxcUAqnn9FmYiIiJCjBgxQhQWForExETRuHFj8emnn1Z1WDXG22+/LcaOHXvX9vj4eGFrays2btwo1Gq1WLdunbCzsxNpaWlCCCFWrFghfHx8xLlz50RxcbF46623ROPGjYVWqxVCCDF27FjRvXt3kZ2dLTIzM0WXLl3Eq6++WqnnVl0cOnRIBAUFCQBi3759QgghcnJyhJubm1i0aJFQq9Vi7969QiaTiWPHjgkhhNi3b5+QyWTi0KFDQqVSiS+//FJ4eHiIwsJCIYQQc+bMEU2aNBEpKSkiPz9fDBs2TPTp00f3nOb8vrhXeQshRMuWLcWKFSvu+ZjHqc8qlUo0aNBAvPvuu0KpVIpTp06JOnXqiD/++MPk51pVioqKhLe3t5g1a5ZQKpXi5s2bok+fPqJv376s2ybwoPIWgnXb2LKysoStra1Yvny50Gg04vr16yIsLEzMmjWr2tdvs0j+EhISBABx7do13ba1a9cKf3//KoyqZuncubNYtGjRXdvfe+890bNnT71tvXr1ErNmzRJCCNGhQwcxf/583T6VSiVkMpmIjIwUhYWFwsrKShw+fFi3/+jRo8LOzk73BjAXK1asEP7+/mLt2rV6ycjPP/8sGjRooHffV155RYwePVoIIcSIESPESy+9pLc/JCRELFu2TAghhK+vr/j99991+zIyMoREIhGJiYlm/b64X3mXlJQIa2trce7cuXs+7nHq8+7du4Wjo6NQKpW6/Z988ono3LmzaU6yGrh48aLo1auXKC0t1W3bvHmzcHJyYt02gQeVN+u2acjlciGEEFqtVpw9e1bUr19ffPfdd9W+fptFt29cXBzc3NxQp04d3bbQ0FCkpKQgLy+vCiOrGbRaLU6dOoXt27ejbt268PX1xcSJE5Gbm4u4uDiEh4fr3T80NBQxMTEAcNd+KysrNGjQADExMUhISIBardbbHxoaiuLiYsTHx1fOyVUTzzzzDBITEzFs2DC97YaW7+378/PzkZaWprff09MTrq6uiI2NNev3xf3KOyYmBmq1GrNmzYKnpyeCg4OxcOFCaLVaAI9Xn+Pi4hAcHKx3Dc/bX8snUcOGDbFz505YWFjotq1fvx4tW7Zk3TaBB5U367ZpyGQyAICfnx/Cw8Ph7e2NcePGVfv6bRbJX0FBARwcHPS22dvbA4BZjy+rqBs3bqB58+YYMmQILly4gKioKCQkJGDkyJH3Ldvycn3Q/oKCAgDQ22+ur4uXl9c9F/g0dvneud9c3xf3K+/8/HxERETgjTfeQFpaGlatWoVvv/0WX3zxBYDHq88Pey2fdEIIvP/++9i6dSu++eYb1m0Tu7O8WbdNKyEhAdeuXYOFhQWGDBlS7eu3WSR/Dg4OKCoq0ttWfrs8a6f78/T0xMGDBzF+/HjY29vD398fn376KXbu3AkhxD3Ltrxc71f2MplMV3lv38/XRd+Dyu9h++9Vvnfu5/tCX8+ePREZGYkuXbrAysoKbdq0wdSpU7Fu3ToAj1efH/ZaPsnkcjmGDBmCVatW4eDBg7oJYqzbpnGv8mbdNi07OzvUqVMHCxcuxK5du6p9/TaL5C8sLAzZ2dnIzMzUbTt//jx8fX3h7OxchZHVDLGxsZg+fTrEbReDUSqVkEqlaNOmDeLi4vTuf/78eYSFhQEoK/vb96vVaiQkJCAsLAwNGzaElZWV3v7z58/D2toawcHBJj6rmuHO8gMeXL6373d1dYWPj4/e/oyMDOTk5CAsLIzvi3vYtGkTfvrpJ71tSqUSdnZ2AB6vPoeFhSE+Pl5vRt7tr+WTKjExEa1bt4ZcLsfJkyd1XVms26Zxv/Jm3Ta+qKgohISEQKVS6bYplUpYW1sjNDS0etfvig9rrNk6duwoXnjhBSGXy0VSUpJo3LixmD17dlWHVSOkpqYKBwcHsXDhQqFWq0VycrJ46qmnxIQJE8SFCxeEra2tWLdunW62r62trbh06ZIQQoilS5cKHx8fcebMGd0Msvr16wuVSiWEEGLkyJEiIiJCZGVliaysLBERESHGjBlThWdb9XDbBISbN28KFxcX8dVXXwmVSiUiIyN1g7CFEGLPnj262yqVSnz11VfC1dVVZGdnCyGEeP/990VYWJhISkoScrlcDBs2THTp0kX3XHxf6Jf3X3/9Jezs7MSePXuEVqsVUVFRwsPDQ/z2229CiMerz2q1WgQEBIi3335bFBcXizNnzog6deqI5cuXV8FZV46cnBzh7+8vxo4dKzQajd4+1m3je1B5s24bX0FBgfDz8xNvvvmmUCqV4urVq6JNmzZi0qRJ1b5+m03yl5GRIYYMGSLc3d1FrVq1xNtvv603I4oebP/+/aJdu3ZCJpOJWrVqiddff10UFxcLIYTYtWuXaNq0qXB0dBSNGzcW27dv1z1Oq9WKzz//XAQEBAhHR0fRtWtXXWIoRNlMqZdeekl4enoKV1dXMXbsWKFQKCr9/KoT3LH0yIkTJ0T79u2FTCYTgYGBd32g/vbbb6Jhw4bCwcFBtGnTRhw9elS3T6VSiXfffVf4+PgIJycnMWDAAJGZmanbz/fF3eX9448/iuDgYGFvby8CAwPF4sWLdfsetz4nJCSIp59+Wjg7OwsfHx/xySefVMo5VpUvvvhCABD29vbCwcFB708I1m1je1h5s24bX1xcnOjZs6dwcXERdevWFe+9954oKSkRQlTv+i0R4ra+PCIiIiJ6opnFmD8iIiIiKsPkj4iIiMiMMPkjIiIiMiNM/oiIiIjMCJM/IiIiIjPC5I+IiIjIjDD5IyIiIjIjTP6IiIiIzAiTPyIiIiIzwuSPiIiIyIww+SMiIiIyI0z+iIiIiMwIkz8iIiIiM8Lkj4iIiMiMMPkjIiIiMiNM/oiIiIjMCJM/IiIiIjPC5I+IiIjIjDD5IyIiIjIjTP6IiIiIzAiTPyIiIiIzwuSPiIiIyIww+SMiIiIyI0z+iIiIiMwIkz8iIiIiM8Lkj4iIiMiMMPkjIiIiMiNM/oiIiIjMCJM/IiIiIjPC5I+IiIjIjDD5IyIiIjIjTP6IiIiIzAiTPyIiIiIzwuSPiIiIyIww+SMiIiIyI0z+iIiIiMwIkz8iIiIiM8Lkj4iIiMiMVGnyd+PGDdSvXx/79+/XbTt27Bjatm0LR0dHBAQE4JdfftF7zMqVK1G/fn04ODigVatWOHLkiEHPKYSAXC6HEMIYp0BERERUo1RZ8nf48GG0a9cOiYmJum25ubno06cPRo8ejby8PPzyyy948803cfz4cQDA/v378frrr2PlypXIy8vDiBEj0L9/fxQVFVX4eQsKCuDs7IyCggKjnxMRERFRdVclyd/KlSsxfPhwzJ8/X2/7hg0b4O7ujtdeew2Wlpbo1q0bRowYgcWLFwMAli5dihdeeAEdOnSAlZUV3nzzTXh4eGDdunVVcRpERERENU6VJH/PPPMMEhMTMWzYML3tcXFxCA8P19sWGhqKmJiYCu2/F6VSCblcrvdHREREZK6qJPnz8vKCpaXlXdsLCgrg4OCgt83e3h4KhaJC++9lwYIFcHZ21v35+fkZ4QyIiIiIaqZqNdvXwcHhrvF7RUVFkMlkFdp/LzNmzEB+fr7uLzU11fiBExEREdUQ1Sr5CwsLQ1xcnN628+fPIywsrEL778XGxgZOTk56f0RERETmqlolf8899xwyMjLw9ddfQ61WY9++ffj9998xfvx4AMD48ePx+++/Y9++fVCr1fj666+RmZmJQYMGVXHkRERERDVDtUr+3N3dsXv3bvz5559wd3fHiy++iG+//RZdu3YFAHTv3h3ff/89Jk2aBFdXV6xZswY7d+6Em5tbFUdOREREVDNIhJmtdiyXy+Hs7Iz8/Hx2ARMREZHZqVYtf0RERERkWkz+iIiIiMwIkz8iIiIiM8Lkj4iIiMiMMPkjIiIiMiNM/oiIiIjMCJM/IiIiIjPC5I+IiIjIjDD5IyIiIjIjTP6IiIiIzAiTPyIiIiIzwuSPiIiIyIww+SMiIiIyI0z+iIiIiMwIkz8iIiIiM8Lkj4iIiMiMMPkjIiIiMiNM/oiIiIjMCJM/IiIiIjPC5I+IiIjIjBiU/CkUCnz99dcAgPPnz6Nt27Z49tlnce3aNZMER0RERETGJRFCiIreeezYsThz5gzOnDmDLl26wNPTE7a2tsjPz8fmzZtNGafRyOVyODs7Iz8/H05OTlUdDhEREVGlMij5CwgIQHR0NCQSCWrVqoXk5GS4u7vD29sbubm5pozTaJj8ERERkTkzqNtXLpfDzc0Ne/fuRVBQEHx8fCCRSCCRSEwVHxEREREZkaUhdw4LC8NHH32EnTt3om/fvigoKMB7772Hli1bmio+IiIiIjIig1r+fvjhB+zZswdOTk6YPXs2Tp06hf3792Px4sWmio+IiIiIjMigMX93EkIYvcv3999/x8svv6y3TaVSQSKRQKlUonfv3ti3bx8sLf9rtFy/fj169epVoeNzzB8RERGZM4PX+du9ezcGDBiAli1bIjMzE++88w5KS0uNFtCIESOgUCh0f5cuXYKHhwd++eUXAMDJkyfx999/692nookfERERkbkzKPlbvXo1RowYgbCwMFy+fBkAsGXLFsycOdMkwQkhMGrUKDz77LMYOXIkrly5gpycHLRo0cIkz0dERET0pDMo+VuwYAG2bNmC+fPnQyqVwsvLC9u3b8fq1atNEtyqVasQFxeHL7/8EgBw4sQJyGQyDBs2DLVq1UJYWBiWLVv2wGMolUrI5XK9PyIiIiJzZVDyl5aWhrZt2wKAbqxf/fr1oVAojB6YVqvFhx9+iPfeew8ymQxAWSLXrl07zJ8/H9evX8eXX36JKVOm4M8//7zvcRYsWABnZ2fdn5+fn9FjJSIiInqQ5PRsHDpzGcnp2VUdimETPtq2bYuZM2diwIABcHNzQ05ODnbv3o2ZM2fixIkTRg1s7969GDhwIDIyMuDg4HDf+7322mvIzMzE+vXr77lfqVRCqVTqbsvlcvj5+XHCBxEREZlcvqIYMxdtRFRMom5b+6ZBWDB5EJwc7aokJoPW+Zs/fz4GDBiAgQMHoqSkBK+++ipWr16NNWvWGD2wDRs2YNCgQXqJ37JlyyCTyfD888/rtimVStjZ3b/wbGxsYGNjY/T4iIiIiB5m5qKNOB53FbLarrCys4a6WIXjcVcxY9FGLJ4+vEpiMqjbt0ePHjhy5AhcXFzQtWtXaDQa/PPPP+jdu7fRAzt06BA6d+6sty0/Px+TJ0/G6dOnodVqdeMNJ06caPTnJyIiInocyenZiIpJhJ2bE2xl9rCwtIStzB52bk6Iikmssi5gg1r+ACAzMxPz5s2Du7s7duzYYbIJFElJSfDx8dHbNnXqVBQWFmLQoEHIyspCYGAgfv31V3Tq1MkkMRARERE9qtTMXACAlZ213vby26mZuajr7V7pcRnU8rd48WI8//zzSE9PBwBkZWVh6NCh+PXXX40emEKhuKtFUSKR4P3338fVq1dRVFSEc+fOYciQIUZ/biIiIqLH5efpCgBQF6v0tpffLt9f2QxK/r788ktERkYiLCwMADB27Fjs3r0b8+fPN0lwRERERDVVXW93tG8ahOIcOUoKiqApLUVJQRGKc+Ro3zSoSlr9AANn+8pkMuTn50Mq/S9n1Gq1cHNzQ15enkkCNDZe3o2IiIgqi1xRjBk1ebZvaGgofvvtN4wZM0a3bfXq1QgJCTF6YEREREQ1nZOjHRZPH47k9GykZubCz9O1ylr8yhnU8rd79270798fLVu2RN26dZGSkoJTp05hx44d6NKliynjNBq2/BEREZE5Myj5A4D4+HisXbsW6enp8PPzwwsvvIDAwEBTxWd0TP6IiIjInBmc/NV0TP6IiIjInBk05i8uLg7Tpk1DfHw8tFqt3r6kpCSjBkZERERExmdQ8vfyyy/D3t4e06dPh5WVlaliIiIiIiITMSj5i42NxbVr1yCTyUwVDxERERGZkEGLPNepUwclJSWmioWIiIiITMyglr/Jkydj0KBBmDJlCjw9PfX2de7c2aiBEREREZHxGTTb9/Yre+gdRCKBRqMxWlCmxNm+REREZM4Mavm7c4YvEREREdUsBo35AwCVSoWNGzfi66+/RlFREWJiYkwRFxERERGZgEEtf4mJiXj66aehUqmQm5uLPn36oFWrVti4cSP69u1rqhiJiIiIyEgMavmbMmUKxo0bh5SUFFhZWSE4OBhLly7FrFmzTBUfERERERmRQRM+PDw8cP36dVhbW8PNzQ05OTnQarVwc3NDXl6eKeM0Gk74ICIiInNmUMufs7MzMjIy9Lalp6fDzc3NqEERERERkWkYlPyNGDECzz33HHbv3g2tVovjx49j5MiReOGFF0wVHxEREREZkUETPj744AMUFxfjueeeQ2FhISIiIjBhwgTMnj3bVPERERERkREZNObvdjdu3ICHhwckEomxYzIpjvkjIiIic2ZQy9+vv/56332jR49+7GCIiIiIyLQMavkLCAjQu52Tk4PCwkJ07NgR+/fvN3ZsJsGWPyIiIjJnBrX8XblyRe+2EAILFy5ETk6OUYMiIiIiItN45DF/5TQaDXx8fO5aAqa6YssfERERmTODr+17p0uXLhl90se6detgaWkJR0dH3d+oUaMAADt27EB4eDgcHBzQqFEjbNu2zajPTURERPQkM6jbt2vXrnqJnkqlQmxsLEaOHGnUoE6cOIFRo0Zh+fLletsTEhIwePBgrFmzBn379sVff/2FoUOHIiEhAT4+PkaNgYiIiOhJZFDyFxERoXfbwsICb775JgYOHGjMmHDixAkMHTr0ru0rV65Ep06ddM83dOhQLF++HEuWLMHcuXONGgMRERHRk8ig5K8yFnPWarU4deoUHBwc8Omnn0Kj0aBPnz5YuHAh4uLiEB4ernf/0NBQxMTE3Pd4SqUSSqVSd1sul5ssdiIiIqLqzqDkb/z48RW637Jlyx4pGKBs8ejmzZtjyJAhWL9+PW7evIkxY8Zg5MiRUCqVcHBw0Lu/vb09FArFfY+3YMECtgoSERER3WLQhA9bW1usWrUKeXl58PT0hFqtxpo1a3D9+nUIIXR/j8PT0xMHDx7E+PHjYW9vD39/f3z66afYuXMnhBAoKirSu39RURFkMtl9jzdjxgzk5+fr/lJTUx8rPiIiIqKazKCWv6SkJKxZswaDBw/WbRszZgw+++yzuyZnPKrY2FisXr0aCxYs0E0uUSqVkEqlaNOmDc6cOaN3//Pnz6NVq1b3PZ6NjQ1sbGyMEhsRERFRTWfQOn8ymQz5+fmQSv9rMNRoNKhVq5bRFnpOX3zBcwAAIABJREFUS0tDSEgIZs2ahbfeegvXr1/HsGHD0LhxY7zzzjto3rw5Vq5cieeeew5//fUXxowZg5iYGAQHB1fo+Fznj4iIiMyZQd2+devWxR9//KG3bfny5WjUqJHRAvL19cX27duxadMmuLm5oVWrVmjdujUWLVqEkJAQbNq0CR9//DFcXV0xb948bNiwocKJHxEREZG5M6jlb8uWLXj++efRpk0b+Pv7IykpCRcvXsSePXvQsmVLU8ZpNHe2/CWnZyM1Mxd+nq6o6+1e1eE9lifpXIgqU0x8GlZujcKFqxnwcHbAoG4tMDCiGaRS4y5gT1RRh89cxqodx3A1PRs+tVzwwjOt0aOt8RpayLwZfHm3ixcvYu3atcjKykJgYCBGjhwJLy8vU8VndOXJX8q1DHzy615ExSTq9rVvGoQFkwfBydGuCiM0XL6iGDMXbXwizoWosh0+cxlTP1+Hut7u6NyiAZKu3cSB6HgMe7oVpo/rXdXhkRnauO805i3ZhvAGPmjTuB5i46/hxPmrmDq8O8b0a1/V4dET4JGu7Zubm4ukpCQ0b94cpaWlsLa2NkVsJlGe/L04+2ecScyCnZsTrOysoS5WoThHjjaN62Hx9OFVHaZBXvtkNY7HXX0izoWoMgkhMGTaj/BwccTiGSNgaVE2Emb1zmP47Nd/sPGLSahXx6OKoyRzUqJSo9dr36BT8waYN6m/buLj57/9gw17ovH34qn8UU+PzaAxfwqFAsOHD4e7uzs6d+6MhIQEBAUF4dKlS6aKz2SOnbsCOzcn2MrsYWFpCVuZPezcnBAVk4jk9OyqDq/CktOzERWT+EScC1Fly8opQNK1mxj2TGtd4gcAQ3q0hJWlhV5rOlFluJCUjnxFMYb3bqN3OdXhvdqgRFWK6IspVRgdPSkMSv6mTZuGwsJCXLx4EdbW1ggMDES/fv0wZcoUU8VnUlZ21ve8nZqZWxXhPJLyWJ+EcyGqbFaWFgCAwmKl3nalqhRarRZWlgathkX02KytyupcUYlKb3vxrdvWt+os0eMwKPnbunUrVqxYgeDgYEgkElhZWeGLL77A8ePHTRWfSamLVfe87efpWhXhPJLyWJ+EcyGqbG7ODmgR4o8VW6KQKy9bQF6j1WLRun2QSCTo2rphFUdI5iYkwAs+tV3w0/qDuoRPpS7F4j/2w8XRDq1C61VtgLdJy8xFYmoWSjXaqg6FDGTQz1qNRqNbMLl8qKBWq62Riyi3DQvAmcQsANAbJ9e+aVCNmilb19sd7ZsG4XjcVQA1+1yIqsKM8b3x0oe/4dk3vkXzEH9cvZ6N6zfyMGNcb3i4OFZ1eGRmLKRSzJ7YD298uga9X/8W4fV9cOFKWVfwp1MGw8a66lujL13NwLyft+F8UjoAwNPNCVOGd0PvDuFVHBlVlEETPkaOHAkrKyssXrwYvr6+yMnJwZtvvomMjAysWbPGlHEaTfmEj9RrGVjwhMz2lSuKMYOzfYke2c08BTbsPYULV9Lh4eKIQV2bo3FQnaoOi8xYWmYu1u+J1i31MrhHCwT61KrqsJCdp8DgaT/Cy90JLz3XCU4Odvhj90nsOXYBi6cPR/umQVUdIlWAQclfVlYW+vfvj9OnT6O0tBT29vZo0KABtm3bhjp1asYHJdf5IyIiejRLN/6LXzYdwo7vpsDVyR5AWU/g2NkrYGtjhZ/eG1nFEVJFGNR+XLt2bRw5cgQnT57E1atX4evrizZt2sDCouYOQK3r7f7EJEpP0rkQEVH1k5CahbD6PrrEDwAkEgk6NAvC2r9PVGFkZAiDJnwAwKlTp9C6dWv07NkTmzZtwldffYXS0lJTxEZERETViKebExLTbkCl1v/ev3Q1A55uTlUUFRnKoORv/vz56NatGwDgjTfe0M3+feedd0wSHBEREVUfg7o2R0FhCT74YTMysvNRWKzE8i2HEXniEp7vUTMu80oGjvlr3LgxVq9ejZCQELi6uuLo0aPw8vJCkyZNkJGRYco4jebOMX9ERERUcX8ficO8Jdt0axFKJRIM790Wb43sobcwNVVfBiV/rq6uyM3NRWRkJEaMGIH09LJp3k5OTpDL5SYL0piY/BERET0eRZESh2Muo0SpRuvG9VCnlktVh0QGMGjCh4+PDw4ePIiVK1eiR48eAIA1a9YgMDDQJMERERFR9eNob4Nn2jWu6jDoERmU/M2dOxe9evWCvb09Dh8+jMjISIwbNw4bNmwwVXxEREREZEQGdfsCQElJCQDA1tYWBQUFUCgU8Pb2NklwpsBuXyIiIjJnBl8nxtbWVve/TCaDTCYzakBEREREZDoGr/NHRERERDUXkz8iIiIiM1Kh5G/06NEAgMjISJMGQ0RERESmVaEJHzKZDDk5OXB3d68x6/ndDyd8EBERkTmr0ISPRo0awc/PD0VFRfdd0y8pKcmogRERERGR8VUo+duyZQv27t2LF198EbNnzzZ1TERERERkIhVK/ry8vDBixAjk5ORgzJgxpo6JiIiIiEzEoNm+r7/+OjZv3ow+ffogNDQU3bt3x+rVq00VGxEREREZmUHJ3+rVqzFmzBiEh4fjjTfeQPPmzTFp0iQsXbrUVPERERERkREZlPwtXLgQGzduxMKFC/HKK6/g888/x6ZNm/Dll18aNaiYmBj07NkTbm5u8PLywujRo3Hz5k0AwKRJk2BjYwNHR0fd35IlS4z6/ERERERPKoOSv+TkZEREROht69KlC9LS0owWUHFxMXr37o327dsjIyMDcXFxyM7Oxrhx4wAAJ06cwJIlS6BQKHR/EydONNrzExERET3JDEr+fH19cfDgQb1tBw8ehL+/v9ECSklJQdOmTTFr1ixYW1vD3d0dL7/8Mg4ePAilUomzZ8+iVatWRns+IiIiInNSodm+5aZOnYqBAwfi5ZdfRmBgIBITE/HTTz/hq6++MlpADRs2xM6dO/W2rV+/Hi1btkRMTAzUajVmzZqFQ4cOwdnZGRMmTMC0adMgld47j1UqlVAqlbrbNX2RaiIiIqLHYVDy9+KLL8LS0hLLly/Hpk2bUK9ePSxduhRDhgwxSXBCCHzwwQfYunUrDh48iIyMDEREROCNN97A2rVrcfr0aQwaNAhSqRTTpk275zEWLFiAuXPnmiQ+IiIiopqmQpd3qwpyuRzjxo1DdHQ0tm7divDw8Hve77PPPsO6detw8uTJe+6/V8ufn58fL+9GREREZsmglr/KkpiYiD59+sDf3x8nT56Eh4cHAGDTpk3IzMzEyy+/rLuvUqmEnZ3dfY9lY2MDGxsbk8dMREREVBMYNOGjMuTm5qJbt25o3749/v77b13iB5R1A7/55pvYu3cvhBA4cuQIvvnmG71kkIiIiIjur9q1/C1fvhwpKSn4448/8Oeff+rtUygU+Oqrr/Dqq68iLS0NXl5emDt3LkaOHFlF0RIRERHVLAaN+fvggw/w4YcfAgBmz55dIydSyOVyODs7c8wfERERmaUKJX/NmjVDv3798PXXXyMzMxP29vZwc3NDTk5OZcRoVEz+iIiIyJxVaMzfzz//DCsrKxQVFcHb2xvt2rVDSUkJ1q1bh/j4eFPHSERENUhyejYOnbmM5PTsqg6FiO7BoG5fJycnZGdn49ixY3j66afRr18/nD59GpmZmcjPzzdlnEbDlj8iMqXyj1SJRFLFkVS+fEUxZi7aiKiYRN229k2DsGDyIDg53n9VBiKqXBVK/v73v/+hb9++mDRpku4KGbd3+yoUCjg6Opo2UiNh8kdEppCSkYPF6/bhQHRZb0iXlsF4bVhX+Hu5VXFklee1T1bjeNxV2Lk5wcrOGupiFYpz5GjTuB4WTx9e1eER0S0V6vbt06cPtm/fDoVCgXr16mH06NFQqVQ4deoU1Gp1jUn8iIhMIStHjvFzVuDs5Wt46blOeOm5Tjh7+RrGz1mBrBzzuKRkcno2omISYefmBFuZPSwsLWErs4edmxOiYhLZBUxUjVQo+Rs1ahRWr14NBwcH7Nu3D926dUNpaSlefPFFuLq6onnz5qaOk4io2lqz6zhUpRqsmj8BEwZ2xISBHbFq/gSoSjVYs+t4VYdXKVIzcwEAVnbWetvLb5fvJ6KqZ9A6f0FBQQgICEBAQAC+++47REdHQ6VS4dy5c6aKj4io2ou+mILOzRvAzclBt83NyQGdmzdA9IWUKoys8vh5ugIA1MUqWMj++2pRF6v09hNR1TPoCh9nzpzR/R8dHQ0AsLa2RosWLYwbFelw1hxR9edoZ4OsnIK7tmflFMDR3jwuL1nX2x3tmwahOEeOkoIiaEpLUVJQhOIcOdo3DUJdb/eqDpGIbql2V/igMpw1R1Rz9OkYjg++34wtB2LQr3MTAMDWg7E4cf4qPnp1QBVHV3kWTB6EGff53CKi6sOgpV6eBDVlti9nzRHVHFqtwOwft2Dbv7Hwci/7XMnIlqNvpyaY+0p/SKXmtexLcno2UjNz4efpyhY/omqIyd8jyMkvxIqtUTh4Kh5SiQQRrRpiTL/2cDZSi1xyejYGvvU9ZLVdYSuz120vKShCQVYuNn35Kj9QiaoZIQROXUzB/pNlS71EtApGixB/s1zvj4iqN3b7GihXXoSxs5cjT1GMZ9o1hlarxR+7T+JAdDxWzBsHmb3tYz9HRWbNVcfkr6CoBPtOXIK8sBjNgv3QOKhOjfviY4sFPcz96ohEIkHLRnXRslHdKozuyZaYmoVj567AxtoKXVs1hJuzw8MfRER3YfJnoNW7jiE7vxB/LJwIn9pls9dG9nkKw6YvwYY9pzC2f/tHOq4QAqUaLawsLWrkrLmDp+Ix47uNKFaqYGNliRJVKTq3aICFUwbD1tqqqsN7KI6xpIdhHak6Gq0W85fuwMZ9p2FtZQGNRouFK3ZhxvjeGNSVS40RGcqg2b4ERMUkomvrhrrEDwACfDzQsVl9HI65bPDxlKpSfLc2Et0mfoE2oz7GkGk/4nxSeo2aNXczT4F3v9mA1o3rYdeiqTi0/F18OmUwjp+7gh//PFDV4VXIzEUbcTzuKmS1XeFW1xOy2q44HncVMxZtrOrQqJpgHak6f+6Oxqb9pzFjfG8cWvYu9v70Nvp2aoIPf96G+OTMqg6PqMZh8mcgK0sLFCvVd20vKlHBytLC4OPNXPQXVu04imc7N8Hsif3gU9sFMxdtRMdm9dGmcT0UZOUiJzkTBVm5aNO4XrWcNbfz8DkIAcx7pT9qu8lgIZWi51OhGPp0a2zcdxrVfVhpdbkyQWxCGr74bTcWLNuJfScvQaPVVsrz0sNVlzpirv6KPIWebUMxtGcrWFlawNnRDjPG94a7syM27T/z8AMQkR52+xqoR9tG+Gb1Xpy6mIIWIf4AgCOxiTgedwXvv9jXoGNduJKOyBOX8PHkQejdIQwAMCCiKWYu2oiVW49g27ev41pWbrUfg5adp0AtV9ldXV+BPh6QF5ZAXaqBtVX1rWpVPcZSCIGvft+D37YfRW03GextrfHH7pNoGxaAb6a9ABvr6lt25qKq64i5y84rRPc2jfS2WVlaoF4dd2TnKaooKqKai98qBhrSoyX2n4zHhLkr0TzEHxqNFrEJaWjfNEi3vldFnb6UCmsrCzzdLlS3TSKRoG+nJtgVFYeMm/mo6+1e7b9UQgK8sHLbEVxKzkDDul4AyhKayBMXEeRbq1onfkDVX5ngeNxV/Lb9KN4c0QMj+zwFqVSCI7GJmPLZOqzacRQTBnY06fPTw1V1HTF3IQFeOBB9CS8O6ggLaVmH1Y3cAsQmpGHSkIiqDY6oBqre38qVLDE1C2v/OYkr127Cp7YLhvZshcZBdfTuY2tthe9nDMeuqDgcOBUPCYD5rw1Ez6dCDe72ldnbQKXWIDtPgdpu/y07k34zHxIJHunKADnyQvz5z0lEX0iBo70N+nQMR/c2ISadddutdQgCfDzw+sK1eHFgR3h7OGP7obM4eCoBH1fDbuo7lV+Z4HjcVQDQW1fRFGMs75wtuuPQWQTUcceoZ5/SvU7tmgThmXaNsePQWSZ/1UBl15GapESlxpb9MdgffQkSSNClVTAGdGlm1Bbrcf074OWPfsMbC9diSI8WkBeWYMXWKDg52GFARDOjPU9N96SsVmCq8yhvlFi98ziS07NhbWWJ1qH1MLpfOwT51jLa89xPdXp9uM7fLYfOXMZbX/wBVyd7NG/oj7jE67h+Iw/zJvXHs50Ma9GrKEWREr0mf41WofUw75X+cHK0w+XULEz+ZA0a1vPEN9NeMOh412/kYfyclZAXFqNd0yDcyC3A2YRrGNy9Bd6b0MekCWBWTgE+Wb4TB6LjoRUCXh7OeGVw5xrzwSxXFN/3ygTGmsl5v9miVhZSFJao8PMHo/Xu/82avdh56Bx2LZ5ilOenx1MZdaSmKVaq8cr8VYhLvIa24YGAAI6eTULTYF98P3OEUWf6H4iOxzer9+DK9bLxlW3CAjB9bC8E+HgY7TlqqidlJropz0MIgY+X7cT6PWWXprWzsYJKXQohAAsLKb6ZNgztmgQ91nPcT3V8fZj8oWwZgb5TFiGwjge+emcorK0sodFqMev7zTh4KgH/fD8VdrbWDznyozl4Kh7vfrMBQgC13Z2QmpGDut5u+Om9UfB0N2wR6vcXb8KJ81fx24fjdS2JG/aewkdLt2P5nLFo1tDPFKegR64ohqJYCU93J133TE1iyl9m97tqi5+nK1LSs7H+s1dQr07ZF1lRiQpDpv2IFiH++Oi1gUaNgx5Pdfr1XtVW7TiKb1bvxbI5YxFe3wcAEBOfhglzV+DtUU/jf73aGPX5hBDIzJHD2soSbk5c46/ck3JFKFOeR0x8GsbOXg5rSws83a4x5rzSD9l5hRj1/i/QaLWwtbHClq8mm+RqPNXx9WG3L4CLVzKQcTMfH782UDc+zUIqxcTBnbHj8Dkcj7uKLi2DTfLcnVsEY+s3r2PHobO4katASD1P9Ggb+khdJntPXMS4/u31upAHdW2On//6F3uPX6yU5M/J0a5G/dK804PGWF6/kYelGw/hSGwirCwt0POpUIzt375CC3uXzxa9/aot5WPHrly7Cd/aLhg3ZyUGd28Oe1sbbDlwBvLCYowf2KFCcReXqLBy2xHsijqHEmUp2oTVw4SBHR8pOYlLvI5fNh3CucTrcJHZYUCXZhj2TGtYWtS8ZN4UasI43MoSefwiOrdooEv8AKBpsC86NKuPyBMXjZ78SSQSeLk7G/WYNd2DPlvKZ6LXhPpqjPNQFCmxYuth/HP0AlSqUrRrGogJAzrC19MVkScuwsnBFvLCErwypDMspFLUdpPh+Z4tsWTDQWTnFyI+OQMhAd7V7rxMgckfoFtSw/KOMXvlY/iKlSps+zcWmdlyNPCvjQ7N6hu1VcvDxRGj+7Z77OMIrYClhf45SCRlTdraO5YNySsowp5jFyAvLEGzhn5o3tCvxl2NozJdv5GHUe8vg1QqQZ+O4SgsVmLNruM4EpuIX2aPhZ3Ng7u3HjZb9NWhXRF9IRl/7o6GSl2Kdk2C8NnUIQj0KRuH8qDWJnWpBpMW/I5LVzPQu0M4nB1ty8akRsdj5bxxutbEijh5/ipeXbAafp6uGBDRFKkZufhy1W6cS7yGjycPYh2pAENbBit6/6wcOfYev4gSlRpPhQeikZG/pB6F5h6fOQBgaSGFRmtWnUomd7968qTMRL/zPIQQUJeoUKoqW1rt4tWMB55HiUqNl+f/hivXbqJPx3A42tlgx+FzOBAdj1/njYdGq4X01vf27XXW0sIC5d+Oxq6z6lINth6MBQBotVoIIXSfoVX9+jD5AxBSzwvuzg5YsSUKn04dDAupFEIILN8SBWsrCyxc8TfyFUWQ2Zf9amjgXxuLpw9HLVdZVYeup2Pz+tgYeRqDu7fQXWd499HzuH4jD23CAnQVb+/xC3h/8SaoSzWws7GGoliJ9k2C8Plbzz80iTFXyzYfhlQqwbqFE3XdTUO6t8Dw95Zix6GzGNy9xQMf/7DZoqGB3ujdIQzvv/is3uMqMlZk99HziIlP0+vaHz+gI4ZNX4Kf//oX8w2YdPPN6r1oFOCNpbNG6378dGgWhFk/bMGI3m0RdlsLz4MoVaUAYFbL1Bg6rseQ+/+5OxqfrtwFiUQCK0sLfLsmEn06hGHupAFV2iLbuUUD/PzXv0i6dkP3Q+Vyahb+PX0Zrw2NeOjj1aUalGq0Ne5zR6PVokSphr2ttcl/ED2snhhjJrpKXQqtEEYboymEQFGJCrY2VhVuKLn9PKQOUuRn5EBdrARuFe+sH7bAQipFj7aN7vn4HYfO4cKVdKz66EWEBpb9MBrbvz2e/7+fsGzzYfRq3xi/7zgGSwsplm+Jwrtjn0FBYQk27I2Gq5M9pBIJGtbzemic+YpiSCUSyBwe3OOTmpmD1xas1iW1hTfzUZyngLO3Oyytrap8pQDz+WS+j52Hz+Lnv/5Fdn4hIk9cRLeJX6DnU6E4n5SOC1fS4WhvA38vN3w8eTx8arsiJj4N075ej5mLNmJMv/bVatzP5GFdMW72Cgx6+3t0ax2CjJv5OByTCEsLKaZ+vg4+tV0wuFsL/LD+ACJaBmP6+N5wcbTHgVPxmLloI374cz/eGtmzqk/jkZh6HNaR2CQ8076x3jijkABvNG/oj6jYxIcmf486W/T2q0qUP6b8qhLlY0WOxCahUYC3Xre+zMEWfTs3wZ+7oyt8jgVFJTiXeB3zJvXXm7nep2M4Pv/1H0TFJj40+buUnIFvVu/FkdgkSCRAh6b1MXVED1haSJ/4cXIVea0e5f6XkjPw8bIdeL5HS7zxv+6wtbHCjkNnMW/JVjQK9MbIPk9V5mnqGfp0K+yKOocRM5eiW+sQCJR1BQf4eGBw95b3fVxWTgG+XLUbe49fQKlGi/D6Pnj9hW5o3bhepcX+KIqVaixetw+b95+BolgJfy83TBjYEf27NDXK8e/1OfawevI4M9GT07Px5ardOHT6MrRCoE3jepgyvIcueTKUEAJ/7onGyq1HcP1GHpwcbDG4ewtMej4CVpYWD/ycvv08ShRFUJcoYevsALWiGC1C/OHiaIeZizaiSQMfvaFN5Y7GJqJZsJ9e7C4ye/TpGI6dh85CUVwCiQQo1Wix7p8T2HEoFiq1BupSDQBg4ZTBD/whtSvqHOb/sgOKIiUAwNnRDh+9OgAdmze4ZznM+G4jJBIJ1n0yEd+s2Ytj565ACAF5RjbsXGQoyS2o0pUCzDb5S8nIwd9bjmHV9qPo0rIBOrVogD1HL+D6zXxsORCDp8IDMezpVlj3z0m4ONri+z/249qNfNzILUBBYTFOnk/GyfPJAMp+hb06pAtyFcXwcHHE2YRrOJd4DY52NnCR2eFaVh4c7MqWXblz6ZjLqVnYejAWeQVFaBxUB307NYH9QyaXCCFw7OwV7D1xERqNFp2a10enFg2QfjMfTzUJRHxyJqJiE1FcooKFhRQvPNMGoYFe+Pf0ZXy7NhKWFlLMmthPt5RMeH0f1Pf1wKodR7HlQAzcnR0w7JnWaNnIH3uPX0RS2g042NmgU/MG6NwiWG9ArDGSrnOXr+GvyNPIUxQh0McDAyKawc/TTe8+l65mYNP+M8jKKUCdWi54ul0jXLmWjaNnk3DmUirSb+br7lvLVYYxfdvhue4tYGVpgQPRl7ArKg4lSjV6PhWKfp2b6H6tFxYrsf3fs4hLug6pRILQwDpoE1bvHucicCQmEbMLt6BFiD+ead8YttZWUBQr4XXbxJyUjBxs3n8GN3ILEFzXE/07N9W14iyYPOies0VnjOuFX7cdweXULNhaWyE0qA6a30rkomISIbWQouBmHgDAxs4WsLRAVEwi1v1zAkN6tISNtSUUxUq9LgWgbPyLzT3WWLx6/SY2H4hBdp4CjQK80bdzE8jsbWFpYQGpRKL7cAPKXt/LqTegVJc+tFUgNTMHE+auhMzeFs0a+kEIgYtX0zHs3Z/0ulPaNw3CR68OwPZDZ7F5fwzUpRq0bxqI14Z2hYOdDUo1Wuw7cRFRsYkoUaoRXNcTXVs1vGf3dXGJCtsPncXZy9fg5GCHfp2bILiu5wPjfJhLVzOw9d9YFBSWoGmwHxoFeOOfo3FITs+Gu7MjBnVthtDAOhBC4OjZJOw9fhFyRTGiYhLhWMvlgeN6yt8vUolEbxxQqUoNiVQKGycH3f3r1HLB3uMX8PNf/8LOxgo924Xq3rP9uzTF0bNJ2LTvDIY93RqRJy7iSGwSbK0t0aNtI6jUpYg8cQlarUCnFg3QuUUDk0zAktnbYtnssVjz93HsPxkPABg/sAOG92oLR3sbRMUk4uzla2jSwEc3k7KoRIUXP/wVxSUqTB7WFc6O9ti47zReXfA7fp41Gs2C//sRk68oxpYDMUhIyYSnmxMGRDSD7wNaSoQQOB53FRsjTyE+JQsAUN+3FtqGB6CWmxPqernhZp4CP204iGtZuXCR2eOFZ9rg2Y7hus81IQSOxCZhy4EzuHI9G86OdujUvAECfTww/5cdyMopQIdmQWgW7IeDpxIw+8ctKC3V4LmH/AB8kHu17jUK8MKEAR119UoilaIotwAAYOVgp1ev7vfZcq8rQqlLNdhz7AIORMfjQPQlODnY4Z3RT8PK0gJ/7D6Jlz78Fas+mqCbRX05NQtbDsQgX1GMsCAfPNspXPcddT7xOn7YcAAp6Tmo5eoIPy83bNp3Bn06hGHysAhcuJKBVTuOITk9ByUq9V3xlX9vln9/LJg8CNO/+wtHYpMAACX5hbrzkEgl+PfVr7Hj0DmM7d/+7vPSaJCZI79rDF2uvBC5BUU4cykNb7zQHYlpN7D76HkUFCnhYGeDpsG+8KntioOn4pGcno0BEc3g4eIIoOwzdPuhWOw+ch7RF1Pues1e/3QtZk/si+Yh/vhubSSiL6RArS6FpaUF8hXFiLg1V0Dv9dEAiht59319KovZzvYNHzyU7udzAAAgAElEQVQTFlZlzbZuTvbIkRfp3c/W2hJKVSnErf9LbnVj3c7awRbWdjYozM7HnaXYwL82ktJuQqPVwsvdCVohkJVTgEnPd8HE5zoDKOvKWbB8B1xlDqhTyxnnk9JRp5YLfp416r6DmrVagTk/bcHWg7Hw93KDlaUFEtNuoJarI27kKlDX2w0WUimSrt0EAMya2FfvwudD3/0Jiak3ELViOmysLXEmPhWvzl+F4nuc3+3sbKxQrFSjS8tgfDZ1CIpKVI89dT1fUYypn6/DmUupum0WFlJIACx4/Tn0aNsI+YpivPPVn7pEGygbT1SqKRul4WBng8Li/5IViVQCcSvRqOvtBheZPWLi0/Set7abDH98MhFFSjVe+vBXpN/Mh72ttV7Sc/u5/LLpEBat2wcAqFen7As8yLcWhnRviU9W7MJXbw9FRKuG+OdIHN5bvAkOdjbw93bDhaR0uMjsseT9UXrLUdyeMKtLNXj5o1UoKCopS+Jui8Hd2R7Z+fr18k4dmtXH/55pjckL12DGuN54vmdLSCQSxCdnYsLclRjUrblea+62f2Mx58ctkNnbws/LDReupMPd2RE/zxoFP083TP18HRJSsvD1O0Px9eq9eq9vixB/fPX20Pu+vh8v24HN+85AVapBvTrukEgkuHKrHlraWcOptivUxSoUZefD+tZlEi2kElhYSKFSa+Bga43fPpqAj5ftwMnzybC3tUZRiUp3/HZNAvHJ68/pnj8rpwAvffgr0jJz0SjQGxk385GdX4j/G/PMI080+G37UXy5ajc8XBxR202G80npZfHfVucAoHlDP3i4yrD76Hn4e7lBXapB+s18WNlZw9nbQ5eEa0pLkZOciU/eeA5bDsTolScAuPjWQlGOHKrbXncAmDdpADbsjUZMfBpk9jZQqkuhUmswok9bvD2yJyQSCX7acBBr/z6OgDoeOH0pFcF1PVFQWKL7IXT7Z0Gn5g3wxVvPP9IlKB9FamYORn+wDHkFxf+dq8wOqz6cgKjYJHyyfCf++mKS7ktaXarByPd/QS1XGRa9+z8AZeuuvjx/FeSFJQgJ8MLV69koLlHhkzeeu+tqH0BZ0vbBD5ux/d+zBsfbspE/fnxvFCQS4IPvN2Pn4XMAAKlEAu0dH+5eHk7IuCnX22ZtZYGd302Bm/OjzUAunw1q6+IIZWGxrkuwnJW9DdRFSljc+jGnUZd9Xn89bRi6tPhvMuLDfowXFivx6oLViE1Ig7uzA3LkhRACGPXsU3hzRA+UKNUY9PYPaNc0ELMn9sMfu0/ik+U74ebkAG+Psu8on9ou+PmD0dhx+Cy+XRMJoGx4R/lQj7re7tj4xSTde2DTvjOYu2QrLCyksHd3hpWdNVSFJXd9b5Z/5trZWqPNqI8xvFcbDH26ld559J+6CBGtGup9pt0rcS7/rEjNzMXYOcuh0Wix6cvX4O9V1qigLtVg+MyfYW9rjYSULEgkEtT3q434lExYSqX4bvr/UNtVhhfn/YrM7Hw86lBACQAB4I0XuuHkheRqtdSLxZw5c+ZUyTM/hqysLIwYMQKvvPIK/r+9+w6Potr/B/7e3lsK6ZUQIAXEEHqJUkSkeK/xi1wBERXBL6iIXrGh3p96QUXE8r0X5ApcUAFREKQTQEroAUwB0kjvbTfZbN/P74/NDlkSmlLEnNfz8Dxkzu7OzJkzZz5zyszHH3+MsrIyDB8+nBvMeTUWiwULFixAxKAxcFgdEMkkMBrN4PF4UPpqofBRw6w3wu5wQi4VwWZ3tryb9lKLCk/AB4jgsNnhdDhBToLKVwcQwelwQsDnQyjgg8cDhiZEIz23FN8teAZKuQRLfziA+xK7wu5w4H//+Q0eGZaAf73+OJKHJ2DUgFhs2n8G+SU1GNkvpt3t33PsHL5cvx/vzhiHt6ePwYSRibBY7Ug9m4fRg+Kw7M3JmPBAIvRNzcjIK0NiTDh6dAnmvl+rN+JkViG6hHZCRJAvnn1vNRpNVjicTvhoFdj+xQv4IeUUrDZXU7hILoFcq4K12YLoUD+cPl8Eb60SX/90GMczC6Dw0ULpo4FQIkZhUSXOF1TgoUHx13Uc537yPU5kFoAv4EMT6A2RTAKbyQKtUoY9x89hwshEzPv8RxzPLACPz4MmwBtimRQWo4mrWGx2B+Q6FWwtQYI20BcimQRWoxkWqw2lVQ3g8/muY+utBjkJBr0RR9IvIiO3BFV1jYgK9UNZtR5ybw14fB4cVjvKavQ4X1CBrmF+eOXTDZj8UF84nYSc4ip0CfVDXkk1Dp7OxZjBPTB17AA0NVvw9D/+i6TeXfGft5/Ao8N7Y/zQe7DraBZOZBZ4PO9Qq5Ij1N8VmL64aB1EQj7CA71RUtUAubcGBILT5uDeIS2SSaDwUsNqNHvk31vPPITvdhxH9wh/BHXSYemPB7D7aBZ2pmbiy/X7ERbojXemj4WkpcWuzmDE9PdWY9SAOHw1fwqShydgzOAe2HYoA+k5pXhocDziOgfhp/1nsOrnVJRU1oPHd5V1iVKGiqqGqx7fBV9vR0OTCQue/ytenzYa/XtEYu3OE1y6wksNoUQEq8kCi9mK3jFh2LjoOUz/6xCIhAIcPpuH/adcd9/dIgJQVd8IubcGQqlrfExZtR4XCi+t/73lW1FYXoc17z+FaeMHYeKoPmgyWfDVjwcxZnA81Iobq1QLy2vx4sfrMGl0P3z+6kQMS+yG73YcB5HrpkvVSQeZVglrS4CVX1KN954bj7eeGYPBvbpg7c4TcNod4Av4ELW0jFiNZliNZtTqm3D6QjF3vvAFAthMFtjMVjisdqh8ddx5ZDNZkJVfhtLqBnz11hQE+3nhWMZFPDV+EFZuSUVC9zD4eavxyepd4PN4KCyvxdI3J+P5x+6Hl0aBlOPnXcfj+b9i7uSRiIkMwPKNB6FTK657zObv9fBLX0JvNHvsV6O+CdsOpUMiFkEsEuLJcZdmsgv4fBiaTNh5JBPTxruWz1m0HgI+H9+8/zQmje6Hxx7og/zSaqzZdhQTRia2eXPQ/lPZ+GLtPu5vqVrhGmTvJAglYjgdDi5N7qWGRCmD3WyBVCxCUUU9/L3VKCqvxdIfDrhm8/ftjtXvPQWVXIKj6RchEvBBcPWUlNfoQSBI1XKIpBJYmi3IyC39Tc81LSyvxUf/3QWFjxZWowl2i43LN3c5cdocUPt5QemjgUyjgNPhhN1iQ1RIJ+41o4Bn3dKeZT8ewP5T2fhq/hTkl9SgS6gf/nJfLyz94QB6x7h6PEqq6pF2vghD7o3GrAXfInl4b/zfa57XqPTcUmzYkwY+n4cv5z2Od2eMQ1RIJ+w+mgV9kwmxnQO5oE0g4OH7PacgUcqg8FKDz+ejud4Ap8PpUT7c14+xQ3pi38kLsNodmDiqD1fXn79Yjq83p2LiqEREhXTi9mnuJ9/jeGYB5N4agOcKjEsq6/FDShrW7ToBlVyKqFA/PNmqtVDA56NOb8T2w5m4JzoEq9+bhgkjE/Ho8AQcy7iIbYczkJFbhjqDEU+MHcB1qV8qP6o2Abrb5WNAdWo5fjmVjbIaPcQKKRxWOxTeGhQVV93Q9fJmuyuf3TBhwgQolUqUlZXh+PHj2LNnDxYvXnxDvyFTuS4M7jsopa8WMrUCfN6lLGk22yASCuBwEoiIu+sih9MVC/J4sFtsUPpoIVHIYG22QK5TQe6tQUWtASP6xeDtZ8dCLBJg7/HzmP7XIfBSK7AjNRO7j56DUCDAC38bxt2NhwV4Y8pD/bHvhGtGX3t2pGYgvksQxg3tyRWy/NJqaJQyFJXXcct6x4QDAH4+eNbj++6xEgtW7MCi1btQXFkPa0se/H3qKNQbmtFovNQK4bDaIVMrIPNS43xBBe7tHoqf9p/53S+5d09/BwCFjwYiqQRSlRxybw1q9UYYTVZs3HcaqWfzQESQa1UQy6SQquQguO7yxSJXvpHT6dpvHg8Wo4mbReWevS33dm2nUCSCupMOfKEAFwoqsP9kNsYM6YHT54sg81JDplZA6aMFeIBQJuG6VrUqOWY/NgzL5z+Bd54dh/AAb3QO8oVELMS7M8aCz+fhl7RsmCw2vDx5JNc96uetxjN/GYy080WoqjO0yYPSqnqk55Tir/ffi5NZhdw2qHy0Hp+zW6ww6ZsgEIsgkl1660tC9zAk9e6KnUey8Nq0B/F/r/0NcVFB8NEq8fpTo7Hy3Sc97ir3nbgAh8OJlyaN4C6cgb5aTHt4IFJ/zUNDYzOC/XT4eE6y68GnYhEkShm0Qb5Q+3ld8/gazVao5FI80D8WPB6PG+gMHg/Uara5rSWofeyBRIiEAvB4PDz18CD4eatRWWtA//jOyMwr4/JDrlVBIBaCLxZy63d3XU0a3ZfrDhYJBZg94T5IxELsOpp1zTJ4uZ1HMqGQSTBrwn0QCvjYe+I8HE73zR8gUcoglkqgaNXNHxcVBB6Px41X4vGA5oYmOOx2mBubYaozoFe3EJw+X+xxvsh1KvBFAjisdih9tB7nkdJHi8paA4be2wX3dA3B+KE9oVPLsf1wOnRqOZZvPIjnPvgGucVVsNjsGD0ongsAdqRm4J6uIQgL8MLOI5kAXI+TGnxvF+xIzbjhPPktUs/moaHR1O5+NTSa0Gy2oLxG79GSCgAlVfXQKl1BS1l1A37NKcH0vw5GJy/XxDqZRISXp4yE0WTFoTO5bdb7Q8ql8a0CsdAVJNkcEEpEHuVPKBFBoVNBplZA4aPlbrJ+SEnD9sOZCAvwhs3uwNyWc7mi1gCJWAiVUsp1Ccu9NZCpFbAazRC01ENp54uuq+67nPs84Qv5sDZbPPJNrlO5HtkAV53ndDhgaTLB3myGVinDicuCkmvZfjgTYwf3wD3RIdCoZCitbsCk0X0R4qfjykdpVQO0Shl2H82CSNj2GjX5oX44+ms+eDweHugXi/49IgEAcVGBrdZzqay5e16EUle9aLfa2uzn5dePZ/4yGKln8/DSJ+uRcvwc1mw7ilkLv0PnYF+PVl/3dcRdV6j9vKD294ZQKkZjsxmzJtyHkf1jUF3XyD3Vwy09txREhBf+Nox7XJdGKcPsx+5HcUUdDp3JwdRxA7DtcNuWZD6fD+EVhmfJdSqoOl0amlDf0qtIfD4sTSbXdU6rvKHr5a1w1wV/ubm52L9/Pz788EPI5XJERkbirbfewhdffHFDv8Pj8yFRyuBsGewpkrnuDN1jq9z4fB43FsQdKIrlUghEQq4tUCQTg+C6QPAFAi74cDpds6ekYhFMZiuEAj7USilMZitMFtdMKJnEswBpVTI4nASbzYH2mCw26C67qzNbbFDIxFwlBgCD7ukCsUiIwvI61LS8+DynqBLLNx5EQvdQJMaGY+2uS60yIX46jOgbc+mC3cJ94eMCKgEfhpYWqKs9WuBaWn+Gz7/UFdX6N8uqL43j47UMxCUitGQ1xEJ3F4gDPD4PfAEPIOLuyC7fdjdBS0XmJOIG+7o/w+PzwOPxwBe61ldT3wSVXAKRUACJWIhxQ3ti4QuP4OH7e7UcI1cpMJlt4PN43CxrN53adaxaHxs39zL3kAL3NvAvG3QslIhgt9jgsNpcs99a5aFWJUez2Qoej4f+PTrj3Rnj8OGLyXhk2L1tZlCaLFYIBYI2rw10txK4bzgamlxddWp/L6h8tVwr1rWOr04tR2OzGet3nYTd4USAT0uQRORqQXRr6UNxzw51c7fUuc8l7pjweODz+eC1nIfFlfWwt8wSvbyFw31Ote4uvl6mltmb7psKV7d02yqy9fFpnRf/nPUX+OpUcNodqCusRGNVPfrEhuOxkYke++Mma5k8dKXzyF221EoZls+fgvBAH9QbmnE8swANTc1Y8spjcDrJIw/MLfWDTq3wyAOdSvGb8uS3SM8tvWw/4PG3l0aJmoYmfPrNHpgsNu6VW1sPpnMTJ0wt26q77EHOmpbgsL19ad3FLBAK4WipQwVioUfw1/r4td7G5pZ6WSIWQsC/dC6bLDZXPS66dD4JxELwBAI4nU401zdC1HJOXU/ddzn3bE9ry013m/qqpTw2VtV7lKveMeFottzYMTVZrNC21EnjhvREdmElVmxOhVopQ7PJgo37TuPQmVyMHdoTJrOVu361plXJ4SQCjwd4ay8dH39vDcIDXa191S1jE6vrG/HT/jMtn3BfRz3rXLfW9cvwvt3xway/IK+kBi8v3oAl36YgoXsY/v3GJI+hC5c/HobH40GikELdkqfRYf54OKkXKusMWPJtCsxWV3nbc+wcjqa7xhW66+jW+wcARK609q7FDruDiwcuxxfw29ThAOC02SHTKqH01bbZ3zvhrgv+MjMz4eXlhcDAS3cZMTExKCoqQkNDQ5vPWywWGAwGj39uSh8Nd1GqK6pEbUEFrM0WyFoGewJAz+gQbtA8ryUgsDab4bDaueDCZrK67gQkIpgNRm78zunzRdhxJAMNTSYkxkXgZFYhCspqkRgbjj6xEdA3mZBy/By3LrvDiY37zqBbuP8Vp5EnxobjaHo+SloVmLioIJRV69G91Swno8kCkdA1TunBWUvw4Kwl+J9Xl0EkFOD/PfcwFr7wCHZ88QKkYiGEQj5KqxpQVFHbZtq5qCWAcAdU6bllSOge6rHM7Uamrrs/w+PzYDYYPfLSbci9rllUfKEAZkMzN6HB3QLbZLIgJjIAdrMVTocTTrsTBMBUZ0C/HpHcs5xsrcZT2a12ros4PNAbaS2DeN3rtTSZQE4Cr6UFeEDPziiurMexjIvcb1htdmz55Sx6x4ZzNwaJsWFwEmHjvtPc54gIG1LS4O+jaXeQeliAN3x1Kpy+bBtMBqPH50RSKWSaljLZarKNRiFFyvHz1z1DMjE2HBab3WNMlNNJ2Lj3NMICvNBJ5wrWWj9yobVrHd+R/WIgEPDxzxXbMXzGJ3jynVVcmkAk5FrD3Mf6bPalsZ4XS2uQW1QJqViIzPwyj/XZrTbYzFZuWEeInw4yqRhxnQOxaf8ZjxakfScvoM5gRJ/YiOvKk9b6xIajqq4Rh07ncn+7bw4EIlcLJRHBbDByrTHuQBFwnb8Wqx1jh/TA569OxKZPnsOX8/7GPT6iTTdRy03MlfL5zIVibjxriJ8XZk24DwDw0qQRWLdgOgbeE4U+seHYkZrJjRVNjA3H4bO5OHOhmMuDupYnGfyWPPkt3A98vtJ+3Z/YFa9MGYlvdxzD8BmfYMTMxZj7yffoFx+JJ8a6nncaFuiDTl4qbEg5hdbD0n9MSQOPB/SOCWuz3j6tzgNry8xOwNX1LmgVwFibLXC0HNfW2zjonigkxoajoKwWDidh0z5X0BIT4Q99kwl2J3EBYUNJNZrrDAC5GhLEMld9/Vse2+FuNbY2Nbebb4KWm9wPZj3Mlau3p4/FkfQ8j32+Homx4dh+KANGkwUDenbGk+MG4It1+5CZV4b9p7Lxj2U/Y9zQnhg/9B70iYtAQ5MJKScuu0btPY0Qfx2cTsKWA66JUQC4cwUATmYV4sHZn2H07M9QUF6LuM6BMNc3wtzYDHfn2rXqlwcHxmHTouew+19z8MvyV7DwhUe4iRhu11NXxUQG4KVJI7Bm21EMe/YTDJ+xGK98ugH94iMhEQm51725/ZCSBplEhPBAb2zcexr3JXZtk4+mhiZXD+Bl+EIBTIZmGBsa26RJNQoovTWXhizd4Ue93HVj/k6ePImDBw9izpw53LKmpiZ89tlnmDVrlsf7egHgvffew6hRo7BgwQIsWLCA6x4OiE+CUCoBX8DnWlMEIiHEciksTc2ukxpAVEgnFJTVugb9OokbA9Wa3WQFXyiASCqCSW+EtdkMP281ymv0SDl2HuGB3jBbbPj02z0tjzQYhgAfDS4UVmDF5lRU1BiQX1qNRat3IzOvFO88Ow4h/p6zXd06B/li19FzWLvzBPSNzTh9vgg/7T8Ls8WGiloD9EYT0s4X4f3/bINQIMCKd6YiMsgXgb5aTHggEa9OHcVVYAqZBGKRqyuNAKzfdRLbDmdwA3cBwOlwwma2wqw3Qix0tRotmP0I8kurUVhUCZ6AD56AB6vR7Aq64iPxtwf7XvM4alVypOeWorSqvqUbwAxbswUmvauV8tHhCZg4qg/Sc0tRUlkPh90OS5MJVpMV9pbgTS4VY/TAeJRU1sHoDhQsNgR30sFisaGqzgA+jw+L0Qy7xQab2QJjy2D4PnHhmJGchP9uOQKFTILmRtdxM+mNrpY2kwX9e0TipckjcOpcEVb/fATV9U3ILqzEh6t2oKC8Fv+YOZ6b6atVyVFd34j/bDqMovJalFTW47O1+3DkbB5enza63RmoAj4fGqUMq34+ArVCCqPBCIvRBEujCUKpmGuVtpktsFtbKreWoufvrcauo+dARHh35rjresuIj1aJksp6fP3TIZRU1aOwvA6ffpuCk1kFePPph7gXm7uPzY0e384hvtiZmgkBn4+IIB/4aJQwWaxwOgk2q811bhjN6NUtBDUNTdh34gJ+OZWN7Ycz8PnavXASMHfyCBxLvwjwAHOTCVaTBc31jeDz+SC7A/17XFp/UCcdVm5OxS+nsmEwmrH1UDq+XL8fA3p0xlMPD7rh568F+eqQnluCVT+norLWgHpDM84VVLiGRRDBZrHDpG+CraWlVSIW4pdT2WhounTO8QAseP4RxEQGcC0IV8pPc0MT1Aopmhub2+Rzjy7BqKw1YOuhdDQ2m7Hv5AV8tGonIoJ88MbTo7nWj8ggX6zddQLbDqXDYDSjur4JGXmlEPB5CPHzwpnsYrz/n23g83h4d8a4Nq2+t0KIvxfW7TqBRn0T+EIBt19NNa7uxDefHoP4LsF4aFAPaJQyRIV2wsxHh2La+IHcDRufz4NWJcPyjYdwMqsQ9QYj1u06gdXbjiJ5mGus6uViOwdh7c7j3M2ApaUFG9RqqE4Lk74JliYTzI2ugEspk2DB839FfFQQdqRmwmqz45dT2dh9/Bz2HDsPm90Bo8mCe7u5bvLcAY9Mq4RYIYWloem66772DL4nCucLKlBcWQ+byeKRb9bGZoiEfKSdL4ZWJUN2YSU++Ho7pGIR3p4+9ppPh2gtMtAH3+08jm2HM9DUbEaz2Ybc4ipoVXIkD0/A3Mkj8T8je7e8ScU11GfVliOuMa4t16is/DK8978Po6xaj8LyWqzbdRLHMy5iyXcpqKgxwN9bjVeeGIlOOpXrNWrTx2L0oHicL6hAXn4ZLC0ttO7r5tXqFx6PB7lUfMWJStdbV/WMDsboQa5xwF3COmHmo0l4+uFBcBJh+cZDuFBYgao6V/299VA6Zj92P0YNiMPXmw7DZnegpr7puvKXnK6ueWerQJjHc22nyWgGT3D1/b2d7rrZvhs3bsQzzzyDmpoabll6ejp69OiBhoYGaDSes2QtFgsslkstP3q9HqGhoYgZNxcC0aVHnZTX6LnuUQDo0SUYCd1CsevYOdQ0NELIF8Bo9pyRFxXsi9mP3Y9vdxz3aBnSKGSw2h1QyiSQSUWoaTBCJZdgRL8YTBnTnztZbXYHvt1+DFsPpqOhqRmxkYGYPKa/xwDe9tQ2NOHrzak4cCobDqcTA3p0xvj7emL74UwcSMuG00kY2LMznhw/AIG+176r2HUkE0t/PIjKWldgxOOhTdeZUMBHUkJXPJs8BAE+WhiaTHh32RaP/e4bF4G3p4+97tlL7f2GVCzCE2MH4PEH+4LP57X7GbFIiPt7d4W+yYRfc0qhlEuQGBsOo9mKjNxSNJusiO8ShClj+0PMF2DeFz96HNteXUPx/v8+DLVShrTzhfh6UyrSc4s9HkfSel9MZitWbz2KnUez0GQ0o2fXYEwdMwAxlz22x+F0YsOeNGzefxrVDU2ICvHDpNF9MaDn1V8WfjAtB6u3HcWFgnKPbegdEw6pRMi1RLU+FjKJGAN7RmHq+P4I7tT+jUJ77A4n1u86gc2/nEWdwYiuYf6Y/FBf9ImL9Pjcbz2+1XUGV9lMywGIMOieKEwdPwA2uxOlVfUI6qRDqL8X8ktr8I+lW5BXWg0iQiedGi/+bRiGJESjoKwWX208gENncuFodXfd3vrPZhfjv1uOID2vFBqlDA8OjMek0X3bTAa4XmarDWu2HsX21Ew0Gc2IiwqEj1aFPcfOwWK71HXfNy4Csyfcj/V7Tl3XOXel/Jw7aQQWrdndbj7XNDRhxebDOJFVCIlIiGF9u2HqmAFt8j+/tAYrN7smYElEQgzo0Rl2hxNH0vNuuC64Wcqq6vHMe6uhb7rUFatRyvDVm5MR2On6t+PQ6Vx8s+MY8oqr4KtTYXzSPUgelnDF968WldfihY/Xcd2O1yOucxDemTEG/t6u7rjqukZ8vfkwdh/NgtVmB5/PQ2SQLwJansjQbLJCLBaivlUL/Y3WfVdyLr8MH6/ejQuFFR6//dyjSVi/5xQOpeUAAAbd2wXTxg+An9eNv+our6QaKzYfxsmsQkjFIgzr0w1Txw5ot7fJarPj2+3Hse2Q6xoV19l1jerVNRQWqx1frt+HbYfSYbbaIBIIMLBXFOZOGtmmK9WtqKLO9YgdpQzLNx36XdcPt99zLSIibD+cge93n0RJVQNC/b0wYWRvjOwfC8D1xqPVPx9FRl4pLO1083prlHhwYBz2nzyPkqq2PY9ioRD/MyIBjw5PwAcrtl91G1Uq1W19g9JdF/zl5OQgOjoaFRUV8PNztaasW7cOL7/8MoqLi6/xbSA/Px+dO1/9YswwDMMwDHO7VFVVwdfX99ofvEnuuuAPAAYPHozg4GAsW7YMNTU1GDt2LJKTk3E9PdgNDQ3Q6XQoKipq00rI3FwGgwEhISEoLi5u0x3P3Hwsv28vlt+3D8vr24vl9+3jzuv2ei5vpbvyDR8bNmzArFmzEBERAT6fjylTpuCtt966ru+6B41rNBpWqG8TtVrN8vo2Yvl9e7H8vn1YXt9eLL9vn9vZ5QvcpcGfn58fvv/++1Ekyv8AABG2SURBVDu9GQzDMAzDMHedu+5RLwzDMAzDMMxvd9c96uVmEAgESEpKglB4VzZ83lVYXt9eLL9vL5bftw/L69uL5fftcyfy+q6c8MEwDMMwDMP8Nqzbl2EYhmEYpgNhwR/DMAzDMEwHwoI/hmEYhmGYDoQFfwzDMAzDMB1Ihwn+qqqq8PDDD0Or1cLHxwcvvvgi7Pa27+pj2rdu3ToIhUIolUru3+TJkwEA27ZtQ3x8PBQKBbp3746ff/7Z47sffvghgoODoVAokJSUhAsXLnBpRqMRTz75JLy9vaHRaDBlyhQ0NV3fS7T/jKqrqxEVFYX9+/dzy44dO4a+fftCqVQiIiIC//nPfzy+s2rVKkRFRUGhUKB37944cuQIl+ZwOPDKK6/Az88PKpUK48ePR3l5OZfe0c+L9vJ75syZkEgkHmV92bJlXPrvKc/Z2dkYNmwYVCoVAgMD8cEHH9yW/byTzp49ixEjRsDLywv+/v6YMmUK9252VrZvvqvlNyvbN9/evXvRt29fqNVq+Pv7Y/bs2TCZXO+0/kOXb+ogkpKS6PHHHyej0Uh5eXkUGxtLH3744Z3erLvG3LlzaerUqW2WZ2dnk1QqpY0bN5LNZqN169aRTCajkpISIiJauXIlBQUFUUZGBplMJnrppZcoNjaWnE4nERFNnTqVhg0bRrW1tVRZWUlDhw6l55577rbu2x/FoUOHqHPnzgSA9u3bR0REdXV15OXlRV988QXZbDZKSUkhlUpFx44dIyKiffv2kUqlokOHDpHVaqVPPvmEfHx8yGg0EhHRO++8Qz169KCioiLS6/U0YcIEGj16NLfOjnxetJffREQJCQm0cuXKdr/ze8qz1WqlLl260KuvvkoWi4XS0tIoMDCQ1q9ff8v39U5pbm6mgIAAmj9/PlksFqqpqaHRo0fTmDFjWNm+Ba6W30SsbN9sVVVVJJVKacWKFeRwOKisrIzi4uJo/vz5f/jy3SGCv5ycHAJApaWl3LK1a9dSaGjoHdyqu8uQIUPoiy++aLP8jTfeoBEjRngsGzVqFM2fP5+IiAYOHEjvv/8+l2a1WkmlUtHevXvJaDSSSCSiw4cPc+lHjx4lmUzGnQAdxcqVKyk0NJTWrl3rEYx89dVX1KVLF4/Pzpgxg6ZMmUJERI8//jg988wzHundunWjr7/+moiIgoOD6ZtvvuHSKioqiMfjUV5eXoc+L66U32azmcRiMWVkZLT7vd9Tnnfv3k1KpZIsFguXvmDBAhoyZMit2ck/gPPnz9OoUaPIbrdzy3766SdSq9WsbN8CV8tvVrZvDYPBQERETqeT0tPTKSoqij7//PM/fPnuEN2+mZmZ8PLyQmBgILcsJiYGRUVFaGhouINbdndwOp1IS0vD1q1bERYWhuDgYEyfPh319fXIzMxEfHy8x+djYmJw9uxZAGiTLhKJ0KVLF5w9exY5OTmw2Wwe6TExMTCZTMjOzr49O/cH8cADDyAvLw8TJkzwWH6j+ds6Xa/Xo6SkxCPdz88POp0Ov/76a4c+L66U32fPnoXNZsP8+fPh5+eH6OhoLFy4EE6nE8DvK8+ZmZmIjo6GWCz2SHcfyz+jrl27Yvv27RAIBNyyDRs2ICEhgZXtW+Bq+c3K9q2hUqkAACEhIYiPj0dAQACefPLJP3z57hDBX2NjIxQKhccyuVwOAB16fNn1qq6uRq9evZCcnIxz584hNTUVOTk5mDRp0hXz1p2vV0tvbGwEAI/0jnpc/P392326+83O38vTO+p5caX81uv1SEpKwvPPP4+SkhKsWbMGn332GRYtWgTg95Xnax3LPzsiwptvvoktW7ZgyZIlrGzfYpfnNyvbt1ZOTg5KS0shEAiQnJz8hy/fHSL4UygUaG5u9ljm/tsdtTNX5ufnhwMHDmDatGmQy+UIDQ3Fhx9+iO3bt4OI2s1bd75eKe9VKhVXeFuns+Pi6Wr5d6309vL38nR2XngaMWIE9u7di6FDh0IkEqFPnz548cUXsW7dOgC/rzxf61j+mRkMBiQnJ2PNmjU4cOAAN0GMle1bo738ZmX71pLJZAgMDMTChQuxY8eOP3z57hDBX1xcHGpra1FZWckty8rKQnBwMDQazR3csrvDr7/+innz5oFavQnQYrGAz+ejT58+yMzM9Ph8VlYW4uLiALjyvnW6zWZDTk4O4uLi0LVrV4hEIo/0rKwsiMViREdH3+K9ujtcnn/A1fO3dbpOp0NQUJBHekVFBerq6hAXF8fOi3Zs2rQJS5cu9VhmsVggk8kA/L7yHBcXh+zsbI8Zea2P5Z9VXl4eEhMTYTAYcPLkSa4ri5XtW+NK+c3K9s2XmpqKbt26wWq1csssFgvEYjFiYmL+2OX7+oc13t0GDRpEjz32GBkMBsrPz6fY2Fh6++237/Rm3RWKi4tJoVDQwoULyWazUWFhIfXr14+eeuopOnfuHEmlUlq3bh0321cqldKFCxeIiGj58uUUFBREZ86c4WaQRUVFkdVqJSKiSZMmUVJSElVVVVFVVRUlJSXRE088cQf39s5DqwkINTU1pNVqafHixWS1Wmnv3r3cIGwioj179nB/W61WWrx4Mel0OqqtrSUiojfffJPi4uIoPz+fDAYDTZgwgYYOHcqti50Xnvn9448/kkwmoz179pDT6aTU1FTy8fGh1atXE9HvK882m40iIiJo7ty5ZDKZ6MyZMxQYGEgrVqy4A3t9e9TV1VFoaChNnTqVHA6HRxor2zff1fKble2br7GxkUJCQmjOnDlksViooKCA+vTpQzNnzvzDl+8OE/xVVFRQcnIyeXt7k6+vL82dO9djRhRzdfv376f+/fuTSqUiX19fmj17NplMJiIi2rFjB/Xs2ZOUSiXFxsbS1q1bue85nU76+OOPKSIigpRKJd13331cYEjkmin1zDPPkJ+fH+l0Opo6dSo1NTXd9v37I8Fljx45ceIEDRgwgFQqFUVGRrapUFevXk1du3YlhUJBffr0oaNHj3JpVquVXn31VQoKCiK1Wk3jx4+nyspKLp2dF23z+9///jdFR0eTXC6nyMhI+vLLL7m031uec3JyaOTIkaTRaCgoKIgWLFhwW/bxTlm0aBEBILlcTgqFwuMfESvbN9u18puV7ZsvMzOTRowYQVqtlsLCwuiNN94gs9lMRH/s8s0jatWXxzAMwzAMw/ypdYgxfwzDMAzDMIwLC/4YhmEYhmE6EBb8MQzDMAzDdCAs+GMYhmEYhulAWPDHMAzDMAzTgbDgj2EYhmEYpgNhwR/DMAzDMEwHwoI/hmEYhmGYDoQFfwzDMAzDMB0IC/4YhmEYhmE6EBb8MQzDMAzDdCAs+GMYhmEYhulAWPDHMAzDMAzTgbDgj2EYhmEYpgNhwR/DMAzDMEwHwoI/hmEYhmGYDoQFfwzDMAzDMB0IC/4YhmEYhmE6EBb8MQzDMAzDdCAs+GMYhmEYhulAWPDHMAxzl8rJybnTm8AwzF2IBX8MwzB3idjYWHzzzTcAgC+//BLTp09vN41hGOZqhHd6AxiGYZjrk5mZyf2/uroaRNRuGsMwzNWwlj+GYe6YPXv2oFevXlCr1Rg4cCBeffVVJCUlcelr165Fjx49oNFokJCQgF27dnFpSUlJeO211zBkyBAolUp0794d69ev59ILCwsxYcIEdOrUCf7+/nj88cdRXl4OANi/fz/Cw8Px0Ucfwd/fHxqNBq+//jo2b96M6OhoqFQqPPLII7BYLAAAq9WK+fPnIzIyEl5eXhg9ejRyc3Pb3aeCggLweDx8/vnn3G9PnjwZBoOB+8zy5csRGxsLtVqN+Ph4jxa7AwcOoHfv3tBqtYiKisKLL74Iu90OAAgPD8fKlSuxatUqfPDBBzh48CC0Wq1HGgCYTCb8/e9/R0hICHQ6HZKSknDixAluHe7t69q1K1QqFQYOHIj09PTfcggZhrkbEcMwzB1w8eJFkkqltHTpUrLZbHTgwAHSaDQ0dOhQIiLaunUraTQa+uWXX8hut9OWLVtIqVRSRkYGERENHTqUvLy8KC0tjSwWC73xxhuk0WjIZDKR1WqlLl260MSJE6mhoYHq6+tp4sSJlJCQQDabjfbt20cAaM6cOWS1Wmnbtm0EgEaMGEF1dXWUn59POp2OVq1aRUREL7/8MvXq1Yvy8/PJZDLRu+++SxEREWQymdrdLwA0dOhQqqqqovLycurbty9NmjSJiIhWrFhBarWaUlJSyG63U0pKCqnVavrxxx+JiCg0NJRWrlzJ/VZgYCBt2LCBiIjCwsJoxYoVRET09ttvc3l1edoTTzxBPXr0oJycHLJYLPTpp5+SSqWiwsJCIiICQP3796fy8nJqaGig4cOH0wMPPHDzDi7DMH9oLPhjGOaOeO+996hfv34ey1555RUuoHnwwQdp3rx5HukTJ06kWbNmEZEr+Hv22We5tOzsbAJAhYWFlJKSQgKBgPR6PZduMBhIIBDQkSNHuOCvurqaiIjsdjsB4AIwIqLBgwfTu+++S06nkxQKBe3YsYNLczqdFBQUxAVlrbmDv7S0NG7Zzp07SSKRkNlspsGDB9Mrr7zSZr/dwVfXrl1p9OjRtGXLFjIYDORwOLjPXU/wZzKZSCwW09atWz3WkZiYSP/85z+JyBX8fffdd1zasmXLKCIios2+MAzz58S6fRmGuSOKi4sRHh7usSwyMpL7f0FBAZYsWQKtVsv927x5M4qKirjP+Pv7c/8XiUQAAKfTicrKSvj4+ECtVnPpKpUK3t7eKCgo4JZ5e3sDAAQCAQBAp9NxaXw+H06nE9XV1TAajXj00Ue57dDpdKirq/P4rct16dKF+39oaCgsFgvq6upQWVnpsZ8AEBERwf1WSkoKAgMD8dxzz8HLywvjx49HSUnJFddzufr6elit1quuA2ibd06n87rXwTDM3Y0FfwzD3BFhYWEoLCz0WNb67+DgYMyfPx8NDQ3cv6ysLCxfvvyavx0eHo6amhqPcXZ6vR41NTUICAjglvF4vGv+lo+PD6RSKXbt2uWxLadPn8azzz57xe+VlpZy/7948SLkcjl8fHwQHh6OvLw8j8/m5eUhICAAZrMZWVlZ+Ne//oWioiJkZmZCr9djzpw519xONz8/P0il0iuug2EYhgV/DMPcEZMnT8aZM2fw3//+Fw6HA8eOHcOyZcu49OnTp+Ozzz7jJiqcPHkSCQkJ+O67767524mJiYiNjcWMGTOg1+uh1+sxc+ZMREVFYeDAgTe0nXw+H0899RTmzZuHkpISOJ1OrFq1CrGxsVd9zt68efNgMBhQVlaG+fPnY8qUKRCJRHj66aexdOlS7N27Fw6HA/v27cOyZcswbdo08Hg8TJw4EYsWLYLdboe/vz9EIhF8fHza/L5UKoXBYPCY8eve3mnTpuH1119Hbm4urFYrlixZgszMTEycOPGG9p1hmD8nFvwxDHNHBAcH44cffsDChQuh0Wjw8ssvY9SoURCLxQCA5ORkfPDBB3jyySehVquRnJyMOXPmYPbs2df8baFQiJ9//hl2ux3R0dEICwuD1WrF7t27IRTe+BOuPv74Y/Tr1w+DBw+GVqvF4sWL8cMPP6BXr15X/E5UVBTi4uIQHx+P/v37Y/HixQCARx99FIsXL8bs2bOh0Wgwc+ZMfPTRR5g8eTIkEgk2b96MTZs2wdvbG+Hh4QgICMCCBQva/P7YsWNRW1sLjUaDhoYGj7SPPvoIDzzwAIYNGwZvb2+sW7cOO3fuRHR09A3vO8Mwfz48uvy2kWEY5jYoLi5GTU2NRwA1d+5cVFRU3NUPKy4oKEBERAQuXrzYZkwjwzDMHwFr+WMY5o6oqanBoEGDcOrUKQDAr7/+ijVr1mDMmDF3eMsYhmH+3NgbPhiGuSN69eqFJUuW4LHHHkN5eTn8/f3x2muvsXFpDMMwtxjr9mUYhmEYhulAWLcvwzAMwzBMB8KCP4ZhGIZhmA6EBX8MwzAMwzAdCAv+GIZhGIZhOhAW/DEMwzAMw3QgLPhjGIZhGIbpQFjwxzAMwzAM04Gw4I9hGIZhGKYDYcEfwzAMwzBMB/L/AYh0me3UzsdSAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 1200x800 with 2 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# generate a figure with sub-plots (one for each gene segment)\n",
    "fig = plt.figure(figsize=(12,8), facecolor='w')\n",
    "plt.style.use('default')\n",
    "mpl.rc('font', family='Arial')\n",
    "gs = gridspec.GridSpec(4, 12)\n",
    "\n",
    "ax1 = fig.add_subplot(gs[1,0:6])\n",
    "ax2 = fig.add_subplot(gs[2,0:6])\n",
    "\n",
    "# get right of top and right frame of each subplot\n",
    "ax1.spines['top'].set_visible(False)\n",
    "ax1.spines['right'].set_visible(False)\n",
    "ax1.spines['bottom'].set_visible(True)\n",
    "ax1.spines['left'].set_visible(True)\n",
    "\n",
    "ax2.spines['top'].set_visible(False)\n",
    "ax2.spines['right'].set_visible(False)\n",
    "ax2.spines['bottom'].set_visible(True)\n",
    "ax2.spines['left'].set_visible(True)\n",
    "\n",
    "# set fontsize of axes \n",
    "ax1.tick_params(axis = 'both', which = 'major', labelsize = 10)\n",
    "ax1.tick_params(axis = 'both', which = 'minor', labelsize = 10)\n",
    "\n",
    "ax2.tick_params(axis = 'both', which = 'major', labelsize = 10)\n",
    "ax2.tick_params(axis = 'both', which = 'minor', labelsize = 10)\n",
    "\n",
    "ax1.set_xlabel('', fontname='Arial', fontsize=10)\n",
    "ax1.set_ylabel('# of sequences', fontname='Arial', fontsize=10)\n",
    "ax1.set_facecolor('white')\n",
    "for tick in ax1.get_xticklabels():\n",
    "    tick.set_fontname(\"Arial\")\n",
    "for tick in ax1.get_yticklabels():\n",
    "    tick.set_fontname(\"Arial\")\n",
    "ax1.set_ylim(-2,102)\n",
    "ax1.set_xlim(-2,30000)\n",
    "ax1.set_xticks([0, 5000, 10000, 15000, 20000, 25000, 30000])\n",
    "\n",
    "ax2.set_xlabel('\\n genome position', fontname='Arial', fontsize=10)\n",
    "ax2.set_ylabel('# of sequences', fontname='Arial', fontsize=10)\n",
    "ax2.set_facecolor('white')\n",
    "for tick in ax2.get_xticklabels():\n",
    "    tick.set_fontname(\"Arial\")\n",
    "for tick in ax2.get_yticklabels():\n",
    "    tick.set_fontname(\"Arial\")\n",
    "ax2.set_ylim(-2,102)\n",
    "ax2.set_xlim(-2,30000)\n",
    "ax2.set_xticks([0, 5000, 10000, 15000, 20000, 25000, 30000])\n",
    "\n",
    "ax1.scatter(x=Dane_group_syn['Position'], y=Dane_group_syn['Sample_count'], marker='o', s=20, facecolors='#C44536', edgecolors='#933831', zorder=1)\n",
    "ax1.scatter(x=Dane_group_nonsyn['Position'], y=Dane_group_nonsyn['Sample_count'], marker='o', s=20, facecolors='none', edgecolors='#933831', zorder=1)\n",
    "\n",
    "ax2.scatter(x=MKE_group_syn['Position'], y=MKE_group_syn['Sample_count'], marker='o', s=20, facecolors='#457B9D', edgecolors='#305882', zorder=2)\n",
    "ax2.scatter(x=MKE_group_nonsyn['Position'], y=MKE_group_nonsyn['Sample_count'], marker='o', s=20, facecolors='none', edgecolors='#305882', zorder=2)\n",
    "\n",
    "fig.savefig(\"../figures/consensus_SNV_differences.pdf\", bbox_inches='tight', dpi=300)\n",
    "\n",
    "# use tight_layout to optimize spacing of axes and titles within each subplot\n",
    "fig.tight_layout()\n",
    "fig.show()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## metadata age stuff -- for UWHC only "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 396,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "Dane_all = pd.concat([pd.read_csv(f, sep='\\t') for f in glob.glob(\"../data_derived/Dane/cleaned/*.csv\")], ignore_index=True)\n",
    "Dane_all = Dane_all[['sample', 'Position']]\n",
    "\n",
    "metadata_age = pd.read_csv('../data_raw/UWHC_ages.csv', sep='\\t')\n",
    "\n",
    "metadata_age = metadata_age[['strain', 'age']]\n",
    "\n",
    "metadata_age['strain'] = metadata_age['strain'].str.replace('/','_')\n",
    "\n",
    "metadata_age_dict = dict(zip(metadata_age.strain, metadata_age.age))\n",
    "\n",
    "Dane_all['ages'] = Dane_all['sample'].map(metadata_age_dict)\n",
    "\n",
    "# Dane_all\n",
    "\n",
    "#"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 397,
   "metadata": {},
   "outputs": [],
   "source": [
    "Dane_ages_temp = Dane_all.groupby('Position')['ages'].apply(','.join)\n",
    "\n",
    "Dane_ages = pd.DataFrame(Dane_ages_temp)\n",
    "\n",
    "Dane_ages = Dane_ages['ages'].str.split(',',expand=True)\n",
    "\n",
    "Dane_ages = Dane_ages.astype(float)\n",
    "\n",
    "Dane_ages['mean'] = Dane_ages.mean(axis=1)\n",
    "Dane_ages['std'] = Dane_ages.std(axis=1)\n",
    "\n",
    "Dane_ages = Dane_ages[['mean', 'std']]\n",
    "\n",
    "Dane_ages = Dane_ages.reset_index()\n",
    "\n",
    "# Dane_ages\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 413,
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/matplotlib/figure.py:445: UserWarning: Matplotlib is currently using module://ipykernel.pylab.backend_inline, which is a non-GUI backend, so cannot show the figure.\n",
      "  % get_backend())\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAA9wAAAGyCAYAAAAWMcweAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAPYQAAD2EBqD+naQAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4zLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvnQurowAAIABJREFUeJzs3Xl8lOW9//939slCEoJFPWSpFhEjVHoQU77WtRo8VikuBzewarXWQz0FbY8CChjR6HFptGIfdTvaShcqSNFaG1qlWk0j0BYVpPSndiZJEWuTTAjZZ+b3R5iYhJkks9xzL/N6Ph48wkwyM9fc93Vd9/W5ry0lEAgEBAAAAAAA4irV7AQAAAAAAOBEBNwAAAAAABiAgBsAAAAAAAMQcAMAAAAAYAACbgAAAAAADEDADQAAAACAAQi4AQAAAAAwAAE3AAAAAAAGIOAGAAAAAMAABNwAAAAAABiAgBsAAAAAAAMkdcAdCATU1tamQCBgdlIAAAAAAA6T1AH3/v37VVBQoP3795udFAAAAACAwyR1wA0AAAAAgFEIuAEAAAAAMIBlAu5//vOfmjx5srZs2TLwXH19vSoqKpSXl6ejjjpKTz755JDXPPPMM5o8ebJyc3N14oknqq6uLsGpBgAAAAAgNEsE3G+88YZmz56t999/f+C5lpYWnXvuubryyivV2tqqJ598UkuWLNFbb70lSdqyZYtuvPFGPfPMM2ptbdUVV1yhuXPnqqOjw6yvAQAAAADAANMD7meeeUaXX3657rrrriHPr1+/XhMmTNCiRYuUnp6uM888U1dccYXWrFkjSXriiSd06aWX6uSTT1ZGRoaWLFmiww47TD//+c/DflZ3d7fa2tqG/AMAAAAAwAimB9xz5szR+++/r0suuWTI8zt37tT06dOHPFdeXq4dO3aM6fehVFdXq6CgYOBfSUlJnL4FAAAAAABDmR5wH3HEEUpPTz/k+f379ys3N3fIczk5OWpvbx/T70NZunSpvF7vwL+GhoY4fAMAAAAAAA51aKRrEbm5uWptbR3yXEdHh8aNGzfw++HztTs6OnTYYYeFfc+srCxlZWXFP7EAAAAAAAxjeg93ONOmTdPOnTuHPLdr1y5NmzZtTL8HAAAAAMBMlg24L7zwQn300UeqqalRb2+vXn31Va1du1bXXHONJOmaa67R2rVr9eqrr6q3t1c1NTXat2+fLrjgApNTDgAAAACAhQPuCRMmaPPmzfrFL36hCRMm6Nprr9XDDz+sM844Q5L05S9/WY8++qhuuOEGjR8/Xj/96U/161//WkVFRSanHAAAAAAAKSUQCATMToRZ2traVFBQIK/Xq/z8fLOTAwAAAABwEMv2cAMAAAAAYGcE3AAAAAAAGICAGwAAAAAAAxBwAwAAAABgAAJuAAAAAAAMQMANAAAAAIABCLgBAAAAADAAATcAAAAAAAYg4AYAAAAAwAAE3AAAAAAAGICAGwAAAAAAAxBwAwAAAABgAAJuAAAAAAAMQMANAAAAAIABCLgBAAAAADAAATcAAAAAAAYg4AYAAAAAwAAE3AAAAAAAGICAGwAAAAAAAxBwAwAAAABgAAJuAAAAAAAMQMANAAAAAIABCLgBAAAAADBAutkJAAA7avG41dbUqILiEhWWlJqdHABAjKjXARiBgBsAItDl9ap25TJ56usGniutmK3Kqmq58vNNTBkAIBrU6wCMxJByAIhA7cplatparzJXmo7PTVeZK01NW+tVu2Kp2UkDAESBeh2AkQi4AWCMWjxueerrVJyZoqKMVGWm9v8szkyRp75OrQ0es5MIAIgA9ToAoxFwR2lP7ctmJ8HWOH6wo7amRklSXlrKkOeDj72NDQlPE4DkwzU0fqjXxy5Z8p2R3zORxzCaz7LbOR6eXqumn4A7Sns2W/OE2gXHD3aUP6lYktTuCwx5Pvi4oLgk4WkCkHy4hsYP9frYJUu+M/J7JvIYRvNZdjvHw9Nr1fQTcAPAGI0vLVNpxWw19gTU3OtXj7//Z2NPQKUVs1nVFgBshnodgNEIuAEgApVV1Zo0q0LuLp92HuiTu8unSbMqVFlVbXbSAABRoF4HYCS2BQOACLjy8zW3Zo1aGzzyNjawXysA2Bz1OgAjEXADQBQKS0ppkAGAg1CvAzACQ8oBAAAAADAAATcAAAAAAAYg4AYAAAAAwAAE3AAAAAAAGICAGwAAAAAAAxBwAwAAAABgAAJuAAAAAAAMwD7cADCKFo9bbU2NKiguYY9WAAAAjBkBNwCE0eX1qnblMnnq6waeK62Yrcqqarny801MGQAAAOyAIeUAEEbtymVq2lqvMleajs9NV5krTU1b61W7YqnZSQMAAIANEHADQAgtHrc89XUqzkxRUUaqMlP7fxZnpshTX6fWBo/ZSQQAAIDFEXADQAhtTY2SpLy0lCHPBx97GxsSniYAAADYCwE3AISQP6lYktTuCwx5Pvi4oLgk4WkCAACAvRBwA0AI40vLVFoxW409ATX3+tXj7//Z2BNQacVsVisHAADAqAi4AYdo8bjlrnuDucVxVFlVrUmzKuTu8mnngT65u3yaNKtClVXVZicNAAAANsC2YIDNsXWVcVz5+Zpbs0atDR55GxvYhxsAAAARoYcbsDm2rjJeYUmpymafTLANAACAiBBwAzbG1lVAfDAlw7o4NwAAO2NIOWBjY9m6il5ZIDymZIxuT+3LmlJ5TsI/l3MDAHACergBG2PrKiA2TMkY3Z7NL5vyuZwbAIATEHADNsbWVUD0mJJhXZwbAIBTEHADNsfWVUB0xjIlA+bg3AAAnII53IDNsXUVEJ3BUzKKUj8N7JiSYT7ODQDAKejhBhyCrauAyDAlw7o4NwAAp7B8wP2nP/1Jp556qgoLC3XkkUfq29/+trq7uyVJL730kqZPn67c3Fwdd9xxevHFF01OLQDATpiSYV2cGwCAE1h6SLnf79d5552nW2+9VVu2bNE//vEPnXXWWTrssMN06aWX6qKLLtJPf/pTnXfeedqwYYPmz5+vv/3tb5o0aZLZSQcA2ABTMqyLcwMAcAJL93C3tLRo79698vv9CgT6522lpqYqJydHzzzzjE455RTNmzdP6enpmj9/vk477TQ99thjJqcaAGA3TMmwLs4NAMDOLB1wT5gwQUuWLNHNN9+srKwslZSUaMqUKVqyZIl27typ6dOnD/n78vJy7dixI+z7dXd3q62tbcg/AAAAAACMYOmA2+/3Kzs7W4888ogOHDigd999V7t27dLKlSu1f/9+5ebmDvn7nJwctbe3h32/6upqFRQUDPwrKWGVUwAAAACAMSwdcD///PNav369brjhBmVlZen444/XypUr9eijjyo3N1cdHR1D/r6jo0Pjxo0L+35Lly6V1+sd+NfQwD6eAAAAAABjWHrRNI/HM7AieVBGRoYyMzM1bdo0/elPfxryu127dunEE08M+35ZWVnKysoyJK0AAAAAAAxm6R7uOXPmaO/evbr77rvl8/n0wQcfaPXq1VqwYIEWLlyoLVu2aN26derr69O6deu0ZcsWLVy40OxkAwAAAABg7YC7vLxcL774ojZt2qQJEybojDPO0Pnnn6+77rpLU6dO1caNG3X33Xdr/Pjxqqqq0vr16zVlyhSzkw0AAAAAgLWHlEvSWWedpbPOOivk7+bMmaM5c+YkOEUAAAAAAIzO8gG3nbR43GpralRBcQn7hZqEcwAAAABYV7K11wm446DL61XtymXy1NcNPFdaMVuVVdVy5eebmLLkwTkAAAAArCtZ2+uWnsNtF7Url6lpa73KXGk6PjddZa40NW2tV+2KpWYnLWlwDgAAAADrStb2OgF3jFo8bnnq61ScmaKijFRlpvb/LM5Mkae+Tq0NHrOT6HicA8B4e2pfNjsJAADAppK5vU7AHaO2pkZJUl5aypDng4+9jQ0JT1Oy4RwAxtuzmYAbAABEJ5nb6wTcMcqfVCxJavcFhjwffFxQXJLwNCUbzgEAAABgXcncXifgjtH40jKVVsxWY09Azb1+9fj7fzb2BFRaMTspVt4zG+cAAAAAsK5kbq8TcMdBZVW1Js2qkLvLp50H+uTu8mnSrApVVlWbnbSkkehzwHxWAAAAYOySNWZiW7A4cOXna27NGrU2eORtbEiaPeWsJNHnYM/mlzWl8hzD3h8AAABwkmSNmQi446iwpDQpMo2VcQ4AAAAA60q29jpDygEAAAAAMAABNwAAAAAABiDgBgAAAJD0WjxudTQ3q7XBY3ZSbIXFhEdGwA0AAAAgaXV5vdq0eJHWXnKBPt71rp6dP0+bFi9SV1ub2UmzhT2bCbhHQsANR2rxuOWue4M7lIANUX6BxKG8wUh2yV+1K5epaWu9ylxpOj43XWWuNDVtrVftiqVmJw0OwCrlcJQur1e1K5fJU1838FxpxWxVVlXLlZ9vYsoAjIbyCyQO5Q1GslP+avG45amvU5krTUUZ/X2RRakpkvxy19eptcGTVCtqI/7o4YZj7Kl9mTuUgI1RfoHEobzBSHbKX21NjZKkvLSUIc8HH3sbGxKeJjgLATccY+emDfLU16k4M0VFGanKTO3/WZyZIs/BO5QArCnYw0D5BYxHeYOR7Ja/8icVS5LafYEhzwcfFxSXJDxNcBYCbjhGb2eXJO5QAnZEDwOQOJQ3GMlu+Wt8aZlKK2arsSeg5l6/evz9Pxt7AiqtmM1wcsSMgBuOkZHtksQdSsCO6GEAEofyBiPZMX9VVlVr0qwKubt82nmgT+4unybNqlBlVbXZSYMDEHDDMTKyc7hDCdgUPQzWxJ60zkR5g5HsmL9c+fmaW7NGC9Zt1MTyaVqwbqPm1qyx3AJvsCcCbjgKdygB+6L8Wgd70jof5Q1Gsmv+KiwpVU5RkSVvCsC+2BYMjhK8Q9na4JG3sUEFxSVUmoBNUH6tY/AKw3lpKWr3BdR4cIXhuTVrzE4e4oDyBiORv4BPEXDDkQpLSqnYAZui/JqLPWntZ0/ty5pSeU5Ur6W8wUjkL4Ah5RhBi8ctd90bzN0DgCQyeIXhLn9A3j6/uvwBy64w7DTRXHv3bH7ZwBQBzkHb1j6cdK7o4cYhurxe1a5cJk993cBzpRWzVVlVzeIRAOBwwRWGP+jsU6f/0+ezD96it+IKw07AtRcwDuUrvFhGyBjBieeKHm4cYvDcveNz01XmSlPTwbl7AABnG19aJldBgbr9GnId6PZLroIChocahGsvYBzKV3hWGyHjxHNFwI0hgnP3ijNTVJSRqszU/p/FmSnyHJy7h+TjpGE9AEbW4nGry+tVycE53MHrQIkrTV1eL/XAMHtqXw75/3B/EwrXXufgemk9lC/7cOq5Ykg5hhg8d2+wwXP36N1IHk4c1gNgZFwHIrNn86fDMQf/P9zfhMIxtz+ul9a164WNkihfduDUupAe7ii0eNzqaG627V2WkQTn7rX7AkOeDz6Ox9w9Jx8/p3HisB4AI0vEdQBDRXLMuYZak9Ovl3bOdx+/t1OS+XVaIo9hNJ9lhXMcS11ohfSHQw93BIbfvXx2/jzH3b0cX1qm0orZatxaL8n/6f6rPQGVVsyO+q7SntqXVVox2/HHz0nYGghITkZdBxDeWI75SG0QmMvJ10sntH0zsnNMrdMSeQyj+Sxfb682LV5kiXMcbV3oKihQl9drevrDoYd7BMPnXDn97mVQZVW1Js2qkLvLp50H+uTu8mnSrIqYLup7Nr+cNMcvERIxR2wsw3oAJN5o84HjwYjrgJVYcZ7taMeca6h1Wfl6GWt94ZR8Z2adlshjGM1nffLX3ZY6x5HWhdmpUo/Xa5n0h0IP9wgGz7ly8t3L4Vz5+Zpbs0atDR55GxtUUFwS83fr7exQ0/ZtSXH8jJTIOWKDh/X0nysNPJYYVgqYZbT5wPEQvA48/63r1dvZqcpVqx1RR1t5nu1I197R2iCTZs4yMeWw8vUylvrCSW1fI9q2Y5HIYxjNZ7V43OpsbbHUOY6kLuzyB9R5cEcNq6Q/FHq4x8jKdy+NUlhSqrLZJ8clo/Z2dklKruNnhETeJR0Y1tMTUHOvXz3+/p8MKwWSR0Z2tnKKihxT3u3QWxfq2jtaG6S3szNxCcQhnHq9dGLbN55t27FI5DGM5rOsfI7HUhd2+wNDHgdZIf2DEXCPkRGLyCRiWKCRw+Yiee+MbJck8xessDMztkpw+rBSJ7HiEFnASsLVoYenS576Onne+qPZSQxrtDZIRnZ2wtOEoZx4vWQBxdgl8hhG81lWPcfh2jTD05t1cESJ1dI/HEPKx8iIRWSMHBZo5LC5aN7b7AUrnMCMrRLMGoKFsbPyEFnASobXoX2BgNydPrUdbJht+vZ/WbbsjNYGSc3IMDuJSc+J10sWUIxdIo9hNJ81vrRM2YXj1djmNTx9YzFamybUd8xOlRq6fJJkevrDoYc7Ana6e2nksLlo39tOx8+KzLwLmeghWBg7OwyRBaxgeB3q7vTpgC9gm7LDNdQenHa9JN/FLpHHMJrPOmzqVMuc47G0aYZ/x06/lFlQYIn0h0MPdwQG372sXXWbZReRMXKBhkjeu8XjVltT40AgaJfjZ1XcaU68wXnYisfXSQvaoH+akdGLoSWzwXVor7+/Z9tOZYdrqLPFcr2J5rUtHrf+8ec/SSnSpC/MDPs6V36+pp57vk69+Rbb5zuzrumJLLvRfFZaeobOu69m1NcYffzG2qYJ9x2tXDcScEehsKTU0ovIGDn0eCzv7covOGQ4SHbheHW1tcmVn2/542dllVXVql2xVO4QQ20QP3YZpm3GNAMYx6hpRgTynwrWocGybceywzXUWWK53kTz2i6vV79e/l01bd825PniE2fpnLvuC/m6YN1k13w3fJ9pKfJrejzq0USW3Wg+K9xrEtUmirRNMzy9Vq4bGVLuQEYOPR7Le4caDtLjbbHsMD07Cd7VW7Buo85/8PtasG6j5tassVQQ6AR2GaZttcVOErEQJCK3ZzPnJShYh86tWSPJOmUHySuW6000r61duUz/2L5NadKQ1+3dvtVy17h4icc+08lcjyaqTWS1Nk08EXA7kJHbU4z23oFAIPRK2llphq2knYycNkfMSsxYDT5aVtuKJpkbJLCX0orZlio7SE6xXG+ieW3wNQFJxQeH7Tq9nRbcZ9oO13QrSmSbyGptmngi4HYoIxdoGOm9rbyfHzAWdsvDLGgDRIeyA7PFcr2JZc/laD/Tjux2TR+rRI0oS/Txc2q9zBxuhzJye4qR3nvwcJCi1E8LpxOGgyA52C0PO3ErGiARKDswWyzXm2heG3xNtJ9pR3a7po+VkVsLD5bo4+fUepmA2+EKS0oNy6ih3jvsStrdPtsPB0FysOtq8EaWdSASdlukjbIDs8RyvYl2z+XSitlqqK9T4/B9ix3aTrPaPtN2Y1abyGn1MkPKEXehhoNkFoy3/XAQJA+nDmkCEoG5/NbQ4nHLXfcGc1QtLpbrTTSvrayq1qSZs+SThrzuyJmzHHuNO2zqVB0+/QSu6VGiTRQ7ergRd6GGg/zh4QdZSRu24dQhTQCczy7bGqJfLNebaF7rys/XvEd+qNYGj5r+vF3SyPtw212X16tPdu9WZ2vLwHNHnjCD8hAB2kSxI+CGYZw2HATJhzwMwG4Gb+EzMPzz4BY+we3QYD2xXG+ieW2yXN9qVy5Tj7dlaHl4523KQxSSJc8YgSHlAAAADmCnbQ0Bow2Uh6w0ygNMRcANADaVqG1BANiDU7dAQvKK5TpHeYBVEHADgE2xOBWAwQZv4TOY3bdAQvKK5TpHeYBVEHADiAi9qpDIB4AVDWzh0xNQc69fPf7+n2yBhGQ0UB66fZQHmIqAG0BE6FWFRD4ArIotfIBPVVZVK7NgPOUBpopqlfJPPvlE9913nzZv3qy9e/fqN7/5jZ5//nnNmDFDX/3qV+OdRgAAAIwBW/gAn3Ll5+vwadP1pf++yTHlocXjVkdzs1obPLb/Lski4oD7ww8/1Mknn6zOzk6dcsop2rFjh/r6+rRnzx7deeed+uUvf6mvfOUrRqQVAAAAY8AWPsCnnFAeurxe1a5cJk99nSTp2fnzVFoxmz3FbSDiIeU333yzJk6cqA8//FAbNmxQINC/8MDatWs1d+5c3X333XFPJAAAAAAkq9qVy9S0tV5lrjQdn5uuMleamrbWq3bFUrOThlFEHHD/7ne/0+23367CwkKlpAxdZv/666/Xu+++G7fEAQAAAEAyG9hTPDOFPcVtKKpF09LTQ49E7+7uPiQIBwAAAABEJxn3FHfSbigRB9ynnHKKqqurdeDAgYHnUlJS5Pf79YMf/EAnn3xyXBMIAInmpEoeAADYW6L3FLdCO8hJu6FEHHDfc889eu+99zR58mQtXLhQKSkpuv/++zVz5kz94Q9/0F133RXXBDY3N+vKK6/UhAkTNH78eM2bN0979+6VJNXX16uiokJ5eXk66qij9OSTT8b1swEkJydV8gAAwN4G9hTvCSRkT3HaQfEVccA9bdo0bd26VWeeeaZeffVVpaWlafPmzZo8ebLefPNNzZgxI64JvOiii9Te3q73339fHo9HaWlpuu6669TS0qJzzz1XV155pVpbW/Xkk09qyZIleuutt+L6+QAAAABgpsqqak2aVcGe4jYU1T7cU6ZM0dq1a+OdlkNs375df/zjH7Vv3z7lH1zu/vHHH9fevXu1fv16TZgwQYsWLZIknXnmmbriiiu0Zs0anXTSSYanDQAAAAASwZWfr7k1a9Ta4FHtqttUuWq17bc6SxYRB9yvvfZa2N+lpqYqLy9PRx999ECAHIu33npL5eXlevzxx/WDH/xABw4c0DnnnKMHHnhAO3fu1PTp04f8fXl5+YjDyru7u9Xd3T3wuK2tLezfJnpT+RaPW21NjSooLqHwAAAAx6GtAyuyW74sLClVTlGRLdKKfhEH3KeffvrASuTBPbglDVmdPDU1VV/72tf0wx/+UGlpaVEnrrm5WW+//bZmzZqlP//5z+ro6NDChQt15ZVX6ogjjlBubu6Qv8/JyVF7e3vY96uurtYdd9wx4mcmelP54Z8niU3sAcTNntqXNaXyHLOTASCJ0daBFYXLlwE2XEKcRTyHe9OmTXK5XPrGN76hLVu2aPfu3Xrttdd04403KiMjQ9/73vdUU1Oj9evX65577okpcVlZWZKkmpoajRs3TocffrjuuusuvfTSSwoEAuro6Bjy9x0dHRo3blzY91u6dKm8Xu/Av4aGQ5fQT/Sm8mxiD8BILHwCwGy0dWBF4fLlJ7t3m500OEzEPdz33nuvbrjhBt1///0Dz02ZMkVf+tKXlJeXpw0bNmjLli3y+/165JFHtHz58qgTV15eLr/fr56eHrlcLkmSz+eTJM2YMUOPPvrokL/ftWuXpk2bFvb9srKyBoL4UIKbype50lSU0X8voig1RZJf7oObysdz+EZvZ4eatm9L2OcBiJ3dhp4BZqCcICjRbSvELprya7cyP2K+bG0hXyaA3fJMLCIOuLdv366VK1eG/N3pp5+uBx54QFJ/QOzxeGJK3Nlnn62jjz5a11xzjZ5++ml1dnZq+fLlmjdvni6//HKtWLFCNTU1WrRokf7whz9o7dq1+uUvfxn1541lU/n4BtxdCf08ANFjSCSS1ZSzxz4locvr1b5339HaSy4YeI5yktwS3bZC9KIZYm3XayP50jx2zTOxiHhI+ZFHHqlXX3015O+2bNmiiRMnSpI++eQTFRYWxpS4jIwM/f73v1d6erqOOeYYTZkyRcXFxXrqqac0YcIEbd68Wb/4xS80YcIEXXvttXr44Yd1xhlnRP15id5UPiPbldDPAxA9hkQi2QQXD514XPmYX1O7cpl6vC1xLSd7apkWYWeJblshetEMsbbrtZF8aR675plYRNzDfcMNN+jWW29VR0eHLr74Yk2cOFEff/yxNmzYoO9///tatWqVGhsbtXr16piC36B/+7d/089+9rOQvzvxxBP1xhtvxPwZQQObym+tl+RXXlqK2n0BwzaVz8jOSejnAYgOQyKRTKJdPNSocrJnMwv/2Vmi21aIzliGWEf0GotfG0fKl9mF4y2bbruzc56JRcQ93N/5zne0YsUKPfbYYzr11FM1depUnXLKKXrqqad0xx13aNmyZXr11VfV3d2t6mr7bcSe6E3l2cQesL6xDD0DnCLa3gfKCcKhrWN90ZRfu5f5cPnysKlTzU6aY9k9z0Qr4h5uSVqxYoW+853v6M0339Qnn3yi4uJizZgxQ3l5eZKkyy+/XAsXLoxrQhMl0ZvKD/48b2NDUiwcANjN4KFn/XdiNfBYYugZnCOW3gfKCcKhrWN90ZRfu5f5cPnyxe8uNjtpjmX3PBOtqAJuqX/P67POOmvIcwcOHNDrr7+uc86x/9CvRG8qX1hSysUHsKhkHxIZnMvr1KFe+FQsCwkNlJO36gZek0zlBKOjrWNd0Qyxdsq1kXyZOE7JM5GKeEi52+3WOeeco+zsbKWlpQ35l5+fr6985StGpBMATJWMQyK7vF5tWrxIay+5QB/velfPzp+nTYsXqautzeykwSCxLiRUWVWtzILxSVVOAKeIZoh1Ml4bEZtkzDMR93AvWbJEb775pr7xjW/ojTfeUE5OjmbPnq3a2lq988472rBhgxHpNN2eWhZtAZJZMg6JHDyXd+Au9MG5vKkZGWYnDwaItffBlZ+vw6dN15f++6akKSeAU0QzxDre10YrtLcj2QrRLHZIYzhGt6eskIeGi7iH+/e//71Wr16thx56SFdffbWysrJ07733atu2bTrttNNi2gfbyvZsZlsSAP1Dz8pmn+z4ICI4l7c4M0VFGanKTO3/WZyZIk99nXo7O81OIgwSj96HZCkngBNFU37jVeat0N62WrAWih3SOBqjrhNWyEPDRRxwt7e3a8aMGZKk8vJy/eUvf5EkpaWladGiRXrllVfim0Iba/G45a57I+RWCoDZyJ8YyWhzeQm4nSvY+7Bg3UZNLJ+mBes2am7NmhG3BIN5Bq+xAACwnoiHlB955JH66KOPJEmTJ09Wc3Oz9u7dqyOPPFJFRUXat29f3BNpN8P3MJU0pj1MgUQgf2IsRltJNCM725Q9+AxRAAAgAElEQVR0IXESvXgoIhPtfukAgMSKuIf7K1/5im6//Xa9+eabKikpUXFxse6//37t379fTz31lCZNmmREOm0l2j1MgUQgf2IsBuby9gTU3OtXj7//Z3AuLwE3YC7qcgCwh4gD7qqqKhUWFmrFihWSpLvvvlsPPfSQCgsLtXbtWt18881xT6SdjDbvkSFfMBP5E5FIxpVEATvo7eygLgcAm4h4SPmECRNUX1+vvXv3SpKuuOIKlZWVqa6uTieddJJOO+20uCfSTmLZwxTRY5/gsSF/IhKDVxKtXXWbKletJn8AFtDb2SUp+rrciqv4AlZ24J8fm50E2FjEAXfQkUceOfD/L33pS/rSl74UlwTZ3WjzHkfbwxSRYQ5bZMifiAZzeQFrych2SYq+Lt+zmYAbiEQ7ATdiEPGQcoxstHmPNFjjizlskSF/AoD9ZWTnUJcDgE0QcBuAeY+JwXzk6Ng5f7KVGZIdW0AhyM51OQAkk6iHlCO8wfMevY0NKigu4W6zAZiPHB075k+2MkOyY/oMhmONBQCwB3q4DVRYUqqy2SdzATTI4PnIgzEfeWzslD+ZOoBkRxlAOKyxAADWFnXA/d577+nhhx/WrbfeqqamJv3hD3/Q/v3745k2YETMR04OTB1AsqMMAABgXxEH3D6fT9ddd52mTZumxYsX67777tO+fft0xx13aMaMGWpsbDQinUBIzGFzvrFMHUhGzOVNHslcBvbUvpzQz4t1nQhWMh5dos8p7CvW6xx5zXoSdU6sdu4jDrhXr16ttWvX6oknntBHH32kQKB/+O4DDzwgn8+n5cuXxz2RQDjBOWwL1m3UxPJpWrBuo+bWrGFOo4MwdWCoLq9XmxYv0tpLLtDHu97Vs/PnadPiRepqazM7aTBIMpeBPZsT02gaXK5euOnGqMsVe/WOLlHnFPYVr+scec16EnVOrHbuIw64n3rqKVVVVenqq6/WhAkTBp7//Oc/r6qqKm3evDmuCQTGgjlszsXUgaGYy5t8KAPGo1wB1kF5tBdG3I0u4oB73759mjFjRsjfFRcXq6WlJeZEAVbe/snsisXKx8YoTB3ox1ze5EUZMA7lCrCO3s4OyqNNMOJu7CLeFmzy5Ml66aWXdNZZZx3yuy1btmjy5MlxSRiSk5W3fzJ7Wx4rHxuj2XErMyOwFZ5zDb6RF+ocsgWUcShXQHiJ7mTo7eySRHm0g8EjEfLSUtTuC6jx4EiEuTVrzE6epUTcw7148WI99NBD+ta3vqXf/va3SklJ0d/+9jc98MADuv/++7Vo0SIj0gkbi6SytvIwIrPTZvbnW4GdtjIzQjLP5XWqSHsImD4Tf5Qr4FCh6qZ9775jeO9lRrZLEuXR6hgZFJmIA+5rr71Wq1ev1tNPP61zzz1XgUBAl112mZYvX67vfOc7+uY3v2lEOhPO7GHDThBpQ9LKhdfstJn9+bAGK8/lpc6MTqQ30jjO8WflcgXEIpb6IlTd1ONtMfwmf0Z2DuXRBpJ594xoRDykXJKWLl2qRYsW6c0331Rzc7MKCwv1xS9+UUVFRfFOX8KFGzYcSBnlhTjESENNUjMyDvl7Kw/rMzttZn8+rKOyqlq1K5bKHWJqgRnMnmphZ8EbaWWuNBVl9N//LkpNkeSX++CNtGC55jgby2rlCohFrPVF+LpJh9RNRqA8Wt/gkUHBvBF8LDESYbioAm5Jys/P1znnnBPPtFhCuCAxM7/A7KTZymgNyUkzZx3yGisXXrPTZvbnwzqsNpeXOVzRi+RGGsfZWNGuEzG4BzEQCMjX03NIMDLa/Hwg3mKtL8y+yR/P6xzlzxgDI4O21kvyf5rPGIkQUsQB91FHHaWUlNDdvampqcrLy9PkyZP13//93zr11FNjTmAijRgktrZQWCMwWmXd29l5yGusXHjNTpvZnx/Ehcs6rDCXN5IeWhxqrDfSOM6JU1hSOqZjGaoHMSjYm3jqd5bqtfurGZWAhIpHfWGVm/yxXOe6vF7te/cdrb3kAkmUPyMwEmHsIp7DfcUVV2jv3r1qb2/X6aefrksvvVRnnnmmuru75Xa7NWXKFHk8Hp155pn63e9+Z0Sa465h21ZJzEeIp9EWocnIzg75OitvfWN22sz8fLZ+QCjUmbEZ69xhjrP1DO9BTJMOmYf/3LULk36hSyRePOqLsHVTt8/0DpCxql25TD3eFsqfgYIjERas26iJ5dO0YN1Gza1Zww2NECLu4f7Xv/6lL3zhC6qtrVVeXt7A811dXTr33HN15JFHat26dbrqqqt055136stf/nJcE2yEl265SXUTijTnznslmX9HzwlG65ENNYdbsvb2T2YP5TXz2DCcFaFYpRfEzsbSQ8BxtpbBPYg5aSlydwUO6U3s9fv0D6+XUQlIuHjVF6HqpuzC8ZboABkNo4ISywoj7qwu4h7udevWaenSpUOCbUlyuVy66aab9Oyzz0qSLr/8cv35z3+OTyoNVupKU4/Xq9/cfkvY3obswvFkpAjF0iNr5e2fzK5YEn1sWCEd4bC6c+zG0kPAcbaWwT2I3f7AwP8HSz/4kFEJSLR41Reh6qbDp023Re8lo4LGjp0vEiPigFuS9u/fH/J5r9ervr4+SVJGRkbYud5WMz4jVSWuNHV5vSqfd1HIIPGwqVNHfI8Wj1vuujfIsIMw1MT+Wjxu/e23v5HEhQuhmT3VwilGu5HHcbaOwT2IWQd7EIdPn+o7+DDctKr2f35MewGGiWd9YXYnQzRGm9bIqKDwUwV9fb1mJ82RIh5SfvbZZ2vp0qUqLy/XjBkzBp5/++23ddttt+nss8+WJG3YsEHHHXdc/FJqsGDw0PzB/xdy2PCL310c8nXDF06RxKIMw9ixsk52ofI1w1kRitlTLZIFx9k6Bk+ZKs7sbz80dvkkaWDKzb4+yVVQoMb9+xWcVuXt86up2y9JerX6Tkm0F2CMZK8vBsroW/1tGCstwmsV7MqUWBH3cNfU1CgnJ0czZ87UMccco5NPPlnHHHOMvvCFLygnJ0cPPfSQNmzYoEcffVTf/e53jUizIYLBwxHTPi9p7EHi4AzLogxwiuH5OjtVaujyMZwVYXFjLTE4ztYwuAex3ReQTzqkN/E/n3x2SC9jY7dfqSmHLq5GewFGSeb6orKqWpkF4xkVFMJIUwU7D+7KhPiKOOA+4ogjtGPHDj3++OP64he/qHHjxum0007T008/rbfffluTJk3S1KlTVVdXpwsvvNCINMddS69fDV0+uQoKVHrSF8f+uhjmtjJnAlYVKl9PzklXVuqhDUouXMmJ+gvJLtSUqQXrNiozL29g+lTBpEkDf1P0uWMkScVZaayFASSAKz9fh0+bzrTGEJjjnngRDymXpMzMTF1zzTW65pprDvldIBBQeXl5zAlLJE+XT+MnFOk/n3w2oteNJcMOv6sYau9OhpTBSkLl6/SUFB2dna6dB/pUUFKm8x94KCnvmCc76i9gqOE9iGmZmYfUjYUlpUrPypIUWXsBQOySuZc/HHa+SLyoAu6f/exn+v3vf6+enh4FAv0nx+/368CBA6qrq1NjY2NcE2m0c+99UMefGfn2ZdFkWLZXgtWNlq/zJk7kwpWkqL+QaL2dHert7LL9Nj4Z2S5JNHABmG+krXvZlckYEQfcd9xxh+644w4VFBSor69PGRkZysjI0D//+U+lpqbquuuuMyKdhio5cVZUrxttr+nhGZZ9AWEH0e6hDmfr7exQ0/ZtI9ZfQLwER1M0bd8myf6jKTKycyJqLwCAkULts15aMVsBe2wwZTsRz+F+5plntGDBAjU3N2vJkiU6//zztW/fPm3dulUTJkzQ8ccfb0Q6LSuSrReYMwG7SOYtiJifHFpvZ5ck6i8khhMXJE3mehWAtYTbujctPYN2kAEi7uFuamrSwoULlZKSopkzZ+qnP/2pJGnmzJlavny5nnjiCX3rW9+Ke0KtZsrZ50gauvWCt7FBBcUlYe9UM2fiU8HjB2uKJF87BfOTR8aQWCSKU0eDxbNe5RoKM5DvpD21L2tKZfTHIZHHcCyfNXiOe5fXq/aP9mrtJRdIMrcdNDjoH6meHP4dp5x9jvZsftno5EUs4h7u3NxcpaT0N7aOOeYYffjhh+rs7JQkzZgxQx9++GF8U2hRwwtbYUmpymafPGKmGBiq2xNI+u2VYqmskDhjyddO4cQetXgaGBJL/QWDOX00WDzqVa6hMAP5TjEHc4k8hpF+Vu3KZWr54H1T20FdXq82LV6ktZdcoI93vatn58/TpsWL1NXWFvLvh39Hq+bRiAPuk046Sc8884wk6XOf+5zS09P129/+VpL03nvvKevgSpwIjSFlgPXEssVfMqH+QiIMHg02GKMpAMAYVmkHObXzI+Ih5cuWLdNZZ52l1tZWvfDCC1qwYIG+9rWv6YwzztBvfvMbXXDBBUak0zEGDymrXXWbKletpmcIMFk0W/wlI+ovJEKkC5ICAGJjhXaQU6cTSVH0cJ966qnatm2bLrnkEknSI488oosvvli7d+/Wf/7nf+rhhx+OeyKdiH0BAeugRy0y1F8wGqMpACBxrNAOcvJ0ooh7uO+++2599atf1YIFCyRJLpdLjz32WNwTBgCJQo8aYC3B0RTPf+t69XZ2MpoCAAxkhXaQkxeXjriH+3//93/V0GDfOwwAEAo9asZo8bjlrnuDefCISkZ2NqMpACABzG4HOXlx6Yh7uKdMmaJ3331X55xjzVXgACAazE+Or+HbrElimzUTjHVrlXDYBghAsmvxuOXr6XH8jWMrtIMqq6pVu2Kp3CHaDnYWccB93nnnafny5XrhhRc0bdo0HX744UN+n5KSottvvz1uCQSARGJ+cnwMXml0YGjawZVG59asMTt5jhevfeWtusUKABgtVD2aXTheXW1tjr5xbGY7yApBvxEiDrhXrVolSXr99df1+uuvH/J7Am4ASG5OXmnULrjhAQCxCVmPeluoRxPAaZ0fEc/h9vv9I/7z+XxGpBMAYBNOXmnUDqyyn2qsgsPhezs7zU7KmCTLsNNYDJ7iABgp1rwWth7NSrNVPWoliSr/vZ0dlqtnIg64B/N6vdq9e7e6u7sJtAHAYsxq3Fphe5FkZvcbHl1erzYtXqS1l1ygj3e9q6btW7Vp8SJ1tbWZnbSQfL29A+ntaW/Xs/PnWTq9ZtQLw8+p1Y8R7CtUXtv37jsR5zW716NWkqjyH/ycpu3bLFfPRBVwb9myRRUVFSoqKtK0adO0c+dOXX755br55pvjnT4AQITMbtw6eaVRO7D7DY/BwziPz01XmStNTQeHw1vRJ3/dbYv0mlkv2O2cwr5C5bWeg8PAI2H3etRKElX+rVzPRBxwv/LKK6qsrFR2drbuvfdeBQL9Ge8LX/iCHnroIT344INxTyQAYOyscNExe3uRZGbnGx52Gw7f4nGrs7XFFuk1q16w2zmFfcVzGPj40jK5CgrU0OUbUo82dPnkKiiwdD1qJYkq/1avZyJeNG358uWaN2+e1q1bp76+Pv3P//yPJOnWW29VW1ubnnjiCd10001xTygAYHRWWbBs8Eqj3sYGFRSX0EBJILturTKWYZxWykd2Sa+Z9YJdjhHsL555rcXjVpfXq+xUyd316bTZ7FSp0+tl8c8xSlT5t3o9E3EP91/+8hddc801kvpXJB+ssrJSf//73+OSMABA5Kw276ywpFRls0+mYZJgwRseC9Zt1MTyaVqwbqPm1qyx/FY2dhvGaZf0mlkv2OUYwf7imdeCZebo7HQdl5uuo7PTDv7s76tkDvfYJKr8W72eiTjgLigo0N69e0P+zuPxqKCgIOZEAQCiY/WLDhLLblur2G04/PjSMmUXjrd8es2sF+x2TmFfYfNaty/ivDa4zLhSU1SQ3h8y/bPHL4lr6VglqvxbvZ6JOOD+6le/quXLl2vbtm0Dz6WkpKixsVF33323zjvvvLgmEAAwdla/6ACjsdv8/8OmTrV8es2uF+x2TmFfofJaZsH4iPPa4DLzzx6f/tbRp/cO9Onj3v6A+7UH7rXE6td2kKjyb+V6JuKA+5577tHEiRNVUVGh0tL+Cvqyyy7Tscceq5SUFFVXm/+lACCZWfmiA4xm+HD4STNnWXo4fFp6xkB6M/PyLDt838x6wa5THGA/ofLa4dOmR5XXgmWmsduvDl/Akqtf20Giyn/wcybNnGW5eibiRdPGjx+v+vp6/ehHP9Irr7yif/3rXyosLNSNN96oq6++Wjk5OUak01SD96ykdwiA1Q1esKx21W2qXLWaugsJEc/rZXA4vF0UlpQqLTPTsmXNCvWC3aY4ILGMqD9ieR9Xfr5Ouel/tPaSC1Ri8kKkTpCI8t/icau3s1MZ2dmWOi8RB9zbt2/XzJkzdd111+m6664zIk0h+Xw+ffnLX9ZnP/tZPf3005Kkl156Sbfccos++OADlZaW6r777ovrkHZfb682LV4kz8FVXp+dP29glVcr3C0BgJHQuEWidHm9ql25jOulDVAvwGpGqj/MZvXVr9FveB6SpE2LF1nmGhTxkPJZs2apvLxc99xzjxoaErdC3x133KHXX3994PHf/vY3XXTRRbrzzjvl9Xp1xx13aP78+WpqaorbZ37y192m72ULAIDVWWHvdwD2ZOX6g4VI7cHKeUiKIuD+1a9+pZkzZ6q6ulpHHXWUzjjjDP3f//2f9u/fb0T6JEmvvPKK1q9fr4suumjguWeeeUannHKK5s2bp/T0dM2fP1+nnXaaHnvssbDv093drba2tiH/wmnxuNXZ2mLZDdQBALCC4B7PXC8BRGq0+qO3s9PU9AUXT2vo8rEQqUXZ4RoUccD9H//xH/rxj3+sffv26Sc/+YkKCgp0ww036PDDD9dll12mX/3qV3FN4Mcff6yvf/3r+slPfjJkfvjOnTs1ffr0IX9bXl6uHTt2hH2v6upqFRQUDPwrKem/K9UaYi89q+1lCwDDTTn7HLOTAHC9BBC10eqPiceVJzxNw1VWVUvp6SxEalF2uAZFHHAHuVwuzZ8/Xxs3btRHH32k6667Ts8995zmzp0bt8T5/X4tWLBAN910k0444YQhv9u/f79yc3OHPJeTk6P29vaw77d06VJ5vd6Bf8Eh8T//2uXatHjRkOX9GUICwOqmVBJww3xcLwFEa7T64/i5FyQ8TcO58vOVlZ/PKvsWZYdrUNQBtyRt3bpVN998sz7/+c/r+9//vmbMmKHvfe978Uqbqqur5XK5dOONNx7yu9zcXHV0dAx5rqOjQ+PGjQv7fllZWcrPzx/yT5JKQ4zzH19apuzC8exlCwDACMze4xmAfdmp/mDBQWuyQx6KeJXyd955Rz/72c/085//XB9++KGKi4u1YMECLVy4UMcdd1xcE/fjH/9Y//jHP1RYWChJAwH2xo0btWjRIv3pT38a8ve7du3SiSeeGPHnjM9IVU4g5ZDl/Q+bOlUpAck9aMW7wasmtnjcamtqVEFxiSVOJgAAZqisqlbtiqVhr5cAEA71B2LR4nHruPPnqa+rS+4dfx543kp5KOKA+4QTTtC4ceN00UUX6fHHH9cZZ5xhRLokSbt37x7y+KqrrpIkPf3009q9e7cefPBBrVu3ThdeeKE2bNigLVu26KGHHorqs0It75+WnqHz7qs5ZM/KLq93yHZhktj+BACQtKywxzMAe6L+QDRCbQV25Akz1NvVpay8cZpbs8bE1A0VccC9du1aXXDBBXK5XEakZ8ymTp2qjRs36pZbbtHXv/51lZWVaf369ZoyZUpU7zfSOP/hQ0gGLz2fl5aidl9AjQeHpFvp5AIAkEgMuTQHCyjCCag/EImQ8dg7byszv0B5n5lodvKGiDjgvuyyyyRJ+/btU09PjwKB/kDV7/frwIEDev311/XNb34zvqk86Omnnx7yeM6cOZozZ07M79vS61ezP3VM4/yDS8+XudJUlNE/Bb4oNUWS/5Ah6QDsqbezQ+66N5J6ughTZgD7YAFFAMkkXDzW6/fpH60t6mxpsVQ7JuKAe8eOHbrsssv017/+NeTvU1JSDAu4jeLp8unYk8c2zn8sS8+bfVIBRCc4PKlp+zY1bd8mKfmmi4QaolVaMVuBlBFeBAAAkCDD47G+QEDuTp/aDo5Y3rfzHa295NMV7s1uy0W8Svl3v/tdtbS06P7779fpp5+uOXPm6JFHHtG5556rlJQUbdmyxYBkGuuSZ34y5uX97bD0PIDoDB6edHxuuspC7GDgdOGOwSfD1tQAAAAww/B4zN3p0wFfYGB4eZpkqbZcxAF3fX29Vq9erSVLlujSSy9Ve3u7brjhBr3wwguaN2+eHn74YSPSaajCCIJkOyw9DyByweFJxZkpKspIVWZq/8/izBR5Dk4XcbqRjkFna0tSHAMAAGBtg+Oxfd39PdvFrjTlHJzLXXxwqLlV2nIRB9zd3d0DC5NNnTpVb7/99sDvrr76atXV1YV7qWNUVlVr0qwKubt82nmgT+4unybNqrDM0vMAIjeW6SJO0+Jxy133xsAFKBmPgR0MP0+wtz21L5udBAA2Fe/rQYvHrY7m5oRcX3Zuel4f796tXS9uCpuWSL5bMB77R49fUn9bpdsfGPj/YGa3YyKew11aWqoPPvhAp5xyio455hi1tbXp73//uz772c8qKytLzc3NRqTTUgZvX+BtbLDEZHwAsRk8PKl/IUQNPJacNV0k3Dztim/8l6TkOAZ2wHx6Z9qz+WUWOQMQkXDXg2jnJQ9/v2fnzzNsnvO+93Zp/Teukr+vT5L0yl2rtOXe1br4iR9r4rHHRn2tC8Zjnvo6bVq8SO2+gHIOBtZWa8dE3MN90UUX6ZZbbtFzzz2nI444QlOnTtXy5cv1zjvv6IEHHtDnPvc5I9JpSYUlpSqbfTLBNuAAyTRdJNw87frHHg17DLILxzvqGNgB8+kBAFL815hJ5Jo1679xldTXN+Sz1Nen565dOGJaxnqtK62YPdB26fAFlJeWosYun6XachEH3CtXrtQpp5yip556SpL0ve99T88//7xmzJih3/3ud1q1alW80wgACZEM00VGm6tecf2ikMfgsKlTzU56UmE+PQBAiv8aM4lcs2bnpufl7+tTybA51SWuNPn7+rTtmaficq0b3H5r9wXkkyzVlot4SLnL5dIvfvEL9fb2SurfC/vdd9/V9u3b9e///u9J1cMNwFmCw5Oe/9b1+vcrrnTkdJHR5ml3tbaEnDLz4ncXJzytyYwtKAEAUvyvB4lcr6Xpz9tH/KyGrX8cNS1j+W6Dp/vWrrpNGdnZOuOW5ZaZ+htxwB2UkZEx8P+jjz5aRx99dFwSBABmy8jOVtnsk81OhiHGOle9sKTU9AtUMkumNQUAAOHF+3qQyOvLpC/M1J6XXwr7WSWzvqim7dvilpbCklLlFBUN/N8q7ZiIh5QDAOwr2rnqU85mkadEGuk8MZ8eAJJHvNeYSeSaNcfPvUCp6elqGDanuqHLp9T0dJ34tWuS4lpHwA0gIgRe9hfNXHVWVU68cOeJ+fQAYA6z2kDxXmMmkWvWXPzEj6X09CGfpfT0/udHSIuTrnVRDykHkJwIvOyPrQ3tIdx5Srb59NzkA2AVZrWB4n3dHj7nuXLVasPaAROPPVb/9fpb2vXiJv3xh4/qi9f/l8rPmxsyLU691hFwwzFolAGRsdL8JoSX7OeJm3wA0C/e14PgnOdEXGPKz5urD37/ypBge3hanHqtY0g5HINGGQAAAAArIeAGAMQFo0xiw/EDYEXUTUBsCLgBAHHBKJPYcPwAWBF1ExAbAm4AAAAkRG9nh9x1b6i1wWN2UgAgIVg0DQAAAIbq8npVu3KZmrZvU9P2bZKk0orZqqyqlis/3+TUAYBx6OEGAACAoWpXLlPT1nqVudJ0fG66ylxpatpar9oVS81OGgAYioAbAABYkt0Wa8r9zESzk2BJLR63PPV1Ks5MUVFGqjJT+38WZ6bIU1/H8HIYwm71RzJI1Dmx2rkn4AYAAJZkt8Wa8gi4Q2prapQk5aWlDHk++Njb2JDwNMH57FZ/JINEnROrnXsCbgAAgARo8bjV0dycdD26+ZOKJUntvsCQ54OPC4pLEp4mAEgUAm4khNWGdgAAkCi+3l5tWrxIay+5QB/velfPzp+nTYsXqautzeykJcT40jKVVsxWY09Azb1+9fj7fzb2BFRaMVuFJaVmJxEADEPAjYSw2tAOALHhJhowdp/8dXfSLxhWWVWtSbMq5O7yaeeBPrm7fJo0q0KVVdVmJw2AQ7V43JbYhpBtwUZAgxIAQuMmmnm4NtlLi8etztYWlbnSVJTR389RlJoiyS/3wQXDkqGH15Wfr7k1a9Ta4JG3sUEFxSVJ8b3NRn2BZBTchtBTXzfwnJnbENLDPQIalAAAq+HaZC8sGDZUYUmpymafTLCdINQXSEZW24aQHm4AGIYeAQChtHjcamtqjKh3dvCCYf092xp4LLFgGIDo6hYjRdsOMut7DP7c3s4ONW3fZqlRRQTcADAMPQIABhtpeOJoxpeWKbtwvBrbvJL8yktLUbsvYOqCYdxUBKzBakOfgyJtBwUXhkz09wh1/DJz8ySNPKoo0fUuQ8oBAABGEOvwxMOmTrXUgmHcVAQik/eZiYa8r9WGPkfLiIUhx3JjMNTx6zvQLsla2xDSww0AABBGi8ctT31d2OGJk2bOGvU90tIzdN59NWpt8Kh21W2qXLXaEsNGAYxNrgEB92h1i10WVDRqYcjRbgyGP36Su8unxh6/JFliVBE93AAAAGGMtuhZb2fnmN+rsKRUOUVFtmhEG2FP7ctmJwFJyKpTKJyyoKJZ32O0zy06ZqplRhXRww0AABDGaIueZWRnm5IuO9qz+WWGsyPhrJrnnLKgolnfY7TPnXNnf3BthW0ICbgBAADCGF9aptKK2WrcWq9Qi55NPff8gb8N15Nm1R42AOYZrW6xy0gYs3XZsosAACAASURBVBaGDHv8un1DPtcKx5Eh5QAAwBFaPG65696Qp75O7ro31Nrgicv7VlZVh130bHDvWbieNKv2sAEw10h1i52YtTBkqOOXWTDecsePHm4AAGBrobaGCYrH1jSu/HzNrVmj1gaPJYYnAnAGp9QtgxeGTOT3CHX8/vDwg6ZuqRYKATcAALC14NYw2alSj18qdqV9Orzw4NY0c2vWxPw5hSWltmwMA7A2u9ctwWkzZn0Pqx8/hpQDAADbCm4Nc3i61Hkw2C7KSFVmaoqKMlJVnJkiz8GtaQAA8ce0mZERcANwpOBcThrZgLMFt4ZJP7hIrd232HGK4XVwi8etjuZm6mQASYch5QAcJdRcznjM4QRgTcGtYfr6d4Kx/RY7dheqDnYVFKjL65UkPTt/HnUygKRCDzcARwnO5Sxzpen43HSVudLUdHAOJwDnCW4Ns69Pyk6VGrt8au71q8cfUHOv33Zb7Njd8Do4O1Xq8XqpkwEkLQJuE7EvJxBfwbmcxZkpzOEEkkhwa5hOv+STbL/Fjl0Nr4P96p9XX8K8egBJjCHlJmKBAfviZok1BedyjjSHk14uwHmGbw2TkpamgM9n2y127Gp4HdztDwx5HESdDCCZEHADUeBmiTUF53IyhxNITlbfGsbphtfBWQfrYepkwLroRDIeQ8oBOEZwLmdjT4A5nACQYMPr4FT1z6tvYF49YFl0IhmPgBuAowTncjKHsx93rgEk0vA6uNMvZRYUUCcDSFoMKQfgKMPncib7HE7uXANIpHB1cGuDR7WrblPlqtVJXScDSD4E3AAcibmcAGCe4XVwYUmpcoqKqJcBJB2GlMNyWjxuueveYLsQAIDjMM3DWLQhkAz21L5sdhIQAXq4YRldXq9qVy6Tp75u4LnSitmqrKqWKz/fxJQBABAfTPMwBm0IJJM9m1+mLrERerhhGbUrl6lpa73KXGk6PjddZa40NW2tV+2KpWYnDQAAWBhtCABWRQ+3gVo8brU1NTpi0Sajv0uLxy1PfZ3KXGkqyui/D9S/Z6df7vo6tTZ4bHsMnZQPgGhRDoDYUY5CG60Nse3pJ9Xs/lDFM09S+XlzzU0sEKMWj1sdzc22bhsnGwLuMYpkzpWThjUl6ru0NTVKkvLSUoY8H3zsbWywXaXipHwARItyAMTOKeXIqPnr4doQqfJLkv74wzWSpD0vv6Qt967WxU/8WBOPPdaQtCA2rHEQ3vB64Nn582xZDxjNinmIIeVjFMk8CScNa0rUd8mfVCxJavcFhjwffFxQXBLXz0sEJ+UDIFqUAyB2TilHRs05DdeG+LAroFRpyHFTX5+eu3ahIelA7JiXHJ5T6gGjWTEPEXDHWXBYU3FmiooyUpWZ2v+zODNFnoNDo+0ikd9lfGmZSitmq7EnoOZev3r8/T8bewIqrZhtu95tJ+UDIFqUAyB2lKPRhWpDuDt7JUklB4eZB49biStN/r4+7Xpxk8mpBsaOesDeCLjjbCxDo+0i0d+lsqpak2ZVyN3l084DfXJ3+TRpVoUqq6rj+jmJ4KR8AESLcgDEjnI0NsPbEM19/c+HO26N299KdBKBqFEP2BtzuONs8LCm/gU7NPBYstfQ6ER/F1d+vubWrFFrg0fexgZbLwrjpHwARItyAMSOcjQ2w9sQ//zrbv3xh2vCHrfimSeZlVQgYtQD9kYPd5w5aWi0Wd+lsKRUZbNPttWxGs5J+QCIFuUAiB3lKDLBNsSJV31dqenpaujyDTluDV0+paans1o5bIV6wN4sH3Dv2LFDZ599toqKinTEEUfoyiuv1CeffCJJqq+vV0VFhfLy8nTUUUfpySefNDm1/Zw0NLqyqlqHTz9hyHeZOP3ztvwuieakfABEi3IAxI5yFJ2Ln/ixlJ4+5LgpPb3/edhCi8ctd90bzFEW9YCdpQQCgcDof2aOzs5Ofe5zn9N1112n5cuXa//+/bryyiuVmpqqH/3oR5o8ebKqqqp0/fXX67XXXtO8efP029/+ViedNLZhQm1tbSooKJDX61W+Acvp231odKhtSILYhmDs7J4PgHigHJjnxe8u1nn31Tj285IJ5Sg6u17cpMbtb7EPt41YbSs8K9VrrQ0e1a66TZWrVlMP2ISl53B7PB6dcMIJWrFihdLS0jRhwgRdf/31WrhwodavX68JEyZo0aJFkqQzzzxTV1xxhdasWTPmgNtohSWlti4Iwe0HslOlHr9U7EpTXlqK2n0BNR7chmBuzRqzk2l5ds8HQDxQDoDYUY6iU37eXAJtmxm8BRZtz6EKS0qVU1REXWAjlh5Sfuyxx+rXv/610tLSBp577rnnNHPmTO3cuVPTp08f8vfl5eXasWNH2Pfr7u5WW1vbkH8ILbj9wOHpUufBYJttCAAAAGAktsCC01g64B4sEAjotttu0wsvvKCHHnpI+/fvV25u7pC/ycnJUXt7e9j3qK6uVkFBwcC/khJW9AsnuP1A+sGFENmGAADsacrZ55idBAAYMytugWW1etRq6cHIbBFwt7W16eKLL9azzz6r1157TdOnT1dubq46OjqG/F1HR4fGjRsX9n2WLl0qr9c78K+hgWAxnOD2A30HZ/gHtx0IYhsCALCHKZU0zADYx+AtsAYzs+1ptXrUaunByCwfcL///vuaNWuW2tratG3btoFh5NOmTdPOnTuH/O2uXbs0bdq0sO+VlZWl/Pz8If8QWnD7gX19Unaq1DhsWw22IQAAAEC8sQUWnMbSAXdLS4vOPPNM/b//9//0m9/8RocddtjA7y688EJ99NFHqqmpUW9vr1599VWtXbtW11xzjYkpdpbg9gOdfsknsQ0BAAAADMcWWHASS28L9uCDD+rmm29WTk6OUlKGzuNob2/Xtm3b9O1vf1vvvPOOPvOZz+j222/XVVddNeb3N3pbMKcIbkOSkpamgM/HdiQAgLCstH0OAHtjKzw4gaUDbqMRcAMAEF8E3AAAfMrSQ8oBAAAAALArAm4AAAAAAAxAwA0AAAAAgAEIuAEAAAAAMAABNwAAAAAABiDgBgAAAADAAATcAAAAAAAYgIAbAAAAAAADEHADAAAAAGAAAm4AAAAAAAxAwA0AAAAAgAEIuAEAAAAAMAABNwAAiJspZ59jdhIAALCMlEAgEDA7EWZpa2tTQUGBvF6v8vPzzU4OAAAAAMBB6OEGAAAAAMAABNwAAAAAABiAgBsAAAAAAAMQcAMAAAAAYAACbgAAAAAADEDADQAAAACAAQi4AQAAAAAwAAE3AAAAAAAGIOAGAAAAAMAABNwAAAAAABiAgBsAAAAAAAMQcAMAAAAAYAACbgAAAAAADEDADQAAAACAAQi4AQAAAAAwAAE3AAAAAAAGIOAGAAAAAMAABNwAAAAAABiAgBsAAAAAAAMQcAMAAAAAYAACbgAAAAAADEDADQAAAACAAQi4AQAAAAAwAAE3AAAAAAAGIOAGAAAAAMAABNwAAAAAABiAgBsAAAAAAAMQcAMAAAAAYAACbgAAAAAADEDADfz/7d17cM1nHsfxTxqJ3E4SIUIkUbfQiN1V1p1ErQpFtdVlx73Wqk7b0VrslqZ2Olr0YqtdtdZ12F06oW5ddpGg6lZrqtsoUm3FLUKQEOT63T/UGcetJTlJyPs1c2Z6nuf5nd/z/Oabr37z+50nAAAAAOAGFNwAAAAAALgBBTcAAAAAAG5AwQ0AAAAAgBtQcAMAAAAA4AYU3AAAAAAAuAEFNwAAAAAAbkDBDQAAAACAG1BwAwAAAADgBhTcAAAAAAC4AQU3AAAAAABuQMENAAAAAIAbUHADAAAAAOAGFNwAAAAAALgBBTcAAAAAAG5wzxfcmZmZ6tOnj4KDg1WjRg2NHj1ahYWF5T0tAAAAAEAld88X3P369VNAQICOHz+uXbt2acOGDZo+fXp5TwsAAAAAUMl5mJmV9yTu1jfffKNGjRrp2LFjCg8PlyQtXbpU48aN0+HDh28Yn5eXp7y8POf7nJwcRUZGKjs7W4GBgWU2bwAAAADA/e+evsOdmpqqkJAQZ7EtSTExMUpPT9e5c+duGP/mm28qKCjI+YqMjCzL6QIAAAAAKpF7uuA+f/68/P39Xdr8/PwkSRcuXLhh/B//+EdlZ2c7X0eOHCmTeQIAAAAAKp8q5T2BkvD399fFixdd2q6+dzgcN4yvWrWqqlatWiZzAwAAAABUbvf0He7Y2FhlZWXp5MmTzrZ9+/YpIiJCQUFB5TgzAAAAAEBld08X3I0aNVKHDh00evRonT9/Xt99951ef/11DR8+vLynBgAAAACo5O7pgluSkpKSVFhYqHr16ql169ZKSEjQq6++Wt7TAgAAAABUcvf0nwUrqezsbAUHB+vIkSP8WTAAAAAAuM85HA55eHiU2fnu6U3TSiorK0uS+PNgAAAAAFAJZGZmKjQ0tMzOV6kL7pCQEElSeno6m6yhVOTk5CgyMpKnJlBqiCmUNmIK7kBcobQRUyhtV2PK29u7TM9bqQvuBx648hX2oKAgfpBRqgIDA4kplCpiCqWNmII7EFcobcQUSltZPk4u3QebpgEAAAAAUBFRcAMAAAAA4AaekyZNmlTekyhPnp6eio+PV5UqlfrpepQiYgqljZhCaSOm4A7EFUobMYXSVh4xVan/LBgAAAAAAO7CI+UAAAAAALgBBTcAAAAAAG5AwQ0AAAAAgBtQcAMAAAAA4AaVsuDOzMxUnz59FBwcrBo1amj06NEqLCws72mhglm6dKmqVKmigIAA52vQoEGSpH/9619q1qyZ/P399dBDD2nNmjUux06bNk0RERHy9/dXfHy8Dhw44OzLzc3VsGHDVL16dQUFBWnw4MG6cOFCma4NZevUqVNq2LChNm3a5GzbuXOnWrdurYCAANWrV09z5851OWbhwoVq2LCh/P391bJlS23fvt3ZV1RUpLFjxyosLEwOh0OPP/64Tpw44ewnx1UON4urUaNGqWrVqi55a/bs2c7+kuSmgwcPqkuXLnI4HAoPD9cbb7xRJuuE++3du1ddu3ZVSEiIatWqpcGDB+v06dOSyFW4O7eLKfIU7lZycrJat26twMBA1apVSy+88IIuXbokqYLnKquE4uPjbcCAAZabm2uHDh2ypk2b2rRp08p7WqhgxowZY0OHDr2h/eDBg+bj42Mff/yxFRQU2NKlS83X19eOHj1qZmYLFiywOnXq2FdffWWXLl2yl19+2Zo2bWrFxcVmZjZ06FDr0qWLZWVl2cmTJy0uLs6ee+65Ml0bys7WrVutQYMGJslSUlLMzOzMmTMWEhJiH3zwgRUUFNjGjRvN4XDYzp07zcwsJSXFHA6Hbd261fLz8+3dd9+1GjVqWG5urpmZTZo0yX72s59Zenq6ZWdnW79+/axHjx7Oc5Lj7n83iyszsxYtWtiCBQtuekxJclN+fr41atTIxo8fb3l5ebZnzx4LDw+3jz76yO1rhXtdvHjRateubYmJiZaXl2enT5+2Hj16WM+ePclVuCu3iykz8hTuTmZmpvn4+Nj8+fOtqKjIjh8/brGxsZaYmFjhc1WlK7jT0tJMkh07dszZtmTJEouKiirHWaEi6tSpk33wwQc3tE+YMMG6du3q0paQkGCJiYlmZta+fXubPHmysy8/P98cDoclJydbbm6ueXl52Weffebs37Fjh/n6+jp/6HH/WLBggUVFRdmSJUtcCqO//e1v1qhRI5exzz77rA0ePNjMzAYMGGAjRoxw6W/SpInNmzfPzMwiIiLs73//u7MvIyPDPDw87NChQ+S4SuBWcXX58mXz9va2r7766qbHlSQ3rV+/3gICAiwvL8/ZP2XKFOvUqZN7Fokys3//fktISLDCwkJn28qVKy0wMJBchbtyu5giT6EkcnJyzMysuLjY/ve//1nDhg3t/fffr/C5qtI9Up6amqqQkBCFh4c722JiYpSenq5z586V48xQkRQXF2vPnj365JNPVLduXUVEROh3v/udzp49q9TUVDVr1sxlfExMjPbu3StJN/R7eXmpUaNG2rt3r9LS0lRQUODSHxMTo0uXLungwYNlsziUmW7duunQoUPq16+fS/udxtC1/dnZ2Tp69KhLf1hYmKpVq6Yvv/ySHFcJ3Cqu9u7dq4KCAiUmJiosLEzR0dGaOnWqiouLJZUsN6Wmpio6Olre3t4u/VdjFveuxo0ba+3atfL09HS2JSUlqUWLFuQq3JXbxRR5CiXhcDgkSZGRkWrWrJlq166tYcOGVfhcVekK7vPnz8vf39+lzc/PT5L4Hi2cTp06pebNm6tv3776+uuvtW3bNqWlpWngwIG3jKGr8XO7/vPnz0uSSz/xd/+qVauWqlSpckN7acfQ9f3kuPvbreIqOztb8fHxevHFF3X06FEtXrxYM2bM0DvvvCOpZLnpx2IW9wcz08SJE7V69Wq999575CqU2PUxRZ5CaUhLS9OxY8fk6empvn37VvhcVekKbn9/f128eNGl7er7q781AcLCwrRlyxY988wz8vPzU1RUlKZNm6a1a9fKzG4aQ1fj51Yx5nA4nD+w1/YTf5XP7WLkx/pvFkPX95PjKqeuXbsqOTlZcXFx8vLyUqtWrTR69GgtXbpUUsly04/FLO59OTk56tu3rxYvXqwtW7Y4NwYlV+Fu3SymyFMoDb6+vgoPD9fUqVO1bt26Cp+rKl3BHRsbq6ysLJ08edLZtm/fPkVERCgoKKgcZ4aK5Msvv9Qf/vAHmZmzLS8vTw888IBatWql1NRUl/H79u1TbGyspCsxdm1/QUGB0tLSFBsbq8aNG8vLy8ulf9++ffL29lZ0dLSbV4WK4voYkW4fQ9f2V6tWTXXq1HHpz8jI0JkzZxQbG0uOq8RWrFihv/71ry5teXl58vX1lVSy3BQbG6uDBw+67Mp6bczi3nbo0CH98pe/VE5Ojnbv3u18tJJchbt1q5giT+Fubdu2TU2aNFF+fr6zLS8vT97e3oqJianYueqnf039/tGhQwfr37+/5eTk2LfffmtNmza11157rbynhQrkyJEj5u/vb1OnTrWCggI7fPiwtWnTxoYPH25ff/21+fj42NKlS527lPv4+NiBAwfMzGzOnDlWp04d++KLL5w7bDZs2NDy8/PNzGzgwIEWHx9vmZmZlpmZafHx8TZkyJByXC3Kgq7Z3Or06dMWHBxs06dPt/z8fEtOTnZuCmNmtmHDBuf7/Px8mz59ulWrVs2ysrLMzGzixIkWGxtr3377reXk5Fi/fv0sLi7OeS5yXOVxbVwtX77cfH19bcOGDVZcXGzbtm2zGjVq2KJFi8ysZLmpoKDA6tWrZ2PGjLFLly7ZF198YeHh4TZ//vxyWDVK05kzZywqKsqGDh1qRUVFLn3kKtyN28UUeQp36/z58xYZGWkvvfSS5eXl2ffff2+tWrWyUaNGVfhcVSkL7oyMDOvbt69Vr17dQkNDbcyYMS47KQJmZps2bbK2bduaw+Gw0NBQe+GFF+zSpUtmZrZu3Tr7+c9/bgEBAda0aVP75JNPnMcVFxfb22+/bfXq1bOAgADr3Lmzsxg3u7LD4ogRIywsLMyqVatmQ4cOtQsXLpT5+lC2dN2fb/r888+tXbt25nA4rH79+jf8D8GiRYuscePG5u/vb61atbIdO3Y4+/Lz8238+PFWp04dCwwMtMcff9xOnjzp7CfHVR7Xx9WsWbMsOjra/Pz8rH79+vaXv/zF2VfS3JSWlmaPPvqoBQUFWZ06dWzKlCllska41zvvvGOSzM/Pz/z9/V1eZuQq3LkfiynyFO5Wamqqde3a1YKDg61u3bo2YcIEu3z5splV7FzlYXbNM7MAAAAAAKBUVLrvcAMAAAAAUBYouAEAAAAAcAMKbgAAAAAA3ICCGwAAAAAAN6DgBgAAAADADSi4AQAAAABwAwpuAAAAAADcgIIbAAAAAAA3oOAGAAAAAMANKLgBAAAAAHADCm4AAAAAANyAghsAAAAAADeg4AYAAAAAwA0ouAEAAAAAcAMKbgAAAAAA3ICCGwAAAAAAN6DgBgAAAADADSi4AQAAAABwAwpuAAAAAADcgIIbAAAAAAA3oOAGAAAAAMANKLgBAAAAAHADCm4AAAAAANyAghsAAAAAADeg4AYAABWGmZX3FAAAKDUU3AAAoFwMHTpUDz74oPP9qlWrNGTIEOf7TZs2ycPDQ5s2bSr7yQEAUAo8jF8lAwCAcnDo0CHl5OSoefPmkqT4+HhJchbYOTk52rdvn2JiYhQYGFhOswQA4O5VKe8JAACAyqlBgwa37Q8MDFSbNm3KaDYAAJQ+HikHAOAa+/fvV48ePRQYGKiwsDBNmDBBzzzzjPPuqyQVFxdrypQpatiwoapWraro6Gi9//77Lp8THx+v3/72t5o6daqioqLk4+Ojdu3aaefOnS7jdu/erYSEBFWvXl2BgYHq1auXUlNTnf1XH6tOTk5W586d5evrq6ioKM2ZM0cnTpzQk08+qYCAAEVGRurPf/6zy2efOXNGI0eOVFhYmHx8fNSmTRtt3LjxtutfsGCBPDw8tHPnTj388MPy9fVVs2bN9NFHH7mMy87O1ssvv6wGDRrIx8dHsbGxmjdvnsuYPXv2qEuXLgoKCpLD4dCvfvUrl/Vf+0h5fHy8Nm/erM2bNzsfI7/ZI+U/9Xpt3LhRjz76qPz8/BQWFqaxY8eqsLDwtmsHAKC0UXADAPCD06dPq1OnTkpPT9f8+fM1Y8YMJSUl6R//+IfLuFGjRikxMVEDBw7U6tWr9fTTT2v06NF6/fXXXcYlJSVpxYoVmjFjhv75z3/q5MmT6tu3r4qKiiRJKSkpateunYqKijRv3jzNmTNHR44cUbt27bR//36Xz+rfv7969eql1atXKzo6Ws8++6w6d+6sZs2aadmyZWrRooVeeukl7dq1S5J0+fJlPfLII1q5cqUmT56s5cuXKyIiQgkJCUpOTv7Ra9GzZ0/16dNHH3/8sZo0aaL+/ftr1apVkqRLly6pQ4cOWrx4sX7/+99r5cqV6tixo4YPH6433nhD0pXHwRMSElSjRg0lJSVpyZIlys3NVbdu3ZSdnX3D+WbOnKnmzZurefPm2r59ux5++OEbxtzJ9RowYIA6duyoNWvWaMCAAXr77bc1d+7cH103AAClygAAgJmZvfrqq+bj42NHjx51tn3//ffm7e1tcXFxZmZ24MAB8/DwsClTprgcO3HiRPPx8bHTp0+bmVlcXJz5+flZdna2c8zChQtNku3evdvMzFq1amVNmjSxwsJC55izZ89a9erV7de//rWZmaWkpJgkGz9+vHPM9u3bTZINGjTI2Xb69GmTZNOnTzczs9mzZ5sk27Fjh3NMcXGxderUyVq2bHnLazB//nyTZH/6059cjvvFL37hPG7mzJkmybZu3epy7PDhw83Hx8eysrKcc7x2zDfffGNjx4619PR0MzMbMmSI1a1b19kfFxfnvM7Xrj0lJeWOr9fEiRNd5lavXj3r2bPnLdcNAIA7cIcbAIAfJCcnq127dqpTp46zrW7dumrXrp3LGDNTr169VFhY6Hz17t1bly9f1qeffuoc27RpU5fNviIiIiRJubm5ys3N1eeff65+/frJ09PTOSY4OFg9e/ZUSkqKy9yunUOtWrUkyeX7zdWrV5cknTt3TpK0ceNG1apVSy1atHDOsaioSL169dLu3bt19uzZ216LQYMGOf/bw8NDTz75pP773//q4sWL2rRpk+rWrav27du7HDNw4EBdvnxZO3bsUGxsrEJDQ9WrVy+NGjVKq1evVu3atTVt2jRFRkbe9tw3c6fXq23bti7vIyIilJube8fnBQCgJCi4AQD4walTp1SzZs0b2q8WuJKUlZUl6Uox7eXl5Xy1atVKknT8+HHnWD8/P5fPeeCBK//sFhcX69y5czIzl8++9nxXC+erbrZL9/Wff62srCxlZGS4zNHLy0tjx46VJJ04ceKWx0pSeHi4y/uaNWvKzJSdna0zZ87cct7SlaI/ICBAn376qR577DEtWbJEvXv3VmhoqEaOHKnLly/f9tw3c6fX62bXvri4+I7PCwBASbBLOQAAP4iIiFBmZuYN7de2BQcHS7pyp9vhcNwwNioq6iedKzg4WB4eHsrIyLih78SJE6pRo8ZPnfYtP79Ro0Y3fP/8qnr16t32+KysLJei++TJk/L09FRISIhCQkKUlpZ203lLcs69cePGWrRokYqKirRr1y4tWrRIH374oerXr6/x48ff8Xrceb0AAHAH7nADAPCDuLg4bdu2zaWoy8jI0Pbt213GSFc2WGvZsqXzlZWVpYkTJzrvgP8Yf39/tWzZUkuXLnVuoiZd2f17zZo16tChQ4nXcuTIEdWsWdNlnhs2bNC0adNUpcrtf+d+dYM0STIzLVu2TB06dFDVqlUVFxenw4cP67PPPnM5ZvHixc67/UlJSQoNDVVGRoY8PT3Vtm1bzZw5U8HBwTpy5MhNz3nto+LXc/f1AgDAHSi4AQD4wYsvviiHw6Fu3bpp2bJlWrZsmbp166a8vDzn4+CxsbEaOHCgRowYobfeekspKSmaNWuWfvOb3+jUqVOKjo7+yed78803lZaWpoSEBK1atUpJSUl65JFHlJeXp9dee61Eaxk2bJjq1q2rrl27auHChUpJSdErr7yiCRMmKDw8XF5eXrc9fty4cXrvvfe0bt069e3bV/v27XPuwj506FDFxMToiSee0KxZs/Sf//xHzz//vObNm6dXXnlFwcHBat++vYqKitSnTx+tWLFCycnJGjlypLKzs/XUU0/d9JzBwcE6ePCgkpOTb/odc3deLwAA3IGCGwCAHwQHByslJUWhoaEaNGiQnnvuOT311FNq3bq1AgICnOPmz5+vMWPGaNasWerWrZsmT56s/v37a/369be9S3u9Ll26aMOGDcrLy1P//v01YsQIRUREaOfOnWratGmJ1uLv768tKqt25AAAAXJJREFUW7aoQ4cOGjdunLp3767ly5drypQpevfdd3/0+A8//FCzZ8/WE088oRMnTmj9+vXq2LGjpCvfj968ebN69+6txMRE9e7dW1u3btXcuXM1adIkSVLt2rX173//W0FBQRo+fLgee+wx7dmzR8uWLVPnzp1ves7nn39eXl5e6t69u9auXXtDvzuvFwAA7uBhZlbekwAAoCLYuXOnzpw5o+7duzvbCgsLFRUVpf79+/+kQvVet2DBAg0bNkzfffedHnzwwfKeDgAA9zQ2TQMA4Afp6enq16+fEhMTFR8fr9zcXM2aNUvnzp3TiBEjynt6AADgHkPBDQDAD55++mllZWVp5syZeuutt+Tt7a02bdpoy5Yteuihh8p7egAA4B7DI+UAAAAAALgBm6YBAAAAAOAGFNwAAAAAALgBBTcAAAAAAG5AwQ0AAAAAgBtQcAMAAAAA4AYU3AAAAAAAuAEFNwAAAAAAbkDBDQAAAACAG/wfw5ty0eQgHKUAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 1200x800 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "# generate a figure with sub-plots (one for each gene segment)\n",
    "fig = plt.figure(figsize=(12,8), facecolor='w')\n",
    "plt.style.use('default')\n",
    "mpl.rc('font', family='Arial')\n",
    "gs = gridspec.GridSpec(2, 10)\n",
    "\n",
    "ax1 = fig.add_subplot(gs[1,0:8])\n",
    "\n",
    "# get right of top and right frame of each subplot\n",
    "ax1.spines['top'].set_visible(False)\n",
    "ax1.spines['right'].set_visible(False)\n",
    "ax1.spines['bottom'].set_visible(True)\n",
    "ax1.spines['left'].set_visible(True)\n",
    "\n",
    "# set fontsize of axes \n",
    "ax1.tick_params(axis = 'both', which = 'major', labelsize = 10)\n",
    "ax1.tick_params(axis = 'both', which = 'minor', labelsize = 10)\n",
    "\n",
    "ax1.set_xlabel('\\n genome position', fontname='Arial', fontsize=12)\n",
    "ax1.set_ylabel('average age', fontname='Arial', fontsize=12)\n",
    "ax1.set_facecolor('white')\n",
    "for tick in ax1.get_xticklabels():\n",
    "    tick.set_fontname(\"Arial\")\n",
    "for tick in ax1.get_yticklabels():\n",
    "    tick.set_fontname(\"Arial\")\n",
    "ax1.set_ylim(-2,102)\n",
    "ax1.set_xlim(-2,30000)\n",
    "ax1.set_xticks([0, 5000, 10000, 15000, 20000, 25000, 30000])\n",
    "\n",
    "ax1.errorbar(Dane_ages['Position'], Dane_ages['mean'], yerr=Dane_ages['std'], fmt='.', zorder=0, lw=0.5, c='#933831')\n",
    "ax1.scatter(x=Dane_ages['Position'], y=Dane_ages['mean'], marker='o', s=20, facecolors='#C44536', edgecolors='#933831', zorder=2)\n",
    "\n",
    "fig.savefig(\"../figures/SNVs_vs_age.pdf\", bbox_inches='tight', dpi=300)\n",
    "\n",
    "# use tight_layout to optimize spacing of axes and titles within each subplot\n",
    "fig.tight_layout()\n",
    "fig.show()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 428,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'range': '0-19', 'Count': 7},\n",
       " {'range': '20-44', 'Count': 424},\n",
       " {'range': '45-64', 'Count': 297},\n",
       " {'range': '65-89', 'Count': 183},\n",
       " {'range': '≥90', 'Count': 22}]"
      ]
     },
     "execution_count": 428,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "Dane_all = pd.concat([pd.read_csv(f, sep='\\t') for f in glob.glob(\"../data_derived/Dane/cleaned/*.csv\")], ignore_index=True)\n",
    "Dane_all = Dane_all[['sample', 'Position']]\n",
    "\n",
    "metadata_age = pd.read_csv('../data_raw/UWHC_ages.csv', sep='\\t')\n",
    "\n",
    "metadata_age = metadata_age[['strain', 'age']]\n",
    "\n",
    "metadata_age['strain'] = metadata_age['strain'].str.replace('/','_')\n",
    "\n",
    "metadata_age_dict = dict(zip(metadata_age.strain, metadata_age.age))\n",
    "\n",
    "Dane_all['ages'] = Dane_all['sample'].map(metadata_age_dict)\n",
    "\n",
    "Dane_all = Dane_all.set_index('sample')\n",
    "\n",
    "Dane_all = Dane_all.astype(float)\n",
    "\n",
    "Dane_all['range'] = pd.cut(Dane_all.ages,[-1,19,44,64,89,100], labels=['0-19','20-44','45-64', '65-89', '≥90'])\n",
    "\n",
    "Dane_all.groupby('range')['ages'].count().reset_index(name='Count').to_dict(orient='records')\n",
    "\n",
    "# age bins\n",
    "\n",
    "# \"0 - 19\"\t1\n",
    "# \"20 - 44\"\t56\n",
    "# \"45 - 64\"\t35\n",
    "# \"65 - 89\"\t24\n",
    "# \"≥ 90\"\t2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 430,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/seaborn/categorical.py:3666: UserWarning: The `factorplot` function has been renamed to `catplot`. The original name will be removed in a future release. Please update your code. Note that the default `kind` in `factorplot` (`'point'`) has changed `'strip'` in `catplot`.\n",
      "  warnings.warn(msg)\n",
      "/Users/katbraun/anaconda3/lib/python3.6/site-packages/matplotlib/figure.py:445: UserWarning: Matplotlib is currently using module://ipykernel.pylab.backend_inline, which is a non-GUI backend, so cannot show the figure.\n",
      "  % get_backend())\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAecAAAGqCAYAAAAxy4zmAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAPYQAAD2EBqD+naQAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4zLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvnQurowAAIABJREFUeJzt3XlYFYXi//EPoIIggh4XSAQhNUWway6U5XqtTG0lw+uCW6lJ7m1armV2zf2q+eTVrLRyKytvmaZyScstygU1cccdlUVFQGB+f/jzfO+5qJdT53gGeL+eZ56nMzNnzmcm5MMsZ8bNMAxDAADANNxdHQAAANiinAEAMBnKGQAAk6GcAQAwGcoZAACToZwBADAZyhkAAJOhnAEAMBnKGQAAkylW5bx06VKVKVNGFSpUsA49evSQJH377beKjIyUj4+P6tevr9WrV7s4LQAAf4xbcbp958svv6wLFy7oww8/tBmfnJyshg0b6rPPPlOnTp30xRdfqFevXkpOTlaNGjVclBYAgD+mWO05b9++XU2aNCk0/qOPPlKLFi301FNPqUyZMnruuefUqlUrffDBBy5ICQDAn1NsyrmgoECJiYn617/+pZCQEAUFBalfv35KS0tTUlKSIiMjbeYPDw/Xzp07b7m8nJwcZWZmWoeMjAylpqaqGB1IAACUUMWmnFNTU9WoUSM9++yz2rdvn3766SclJyere/fuunTpknx8fGzm9/b21uXLl2+5vEmTJsnPz886+Pv7q1q1arp06ZKzVwUAgNsqNuVcvXp1JSQkqE+fPvL29lZwcLAmT56s7777ToZhKCsry2b+rKws+fr63nJ5I0eOVEZGhnVISUlx9ioAAFAkxaacd+3apddff93msHNOTo7c3d3VrFkzJSUl2cy/d+9eRURE3HJ5np6eqlixos0AAIAZFJtyrly5smbPnq333ntPeXl5On78uF555RX16tVLPXv2VHx8vJYtW6a8vDwtW7ZM8fHx1q9ZAQBQnBSbcg4KCtK//vUvrVq1SpUrV1aTJk3UtGlTzZ49W/Xq1dOqVav0zjvvqFKlSpowYYJWrlypunXrujo2AAB2K1bfc3amzMxM+fn5KSMjg0PcAACXKuPqAAAA88nPz9e1a9dcHaNYKFeunNzdHXsgmnIGAFgZhqEzZ84oPT3d1VGKDXd3d4WGhqpcuXIOWyblDACwulHM1apVk7e3t9zc3FwdydQKCgp06tQpnT59WsHBwQ7bXpQzAEDS9UPZN4rZYrG4Ok6xUbVqVZ06dUp5eXkqW7asQ5ZZbK7WBgA4141zzN7e3i5OUrzcOJydn5/vsGVSzgAAGxzKto8zthflDABwiezsbJ04ccLVMUyJcgYAuESLFi30ww8/uDqGKVHOAACXSE1NdXUE06KcAQB33COPPKLjx49rwIABateunfr162czvVOnThozZozi4+MVFBSkMWPGyGKxyGKxaOjQocrNzZV0/XvZs2bN0j333CN/f3+1aNFCv/zyiytWyaEoZwDAHbd27VoFBwdr3rx56tevn5YvX66cnBxJ0tmzZ7V27Vr16tVLknTy5En9/vvvOnr0qLZs2aLVq1frrbfekiTNnTtXU6dO1fLly5WamqrevXurXbt2Onv2rKtWzSEoZwCASz355JNyd3fXN998I0lasmSJHnzwQYWFhUm6fjX03Llz5evrqzp16ujVV1/VJ598IkmaM2eORo0apYYNG6ps2bLq06eP6tevr8WLF7tsfRyBm5Cg1NmzZ4+rI9z2WeNAaePp6amuXbvqk08+0bPPPquPPvpII0aMsE6vXLmyzU1RgoODdfr0aUnS0aNHNWLECL322mvW6deuXVOTJk3u3Ao4AeUMAHC5Pn36KCoqSj///LOOHDmiZ5991jotPT1dWVlZ1pujHDlyRMHBwZKuP054woQJ6tKli3X+Q4cOFfs7nHFYGwDgEl5eXsrIyJAkNWrUSA0aNFBcXJxiYmJs7lKWn5+vl19+WdnZ2fr999/13nvv6fnnn5ck9evXT2+//bb2798vSfr+++/VoEEDJSQk3PkVciD2nAEALtG3b1+NGjVK27dv1+LFi9W7d28NGTJEc+bMKTRvpUqVFBoaKkkaMGCAXnnlFUnSsGHDZBiGnnjiCZ0+fVo1atTQ7Nmz9cQTT9zRdXE0N8MwDFeHMIPMzEz5+fkpIyNDFStWdHUcOBHnnIGby87O1pEjRxQaGiovL687/vlff/21XnvtNe3bt886Lj4+Xm3atJGZq8oZ2409ZwCAS124cEEpKSl6++239eKLL7o6jilwzhkA4FI7duxQ8+bNFRgYqAEDBrg6jimw5wwAcKlHH31UWVlZN53WunVrUx/Sdhb2nAEAMBnKGQAAk6GcAQAwGcoZAACToZwBADAZyhkAAJPhq1S4o6r1LXxbvjttw7BWro4AFCt3+t/tuQVxd/TzzIg9ZwBAsZeWlqaBAweqZs2a8vHxUWBgoHr27KkTJ05Iuv596Zo1a+rChQs27zt69Kjc3Nx09OhRvfPOOwoICNC1a9cKLf/AgQNyd3fXb7/9dkfWh3IGABR7MTExOn/+vLZv364rV67ot99+U05Ojh5++GHl5eVJkk6cOKHY2Nhb3tSkX79+ysjI0Ndff11o2rx589SiRQv95S9/cep63EA5AwCKvU2bNunpp59WQECAJKl69eqaMWOGoqKilJaWJkmKjY3V5s2b9d577910GVWqVFGXLl00f/58m/FXr17VokWLNGTIEOeuxH/gnDMAoNj729/+pgEDBujHH39U69atFRUVpZCQEC1atMg6T2hoqBYsWKAuXbrooYceUvPmzQstZ9CgQWratKmOHTumkJAQSdLnn38uX19fPfnkk3dqddhzBgAUf/Pnz9ecOXN0/Phx9evXT7Vq1VLt2rW1ZMkSm/mio6PVv39/denSRRcvXiy0nPvuu08PPPCAFixYYB33/vvv66WXXpKHh4fT1+MGyhkAUOy5u7ure/fuWr16tdLS0pSUlKTOnTurR48eWr9+vc28U6dOVdWqVdWzZ8+bnn8eNGiQFi5cqPz8fP3yyy9KSkrS888/f6dWRRLlDAAo5r7//ntVqFDBuifs5uam8PBwTZo0SY0aNdKvv/5qM7+np6eWLl2qhIQETZ06tdDyoqOjJUlr1qzRvHnz1KNHD1WqVMn5K/IfKGcAQLHWsmVLVa9eXb1799bu3bt17do1Xbp0SUuWLFFycrI6duxY6D21a9fWBx98oDlzCn+Hu0yZMhowYIDmz5+vFStWaPDgwXdiNWxQzgCAYq18+fLatGmTAgIC9Pjjj8vPz081a9bU4sWLtW7dOtWvX/+m74uJiVH//v1vOq1fv35as2aNmjRpovDwcGfGvyk3ozQ+xfomMjMz5efnp4yMDFWsWNHVcUos7hB2XUREhKsjAIVkZ2fryJEjCg0NlZeXl6vjFBvO2G7sOQMAYDKUMwAAJkM5AwBgMpQzAAAmQzkDAGAylDMAACZDOQMAYDKUMwAAJkM5AwBgMjzPGQBwW3v27Lmjn8cd9NhzBgDAdChnAABMhnIGABR733zzjZo3b65q1arJ29tbrVq1UnJysiTp888/1z333CN/f3+1b99e/fr1U69evSRJhmFo1qxZ1uktWrTQL7/84sI1uY5yBgAUaydOnFDnzp01cuRInTt3TikpKTIMQxMmTNDPP/+snj17asqUKTp//rz69++vDz/80PreuXPnaurUqVq+fLlSU1PVu3dvtWvXTmfPnnXhGlHOAIBirlq1akpKStLjjz+uS5cuKSUlRVWqVNHJkye1cOFCPfPMM3r88cdVpkwZPf3003r66aet750zZ45GjRqlhg0bqmzZsurTp4/q16+vxYsXu3CNKGcAQDFXtmxZffbZZwoKClJ4eLhGjRqlc+fOqaCgQCkpKapVq5bN/GFhYdb/Pnr0qEaMGCF/f3/rsHPnTh07duwOr4UtvkoFACjWli1bpn/84x/avHmzateuLUkaNGiQdu/erZCQkEJFe+zYMXl6ekqSgoKCNGHCBHXp0sU6/dChQ7JYLHduBW6CPWcAQLGWkZEhDw8PlS9fXoZhaM2aNfr444+Vm5urF154QV9++aW+//575efn67vvvtMXX3xhfW+/fv309ttva//+/ZKk77//Xg0aNFBCQoKrVkcSe84AgP/B7DcF6dmzpzZt2qQGDRqoTJkyqlevnoYOHarZs2erYcOGmjdvnl588UWdP39eLVq00F//+leVK1dOkjRs2DAZhqEnnnhCp0+fVo0aNTR79mw98cQTLl0nyhkAUKx5enrq448/LjR+/PjxOnDggKKionT48GHr+OjoaFWtWlWS5OHhoVdeeUWvvPLKHctbFBzWBgCUWElJSWrbtq0OHTokSYqPj9eaNWvUsWNHFye7PfacAQAl1tNPP629e/eqTZs2unjxomrVqqX58+erefPmro52W5QzAKBEe+ONN/TGG2+4OoZdiuVh7fz8fLVu3dp6+zVJ+vbbbxUZGSkfHx/Vr19fq1evdl1AAAD+hGJZzuPHj9ePP/5ofZ2cnKzo6Gi99dZbysjI0Pjx4/Xcc8/p5MmTLkwJAMWTYRiujlCsOGN7Fbty3rBhg1auXKno6GjruI8++kgtWrTQU089pTJlyui5555Tq1at9MEHH7gwKQAUL2XLlpUkZWVluThJ8ZKbmyvp+pXfjlKszjmfO3dOffv21apVqzR9+nTr+KSkJEVGRtrMGx4erp07d95yWTk5OcrJybG+zszMdHxgAChGPDw85O/vr3PnzkmSvL295ebm5uJU5lZQUKDU1FR5e3urTBnHVWqxKeeCggJ1795dw4cP17333msz7dKlS/Lx8bEZ5+3trcuXL99yeZMmTdL48eOdkhUAiquAgABJshY0/jd3d3cFBwc79A+ZYlPOkyZNkpeXlwYNGlRomo+PT6HDMFlZWfL19b3l8kaOHKnhw4dbX2dmZqpmzZqOCwwAxZCbm5sCAwNVrVo1Xbt2zdVxioVy5crJ3d2xZ4mLTTl/8sknOnXqlPz9/SX93zmRVatWKS4uTomJiTbz7927V02aNLnl8jw9Pa03PgcA2PLw8HDoOVTYp9hcELZ//35lZmYqPT1d6enp6tq1q7p27ar09HT16NFD8fHxWrZsmfLy8rRs2TLFx8erR48ero4NAIDdis2e8+3Uq1dPq1at0muvvaa+ffsqJCREK1euVN26dV0dDQAAuxXbcl60aJHN60cffVSPPvqoa8IAAOBAxeawNgAApQXlDACAyVDOAACYDOUMAIDJUM4AAJgM5QwAgMlQzgAAmAzlDACAyVDOAACYDOUMAIDJUM4AAJgM5QwAgMlQzgAAmAzlDACAyVDOAACYDOUMAIDJUM4AAJgM5QwAgMlQzgAAmAzlDACAyVDOAACYDOUMAIDJUM4AAJgM5QwAgMlQzgAAmAzlDACAyVDOAACYDOUMAIDJUM4AAJgM5QwAgMlQzgAAmAzlDACAyVDOAACYjF3lfPnyZc2YMUOStHfvXkVFRaljx446efKkU8IBAFAa2VXOL730khYtWiRJevHFFxUSEiKLxaKBAwc6IxsAAKVSGXtm/ve//61ffvlFaWlp2rx5s44dOyaLxaLAwEBn5QMAoNSxa885MzNTlStX1vr163X33XerRo0acnNzk5ubm7PyAQBQ6ti15xwREaG3335b3333nTp16qRLly7pjTfeUOPGjZ2VDwCAUseuPef3339fP/zwgypWrKhx48YpMTFR8fHxmjNnjrPyAQBQ6hRpz3nXrl1q2LChwsPDFR8fbx3fsmVL7dq1y1nZAAAolYq059yoUSO1adNGX331lQzDsI7nXDMAAI5XpHLet2+fGjVqpD59+igsLEzTpk1TRkaGs7MBAFAqFamc69atq2nTpunkyZMaM2aMli5dqqCgIA0cOFD79+93dkYAAEoVuy4I8/LyUu/evbV161bFx8crLy9PDz74oNq3b++sfAAAlDp2fZXqP91333166qmnlJqaqjVr1jgyEwAApZrdD75ISUnRuHHjFBISogEDBqhp06Y6fvy4M7IBAFAqFWnPOT8/X19//bXmz5+vdevW6f7779eUKVMUHR0tDw8PZ2cEAKBUKVI5BwUFKTMzUzExMdq2bZsaNWrk7FwAAJRaRSrnwYMHq1+/frJYLM7OAwBAqVekcu7WrZuuXLmiK1eu3HKe4OBgh4UCAKA0K1I516pVq9DdwAzDsI5zc3NTXl6e49MBAFAKFamcjxw5UmicYRiaOnWq5syZo379+jk8GIBbq9bXtQ+bObcgzqWfD5R0RSrnkJAQm9fp6enq2bOnEhIStHTpUnXu3Nkp4QAAKI3svgnJli1bFBMTo2rVqmnHjh26++67nZELAIBSy66bkLz33ntq2bKlnnjiCW3evJliBgDACYq055yWlqbY2Fht3rxZS5cu1dNPP+3sXAAAlFpFKud7771XJ0+eVGxsrHbv3q3du3cXmmfMmDEODwcAQGlUpHIOCwvT3XffraNHj+ro0aOFpru5uVHOAAA4SJHKOT4+3skxAADADUW+IKygoEAXLlywvt64caOmTZumAwcOOCXYzWzYsEFRUVGqWLGiAgICNGjQIF29elWStHXrVkVFRalChQoKDQ3VggUL7lguAAAcqUjlfPLkSUVGRuqVV16RJH366adq166dPv30UzVr1kzbt293akhJSk1NVceOHfXiiy8qPT1dv/76q+Lj4/Xuu+8qLS1NHTp0UGxsrNLT07VgwQINGzZM27Ztc3ouAAAcrUjl/MYbb6hhw4Z69913JUljx47V66+/rh07dmjOnDkaN26cMzNKkqpWrapz586pV69ecnNz04ULF5Sdna2qVatq5cqVslgsiouLU5kyZdS2bVt169ZNc+a49i5KAAD8EUUq53Xr1mnWrFmqVq2ajh8/rkOHDqlHjx6SpCeffFI///yzU0Pe4OvrK0mqWbOmIiMjFRgYqN69eyspKUmRkZE284aHh2vnzp13JBcAAI5UpHLOzMxU1apVJV0/t+vv76969epJkry8vJSbm+u8hDeRnJyskydPysPDQ88++6wuXbokHx8fm3m8vb11+fLlWy4jJydHmZmZNgMAAGZQpHKuVKmSUlNTJV2/cvuhhx6yTtu/f7+1uO+U8uXL66677tLf//53rVmzRj4+PsrKyrKZJysry7qnfTOTJk2Sn5+fdahZs6azYwMAUCRFKufHH39cgwYN0tKlS7VkyRJ16dJF0vUHYIwePVrt27d3akhJ+umnn1SvXj2bvfScnByVK1dO4eHhSkpKspl/7969ioiIuOXyRo4cqYyMDOuQkpLitOwAANijSN9znjhxop577jn16dNHf/vb39S1a1dJ18/9BgQEaN68eU4NKUkNGzZUVlaWXn/9db377rs6ffq0Xn75ZfXt21fPPvusXn/9dc2YMUNxcXHatGmTlixZoq+++uqWy/P09JSnp6fTcwMl0Z49e1wd4bZ/fAPFXZHK2d/fX2vXri00fuXKlWrZsqW8vLwcHuy/VahQQWvWrNHQoUNVvXp1+fn5qXv37ho9erQ8PT21bt06DRkyRGPGjFHVqlU1a9YstWnTxum5AABwNLsfGfmfHnnkEUflKJLw8PCb/pEgSU2aNNHmzZvvaB4AAJzBrkdGAgAA56OcAQAwGcoZAACTsbucf/nlF0nXv0b12muvacqUKcrLy3N4MAAASiu7LgibOHGiJk+erIyMDA0ePFg7duyQu7u7Tpw4oRkzZjgrIwAApYpd5fzpp58qISFBOTk5WrFihbZs2aKAgAA1bNiQcgYAwEHsKudTp07p3nvv1YYNG+Tn56eGDRtKUqFbZwIAgD/OrnPONWrUUEJCgj766CO1a9dOkvTZZ58pLCzMKeEAACiN7NpzHj9+vNq3by9vb29t3rxZGzZsUO/evbVy5Upn5QMAoNSxq5yjo6PVsWNHSdcfFXnXXXfpyJEjCgwMdEo4AABKI7vKOSEh4abjk5OT1bJlS4cEAgCgtLOrnFu3bl1onLu7u4KDg3X48GFHZQIAoFSzq5wLCgpsXp8/f17jx49XrVq1HJkJAIBS7U/dvrNKlSqaPHky33EGAMCB/vS9tdPS0pSdne2ILAAAQHYe1u7Tp4/N69zcXP3444/W7zwDAIA/z65yNgzD5rWXl5cGDx6s/v37OzQUAAClmV3l/OGHHzorBwAA+P/sOud89uxZDRs2TJK0adMmVa9eXREREdq7d69TwgEAUBrZtec8cOBAXblyRYZhaPDgwYqJiZGPj48GDRqk9evXOysjAAClil3lvH37du3bt09nzpzRzp07tW7dOvn5+clisTgrHwAApY5dh7WzsrJUvnx5rV+/XpGRkbJYLLp69arKli3rrHwAAJQ6du05N2vWTC+++KI2bdqkmJgYnT17VnFxcWrVqpWz8gEAUOrYtee8YMEC5eTkqGXLlho5cqSOHj2q3NxczZ0711n5AAAodezacw4MDNSiRYusr6OiovT11187OhMAAKWaXeV8+fJlzZ07VwcOHCj0EIyFCxc6NBgAAKWVXYe1e/XqpZkzZyo7O1uGYdgMAADAMezac46Pj9e2bdsUFhbmrDwAAJR6du05e3l5KSgoyFlZAACA7CzngQMHavjw4UpNTXVWHgAASj27DmvPnz9fx44d0/vvv19oWn5+vsNCAQBQmtlVzv/5NSoAAOAcdpXzjTuBpaWl6fDhw2rUqJHy8vJUrlw5p4QDAKA0suuc8+XLl9W1a1dZLBa1bNlSycnJuvvuu/X77787Kx8AAKWOXeX8yiuv6MqVK9q/f7/KlSunsLAwPf744xoyZIiz8gEAUOrYdVj7m2++0e7du1WpUiW5ubmpbNmymjp1qmrUqOGsfAAAlDp27Tnn5+fL09NTkqx3BSsoKLCOAwAAf55d5fzXv/5VcXFxysrKkpubmyTpzTffVOvWrZ2RDQCAUsmuw9rTpk3TE088oUqVKikvL0++vr6qU6eOVq9e7ax8AACUOnaVc7Vq1fTzzz9rx44dOnr0qIKCgtSsWTN5eHg4Kx8AAKWOXYe1JSkxMVFNmzbVww8/rFWrVmn69OnKy8tzRjYAAEolu/acJ06cqMmTJysjI0ODBw/Wjh075O7urhMnTmjGjBnOyggAQKliVzl/+umnSkhIUE5OjlasWKEtW7YoICBADRs2pJwBAHAQu8r51KlTuvfee7Vhwwb5+fmpYcOGkqSsrCynhAMAoDSy65xzjRo1lJCQoI8++kjt2rWTJH322WcKCwtzSjgAAEoju/acx48fr/bt28vb21ubN2/Whg0b1Lt3b61cudJZ+QAAKHXsKufo6Gh17NhRkuTl5aW77rpLR44cUWBgoFPCAQBQGtlVztL1x0UeOnRIBQUF1nHJyclq2bKlQ4MBAFBa2VXO//jHPzR8+HDl5+fbjHdzcys0DgAA/DF2XRA2Y8YMzZkzR7m5uSooKLAOFDMAAI5j155zamqqnn/+ebm7231jMQAAUER2tWzr1q0VHx/vpCgAAECyc8+5Ro0a6tixo9q0aaOAgACbaQsXLnRoMAAASiu7yjk7O1tdunSRJBmG4ZRAAACUdnaV84cffuisHAAA4P+z+8qumTNnKjw8XN7e3rr77rv1zjvvsBcNAIAD2bXnPHPmTE2dOlUjR45UaGioDh48qMmTJ8vDw0OvvfaaszICAFCq2FXO8+bN01dffaVGjRpZxz344IOKjo6mnAEAcBC7DmvfeGTkf7r33nt14cIFh4YCAKA0s6uc69Spoy+//NJm3Jdffqk6deo4NBQAAKWZXeX85ptvqmvXrurSpYtGjRqlmJgYdevWTePGjXNSPFs7d+7Uww8/rMqVKysgIECxsbE6f/68JGnr1q2KiopShQoVFBoaqgULFtyRTAAAOJpd5fzUU09pzZo18vT0VGJioipVqqSEhAR16tTJWfmsrl69qscee0zNmzfXmTNnlJSUpAsXLqh3795KS0tThw4dFBsbq/T0dC1YsEDDhg3Ttm3bnJ4LAABHs/uRkXl5eZo2bZosFou+/fZbZWZmOiNXIcePH9e9996rMWPGyMPDQxaLRf3791ePHj20cuVKWSwWxcXFSZLatm2rbt26ac6cOWrWrNkdyQcA9tqzZ4+rIygiIsLVEXATdu05z5kzR507d9bp06clSefOndNzzz2njz/+2Cnh/tM999yj7777Th4eHtZxK1asUOPGjZWUlKTIyEib+cPDw7Vz585bLi8nJ0eZmZk2AwAAZmBXOU+bNk0bNmyw/qXVq1cvrVu3ThMnTnRKuFsxDENvvvmmvvnmG82cOVOXLl2Sj4+PzTze3t66fPnyLZcxadIk+fn5WYeaNWs6OzYAAEViVzmfO3dOf/nLX2zGNWrUSGfPnnVoqNvJzMzUs88+q8WLFyshIUGRkZHy8fFRVlaWzXxZWVny9fW95XJGjhypjIwM65CSkuLs6AAAFIld5RweHq5PPvnEZtynn36qevXqOTTUrRw6dEhNmzZVZmamduzYYT2UHRERoaSkJJt59+7de9tzKZ6enqpYsaLNAACAGdh1Qdjbb7+tJ554QvPnz1dISIiOHz+uxMREffvtt87KZ5WWlqa2bduqbdu2WrBggdzd/+/vimeeeUavvvqqZsyYobi4OG3atElLlizRV1995fRcAIqnan3nuDqCNgxr5eoIMCm79pwffvhh7dq1S4888ogqVqyoxx57TLt371arVs7/Afvwww91/PhxLVu2TBUrVlSFChWsg8Vi0bp167R8+XJZLBY9//zzmjVrltq0aeP0XAAAOJrdX6WqU6eOxowZ44wstzV8+HANHz78ltObNGmizZs338FEAAA4h92PjAQAAM5FOQMAYDJFKuc7ccEXAAC4rkjl3K1bN0ni6VMAANwBRbogrFy5cho6dKiOHz+uCRMm3HQeV1wkBgBASVSkcp49e7bmz5+vgoICbdy4sdB0Nzc3yhkAAAcpUjl37txZnTt3VlRU1E3LGQAAOI5d33PeunWrLl++rG+//VZHjx7VXXfdpU6dOsnf399P3Cp5AAAZbklEQVRZ+QAAKHXsKueDBw/qr3/9q65du6bg4GAdO3ZMI0aM0IYNG9SgQQNnZQQAoFSx63vOw4cP13PPPacTJ05oy5YtOnnypHr06KERI0Y4Kx8AAKWOXXvOW7Zs0fLly60PnXB3d9dbb72lwMBAp4QDAKA0smvPuUyZMsrMzLQZl5mZKR8fH4eGAgCgNLOrnDt16qSuXbvq999/V25urvbt26fu3burU6dOzsoHAECpY1c5v/vuu7p27Zrq16+v8uXLKyIiQl5eXvr73//urHwAAJQ6dp1zrly5suLj43XkyBGdPXtWtWrVUkBAgLOyAQBQKtn9PGdJCg0NVWhoqKOzAAAA8chIAABMh3IGAMBkKGcAAEzmD51zlqS1a9dq9+7dql27tp588klHZgIAoFT7Q3vO48aN08svv6zExESNGjVKAwcOdHQuAABKrSLtOV+8eFGVK1e2vv7mm2+UmJioMmXKKCsrSzVr1tTcuXOdFhIAgNKkSHvOLVq00MKFC62vq1WrpmnTpumHH37QlClTFBIS4rSAAACUNkUq559++kmJiYlq0aKFdu3apQ8++EA7duzQ4MGDlZiYqCVLljg7JwAApUaRDmv7+flp9uzZSkxM1MCBA9W0aVMtXLhQFSpUcHY+AABKHbsuCLvvvvu0adMm1a9fX82bN9eyZcuclQsAgFKrSOW8atUqBQYGqmbNmlq7dq369eunjRs3at26dXr00Ud18OBBZ+cEAKDUKNJh7WHDhumDDz5QQUGBhgwZon379slisWj+/PnasmWLYmNj9dNPPzk7KwAApUKRyjkjI0PNmjVTQUGB0tPTbabdf//92rRpk1PCAQBQGhWpnJ9//nnVrl1b7u7uGj58eKHp7u7cBRQAAEcpUjlPnjxZffr0kYeHh+rUqePsTAAAlGpFvrd2vXr1nJkDAAD8fxyPBgDAZChnAABMhnIGAMBkKGcAAEyGcgYAwGQoZwAATIZyBgDAZChnAABMhnIGAMBkKGcAAEyGcgYAwGQoZwAATIZyBgDAZChnAABMhnIGAMBkKGcAAEyGcgYAwGQoZwAATIZyBgDAZChnAABMhnIGAMBkKGcAAEyGcgYAwGQoZwAATIZyBgDAZChnAABMhnIGAMBkKGcAAEymWJZzamqqateurfj4eOu4rVu3KioqShUqVFBoaKgWLFjguoAAAPwJxa6cN2/erAceeECHDh2yjktLS1OHDh0UGxur9PR0LViwQMOGDdO2bdtcmBQAgD+mWJXzRx99pK5du2rixIk241euXCmLxaK4uDiVKVNGbdu2Vbdu3TRnzhwXJQUA4I8rVuX86KOP6tChQ4qJibEZn5SUpMjISJtx4eHh2rlz5y2XlZOTo8zMTJsBAAAzKFblHBAQoDJlyhQaf+nSJfn4+NiM8/b21uXLl2+5rEmTJsnPz8861KxZ0+F5AQD4I4pVOd+Kj4+PsrKybMZlZWXJ19f3lu8ZOXKkMjIyrENKSoqzYwIAUCSFd0OLoYiICK1du9Zm3N69exUREXHL93h6esrT09PZ0QAAsFuJ2HN+5plndObMGc2YMUPXrl3Txo0btWTJEvXp08fV0QAAsFuJKGeLxaJ169Zp+fLlslgsev755zVr1iy1adPG1dEAALBbsT2sbRiGzesmTZpo8+bNLkoDAIDjlIg9ZwAAShLKGQAAk6GcAQAwGcoZAACToZwBADAZyhkAAJOhnAEAMBnKGQAAk6GcAQAwGcoZAACToZwBADAZyhkAAJOhnAEAMBnKGQAAk6GcAQAwGcoZAACToZwBADAZyhkAAJOhnAEAMBnKGQAAk6GcAQAwGcoZAACToZwBADAZyhkAAJOhnAEAMBnKGQAAk6GcAQAwGcoZAACToZwBADAZyhkAAJOhnAEAMBnKGQAAk6GcAQAwGcoZAACToZwBADAZyhkAAJOhnAEAMBnKGQAAk6GcAQAwGcoZAACToZwBADAZyhkAAJOhnAEAMBnKGQAAk6GcAQAwGcoZAACToZwBADCZMq4OAAAovqr1nePqCDq3IM7VERyOPWcAAEyGcgYAwGQoZwAATIZyBgDAZLggDABQrO3Zs8elnx8REeHwZbLnDACAyVDOAACYDOUMAIDJUM4AAJgM5QwAgMlwtXYx4uorEiXnXJUIALBFOdvB1feQ3TCslUs/HwBwZ5Sow9rnzp3TU089JX9/f1WpUkVDhw5VXl6eq2MBAGCXElXOMTExqlChgk6dOqVt27bphx9+0PTp010dCwAAu5SYcj548KDi4+M1efJkeXt7KywsTKNHj9bs2bNdHQ0AALuUmHPOSUlJqly5su666y7ruPDwcB0/flzp6eny9/e3mT8nJ0c5OTnW1xkZGZKkzMzMW35GQe5VB6e2z+XLl136+dLtt09RuHobSmxHR2AbOgbb0TFcvR2Lsg19fX3l5uZW9IUaJcQnn3xi1KxZ02bcwYMHDUlGSkpKofnHjh1rSGJgYGBgYHD6kJGRYVenlZg9Zx8fH2VlZdmMu/Ha19e30PwjR47U8OHDra8LCgp08eJFWSwW+/66uUMyMzNVs2ZNpaSkqGLFiq6OU2yxHf88tqFjsB0do7hsx5v10O2UmHKOiIjQhQsXdPbsWVWvXl2StHfvXgUFBcnPz6/Q/J6envL09LQZ99+Hvs2oYsWKpv4BLC7Yjn8e29Ax2I6OUdK2Y4m5IKxOnTp66KGHNHToUF26dElHjhzRW2+9pb59+7o6GgAAdikx5SxJK1asUF5enkJDQxUVFaX27dtr9OjRro4FAIBdSsxhbUmqXr26li9f7uoYTuHp6amxY8cWOhQP+7Ad/zy2oWOwHR2jpG5HN8MwDFeHAAAA/6dEHdYGAKAkoJwBADAZyhkAYDo37tpYWlHOLvRHn6I1ZMgQ9erVy2bcqVOn1LlzZ1ksFgUGBmr48OHKzs52UnLX2rlzpx5++GFVrlxZAQEBio2N1fnz5yVJW7duVVRUlCpUqKDQ0FAtWLCgyMvt0aOHWrdufdNpp0+fVvXq1bVo0SIHrIF55Ofnq3Xr1jY/T4899pi8vLxUoUIF67BmzZpbLuPYsWPWn2OLxaKePXve9HaKe/bskbe3t+Lj452wJq5x8eJFxcbGymKxqFKlSnrqqad0+vRpSdKLL74oT09Pm+34wQcf3HJZiYmJatmypfz9/RUYGKghQ4bY3GJ40aJFqlevnnx9ffXAAw8oISHB6evnSv/85z8VGRmpefPm6cqVK4Wm/6/tMXnyZAUFBcnHx0etW7fW77//fqeiO8afuGMm/qTWrVsb3bp1M65cuWIcOnTIaNCggTF58uRbzn/+/HmjW7duhiSjZ8+e1vH5+flGkyZNjE6dOhnnz583UlNTjbZt2xq9evW6A2txZ2VlZRmBgYHGmDFjjJycHOP8+fNGhw4djE6dOhkXL140KleubMyePdu4du2asX79esPX19fYunXr/1zuggULDHd3d6NVq1aFpuXn5xtt2rQx3N3djQ8//NDxK+VCo0ePNtzd3W1+nqpUqWLEx8cX6f05OTlG3bp1jbi4OOPKlSvGuXPnjObNmxtxcXE28125csWIiIgwJBkbN2504Bq4VuvWrY2nn37aSEtLMzIzM41nnnnG6Nixo2EYhtG4cWNj0aJFRVpOfn6+ERgYaMycOdPIz883UlJSjHvuuceYMGGCYRiG8dVXXxmenp7GN998Y+Tl5RkrV640vL29jf379ztt3VwtKyvLWLhwodG0aVPDz8/PGDJkiHHgwAHDMP739li0aJFRo0YNY8+ePcbVq1eN4cOHGw0aNDAKCgpcuUp2oZxdJDk52ZBknDx50jru888/N4KDg286/6VLlwyLxWLExcUZ0dHRNr9M9+3bZ0gyjh07Zh23ZcsWo2zZskZ6errT1sEV9u/fb7Rv397Iy8uzjvvqq6+MihUrGvPnzzfq1KljM/+AAQOM2NjY2y4zKSnJqFWrltG/f/+blvPYsWONHj16GCEhISWqnNevX2+Eh4cbnTt3tv48HT582HB3dzcyMzOLtIwVK1YYwcHBNv8/Tp06Zf0lekPPnj2N0aNHl6hy3rFjh+Hl5WVzz+QLFy4Ye/bsMbKzs41y5coZe/bsKdKyzp8/b0gypk+fbuTl5RkpKSlG/fr1jSlTphiGYRgxMTFG7969bd7Tvn1749VXX3XcCpnYjh07jD59+hg+Pj5G+/btjaeeeuq22+PBBx80Jk6caJ2Wm5tr+Pr6Ghs2bLijuf8MDmu7yP96itZ/8/LyUlJSkmbPnq0KFSrYTMvPz5d0/f7iN7i7u+vatWs6fPiwk9bANe655x5999138vDwsI5bsWKFGjdurKSkJEVGRtrMHx4erp07d95yeVevXlVMTIzmzp2rgICAQtM3btyozz//XHPnznXcSpjAuXPn1LdvX3366afy9va2jt++fbt8fX0VExOjqlWrKiIiQgsXLrzlcrZt26a//OUvGj16tIKDgxUcHKz33ntPQUFB1nk+/vhjHTx4UGPHjnXqOt1p27ZtU3h4uObPn6/atWsrMDBQI0aMUGBgoHbu3Klr165pzJgxql69uurWrau///3vKigouOmyLBaLhg0bphEjRsjT01M1a9ZU3bp1NWzYMEnX/43/579v6fq/8f379zt9Pc0gMjJS7dq1U926dbVjxw6lp6ffdnv89++CsmXLqk6dOrf9XWA2lLOLXLp0qdAP141fkjc7X1emTBnrPcP/W7169dSgQQMNGzZM6enpSk1N1bhx4yRdL5+SyjAMvfnmm/rmm280c+bMW27T2z1O7qWXXtIjjzyixx57rNC0c+fOqXfv3lqyZEmhP4iKs4KCAnXv3l3Dhw/XvffeazMtJydHDzzwgCZOnKhTp05p2rRpGjJkyC1v7nPx4kV9++23Kl++vA4cOKD4+Hht3LhRr7zyiiRp//79euONN/Tpp5/a/EFVEly8eFG7du1ScnKyfv31V/322286efKkYmNjlZGRodatW2vw4ME6ceKEFi9erFmzZmnq1Kk3XVZBQYHKly+v2bNn68qVK9qzZ4/27t1r/YMmOjpaH3/8sf79738rLy9PX331ldavX1+i/31LUkpKit58803VrFlTkydP1ksvvaSUlBT179//ttvjj/wuMJsSdYew4uR2T9Hq0aOHtm/fbh3/v36gPDw89PXXX2vIkCGqU6eOAgICNGLECH377beqVKmS48ObQGZmpnr37q1ffvlFCQkJioyMlI+PT6GjDllZWdanwfxnwbZo0ULdu3fXzp079dNPPxVavmEY6tGjhwYPHqzGjRs7d2XusEmTJsnLy0uDBg0qNK1Hjx7q0aOH9fUjjzyi2NhYLV26VAEBATZ/xIwaNUqenp4KCAiw3iY3LCxMo0aNUlxcnKZMmaKYmBjNmDFDwcHBzl+xO+zGHalmzJghLy8v+fr6auLEiYqKitLnn3+uDRs2WOdt1qyZhg4dqqVLlyomJkbh4eHWad27d9fDDz+slStXWvf8GjRooLFjx2rw4MF666231KVLF6WmpuqFF15QWlqaOnTooL/97W+FfoeUJIsXL9YLL7ygxx9/XCtWrFCLFi2s0/7X9rjV71d7nwzlSpSzi9zuKVobN260a1mGYSgtLU3Lli1T+fLlJUnfffedfH19VadOHYdnd7VDhw6pQ4cOCg4O1o4dO1SlShVJ17fp2rVrbebdu3evIiIiJBX+I+fRRx/V77//rmrVqkmSsrOzlZeXJ39/f+3atUv//ve/tXXrVk2YMEHS9T8IBg4cqBUrVmj16tXOXk2n+eSTT3Tq1CnrU9hu/BJbtWqVpk2bJl9fX3Xu3Nk6f05OjsqXL68WLVoU2oZz587V8uXLVVBQIHf36wfi8vPzZRiGtm/frgMHDqhv3742D6Dp1KmTYmNji/2pgvDwcBUUFCg3N1deXl6S/u8U05dffqmsrCz179/fOv+N7RgcHFxoO06fPt3mymzp+qHYcuXKSZLOnDmj9u3b2/xBdf/99ys6Otop62YGDRs2VHJyss0pkhv+1/aIiIhQUlKSOnXqJEm6du2akpOTrb8LigXXnvIu3R566CGjS5cuRmZmpnH48GGjQYMGxtixY//n+3r27GlzQZhhGEb9+vWNMWPGGPn5+caBAweMiIgIY9SoUc4J7kIXL140goODjV69ehn5+fk2086fP2/4+/sb06dPN3Jzc40NGzbYdRHI2LFjb3pB2A0l7YKwG/7z52natGlGtWrVjMTERCM/P99YvXq1Ub58eSMhIeGm701NTTUsFosxaNAgIzs72zhy5IgRGRlpDBs27KbzqwRdEJabm2vUrl3biI6ONi5dumScO3fOaNu2rfH0008bX3zxhVG+fHnjhx9+MAoKCoyffvrJqFKlivHJJ5/cdFlJSUmGp6enMXHiRCMvL884dOiQERkZabz88suGYVy/WPSuu+4yjh49aly9etWYPn264e/vb5w9e/ZOrvId9d577xmSbjosWrTottvjn//8p1GjRg3jt99+s16tXbt2bSM3N9fFa1V0lLMLnTlzxnj22WcNi8ViVK1a1RgxYoTNVa+3crNy3r17t9GiRQvD19fXqFGjhrWoS5qpU6cakgxvb2/Dx8fHZjAMw9i+fbvRvHlzw9fX1wgLC7OrTClnwygoKDDeeustIyQkxChfvrzRoEEDY/ny5bd9//79+43HHnvMqFKlilGlShVj6NChRnZ29k3nLUnlbBiGcfLkSSMmJsYICAgw/P39jdjYWCMtLc0wDMOYN2+eUbduXcPb29sICwsz5syZc9tlrVu3zoiKijL8/PyM4OBgY9SoUUZOTo51+rhx44zAwEDD19fXaNWqlZGYmOjUdXO1tLQ0Izk5+aZDXl7ebbdHQUGBMWXKFCM0NNSoUKGC0aZNG+P333934drYjwdfAABgMlytDQCAyVDOAACYDOUMAIDJUM4AAJgM5QwAgMlQzgAAmAzlDACAyVDOAACYDOUMAIDJUM4AAJgM5QwAgMlQzgAAmAzlDACAyVDOAACYDOUMAIDJUM4AAJgM5QwAgMlQzgAAmAzlDACAyVDOAACYDOUMAIDJUM4A/pSjR4/Kzc1NR48even0JUuWqEGDBnc2FFDMuRmGYbg6BIDi6+jRowoNDdWRI0dUq1YtV8cBSgT2nIESauHChWrcuLEsFot8fX3VqVMnpaamWqfPmjVLISEhslgs6tKli6KjozVu3DhJUm5ursaMGaOwsDBVrlxZHTp00MGDB2/7eR9//LHuvvtuVa9eXS+88IIyMzMlSYsWLbKWdnx8vGrVqqWJEyeqRo0aqly5sqKjo63znjp1So899pgqV66soKAgPfPMMzp9+rTjNw5gcpQzUAJt27ZNgwYN0vvvv68LFy5o3759Sk5O1qxZsyRJn3/+ucaNG6fPPvtMZ86cUcuWLfXFF19Y3//GG29o9erVWr9+vU6dOqX7779fjzzyiLKzs2/5mQkJCdq6dat27dql3bt3a9iwYTed79ixYzp58qQOHjyobdu26eeff9bcuXMlSSNHjlRQUJDOnj2rffv26fLly3r33XcduGWA4oFyBkqgyMhIJSUlqVmzZkpLS9OpU6dUtWpVnTx5UpK0YMEC9e/fX82bN1fZsmU1cOBANW3aVJJkGIbef/99TZo0SaGhofLy8tLo0aOVm5urf/3rX7f8zKlTp6pKlSqqXr26JkyYoCVLlqigoOCm844ZM0bly5dX7dq11aZNGx04cECSVL58eW3atEmff/65Ll26pDVr1mjmzJkO3jqA+VHOQAnk4eGhmTNnqlq1amrcuLHeeecdZWZmWssyJSWl0PnhsLAwSVJqaqquXLmizp07y9/fX/7+/qpUqZIuXrx4y4u+JCk0NNT638HBwcrJydGFCxduOm9AQID1v8uWLWvNNWvWLMXExOi9995TUFCQGjdurB9//PGPbAKgWKOcgRJo+vTpWrt2rXbv3q3Dhw9r1apVCgkJsU4PCQnRsWPHbN5z43WVKlXk5eWltWvXKj093Tr8+uuv6t+//y0/89SpU9b/Pnz4sHx8fFS1alW7cicmJqp///7atWuXzp49q4ceekjPPPOMXcsASgLKGSiBMjIyVLZsWZUrV055eXlavHix1qxZo9zcXElSv379NH/+fG3fvl15eXn68MMPtWXLFkmSu7u7+vbtq9dff10nTpxQQUGBPvroIzVo0EDJycm3/MxXX31VaWlpOnHihEaPHn3bIr+ViRMn6qWXXlJmZqYqVaokHx8fValS5Y9tBKAYK+PqAAAc7+WXX9bu3bsVEhIiLy8v3XfffYqLi9P69eslSdHR0Tp06JCefPJJZWdnq0OHDmrSpInKlSsnSZoyZYrGjRunFi1a6MKFCwoLC9PKlSvVqFGjW37mAw88oHvuuUfu7u7q2rWrJk6caHfuDz74QAMHDlRoaKhyc3PVpEkTLV++/I9tBKAY43vOQCm0c+dO+fv72xzqbty4sQYMGKAXXnjBhckASBzWBkqlDRs26PHHH9eZM2dkGIaWLl2qvXv3ql27dq6OBkAc1gZKpUGDBunYsWNq1KiRLl++rHr16unrr7+2ueIagOtwWBsAAJPhsDYAACZDOQMAYDKUMwAAJkM5AwBgMpQzAAAmQzkDAGAylDMAACZDOQMAYDKUMwAAJvP/APh7xZcz735iAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 1400x800 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAesAAAH0CAYAAAAdcwXvAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAAPYQAAD2EBqD+naQAAADl0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uIDMuMC4zLCBodHRwOi8vbWF0cGxvdGxpYi5vcmcvnQurowAAIABJREFUeJzt3XtwVPX5x/HPkkAkm2TDRaBpgkINSAQK9ZKhRdHaDoigUSwUECmiLVJpU7loFArICFNEkEoZoRZFTa2Q2IylKJpCCoPSKDCoIMolEBqEIJDdDYmBTc7vD8v+XCFKZLP7BN+vmR3N8ezuk++wvjlnby7HcRwBAACzmkV7AAAA8NWINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA46Ia6yNHjuiyyy5TUVFRvfusXr1aPXr0kNvtVrdu3bRq1arIDQgAgAFRi/XGjRvVp08f7dmzp959du3apSFDhmjWrFnyer2aOXOmhg4dqrKysghOCgBAdEUl1suXL9eIESP02GOPfe1+1157rbKyshQbG6uhQ4eqX79+Wrp0aYQmBQAg+qIS6/79+2vPnj0aNmzYV+63fft29ejRI2RbRkaGtm3bVu91ampq5PP5ghev16sjR46I7ysBADRVUYl1hw4dFBsb+7X7+f1+ud3ukG3x8fGqrKys9zpz5syRx+MJXpKTk9WuXTv5/f7znhsAgGgw/Wpwt9utqqqqkG1VVVVKTEys9zo5OTnyer3By4EDBxp7TAAAGtXXH95GUffu3bVly5aQbTt27NBVV11V73Xi4uIUFxfX2KMBABAxpo+sR40apaKiIq1YsUKBQEArVqxQUVGRRo0aFe3RAACIGHOxTkhIUG5uriTp8ssvV0FBgWbPnq1WrVrp0UcfVX5+vrp06RLlKQEAiByXc4G/TNrn88nj8cjr9SopKSna4wAA0GDmjqwBAEAoYg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwLiqxLi8vV1ZWlpKTk9W2bVtlZ2crEAicdd+FCxeqU6dOSkpKUs+ePZWfnx/haQEAiK6oxHrYsGFKSEjQwYMHVVxcrMLCQi1YsOCM/V577TXNnj1br7/+unw+n6ZPn66hQ4dq3759kR8aAIAoiXisd+/eraKiIs2dO1fx8fHq3Lmzpk2bpkWLFp2x74cffijHcVRXVyfHcRQTE6MWLVooNjY20mMDABA1Ea/e9u3b1bp1a6WkpAS3ZWRkqLS0VBUVFUpOTg5uHz58uJ599lllZGQoJiZGLpdLL774olJTU+u9/ZqaGtXU1AR/9vl8jfOLAAAQIRE/svb7/XK73SHb4uPjJUmVlZUh20+ePKlevXqpuLhYVVVVWrp0qcaOHav333+/3tufM2eOPB5P8JKWlhb+XwIAgAiKeKzdbreqqqpCtp3+OTExMWT7/fffryuuuEJXX321WrRooTFjxqhPnz567rnn6r39nJwceb3e4OXAgQNh/x0AAIikiMe6e/fuOnr0qA4fPhzctmPHDqWmpsrj8YTsW1paGnJKW5KaN2+uFi1a1Hv7cXFxSkpKCrkAANCURTzW6enp6tu3r7Kzs+X3+1VSUqJZs2Zp7NixZ+x7yy23aNGiRdqyZYvq6uqUl5endevWadiwYZEeGwCAqInKy6rz8vJ0//33q1OnTmrWrJnuuusuTZs2TZKUkJCgJUuWaOTIkZo+fbpiYmI0ZMgQHTt2TOnp6SooKFCvXr2iMTYAAFHhchzHifYQjcnn88nj8cjr9XJKHADQJPFxowAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMZFJdbl5eXKyspScnKy2rZtq+zsbAUCgbPu++9//1uZmZlKSEhQWlqa5syZE+FpAQCIrqjEetiwYUpISNDBgwdVXFyswsJCLViw4Iz9du7cqYEDB2r8+PHy+/365z//qSeeeEJ5eXlRmBoAgOhwOY7jRPIOd+/erfT0dJWVlSklJUWS9PLLL2vKlCnav39/yL4TJkzQsWPHlJubG9z28ccfKykpSR06dDin+/P5fPJ4PPJ6vUpKSgrfLwIAQIRE/Mh6+/btat26dTDUkpSRkaHS0lJVVFSE7FtcXKxLL71Uw4cPV9u2bdWtWzcVFRV9Zahramrk8/lCLgAANGURj7Xf75fb7Q7ZFh8fL0mqrKwM2X7s2DH98Y9/1J133qlDhw5pyZIlmjRp0leeBp8zZ448Hk/wkpaWFv5fAgCACIp4rN1ut6qqqkK2nf45MTExZHtcXJxuvfVW3XzzzYqNjdV1112nUaNGacWKFfXefk5Ojrxeb/By4MCB8P8SAABEUGyk77B79+46evSoDh8+rPbt20uSduzYodTUVHk8npB9MzIyVFNTE7KttrZWX/U0e1xcnOLi4sI/OAAAURLxI+v09HT17dtX2dnZ8vv9Kikp0axZszR27Ngz9h03bpwKCgr04osvynEcrV+/Xrm5uRo1alSkxwYAIGqi8tatvLw8BQIBderUSZmZmRowYICmTZsmSUpISAi++vvHP/6xXn31VS1cuFAej0djxozRvHnzdMstt0RjbAAAoiLib92KNN66BQBo6vi4UQAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYFxsQ3YuLS09p/06duz4jYYBAABnalCsL730UrlcLkmS4zjBfz/t9Lba2trwTQgAwLdcg06DL126VDfccIO2bt2q6upqffTRR7rllls0efJk7d27VyUlJdq7d29jzQoAwLeSy3Ec51x3Tk1N1bZt29SmTZvgNp/Pp549e2rfvn2NMd958/l88ng88nq9SkpKivY4AAA0WIOOrCsrK/Xltvt8Pp06dSqsQwEAgP/XoFhnZWUpKytLa9eu1a5du7RmzRrddtttGj16dGPNBwDAt16DToP7/X6NHz9eK1eu1MmTJ+V2uzV+/Hg99thjio1t0GvVIobT4ACApq5BsT7t5MmTqqioULt27RpjprAi1gCApq7BH4ry4YcfavLkyRo3bpyOHj2qRYsWNcZcAADgfxoU6zfffFOZmZn69NNPVVhYqKqqKj366KP6wx/+0FjzAQDwrdegWD/88MP629/+ptzcXMXExCgtLU2rV6/WkiVLGms+AAC+9Rr0nHVycrKOHz8ul8ul1q1b69ixY8HtFRUVjTbk+eA5awBAU9egI+tLLrlEb731Vsi2d999V2lpaWEdCgAA/L8GxTonJ0eDBw/WI488opMnT2ru3LnKysrS5MmTG2s+AAC+9Rr81q3Vq1frT3/6k/bt26fU1FT98pe/1JAhQxprvvPGaXAAQFPXoFhPmDBBjz32WJOKHrEGADR1DToNnpubq/j4+MaaBQAAnEWDjqwnTZqkyspKjR49WikpKSHfZ92xY8dGGfB8cWQNAGjqzinWq1ev1sCBA9Ws2dkPxF0ul2pra8M+XDgQawBAU3dO374xcuRIHT9+XM2aNdPevXvP+JpMAADQeM4p1i1atFB2drZiY2P13HPPnXWf3//+9+GcCwAA/M85xXrRokX685//rNraWq1bt+6M/+5yuYg1AACNpEEvMMvMzNR//vOfxpwn7HjOGgDQ1DXorVtNLdQAAFwIGvx91gAAILKINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxkUl1uXl5crKylJycrLatm2r7OxsBQKBr7zOBx98oPj4eBUVFUVmSAAAjIhKrIcNG6aEhAQdPHhQxcXFKiws1IIFC+rdv6qqSsOHD1d1dXUEpwQAwIaIx3r37t0qKirS3LlzFR8fr86dO2vatGlatGhRvdcZP368brvttghOCQCAHRGP9fbt29W6dWulpKQEt2VkZKi0tFQVFRVn7P/8889r9+7dmj59eiTHBADAjNhI36Hf75fb7Q7ZFh8fL0mqrKxUcnJycPvOnTv1yCOPaOPGjYqJiTmn26+pqVFNTU3wZ5/PF4apAQCInogfWbvdblVVVYVsO/1zYmJicNtnn32mYcOG6cknn1THjh3P+fbnzJkjj8cTvKSlpYVncAAAoiTise7evbuOHj2qw4cPB7ft2LFDqamp8ng8wW3vvPOOPv74Y40dO1bJycnBI+5BgwZp/Pjx9d5+Tk6OvF5v8HLgwIHG+2UAAIgAl+M4TqTv9Nprr1VqaqqWLl2qTz/9VIMHD9Ydd9yhGTNmfOX1XC6X1q1bp+uvv/6c78vn88nj8cjr9SopKen8BgcAIAqi8tatvLw8BQIBderUSZmZmRowYICmTZsmSUpISFBubm40xgIAwKSoHFlHEkfWAICmjo8bBQDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMC4qsS4vL1dWVpaSk5PVtm1bZWdnKxAInHXfp59+Wl27dlViYqK6dOmixYsXR3haAACiKyqxHjZsmBISEnTw4EEVFxersLBQCxYsOGO/goIC5eTkaPny5fL5fFq+fLkeeeQR5efnR2FqAACiw+U4jhPJO9y9e7fS09NVVlamlJQUSdLLL7+sKVOmaP/+/SH7Ll68WH6/Xw8++GBw2+233660tDQtXLjwnO7P5/PJ4/HI6/UqKSkpfL8IAAAREhvpO9y+fbtat24dDLUkZWRkqLS0VBUVFUpOTg5uHz9+fMh1y8vLtX79es2fP7/e26+pqVFNTU3wZ5/PF8bpAQCIvIifBvf7/XK73SHb4uPjJUmVlZX1Xu/QoUO66aabdOWVV2rEiBH17jdnzhx5PJ7gJS0tLTyDAwAQJRGPtdvtVlVVVci20z8nJiae9TqbNm3S1Vdfra5du+rVV19VbGz9JwRycnLk9XqDlwMHDoRveAAAoiDise7evbuOHj2qw4cPB7ft2LFDqamp8ng8Z+y/bNky3XjjjcrOztZf//pXxcXFfeXtx8XFKSkpKeQCAEBTFvFYp6enq2/fvsrOzpbf71dJSYlmzZqlsWPHnrFvfn6+7rvvPr3yyiuaOHFipEcFAMCEqLx1Ky8vT4FAQJ06dVJmZqYGDBigadOmSZISEhKUm5srSZo5c6YCgYCGDBmihISE4GXcuHHRGBsAgKiI+Fu3Io23bgEAmjo+bhQAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMA4Yg0AgHHEGgAA44g1AADGEWsAAIwj1gAAGEesAQAwjlgDAGAcsQYAwDhiDQCAccQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxhFrAACMI9YAABhHrAEAMI5YAwBgHLEGAMC4qMS6vLxcWVlZSk5OVtu2bZWdna1AIHDWfVevXq0ePXrI7XarW7duWrVqVYSnBQAguqIS62HDhikhIUEHDx5UcXGxCgsLtWDBgjP227Vrl4YMGaJZs2bJ6/Vq5syZGjp0qMrKyqIwNQAA0eFyHMeJ5B3u3r1b6enpKisrU0pKiiTp5Zdf1pQpU7R///6QfadOnari4mK98cYbwW033XSTrrnmGs2cOfOc7s/n88nj8cjr9SopKSl8vwgAABESG+k73L59u1q3bh0MtSRlZGSotLRUFRUVSk5ODtm3R48eIdfPyMjQtm3b6r39mpoa1dTUBH/2er2SPo82AACRkJiYKJfLFbbbi3is/X6/3G53yLb4+HhJUmVlZUis69u3srKy3tufM2fOWY+609LSzmdsAADOWXl5uS6++OKw3V7EY+12u1VVVRWy7fTPiYmJ57Tvl/f7opycHD3wwAPBnysqKnTJJZeotLRUHo/nfMf/1vL5fEpLS9OBAwd4OuE8sZbhw1qGD2sZHqfXsUWLFmG93YjHunv37jp69KgOHz6s9u3bS5J27Nih1NTUM2LavXt3bdmyJWTbjh07dNVVV9V7+3FxcYqLiztju8fj4Q9gGCQlJbGOYcJahg9rGT6sZXiE8xS4FIVXg6enp6tv377Kzs6W3+9XSUmJZs2apbFjx56x76hRo1RUVKQVK1YoEAhoxYoVKioq0qhRoyI9NgAAUROVt27l5eUpEAioU6dOyszM1IABAzRt2jRJUkJCgnJzcyVJl19+uQoKCjR79my1atVKjz76qPLz89WlS5dojA0AQFRE/DS4JLVv314rV64863/78ovH+vfvr/79+3/j+4qLi9P06dPPemoc5451DB/WMnxYy/BhLcOjsdYx4u+zBgAADcNngwMAYByxBgDAOGINAIBxxBrABYGX3+BC9q2MNQ/q8GAdYUm4P4QCsCQqb92KtM2bN8vv9+vaa6+V4ziKjY2V4zg8uBvotdde0/Hjx9W5c2f16tVLF110UbRHarJef/119e3bVwkJCdEepcnLz8/XoUOHdOWVVyozM5PH9XnYtGmTvF6vbrjhhrB/XCbOzwUf66ysLP33v/9VSUmJevfurT59+mjy5MlKSkoi2A1w2223qby8XJ9++qnatWunn//85xo/frwkjmgaav369Ro4cKAWLVqk0aNHn/FlNTh3gwcP1ieffKKTJ09q+vTpIX8WeXw3zB133KHDhw9r69at+uEPfxjy1cSsZcM8/fTTKi8vV48ePdS9e3elp6dLOr91vKBPgz/00EM6ePCg3nrrLX388cf66U9/qk2bNmnChAnyer1yuVycyj0Hv/3tb1VeXq6NGzdq69atuuKKK5Sfny+Xy8UDuIEcx1G7du0kSZMmTdLixYtVXV19xj74ejNnztQnn3yid999V++99566deumDRs26MMPP9Rnn33G47sBfvWrX6msrExr167VP/7xD7ndbu3du1fV1dWqra1lLRvg1ltv1ZIlS7Rnzx5NnTpVDzzwgJYvXy7p/A5sLugj6yNHjuh3v/udWrRooTZt2mjixInq0KGDcnNzNXPmTM2ePZtTuV+jurpaJSUlmj17tqTPv6L07rvv1pAhQ1ReXq7WrVsrNvaC/mMUVi6XS+np6brvvvuUlpamBx98UKdOndLDDz+s/fv3q127dmrZsmW0x2wSDh8+rHvvvVeSNGPGDOXn5+vEiRNq06aNevXqpfnz53/lN/Thcz6fT+Xl5VqyZImaN2+uVatW6V//+pduuOEGpaSk6Oabb9akSZP4f+U5WLFihQ4ePKitW7dK+vyLp5YuXaqnn35agUDgrN+Bca4u6CPrqqoqvfDCC8GfY2NjNWLECA0ePFjvvfeeSktLozhd09C8eXOdOHFCr732WnBbIBCQx+NRYmKiYmJiJEm1tbXRGrFJcRxHJ06c0DvvvKM777xTy5cv19SpUzV27FgNGjRI+/bti/aITUZFRYXWr1+vwsJC/f3vf1d+fr7eeecdTZgwQXv27NGbb74Z7RGbhISEBFVXV2vMmDG65557tGzZMq1Zs0aFhYW68cYbtXbtWu3cuTPaYzYJpw9gJKmurk4ZGRmaMGGCfvSjH2nFihXasGHDN77tCzrWw4cP18mTJ7V06VIFAgFJn8dn3Lhx2rVrl1atWhXlCe2LiYnRPffco5/85CfBbXv27JHjOGrZsqVcLpdeeOEFzZs3j9Nk58BxHCUlJal3797auXOnRo0apUmTJunZZ59Vz5491a1bt2iP2GQMGjRI1dXVeuONN3TfffepS5cuatOmjUaOHKnq6mq9/fbb0R6xSWjWrJnuuecede7cWatXr9bChQvVp08fpaena8aMGdq/f3/I89eoX69evbRv3z6tXbtWzZp9ntfvfe97uvfee+VyubRp06ZvfNsXVKxPx+L0P2+88Ualp6dr5cqVeuGFF4Lbmzdvrn79+gW/TxuhTq9TXV2dXC6Xfvazn4XE2u/3q2PHjpKkhQsX6r777tPNN9/M89dn8eW/wJx+ADdv3lwlJSUqLCzUsmXL9Otf/1ovvfSSFixYoLq6umiMat6XH9+DBg1SIBDQvHnztHbt2uB+MTEx+sEPfqC2bdtGZc6m4IuPcUm6/fbbtXLlSk2dOlUVFRXB/WJjY3X99dfrkksuicqcTU2nTp3Uq1cvPf/889q8eXNwe9euXXXNNddozZo13/gsZMyMGTNmhGnOqDsdC5fLpVOnTumiiy7Sddddp7ffflsbN27U5s2b1apVK61YsULLli3T9OnT1aZNmyhPbc8X17G2tjb4nHRtba2aNWumV155RV27dlVJSYkeeughrVu3Tt///vejObJZX3518ult+/bt07Jly/SXv/xFTz31lKZMmRL8rvfTL0BDqC8/vuPj4zVw4EBt27ZNe/bs0aFDhxQTE6O8vDwtXrxYjz/+OMGux5cf482aNZPL5dKuXbuUl5en5s2b6+KLL9aLL76oZ555RlOnTg2e3kX9kpKS1KpVKxUUFKikpERt2rQJHtisW7dOsbGxGjx48Dc6sLkgvnXr4YcfVmlpqTp06KBrrrlGQ4cOlfR5XGJiYlRZWanly5eroKBAx48fV2JioubPn6/evXtHeXJb6lvHL3vwwQf1+OOPq3Xr1lqzZo2uvPLKCE9q39et5fvvv69BgwbpySef1G233aa6urrgUTdC1beWgUBAsbGxqqio0MyZM7VhwwadOnVK7du319y5c9WrV68oT27P1/25PH78uEaOHKnt27fL7XYrNjZWy5cv5/+VZ/HZZ5/V+6K7119/XYsWLVJlZaXS0tKUkpKiZ555RuvWrVPPnj2/0f01+VjffvvtOnjwoG699VaVlZVp+fLluv/++zVnzhxJ0smTJ0Pe3F9RUaG4uDhecfslX7eOp//iI0nz5s3TY489prfeeovnWM/i69by1KlTchxHn376qVJSUkKObBDq69aypqZGcXFxqq2tVU1NjWpqanTRRRfx+D6Lc13LEydO6KOPPpLL5dJ3v/tdzvScxSOPPKKWLVvqN7/5jZKSkoLbv/g+6g8++EDvvfeeCgoK1KVLFw0fPlxXXHHFN79TpwnbsGGD06NHD6eurs5xHMepq6tzCgoKnJYtWzqTJ08O2beioiIaIzYJDVnHQCDgVFZWOiUlJVGY1L6mJtSOAAACh0lEQVSGrKXP54vGiE1GQ9bS6/VGY8QmoyFreeLEiWiM2KTcfffdTosWLZy5c+eesV6BQKBR7rNJn3eLjY1VixYtdOLEieCT9rfeeqtWrlypp556Sk8//bQkafHixZo8ebJOnjwZzXHNasg63nvvvWrZsqUuvfTSKE5sV0PWcuLEifyZ/AoNWctJkyaxll+hIWuZnZ3NWtbD+d+J6EAgoIEDB+rBBx/U/PnzQz7Y6PQZyJ07d6qysjKsd95k7d2713G73c6SJUuC22prax3HcZwnnnjC6d+/vxMIBJw33njD2bx5c7TGNI91DB/WMnxYy/BhLcPn1KlTTu/evZ0tW7Y4L730kuNyuZxZs2Y5VVVVwX3WrFnjfOc733GOHj0atvtt0rF2HMdZvHix07ZtW+ell15yHMcJnuZ55ZVXnMzMTE7pnCPWMXxYy/BhLcOHtQwPv98f8pee3NzcYLCrq6uD28P9VGGT/5zIu+66S2VlZZo4caJqamo0evRoSVJZWZlatmzJJ2udI9YxfFjL8GEtw4e1DI+EhASNGTNG0uenw0eMGCGXy6Vf/OIXqqqqUk5OjhITE8P/3vSwpj9KfD6fM3fuXCcmJsbp16+fk5WV5XTo0MHZsmVLtEdrUljH8GEtw4e1DB/WMrzq6uqCZyieffZZp1WrVs6RI0ca5b6a/Fu3vmjLli1at26dkpOT1a9fP1122WXRHqlJYh3Dh7UMH9YyfFjL8HG+8GFHPp8v5K1c4XRBxRoAgEhz/vf+aqcRv/ebWAMAYFyTfp81AADfBsQaAADjiDUAAMYRawAAjCPWAAAYR6wBADCOWAMAYByxBgDAOGINAIBxxBoAAOOINQAAxv0fq8mhLpqyB8YAAAAASUVORK5CYII=\n",
      "text/plain": [
       "<Figure size 500x500 with 1 Axes>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "import seaborn as sns\n",
    "\n",
    "# generate a figure with sub-plots (one for each gene segment)\n",
    "fig = plt.figure(figsize=(14,8), facecolor='w')\n",
    "plt.style.use('default')\n",
    "mpl.rc('font', family='Arial')\n",
    "gs = gridspec.GridSpec(2, 12)\n",
    "\n",
    "ax3 = fig.add_subplot(gs[0,8:12]) # beta-binomial estimates \n",
    "\n",
    "# get right of top and right frame of each subplot\n",
    "ax3.spines['top'].set_visible(False)\n",
    "ax3.spines['right'].set_visible(False)\n",
    "ax3.spines['bottom'].set_visible(True)\n",
    "ax3.spines['left'].set_visible(True)\n",
    "    \n",
    "# set fontsize of axes \n",
    "ax3.tick_params(axis = 'both', which = 'major', labelsize = 10)\n",
    "ax3.tick_params(axis = 'both', which = 'minor', labelsize = 10)\n",
    "\n",
    "\n",
    "BB = pd.DataFrame({\"\":['0-19', '20-44', '45-64', '65-89', '≥90'], \"SNV\":[0.75, 45.44, 31.83, 19.61, 2.36], \"age\":[0.847, 47.45, 29.66, 20.34, 1.69]})\n",
    "BB = pd.melt(BB, id_vars=\"\", var_name=\"type\", value_name=\"freq\")\n",
    "\n",
    "flatui = ['#0066A9', 'lightgrey', '#5BACCF', 'lightgrey', '#539648', 'lightgrey', '#C8C76F', 'lightgrey', '#E99F2E', 'lightgrey']\n",
    "sns.set_palette(flatui)\n",
    "sns.factorplot(x='', y='freq', data=BB, hue='type', kind='bar', ax=ax3, legend=None)\n",
    "\n",
    "# edit the axes - ax3\n",
    "ax3.set_title('', fontname='Arial', fontsize=14)\n",
    "ax3.set_ylabel('% of consensus SNVs \\n', fontname='Arial', fontsize=10)\n",
    "ax3.set_xlabel('\\n age bins', fontname='Arial', fontsize=10)\n",
    "ax3.set_facecolor('white')\n",
    "for tick in ax3.get_xticklabels():\n",
    "    tick.set_fontname(\"Arial\")\n",
    "for tick in ax3.get_yticklabels():\n",
    "    tick.set_fontname(\"Arial\")\n",
    "ax3.set_ylim(0,50)\n",
    "ax3.set_yticks([0,10,20,30,40,50])\n",
    "\n",
    "plt.xticks(rotation=45)\n",
    "\n",
    "# use tight_layout to optimize spacing of axes and titles within each subplot\n",
    "fig.tight_layout()\n",
    "fig.show()\n",
    "\n",
    "# save this plot to the figures folder \n",
    "fig.savefig(\"../figures/SNVS-per-age-bins.pdf\", bbox_inches='tight', dpi=300)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.8"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
